
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
        <meta name="description" content="Computational Oncogenomics Group @ UZH & SIB">
      
      
        <meta name="author" content="Michael Baudis">
      
      
      
      
        <link rel="next" href="projects-and-positions/">
      
      
        
      
      
      <link rel="icon" href="img/favicon.ico">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-9.7.3">
    
    
      
        <title>baudisgroup @ UZH & SIB</title>
      
    
    
      <link rel="stylesheet" href="assets/stylesheets/main.484c7ddc.min.css">
      
        
        <link rel="stylesheet" href="assets/stylesheets/palette.ab4e12ef.min.css">
      
      
  
  
    
      
      
    
  
  
  <style>:root{--md-admonition-icon--example:url('data:image/svg+xml;charset=utf-8,%3Csvg%20xmlns%3D%22http%3A//www.w3.org/2000/svg%22%20viewBox%3D%220%200%2024%2024%22%3E%3Cpath%20d%3D%22M3.604%203.089A.75.75%200%200%201%204%203.75V8.5h.75a.75.75%200%200%201%200%201.5h-3a.75.75%200%201%201%200-1.5h.75V5.151l-.334.223a.75.75%200%200%201-.832-1.248l1.5-1a.75.75%200%200%201%20.77-.037M8.75%205.5a.75.75%200%200%200%200%201.5h11.5a.75.75%200%200%200%200-1.5zm0%206a.75.75%200%200%200%200%201.5h11.5a.75.75%200%200%200%200-1.5zm0%206a.75.75%200%200%200%200%201.5h11.5a.75.75%200%200%200%200-1.5zM5.5%2015.75c0-.704-.271-1.286-.72-1.686a2.3%202.3%200%200%200-1.53-.564c-.535%200-1.094.178-1.53.565-.449.399-.72.982-.72%201.685a.75.75%200%200%200%201.5%200c0-.296.104-.464.217-.564A.8.8%200%200%201%203.25%2015c.215%200%20.406.072.533.185.113.101.217.268.217.565%200%20.332-.069.48-.21.657-.092.113-.216.24-.403.419l-.147.14c-.152.144-.33.313-.52.504l-1.5%201.5a.75.75%200%200%200-.22.53v.25c0%20.414.336.75.75.75H5A.75.75%200%200%200%205%2019H3.31l.47-.47c.176-.176.333-.324.48-.465l.165-.156a6%206%200%200%200%20.536-.566c.358-.447.539-.925.539-1.593%22/%3E%3C/svg%3E');}</style>



    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="css/theme_overrides.css">
    
    <script>__md_scope=new URL(".",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="white">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#latest-news-presentations-publications" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="." title="baudisgroup @ UZH &amp; SIB" class="md-header__button md-logo" aria-label="baudisgroup @ UZH & SIB" data-md-component="logo">
      
  <img src="img/site_logo.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            baudisgroup @ UZH & SIB
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Baudisgroup Home
            
          </span>
        </div>
      </div>
    </div>
    
      
    
    
    
    
      
      
        <label class="md-header__button md-icon" for="__search">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        </label>
        <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
          <a href="javascript:void(0)" class="md-search__icon md-icon" title="Share" aria-label="Share" data-clipboard data-clipboard-text="" data-md-component="search-share" tabindex="-1">
            
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M18 16.08c-.76 0-1.44.3-1.96.77L8.91 12.7c.05-.23.09-.46.09-.7s-.04-.47-.09-.7l7.05-4.11c.54.5 1.25.81 2.04.81a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3c0 .24.04.47.09.7L8.04 9.81C7.5 9.31 6.79 9 6 9a3 3 0 0 0-3 3 3 3 0 0 0 3 3c.79 0 1.5-.31 2.04-.81l7.12 4.15c-.05.21-.08.43-.08.66 0 1.61 1.31 2.91 2.92 2.91s2.92-1.3 2.92-2.91A2.92 2.92 0 0 0 18 16.08"/></svg>
          </a>
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" tabindex="0" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
      
    
    
      <div class="md-header__source">
        <a href="https://github.com/baudisgroup/baudisgroup.github.io" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><!--! Font Awesome Free 7.1.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M202.1 328.7c0 20.9-10.9 55.1-36.7 55.1s-36.7-34.2-36.7-55.1 10.9-55.1 36.7-55.1 36.7 34.2 36.7 55.1M496 278.2c0 31.9-3.2 65.7-17.5 95-37.9 76.6-142.1 74.8-216.7 74.8-75.8 0-186.2 2.7-225.6-74.8-14.6-29-20.2-63.1-20.2-95 0-41.9 13.9-81.5 41.5-113.6-5.2-15.8-7.7-32.4-7.7-48.8 0-21.5 4.9-32.3 14.6-51.8 45.3 0 74.3 9 108.8 36 29-6.9 58.8-10 88.7-10 27 0 54.2 2.9 80.4 9.2 34-26.7 63-35.2 107.8-35.2 9.8 19.5 14.6 30.3 14.6 51.8 0 16.4-2.6 32.7-7.7 48.2 27.5 32.4 39 72.3 39 114.2m-64.3 50.5c0-43.9-26.7-82.6-73.5-82.6-18.9 0-37 3.4-56 6-14.9 2.3-29.8 3.2-45.1 3.2-15.2 0-30.1-.9-45.1-3.2-18.7-2.6-37-6-56-6-46.8 0-73.5 38.7-73.5 82.6 0 87.8 80.4 101.3 150.4 101.3h48.2c70.3 0 150.6-13.4 150.6-101.3m-82.6-55.1c-25.8 0-36.7 34.2-36.7 55.1s10.9 55.1 36.7 55.1 36.7-34.2 36.7-55.1-10.9-55.1-36.7-55.1"/></svg>
  </div>
  <div class="md-source__repository">
    baudisgroup.github.io
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    



  

<nav class="md-nav md-nav--primary md-nav--integrated" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="." title="baudisgroup @ UZH &amp; SIB" class="md-nav__button md-logo" aria-label="baudisgroup @ UZH & SIB" data-md-component="logo">
      
  <img src="img/site_logo.png" alt="logo">

    </a>
    baudisgroup @ UZH & SIB
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/baudisgroup/baudisgroup.github.io" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><!--! Font Awesome Free 7.1.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M202.1 328.7c0 20.9-10.9 55.1-36.7 55.1s-36.7-34.2-36.7-55.1 10.9-55.1 36.7-55.1 36.7 34.2 36.7 55.1M496 278.2c0 31.9-3.2 65.7-17.5 95-37.9 76.6-142.1 74.8-216.7 74.8-75.8 0-186.2 2.7-225.6-74.8-14.6-29-20.2-63.1-20.2-95 0-41.9 13.9-81.5 41.5-113.6-5.2-15.8-7.7-32.4-7.7-48.8 0-21.5 4.9-32.3 14.6-51.8 45.3 0 74.3 9 108.8 36 29-6.9 58.8-10 88.7-10 27 0 54.2 2.9 80.4 9.2 34-26.7 63-35.2 107.8-35.2 9.8 19.5 14.6 30.3 14.6 51.8 0 16.4-2.6 32.7-7.7 48.2 27.5 32.4 39 72.3 39 114.2m-64.3 50.5c0-43.9-26.7-82.6-73.5-82.6-18.9 0-37 3.4-56 6-14.9 2.3-29.8 3.2-45.1 3.2-15.2 0-30.1-.9-45.1-3.2-18.7-2.6-37-6-56-6-46.8 0-73.5 38.7-73.5 82.6 0 87.8 80.4 101.3 150.4 101.3h48.2c70.3 0 150.6-13.4 150.6-101.3m-82.6-55.1c-25.8 0-36.7 34.2-36.7 55.1s10.9 55.1 36.7 55.1 36.7-34.2 36.7-55.1-10.9-55.1-36.7-55.1"/></svg>
  </div>
  <div class="md-source__repository">
    baudisgroup.github.io
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  
  <span class="md-ellipsis">
    
  
    Baudisgroup Home
  

    
  </span>
  
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="." class="md-nav__link md-nav__link--active">
        
  
  
  <span class="md-ellipsis">
    
  
    Baudisgroup Home
  

    
  </span>
  
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#latest-news-presentations-publications" class="md-nav__link">
    <span class="md-ellipsis">
      
        Latest News, Presentations &amp; Publications
      
    </span>
  </a>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="address" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Address
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="group" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Group Members
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="publications" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Publications
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="presentations" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Presentations
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="recipes" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Recipes
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="projects-and-positions/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Projects and Open Positions
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
      
        
        
      
    
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_8" >
        
          
          <label class="md-nav__link" for="__nav_8" id="__nav_8_label" tabindex="">
            
  
  
  <span class="md-ellipsis">
    
  
    Teaching and Seminars
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_8_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_8">
            <span class="md-nav__icon md-icon"></span>
            
  
    Teaching and Seminars
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="https://compbiozurich.org/teaching/UZH-BIO390/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    BIO390 Lectures &#8599;
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="https://compbiozurich.org/teaching/UZH-BIO392/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    BIO392 Block Course &#8599;
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="https://compbiozurich.org/news/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    ZH Bioinfo Seminars &#8599;
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="http://progenetix.org" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Progenetix &#8599;
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="http://cancercelllines.org" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Cancercelllines &#8599;
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="https://compbiozurich.org" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    CompbioZurich &#8599;
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="https://schemablocks.org" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    SchemaBlocks {S}[B] &#8599;
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="https://beacon-project.io" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Beacon Project &#8599;
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="https://www.mls.uzh.ch/en/research/baudis.html" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Michael Baudis @ UZH &#8599;
  

    
  </span>
  
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
          
          
            <div class="md-content" data-md-component="content">
              
              <article class="md-content__inner md-typeset">
                
  <h1 id="welcome-to-the-baudisgroup-pages"><a href="http://baudisgroup.org">Welcome to the <em>baudisgroup</em> Pages</a><a class="headerlink" href="#welcome-to-the-baudisgroup-pages" title="Permanent link">&para;</a></h1>
<p>The <em>baudisgroup</em> website represents projects and information by the <strong>Computational Oncogenomics Group</strong> of the <a href="https://www.mls.uzh.ch/en/research/baudis/">University of Zurich (UZH)</a> and the <a href="https://www.sib.swiss/baudis-michael/">Swiss Institute of Bioinformatics (SIB)</a>. For visitors more interested in Particle Astrophysics, we strongly recommend the website of another, although related, <a href="https://www.physik.uzh.ch/en/groups/baudis.html">Professor Baudis</a>.</p>
<p><img style="float: right; width: 320px; margin-left: 15px; margin-bottom-10px;" src="/img/2020-06-10___Michael-Baudis-Beacon-evolution.005.png" />The Computational Oncogenomics Group's research focus lies in the exploration of structural genome variations in cancer. Our work centres around our <a href="http://progenetix.org">Progenetix</a> resource of curated molecular-cytogenetic and sequencing data. Specific projects explore computational methods, genomics of selected tumour entities and genomic variant patterns across malignancies. As members of the <a href="http://ga4gh.org">Global Alliance for Genomics and Health</a>, the group is developing standards in biocuration and data sharing for genomic variants and phenotypic data, for instance in driving development of the <a href="https://genomebeacons.org">ELIXIR Beacon</a> project. Other research is related to genome data epistemology, e.g. geographic and diagnostic sampling biases in cancer studies. Some general information can be found in <a href="https://docs.baudisgroup.org/pdf/_baudisgroup_things.pdf">these slides</a>.</p>
<h2 id="latest-news-presentations-publications">Latest News, Presentations &amp; Publications<a class="headerlink" href="#latest-news-presentations-publications" title="Permanent link">&para;</a></h2>
  

  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
        
          
        
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  

  

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      February 17, 2026
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Migrating the Site's Server</h3>
    
<h5 class="title_news">Increasing data sovereignity...</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <p>We are in the process of migrating the site from Github pages to self hosting (on
Progenetix server instances). At the moment the site's source is still hosted at
Github but the build process &amp; hosting are already running on our own infrastructure.</p>
<p>Particularly, all article &amp; presentation PDF files are now accessed through the
<code>docs.baudisgroup.org</code> subdomain which is hosted on our own server; please adjust old
URLs accordingly (<em>i.e.</em> change <code>https://baudisgroup.org/pdf/...</code> to <code>https://docs.baudisgroup.org/pdf/...</code>).</p>
<a href="news/2026-02-17-server-move/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      February 09, 2026
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Welcome to Juliette Trouilloud</h3>
    
<h5 class="title_news">Erasmus+ MSc Intern from the Université Côte d'Azur</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <p>Welcome to Juliette for her 6 month intership in our group!</p>
<a href="news/2026-02-09-welcome-Juliette/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      January 29, 2026
      
    </span>
</aside>

    
    
    

<h3 class="title_news">The GA4GH Beacon Protocol - Federating Data Discovery and Sharing in Biomedical Genomics</h3>
    
<h5 class="title_news">Festival of Genomics and Biodata, London 2026</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <p><img alt="FOG Logo" src="/img/logo-Festival_of_Genomics.png" style="width: 210px; float: left; margin: 0px 10px 10px 0px;" /></p>
<ul>
<li>The Global Alliance for Genomics and Health - GA4GH - promotes progress in genomic research and human health through a framework of standards for effective and responsible sharing of genomic and related health data </li>
<li>The GA4GH Beacon protocol is a key element in these efforts by enabling the search of genomic variants and associated information across international, distributed resources while leaving full control with the resource providers </li>
<li>The presentation will introduce the Beacon protocol, its current status, place in the GA4GH ecosystem as well as planned developments and implementations </li>
<li>The Beacon protocol’s data discovery and especially sharing abilities will be demonstrated through its use in the Progenetix resource, a reference database for copy number variations (CNV) in cancer </li>
</ul>

<a href="presentations/2026-01-29-Michael-FOG-London/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      January 15, 2026
      
    </span>
</aside>

    
    
    

<h3 class="title_news">CNAttention: an attention-based deep multiple-instanc method for uncovering copy number aberratio signatures across cancers</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="ziying-yang-and-michael-baudis">Ziying Yang and Michael Baudis<a class="headerlink" href="#ziying-yang-and-michael-baudis" title="Permanent link">&para;</a></h4>
<h5 id="briefings-in-bioinformatics-2026-271-bbaf696-2026-01-15">Briefings in Bioinformatics, 2026, 27(1), bbaf696, 2026-01-15<a class="headerlink" href="#briefings-in-bioinformatics-2026-271-bbaf696-2026-01-15" title="Permanent link">&para;</a></h5>
<ul>
<li>doi: <a href="https://doi.org/10.1093/bib/bbaf696">10.1093/bib/bbaf696</a></li>
</ul>
<p><strong>Abstract</strong>: Somatic copy number aberrations (CNAs) represent a distinct class of genomic mutations associated with oncogenetic effects. Over
the past three decades, significant volumes of CNA data have been generated through molecular-cytogenetic and genome sequencing-
based techniques. These data have been pivotal in identifying cancer-related genes and advancing research on the relationship between
CNAs and histopathologically defined cancer types. However, comprehensive studies of CNA landscapes and disease parameters are
challenging due to the vast diagnostic and genomic heterogeneity encountered in ”pan-cancer” approaches. In this study, we introduce
CNAttention, an attention-based deep multiple instance learning method designed to comprehensively analyze CNAs across different
cancers and uncover specific CNA patterns within integrated gene-level CNA profiles of 30 cancer types. CNAttention effectively learns
CNA features unique to each cancer type and generates CNA signatures for 30 cancer types using attention mechanisms, highlighting
the distinctiveness of their CNA landscapes. CNAttention demonstrates high accuracy and exhibits stable performance even with
the incorporation of external datasets or parameter adjustments, underscoring its effectiveness in tumor identification. Expanding
these signatures to cancer classification trees reveals common patterns not only among physiologically related cancer types but
also among clinico-pathologically distant types, such as different cancers originating from neural crest derived cells. Additionally,
detected signatures also uncover genomic heterogeneity in individual cancer types, for instance in brain lower grade glioma. Additional
experiments with classification models underscore the efficacy of these signatures in representing various cancer types and their
potential utility in clinical diagnosis.</p>
<a href="publications/2026-01-15-CNAttention__Briefings-in-Bioinformatics/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      December 15, 2025
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Genomische Daten in Biomedizinischer Forschung – Chancen oder Risiken?</h3>
    
<h5 class="title_news">Kompetenzzentrum Medizin - Ethik - Recht Helvetiae (MERH)</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <p><img alt="Logo MERH" src="/img/logo-MERH.png" style="float: right; margin: 10px 0px 10px 30px; width: 200px" /></p>
<p>Im Rahmen der Vortragsreihe des Kompetenzzentrums Medizin - Ethik - Recht Helvetiae (MERH)
findet am 15. Dezember 2025 um 12:15 Uhr im Hörsaal KOL-G-204 der
Universität Zürich ein Vortrag von Prof. Michael Baudis zum Thema</p>
<p><strong>Genomische Daten in Biomedizinischer Forschung – Chancen oder Risiken?</strong></p>
<p>statt.
<a href="presentations/2025-12-15-Michael-Baudis__Genomische-Daten-in-Biomedizinischer-Forschung_MERH/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      December 04, 2025
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Data Discovery and Data Sharing in Biomedical Genomics</h3>
    
<h5 class="title_news">Baudisgroup projects and work profile for BioTAPESTRY</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <!-- ![BiH logo](/img/bih-logo.svg){ style="width: 100%; float: left; margin: 0px 10px 10px 0px;" } -->

<p>Baudisgroup projects and work profile for BioTAPESTRY...</p>

<a href="presentations/2025-12-04-Michael-BioTAPESTRY/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      December 04, 2025
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Toward Precision Medicine: Genomics Thailand Project</h3>
    
<h5 class="title_news">Zürich Seminars in Bioinformatics</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <p><img alt="GeTH Logo" src="https://baudisgroup.org/img/GeTH_logo.png" style="width: 210px; float: right; margin: -40px 0px 10px 20px;" /></p>
<p><strong>Abstract</strong> Genomics Thailand (GeTH) is the first national scale population genomics project fully funded by the government to drive the implementation and deployment of genomic medicine in Thailand in accordance with the 5-year plan medical hub strategy of the Thailand 4.0 strategic initiative. One of the Genomics Thailand’s goals is to establish the Thai reference genetic variation database collected from 50,000 whole genome sequencing (WGS) data of Thai volunteers. The database also houses clinical data as well as some important phenotypes of these volunteers that can greatly leverage home-grown genomic medicine research as well as other potential public health services in Thailand. 
<a href="presentations/2025-12-04-Worawich-Genomic-Thailand/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      November 20, 2025
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Progenetix as a case study for development and implementation of GA4GH standards</h3>
    
<h5 class="title_news">Colloquium at Berlin Institute of Health Sciences BiH</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <p><img alt="BiH logo" src="/img/bih-logo.svg" style="width: 100%; float: left; margin: 0px 10px 10px 0px;" /></p>
<p>In this colloquim at the Berlin Institute of Health Sciences (BiH), Michael presented
an overview of the Progenetix resource, its role in the global cancer genomics
data landscape, and its development and implementation of GA4GH standards with an 
emphasis on current status and future developments of the Beacon API.</p>

<a href="presentations/2025-11-20-Michael-BiH-Chait%C3%A9/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      November 11, 2025
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Genomdaten - Chancen und Risiken für Medizin und Gesellschaft</h3>
    
<h5 class="title_news">Vortrag am Seniorenforum Werdenberg in Buchs (SG)</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <p><img alt="Logo Seniorenforum Werdenberg" src="/img/Logo-Seniorenforum-600.png" style="float: right; margin: 0px 0px 10px 20px; width: 200px" /></p>
<p>Die Analyse des menschlichen Erbguts (Genoms) wird zunehmend in medizinischen Diagnosen eingesetzt und ist wesentlicher Bestandteil der “Präzisionsmedizin”. Während sich Erbgutanalysen  ausserhalb medizinischer Anwendungen schon länger in der Strafverfolgung etabliert haben hat sich in den letzten Jahren zusätzlich der Bereich der “Lifestyle-Genomik”, das heisst der Analyse des Erbgutes zur Einschätzung individueller Fähigkeiten und Risiken oder zur Feststellung von Verwandtschaftsbeziehungen.
<a href="presentations/2025-11-11-Michael-Baudis__Genomdaten-%E2%80%93-Chancen-und-Risiken-f%C3%BCr-Medizin-und-Gesellschaft__Seniorenforum-Werdenberg-Buchs/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      October 07, 2025
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Implementing the GA4GH Beacon API for Genomic Discovery and Sharing in an Open Data Paradigm</h3>
    
<h5 class="title_news">Poster at GA4GH 2025 Plenary</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <p><img alt="GA4GH Circle Logo" src="https://baudisgroup.org/img/ga4gh_circle_420x420.png" style="width: 210px; float: right; margin: -40px 0px 10px 20px;" /></p>
<p><strong>Abstract</strong> Besides the discovery of genomic and pheno-clinic data in different access settings, the GA4GH Beacon protocol also enables the retrieval of matched records through a variety of open or protected mechanisms, depending on the resources’ data structure as well as data protection and regulatory requirements. While data discovery scenarios with limited and/or aggregated responses will constitute a large fraction of Beacon and Beacon Network scenarios, the protocol also empowers the sharing of data e.g. in open research data settings.</p>
<p>Since 2016 the Progenetix cancer genomics reference resource has served as a platform for the implementation driven development and testing of the Beacon protocol. 
<a href="presentations/2025-10-07-Michael-GA4GH-poster/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      September 26, 2025
      
    </span>
</aside>

    
    
    

<h3 class="title_news">CNAdjust: Enhancing CNA Calling Accuracy through Systematic Baseline Adjustment</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="hangjia-zhao-and-michael-baudis">Hangjia Zhao and Michael Baudis<a class="headerlink" href="#hangjia-zhao-and-michael-baudis" title="Permanent link">&para;</a></h4>
<h5 id="frontiers-in-genetics-2025-sep-26">Frontiers in Genetics. 2025 Sep 26.<a class="headerlink" href="#frontiers-in-genetics-2025-sep-26" title="Permanent link">&para;</a></h5>
<ul>
<li>doi: <a href="https://doi.org/10.3389/fgene.2025.1674138">10.3389/fgene.2025.1674138</a></li>
</ul>
<p><strong>Abstract</strong>: Accurate determination of the genomic copy number baseline is crucial for identifying copy number alterations (CNAs) in cancer, yet it remains a significant challenge in tumors with complex karyotypes. To address this, we present CNAdjust, an integrated method to systematically detect and correct baseline inaccuracies in CNA data. CNAdjust employs a Bayesian framework that integrates cohort-specific CNA frequency priors with a data-driven plausibility score, ensuring that adjusted calls align with both biological cohort patterns and study-specific data. 
<a href="publications/2025-09-10-CNAdjust/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      September 19, 2025
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Theoretical Cytogenetics and Oncogenomics</h3>
    
<h5 class="title_news">Research Profile and Project Opportunities @ <i>baudisgroup</i></h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <p><img alt="CBB logo bar" src="/img/CBB-logo-bar.png" style="width: 100%; float: left; margin: 0px 0px 10px 0px;" /></p>
<p>The presentation at the ETH Department of Biosystems Science and Engineering D-BSSE in Basel introduced the baudisgroup's projects to incoming students of the <a href="https://cbb.ethz.ch">Computational Biology und Bioinformatics MSc program</a>.</p>

<a href="presentations/2025-09-19-Michael-CBB-MSc-presentation/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      July 16, 2025
      
    </span>
</aside>

    
    
    

<h3 class="title_news">pgxRpi: an R/Bioconductor package for user-friendly access to the Beacon v2 API</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="hangjia-zhao-and-michael-baudis">Hangjia Zhao and Michael Baudis<a class="headerlink" href="#hangjia-zhao-and-michael-baudis" title="Permanent link">&para;</a></h4>
<h5 id="bioinformatics-advances-oxford-2025-july-16">Bioinformatics Advances (Oxford). 2025 July 16.<a class="headerlink" href="#bioinformatics-advances-oxford-2025-july-16" title="Permanent link">&para;</a></h5>
<ul>
<li>doi: <a href="https://doi.org/10.1093/bioadv/vbaf172">10.1093/bioadv/vbaf172</a></li>
<li>PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40761325/">40761325</a></li>
</ul>
<p><strong>Abstract</strong>: The Beacon v2 specification, established by the Global Alliance for Genomics and Health (GA4GH), consists of a standardized framework and data models for genomic and phenotypic data discovery. By enabling secure, federated data sharing, it fosters interoperability across genomic resources. Progenetix, a reference implementation of Beacon v2, exemplifies its potential for large-scale genomic data integration, offering open access to genomic mutation data across diverse cancer types. Here we present pgxRpi, an open-source R/Bioconductor package that provides a streamlined interface to the Progenetix Beacon v2 REST API, 
<a href="publications/2025-03-23-pgxRpi/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      July 09, 2025
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Beacon - History, Future, Implementations</h3>
    
<h5 class="title_news">Beacon Day at McGill University</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="workshop-summary">Workshop Summary<a class="headerlink" href="#workshop-summary" title="Permanent link">&para;</a></h4>
<p><img alt="McGill logo" src="/img/mcgill-university-logo.png" style="width: 200px; float: right; margin: -20px 0px 10px 20px;" /></p>
<p>This workshop explores genomic resources from the Canadian context and how current and
future versions of the Beacon  standard can be utilized for data discovery and exchange.</p>

<a href="presentations/2025-07-09-Beacon-day-at-McGill/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      June 13, 2025
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Transfer Learning for Large-Scale Genomic AI in Cancer Genomics</h3>
    
<h5 class="title_news">AI in Healthcare Conference – Legal, Ethical &amp; Implementation Challenges</br>June 2025 – Zurich, Switzerland</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="jiahui-yu">Jiahui Yu<a class="headerlink" href="#jiahui-yu" title="Permanent link">&para;</a></h4>
<p><strong>Summary</strong> Recent advances in large-scale AI models adapted for genomic analysis – including DNA-BERT, Evo, and Nucleotide Transformer – have demonstrated promising capabilities in end-to-end prediction of functional genomic elements without relying on handcrafted features. However, as these models transition from analyzing population-scale germline data to the more complex landscape of tumor genomes, two critical concerns emerge: whether their learned representations can maintain accuracy across diverse sequencing technologies, and whether clinicians can develop sufficient trust in the latent patterns driving the model predictions.</p>
<p>Our ongoing research addresses these challenges through a transfer learning approach, adapting existing genomic language models using whole genome sequencing (WGS) data from the ICGC Pan-Cancer cohort. 
<a href="presentations/2025-06-13-Jiahui__Transfer-learning-for-large-scale-genomic-AI-in-cancer-Genomics__AI-in-healthcare-conference/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      June 10, 2025
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Congratulations to Ziying Yang!</h3>
    
<h5 class="title_news">Ziying today passed her PhD defence</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <p>Congratulations to Ziying for passing her defense for a PhD with the 
topic</p>
<p><strong>Heterogeneity and Signature Patterns of Copy Number Variants in Cancer Classification</strong></p>
<p>Ziying has been main developer behind the new/emerging
refcnv.org (a reference CNV track resource) but chiefly worked on questyions of
the heterogeneity of CNV patterns in cancer and their correspondance to diagnostic
codes.</p>
<p><img alt="Ziying presenting" src="/img/2025-06-10-Ziying-thesis-presenting.jpeg" /></p>
<a href="news/2025-06-10-Ziying-thesis-defense/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      June 04, 2025
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Driving oncogenomic and CNV reference resources through Beacon<br/>Data Discovery :: Data Sharing :: Analysis Support</h3>
    
<h5 class="title_news">Cancer data in Genomic Beacons at ELIXIR All Hands - Thessaloniki June 2025</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="workshop-summary">Workshop Summary<a class="headerlink" href="#workshop-summary" title="Permanent link">&para;</a></h4>
<p><img alt="ELIXIR logo" src="/img/elixir-dark-background-230x160.png" style="float: right; margin: -20px 0px 10px 20px;" />
Across Europe, several large-scale projects and national initiatives are embracing the Beacon V2 standard, actively contributing to its implementation and evolution. Among these efforts is the Genomic Data Initiative (GDI), a prominent international project involving numerous ELIXIR institutes. This collaboration underscores the growing adoption of genomic beacons beyond their initial purpose in Beacon V1—facilitating genomic variant discovery for rare diseases—towards addressing a broader spectrum of genetic data use cases.</p>

<a href="presentations/2025-06-04-Beacon-for-Cancer-ELIXIR-AHM/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      May 22, 2025
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Multicancer analyses of short tandem repeat variations reveal shared gene regulatory mechanisms</h3>
    
<h5 class="title_news">Repeat lengths association with the expression of nearby genes (eSTRs) in CRC, STAD and UCEC tumors</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="feifei-xia-max-verbiest-oxana-lundstrom-tugce-bilgin-sonay-michael-baudis-and-maria-anisimova">Feifei Xia, Max Verbiest, Oxana Lundström, Tugce Bilgin Sonay, Michael Baudis and Maria Anisimova<a class="headerlink" href="#feifei-xia-max-verbiest-oxana-lundstrom-tugce-bilgin-sonay-michael-baudis-and-maria-anisimova" title="Permanent link">&para;</a></h4>
<h5 id="briefings-in-bioinformatics-doi-httpsdoiorg101093bibbbaf219">Briefings in Bioinformatics, doi: <a href="https://doi.org/10.1093/bib/bbaf219">https://doi.org/10.1093/bib/bbaf219</a><a class="headerlink" href="#briefings-in-bioinformatics-doi-httpsdoiorg101093bibbbaf219" title="Permanent link">&para;</a></h5>
<h5 id="biorxiv-preprint-2025-01-06-doi-httpsdoiorg10110120250106629343">BiorXiv preprint 2025-01-06, doi: <a href="https://doi.org/10.1101/2025.01.06.629343">https://doi.org/10.1101/2025.01.06.629343</a><a class="headerlink" href="#biorxiv-preprint-2025-01-06-doi-httpsdoiorg10110120250106629343" title="Permanent link">&para;</a></h5>
<p><img alt="OUP logo" src="/img/logo-oxford-academic.svg" style="width: 180px; float: right; margin-left: 20px; margin-bottom: 10px; margin-top: -10px;" />
<img alt="biorXiv logo" src="/img/logo_biorXiv.jpg" style="float: right; margin-left: 20px; margin-bottom: 10px; margin-top: -10px;" />
<em>Abstract</em> Short tandem repeats (STRs) have been reported to influence gene expression across various human tissues. While STR variations are enriched in colorectal, stomach, and endometrial cancers, particularly in microsatellite instable tumors, their functional effects and regulatory mechanisms on gene expression remain poorly understood across these cancer types. Here, we leverage whole-exome sequencing and gene expression data to identify STRs for which repeat lengths are associated with the expression of nearby genes (eSTRs) in colorectal, stomach, and endometrial tumors. 
<a href="publications/2025-05-22-multicancer-repeats/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      May 15, 2025
      
    </span>
</aside>

    
    
    

<h3 class="title_news">The hCNV Community presents itself at ELIXIR Rare Diseases</h3>
    
<h5 class="title_news">ELIXIR RD Meeting, Bologna 2025</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <p>The ELIXIR hCNV community was a guest at the Rare Disease community meeting in
Bologna on May 15, 2025. The hCNV community presented an overview of its activities
and several members gave talks on specific topics related to their activities in 
the hCNV community.</p>
<a href="presentations/2025-05-15-hCNV-at-ELIXIR-RD/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      March 11, 2025
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Beacon v2 and Beyond - A Standard Framework for Data Discovery and Sharing in Biomedical Genomics</h3>
    
<h5 class="title_news">GA4GH GIF Beacon AMA</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <!-- ![TBRN logo](/img/2024-TBRN-logo.png){ style="float: right; margin: -20px -20px 0px 0px;" } -->

<p>This presentation is a minimal introduction to Beaon as basis for open discussions
during the "Ask Me Anything" session.</p>
<a href="presentations/2025-03-11-Michael-Baudis__GA4GH-GIF-Beacon-AMA/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      January 23, 2025
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Data Discovery in Biomedical Genomics and Cancer Research - Implementing a New Paradigm</h3>
    
<h5 class="title_news">2nd Thailand Bioinformatics Research Network (TBRN) Conference</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <p><img alt="TBRN logo" src="/img/2024-TBRN-logo.png" style="float: right; margin: -20px -20px 0px 0px;" /></p>
<p>The past few decades have highlighted the need for large-scale approaches to data
analysis in both rare disease genetics and cancer genomics due to the vast diversity
of genomic variations. While individual and collaborative studies have advanced
research, addressing the different levels of genomic diversity requires both data
curation and the integration of disparate genomic resources.</p>
<p>In this presentation, I will introduce some concepts and examples of genomic data
repositories and explain how our work on the Progenetix oncogenomics resource
aligns with standards development for the Global Alliance for Genomics and Health,
particularly the GA4GH Beacon API.</p>
<a href="presentations/2025-01-23-Michael-Baudis__TBRN-2024/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      December 10, 2024
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Genomdaten - Chancen und Risiken für Medizin und Gesellschaft</h3>
    
<h5 class="title_news">Vortrag an der Senior:innen-Universität Zürich UZH3</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <p><img alt="UZH logo" src="/img/UZH_logo_180.png" style="float: right; margin: 0px 0px 10px 20px; width: 180" />
In der medizinischen Forschung werden Genomdaten für genauere Diagnosen und zur
Auswahl personalisierter Therapien herangezogen. Genomdaten finden aber auch
ausserhalb der Medizin Verwendung – etwa in der genealogischen Forschung oder
in der Forensik. Diesem Potential stehen jedoch auch Risiken gegenüber.</p>
<p>Wenn genomische Daten von Millionen von Menschen erhoben werden, wer soll den
Einzelnen und die Gesellschaft vor dem Missbrauch solcher Daten schützen? Der
Vortrag beantwortet Fragen im Hinblick auf den Bedarf an solchen Daten in Medizin
und Forschung, und die dadurch eventuell entstehenden Risiken.</p>
<p>Warum brauchen wir genomische Daten von Millionen von Individuen? Wie können
meine Daten eine Gefahr für mich darstellen? Und wer sollte mich als Individuum
vor einem Missbrauch schützen?</p>
<a href="presentations/2020-2024/2024-12-10-Michael-Baudis__Genomdaten-%E2%80%93-Chancen-und-Risiken-f%C3%BCr-Medizin-und-Gesellschaft__Seniorenuniversitaet-Z%C3%BCrich/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      November 14, 2024
      
    </span>
</aside>

    
    
    

<h3 class="title_news">A systematic benchmark of copy number variation detection tools for high density SNP genotyping arrays</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="mn-van-baardwijk-lsem-heijnen-h-zhao-m-baudis-and-ap-stubbs">M.N. van Baardwijk, L.S.E.M. Heijnen, H. Zhao, M. Baudis and A.P. Stubbs<a class="headerlink" href="#mn-van-baardwijk-lsem-heijnen-h-zhao-m-baudis-and-ap-stubbs" title="Permanent link">&para;</a></h4>
<h5 id="genomics-elsevier-2024-nov-14">Genomics (Elsevier). 2024 Nov 14.<a class="headerlink" href="#genomics-elsevier-2024-nov-14" title="Permanent link">&para;</a></h5>
<ul>
<li>doi: <a href="https://doi.org/10.1016/j.ygeno.2024.110962">10.1016/j.ygeno.2024.110962</a></li>
<li>PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/39547585/">39547585</a></li>
</ul>
<p><img alt="elsevier genomics logo" src="/img/logo-elsevier-genomics.gif" style="float: right; margin: 0px 0px 20px 20px;" />
<strong>Abstract</strong> Copy Number Variations (CNVs) are crucial in various diseases, especially cancer, but detecting them accurately from SNP genotyping arrays remains challenging. Therefore, this study benchmarked five CNV detection tools-PennCNV, QuantiSNP, iPattern, EnsembleCNV, and R-GADA-using SNP array and WGS data from 2002 individuals of the DRAGEN re-analysis of the 1000 Genomes project. 
<a href="publications/2024-11-14-benchmark/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      November 06, 2024
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Copy number variation heterogeneity reveals biological inconsistency in hierarchical cancer classifications</h3>
    
<h5 class="title_news">Research Article</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="ziying-yang-paula-carrio-cordo-and-michael-baudis">Ziying Yang, Paula Carrio-Cordo and Michael Baudis<a class="headerlink" href="#ziying-yang-paula-carrio-cordo-and-michael-baudis" title="Permanent link">&para;</a></h4>
<h5 id="molecular-cytogenetics-spring-nature-doi-101186s13039-024-00692-2">Molecular Cytogenetics (Spring Nature). <a href="https://doi.org/10.1186/s13039-024-00692-2">doi: 10.1186/s13039-024-00692-2</a><a class="headerlink" href="#molecular-cytogenetics-spring-nature-doi-101186s13039-024-00692-2" title="Permanent link">&para;</a></h5>
<p><strong>Abstract</strong>: Cancers are heterogeneous diseases with unifying features of abnormal and consuming cell growth, where the deregulation of normal cellular functions is initiated by the accumulation of genomic mutations in cells of - potentially - any organ. At diagnosis malignancies typically present with patterns of somatic genome variants on diverse levels of heterogeneity. Among the different types of genomic alterations, copy number variants (CNV) represent a distinct, near-ubiquitous class of structural variants. Cancer classifications are foundational for patient care and oncology research. Terminologies such as the National Cancer Institute Thesaurus provide large sets of hierarchical cancer classification vocabularies and promote data interoperability and ontology-driven computational analysis. To find out how categorical classifications correspond to genomic observations, we conducted a meta-analysis of inter-sample genomic heterogeneity for classification hierarchies on CNV profiles from 97,142 individual samples across 512 cancer entities, and evaluated recurring CNV signatures across diagnostic subsets. Our results highlight specific biological mechanisms across cancer entities with the potential for improvement of patient stratification and future enhancement of cancer classification systems and provide some indications for cooperative genomic events across distinct clinical entities.</p>
<a href="publications/2024-11-06-CNV-heterogeneity/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      October 30, 2024
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Data Resources, Sharing, Discovery in Biomedical Genetics and Cancer Genomics</h3>
    
<h5 class="title_news">セミナー at National Cancer Center Hospital Tsukiji Campus</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <p><img alt="" src="/img/logo-NCC-Japan.png" style="float: right; margin: -20px -20px 0px 0px; width: 184px" /></p>
<p>In this seminar you will learn about Progentix, a large open resource in cancer
genomics focussing on curation and sharing of somatic copy number variation (CNV)
data as well as other resources and research topics of the <em>Theoretical Cytogenetics and Oncogenomics</em>
group at the University of Zucrich and teh Swiss Institute of Bioinformatics.</p>
<p>The second part of the presentation will make a case why such resources cannot be
sufficient and introduce the approaches shepherded by the Global Alliance for Genomics
and Health (GA4GH)
<a href="presentations/2020-2024/2024-10-30-Michael-Baudis__Chiba-University-Tsukiji/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      October 16, 2024
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Global Alliance for Genomics and Health - Promoting a New Paradigm for Data Discovery in Biomedical Genomics</h3>
    
<h5 class="title_news">Seminar at Tohoku University Sendai</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <p><img alt="" src="/img/logo_Tohoku-Megabank.png" style="float: right; margin: -20px -20px 0px 0px; width: 395px" /></p>
<p>The Global Alliance for Genomics and Health (GA4GH)
has developed into a major international organization to address the challenges of
data discovery in biomedical genomics. The <i>Beacon</i>
standard developed by GA4GH with technical support by the European bioinformatics orgabnization
ELIXIR and many international stakeholders has become a default for implementing
modern concepts of federated data discovery and access.
<a href="presentations/2020-2024/2024-10-16-Michael-Baudis__Tohoku-Sendai/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      October 03, 2024
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Data Discovery in Biomedical Genomics - Time for a New Paradigm</h3>
    
<h5 class="title_news">The XIVth MEDICAL GENETICS TÂRGU MUREȘ<br/>SOCIETATEA ROMÂNĂ DE GENETICĂ MEDICALĂ</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <p><img alt="" src="/img/logo-romanian-genetics.png" style="float: right; margin: 0px 0px 10px 20px; width: 205px" /></p>
<p>In biomedical genetics and genomics the last decades have led to the recognition that the
vast amaount and diversity of genomic variations requires large-scale approaches to
data analysis for both areas of rare disease genetics as well as cancer genomics.
While projects establishing high quality data from large cohorts are highly valuable
and necessary to drive the field, the true power of genomics can only be leveraged
through access to data from different studies and cohorts beyond geographic or
regulatory boundaries.</p>
<p>Since its foundation in 2013 the Global Alliance for Genomics and Health (GA4GH)
has emerged as the main international organization to address the challenges of
data discovery, access and utilization in genomics and related fields. The <em>Beacon</em>
standard developed by GA4GH with technical support by the European bioinformatics organization
ELIXIR and many international stakeholders has become a default for implementing
modern concepts of federated data discovery and access.</p>
<p>This presentation will provide an introduction into data federation in biomedical genomics
with emphasis on the use of the Beaconm protocol for genomic data discovery, together
with a general introduction into principles and mechanisms of genomics standards development by GA4GH.</p>
<a href="presentations/2020-2024/2024-10-03-Michael-Baudis__Romanian-Genetics-Society/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      July 02, 2024
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Congratulations to Rahel!</h3>
    
<h5 class="title_news">Rahel Paloots today passed her PhD defence</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <p>Congratulations to Rahel Paloots for passing her defence for a PhD with the 
topic</p>
<p><strong>Molecular Heterogeneity among Cancer Cell Lines and their Correspondence to Primary Neoplasias</strong></p>
<p>Rahel has been a contributor to progenetix.org and the main developer behind
cancercelllines.org (a resource for, well, cancer cell lines…) - but also to 
the GA4GH &amp; ELIXIR <a href="https://docs.genomebeacons.org">Beacon</a> project.</p>
<a href="news/2024-07-02-Rahel-defence/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      June 13, 2024
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Opening Cancer Genomics - Deploying the GA4GH Beacon protocol</h3>
    
<h5 class="title_news">Zürich Cancer ORD Workshop</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <!--
![](/img/ZHAW-Winterthur-image-470x235.jpg){ style="float: right; margin: 0px 0px 10px 20px; width: 235px" }
-->

<p>The "Moving ORD practices into cancer care" project explores and tests ORD principles in the context of cancer care, through a combination of engagement with relevant communities at the UZH as well as a technical demonstrator of a cutting edge data discovery technology. The cancer community at UZH is being mobilized through a dedicated workshop and topical meetings, to discuss challenges and opportunities of ORD in the oncology domain. A pilot installation of an established ORD approach (Beacon technology by the Global Alliance for Genomics and Health) is being implemented to test the sharing of cancer related -omics and associated data with a focus on the specific data access and security aspects presenting here.</p>
<a href="presentations/2020-2024/2024-06-13-Michael-Baudis__ZH-ORD-colloquium/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      May 16, 2024
      
    </span>
</aside>

    
    
    

<h3 class="title_news">The ELIXIR hCNV Community - Making complex genomics accessible</h3>
    
<h5 class="title_news">ELIXIR Webcast</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="michael-baudis-antonio-rausell-krzysztof-poterlowicz">Michael Baudis, Antonio Rausell &amp; Krzysztof Poterlowicz<a class="headerlink" href="#michael-baudis-antonio-rausell-krzysztof-poterlowicz" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> Genomic copy number variants (CNV) are a major contributor to human genome variation and important factors in rare disease genetics and cancer genomics. However, the complexity of CNV detection technologies, the lack of standardised annotation formats and the fragmentation of cytogenetic and genomic communities so far has limited large scale utilization of CNV profiles in computational genomics.  
<a href="news/2024-05-16-hCNV-webcast/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      May 15, 2024
      
    </span>
</aside>

    
    
    

<h3 class="title_news">In Search of the Perfect Model: How Cance Cell Lines Relate to Native Cancers</h3>
    
<h5 class="title_news">bioRxiv preprint</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="rahel-paloots-ziying-yang-and-michael-baudis">Rahel Paloots , Ziying Yang and Michael Baudis<a class="headerlink" href="#rahel-paloots-ziying-yang-and-michael-baudis" title="Permanent link">&para;</a></h4>
<h5 id="bioarxiv-preprint-2024-05-15-httpsdoiorg10110120240515594310">bioarXiv preprint (2024-05-15): <a href="https://doi.org/10.1101/2024.05.15.594310">https://doi.org/10.1101/2024.05.15.594310</a><a class="headerlink" href="#bioarxiv-preprint-2024-05-15-httpsdoiorg10110120240515594310" title="Permanent link">&para;</a></h5>
<p><img alt="biorXiv logo" src="/img/logo_biorXiv.jpg" style="float: right; margin-left: 20px; margin-bottom: 10px; margin-top: -10px;" />
<strong>Abstract</strong>: Cancer cell lines are frequently used in biological and transla-
tional research to study cellular mechanisms and explore treat-
ment options. However, cancer cell lines may display mutational
profiles divergent from native cancers or may be misidentified
or contaminated. We explored how similar cancer cell lines are
to native cancers to find the most suitable representations for
the corresponding diseases by utilising large collections of copy
number variation (CNV) profiles and applied machine learning
(ML) algorithms to predict cell line classifications.
<a href="publications/2024-05-15-cell-lines-cancer-heterogeneity/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      May 14, 2024
      
    </span>
</aside>

    
    
    

<h3 class="title_news">cancercelllines.org listed in Expasy</h3>
    
<h5 class="title_news">Entry in the Swiss Institute of Bioinformatics Catalogue</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <p><img alt="Expasy logo" src="/img/expasy-addon@2x.png" style="width: 250px; float: right; margin: 0px 0px 10px 20px;" />
Our recently launched cancer cell line genomics site <a href="https://cancercelllines.org">cancercelllines.org</a> is now listed as one of the resources in the Swiss Institute of Bioinformatics’ <a href="https://expasy.org">Expasy</a> catalogue. </p>

<a href="news/2024-05-14-cancercelllines-in-SIB-Expasy/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      April 29, 2024
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Genomic Data Mining and The Case for Open Data Standards</h3>
    
<h5 class="title_news">ZHAW ICLS colloquium</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <p><img alt="ZHAW logo" src="/img/ZHAW-Winterthur-image-470x235.jpg" style="float: right; margin: 0px 0px 10px 20px; width: 235px" />
The last decades have an enormous amount of data generation in the area of biomedical genomics, driven by a feedback loop of technological developments and increasing recognition of human genomic complexities, in health and disease. While individual projects frequently are tailored towards addressing specific research questions, the approach to many problems especially in rare diseases and cancer genomics requires both curated data collections and the access to data from international repositories and study centers.</p>
<p>Here I will present our oncogenomic data resources and research projects, and how our work on data discovery and exchange standards as part of an international community can improve data accessibility through the support of federated discovery and retrieval protocols.</p>
<a href="presentations/2020-2024/2024-04-29-Michael-Baudis__ZHAW-ICLS-colloquium/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      April 21, 2024
      
    </span>
</aside>

    
    
    

<h3 class="title_news">GA4GH Connect April 2024 in Ascona</h3>
    
<h5 class="title_news">Spring 2024 GA4GH Connect working meeting co-organized by our group</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <p><img alt="elixir logo" src="/img/2024-GA4GH-Connect-Ascona-Logo.png" style="width: 250px; float: right; margin: 0px 0px 10px 20px;" />
We're proud to host the next Spring GA4GH Connect meeting in April 2024 at the
<em>Congressi Stefano Franscini</em> on the Monte Verità in Ascona. This will provide an
excellent opportunity for Swiss genomics and bioinformatics to, well, connect with
the international "genomics and health" community and projects.</p>

<a href="news/2024-04-21-GA4GH-connect-Ascona/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      April 10, 2024
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Genomdaten - Chancen und Risiken für Medizin und Gesellschaft</h3>
    
<h5 class="title_news">Vortrag an der Seniorenuniversit&auml;t Winterthur</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <p><img alt="ZHAW logo" src="/img/ZHAW-Winterthur-image-470x235.jpg" style="float: right; margin: 0px 0px 10px 20px; width: 235px" />
In der medizinischen Forschung werden Genomdaten für genauere Diagnosen und zur
Auswahl personalisierter Therapien herangezogen. Genomdaten finden aber auch
ausserhalb der Medizin Verwendung – etwa in der genealogischen Forschung oder
in der Forensik. Diesem Potential stehen jedoch auch Risiken gegenüber.</p>
<p>Wenn genomische Daten von Millionen von Menschen erhoben werden, wer soll den
Einzelnen und die Gesellschaft vor dem Missbrauch solcher Daten schützen? Der
Vortrag beantwortet Fragen im Hinblick auf den Bedarf an solchen Daten in Medizin
und Forschung, und die dadurch eventuell entstehenden Risiken.</p>
<p>Warum brauchen wir genomische Daten von Millionen von Individuen? Wie können
meine Daten eine Gefahr für mich darstellen? Und wer sollte mich als Individuum
vor einem Missbrauch schützen?</p>
<a href="presentations/2020-2024/2024-04-10-Michael-Baudis__Genomdaten-%E2%80%93-Chancen-und-Risiken-f%C3%BCr-Medizin-und-Gesellschaft__Seniorenuniversitaet-Winterthur/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      March 27, 2024
      
    </span>
</aside>

    
    
    

<h3 class="title_news">cancercelllines.org - a Novel Resource for Genomic Variants in Cancer Cell Lines</h3>
    
<h5 class="title_news">DATABASE Article</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="rahel-paloots-and-michael-baudis">Rahel Paloots and Michael Baudis<a class="headerlink" href="#rahel-paloots-and-michael-baudis" title="Permanent link">&para;</a></h4>
<h5 id="database-oxford-2024-apr-302024baae030-doi-101093databasebaae030">Database (Oxford). 2024 Apr 30:2024:baae030.  <a href="https://doi.org/10.1093/database/baae030">doi: 10.1093/database/baae030</a><a class="headerlink" href="#database-oxford-2024-apr-302024baae030-doi-101093databasebaae030" title="Permanent link">&para;</a></h5>
<h5 id="bioarxiv-preprint-2023-12-13-httpsdoiorg10110120231212571281">bioarXiv preprint (2023-12-13): <a href="https://doi.org/10.1101/2023.12.12.571281">https://doi.org/10.1101/2023.12.12.571281</a><a class="headerlink" href="#bioarxiv-preprint-2023-12-13-httpsdoiorg10110120231212571281" title="Permanent link">&para;</a></h5>
<p><img alt="DATABASE logo" src="/img/logo-database-journal-300x150.png" style="float: right; width: 150px; margin-top: -100px;" />
<strong>Abstract</strong>: Cancer cell lines are an important component in biological and medical research, enabling studies of cellular mechanisms as well as the development and testing of pharmaceuticals. Genomic alterations in cancer cell lines are widely studied as models for oncogenetic events and are represented in a wide range of primary resources. We have created a comprehensive, curated knowledge resource - cancercelllines.org - with the aim to enable easy access to genomic profiling data in cancer cell lines, curated from a variety of resources and integrating both copy number and single nucleotide variants (SNVs) data. We have gathered over 5,600 copy number profiles as well as SNV annotations for 16,000 cell lines and provide this data with mappings to the GRCh38 reference genome. Both genomic variations and associated curated metadata can be queried through the GA4GH Beacon v2 API and a graphical user interface with extensive data retrieval enabled using GA4GH data schemas under a permissive licensing scheme.</p>
<p><strong>Availability and Implementation</strong>: Our resource is publicly available on the web at <a href="https://cancercelllines.org">cancercelllines.org</a>.</p>
<a href="publications/2024-03-27-cancercelllines.org/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      March 20, 2024
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Federated genomic discoveries: Deploying the GA4GH Beacon protocol</h3>
    
<h5 class="title_news">Virtual Seminar<br/>GHGA Lecture Series</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <p><img alt="GHGA logo" src="/img/logo-ghga.webp" style="float: right; margin: 0px 0px 10px 20px;" />
With the ever increasing amount of genomic data produced in the context of research studies, population analyses and medical diagnostics the need for access to genomic information beyond administrative or geographic boundaries has become a matter of eminent importance. 
<a href="presentations/2020-2024/2024-03-20-Michael-GHGA-virtual-seminar/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      March 16, 2024
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Data-Driven Information Extraction and Enrichment of Molecular Profiling Data for Cancer Cell Lines</h3>
    
<h5 class="title_news">Literature-derived annotations as entry point for data exploration</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="ellery-smith-rahel-paloots-dimitris-giagkos-michael-baudis-and-kurt-stockinger">Ellery Smith, Rahel Paloots, Dimitris Giagkos, Michael Baudis and Kurt Stockinger<a class="headerlink" href="#ellery-smith-rahel-paloots-dimitris-giagkos-michael-baudis-and-kurt-stockinger" title="Permanent link">&para;</a></h4>
<h5 id="bioinformatics-advances-vbae045-doiorg101093bioadvvbae045">Bioinformatics Advances, vbae045, <a href="https://doi.org/10.1093/bioadv/vbae045">doi.org/10.1093/bioadv/vbae045</a><a class="headerlink" href="#bioinformatics-advances-vbae045-doiorg101093bioadvvbae045" title="Permanent link">&para;</a></h5>
<h5 id="previous-arxiv-preprint-2023-07-03-httpsdoiorg1048550arxiv230700933">Previous arXiv preprint (2023-07-03): <a href="https://doi.org/10.48550/arXiv.2307.00933">https://doi.org/10.48550/arXiv.2307.00933</a><a class="headerlink" href="#previous-arxiv-preprint-2023-07-03-httpsdoiorg1048550arxiv230700933" title="Permanent link">&para;</a></h5>
<p><img alt="OUP logo" src="/img/logo-oxford-academic.svg" style="width: 180px; float: right; margin-left: 20px; margin-bottom: 10px; margin-top: -10px;" />
<img alt="arXiv logo" src="/img/logo_arXiv.png" style="float: right; margin-left: 20px; margin-bottom: 10px; margin-top: -10px;" />
<strong>Motivation</strong>: With the proliferation of research means and computational methodologies, published biomedical literature
is growing exponentially in numbers and volume (Lubowitz et al., 2021). As a consequence, in the fields of biological,
medical and clinical research, domain experts have to sift through massive amounts of scientific text to find relevant
information. However, this process is extremely tedious and slow to be performed by humans. Hence, novel computational
information extraction and correlation mechanisms are required to boost meaningful knowledge extraction. Results: In
this work, we present the design, implementation and application of a novel data extraction and exploration system. This
system extracts deep semantic relations between textual entities from scientific literature to enrich existing structured
clinical data in the domain of cancer cell lines. We introduce a new public data exploration portal, which enables automatic
linking of genomic copy number variants plots with ranked, related entities such as affected genes. Each relation is
accompanied by literature-derived evidences, allowing for deep, yet rapid, literature search, using existing structured
data as a springboard.</p>
<p><strong>Availability and Implementation</strong>: Our system is publicly available on the web at <a href="https://cancercelllines.org">cancercelllines.org</a>.</p>
<p><strong>Contact</strong>: The authors can be contacted at <a href="&#109;&#97;&#105;&#108;&#116;&#111;&#58;&#101;&#108;&#108;&#101;&#114;&#121;&#46;&#115;&#109;&#105;&#116;&#104;&#64;&#122;&#104;&#97;&#119;&#46;&#99;&#104;">&#101;&#108;&#108;&#101;&#114;&#121;&#46;&#115;&#109;&#105;&#116;&#104;&#64;&#122;&#104;&#97;&#119;&#46;&#99;&#104;</a> or <a href="&#109;&#97;&#105;&#108;&#116;&#111;&#58;&#114;&#97;&#104;&#101;&#108;&#46;&#112;&#97;&#108;&#111;&#111;&#116;&#115;&#64;&#117;&#122;&#104;&#46;&#99;&#104;">&#114;&#97;&#104;&#101;&#108;&#46;&#112;&#97;&#108;&#111;&#111;&#116;&#115;&#64;&#117;&#122;&#104;&#46;&#99;&#104;</a>.</p>
<a href="publications/2023-07-03-zhaw/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      March 01, 2024
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Twelve quick tips for deploying a Beacon</h3>
    
<h5 class="title_news">Some hints for Beacon developers & implementers</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="lauren-a-fromont-mauricio-moldes-michael-baudis-anthony-j-brookes-arcadi-navarro-and-jordi-rambla">Lauren A Fromont, Mauricio Moldes, Michael Baudis, Anthony J Brookes, Arcadi Navarro and Jordi Rambla<a class="headerlink" href="#lauren-a-fromont-mauricio-moldes-michael-baudis-anthony-j-brookes-arcadi-navarro-and-jordi-rambla" title="Permanent link">&para;</a></h4>
<h5 id="plos-comput-biol-2024-mar-1203e1011817">PLoS Comput Biol. 2024 Mar 1;20(3):e1011817.<a class="headerlink" href="#plos-comput-biol-2024-mar-1203e1011817" title="Permanent link">&para;</a></h5>
<ul>
<li>doi: <a href="https://doi.org/10.1371/journal.pcbi.1011817">10.1371/journal.pcbi.1011817</a>.</li>
<li>PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/38427629/">38427629</a></li>
</ul>
<p><strong>Introduction</strong>: In the age of data-driven biomedical research and clinical practice, the sharing of genomic and clinical data for health research and personalized medicine has become an important contribu- tor to improved diagnosis and treatment. From the data owner’s perspective, potential benefits include improved treatments, personalization of healthcare practice, and more effective con- trol of disease proliferation. However, the requirement for high levels of data security to pro- tect sensitive information presents a barrier to data discovery and sharing.</p>
<p>Beacon is designed to enable the benefits of data discovery while minimizing the associated risks...</p>
<a href="publications/2024-03-01-Beacon-Tips/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      February 27, 2024
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Genomic Data Sharing Standard Development with GA4GH and ELIXIR<br/>Opportunities and Pitfalls in Federated Data Discovery</h3>
    
<h5 class="title_news">DMLS Lecture Series<br/>University of Zurich Department of Molecular Life Sciences</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <p><img alt="UZH logo" src="/img/UZH_logo_180.png" style="float: right; margin: 0px 0px 10px 20px;" />
In this presentation Michael talks about the role of GA4GH (and ELIXIR) in the development
of standards and practices or genomic data exchange, some general principles, how his group got involved into
these efforts - but also some pitfalls ...</p>

<a href="presentations/2020-2024/2024-02-27-Michael-Baudis__Open-Data-Sharing-Protocols-with-GA4GH-and-ELIXIR__DMLS-lecture-series/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      February 08, 2024
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Structural Genome Variations in Cancer and the Case for Open Data Standards</h3>
    
<h5 class="title_news">Cancer Genomics Seminar at Utrecht<br/>Hubrecht Institute and Princess Maxima Center for Pediatric Oncology</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <p><img alt="UMC logo" src="/img/logo-UMC-Utrecht.png" style="height: 45px; float: right; margin: 0px 0px 10px 20px;" />
<img alt="Princess Maxima logo" src="/img/logo-PMC-Utrecht.png" style="height: 45px; float: right; margin: 0px 0px 10px 20px;" />
The presentation includes notes about work on improving the representation and
of genomic copy number variations (CNV), GA4GH and its <em>Beacon</em> protocol as well
as challenges towards genomic data privacy.</p>

<a href="presentations/2020-2024/2024-02-08-Michael-Utrecht-cancer-school/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      December 28, 2023
      
    </span>
</aside>

    
    
    

<h3 class="title_news">labelSeg: segment annotation for tumor copy number alteration profiles</h3>
    
<h5 class="title_news">A tool to assign relative SCNA levels to segments</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="hangjia-zhao-and-michael-baudis">Hangjia Zhao and Michael Baudis<a class="headerlink" href="#hangjia-zhao-and-michael-baudis" title="Permanent link">&para;</a></h4>
<h5 id="briefings-in-bioinformatics-oxford-2024-jan-312024bbad541">Briefings in Bioinformatics (Oxford). 2024 Jan 31;2024:bbad541.<a class="headerlink" href="#briefings-in-bioinformatics-oxford-2024-jan-312024bbad541" title="Permanent link">&para;</a></h5>
<ul>
<li>doi: <a href="https://doi.org/10.1093/bib/bbad541">10.1093/bib/bbad541</a></li>
<li>PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/38300514/">38300514</a></li>
<li>bioRxiv. doi: <a href="https://doi.org/10.1101/2023.05.17.541097">10.1101/2023.05.17.541097</a></li>
</ul>
<p><strong>Abstract</strong> Somatic copy number alterations (SCNAs) are a predominant type of oncogenomic alterations that affect a large proportion of the genome in the majority of cancer samples. Current technologies allow high-throughput measurement of such copy number aberrations, generating results consisting of frequently large sets of SCNA segments. However, the automated annotation and integration of such data are particularly challenging because the measured signals reflect biased, relative copy number ratios. In this study, we introduce labelSeg, an algorithm designed for rapid and accurate annotation of CNA segments, with the aim of enhancing the interpretation of tumor SCNA profiles. 
<a href="publications/2023-12-28-labelseg/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      December 14, 2023
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Beaconize this: Databases for Cancer Genomics and the Development of Open Data Standards</h3>
    
<h5 class="title_news">Seminar at the Bioinformatics club of the Centre de Recherche des Cordeliers (CRC)<br/>Université Paris Cité</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <p>In this seminar at the Centre de Recherche des Cordeliers in Paris Michael presents
the work of the group, with special emphasis on the role of the Progenetix oncogenomics
resources and tools in the development, implementation and testing of the Beacon
standard of the Global Alliance for Genomics and Health (GA4GH).</p>

<a href="presentations/2020-2024/2023-12-14-Paris-Medecine/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      October 31, 2023
      
    </span>
</aside>

    
    
    

<h3 class="title_news">pgxRpi Accepted by Bioconductor</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <p>The R wrapper for Peogenetix API <em>pgxRpi</em> is now part of the 'devel' branch and will be included in the upcoming 3.19 release in mid-April 2024.</p>
<a href="news/2023-10-31-pgxRpi-in-bioc/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      October 30, 2023
      
    </span>
</aside>

    
    
    

<h3 class="title_news">CNV Project at biohackathon23</h3>
    
<h5 class="title_news">Participating at #BioHackEU23 in Barcelona with a CNV reference resource project</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <p><img alt="biohackathon logo image" src="/img/biohackathon-europe-image.png" style="width: 250px; float: left; margin: 0px 20px 10px 0px;" />
With other members of the hCNV community some of us will participate at this year's
Biohackathon Europe event. The main project will address the creation of the template
for a "beaconized" public resource for reference (<em>i.e.</em> not disease associated)
copy number variation data, including the necessary tooling for the import from
e.g. VCF or BED file variants into Beacon backends (such as our <a href="https://github.com/progenetix/bycon/"><code>bycon</code></a> environment).</p>

<a href="news/2023-10-30-biohackathon/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      October 12, 2023
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Swiss-Korean Life Science Symposium</h3>
    
<h5 class="title_news">The 10th Swiss-Korean Life Science Symposium in Seoul</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <p><img alt="Swiss-Korean logo image" src="/img/2023-seoul-swiss-korean-logo.png" style="width: 200px; float: left; margin: 0px 0px 10px 20px;" />
As representative of the Swiss delegation and particularly of the University of Zurich
<a href="https://uzh.ch">UZH</a> Michael will be presenter and panel discussion participant at the
10th Swiss-Korean Life Science Symposium in Seoul, together with members of the Swiss
and Korean life sciences and personalized health academic and industrial communities.</p>

<a href="news/2023-10-12-swiss-korean/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      July 20, 2023
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Progenetix as SIB and ELIXIR Resource</h3>
    
<h5 class="title_news">Recognizing the Progenetix platform as Swiss contribution to the European bioinformatics resources ecosystem</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <p><img alt="elixir logo" src="/img/elixir-dark-background-230x160.png" style="float: right; margin: 0px 0px 10px 20px;" />
The Progenetix resource has finally been recognized as an official contribution to the ELIXIR European bioinformatics informatics ecosystem. Besides <a href="https://expasy.org">Expasy</a> Progenetix now is linked through ELIXIR's resource page. Or just go directly to <a href="https://progenetix.org">progenetix.org</a> (and its daughter project <a href="https://cancercelllines.org">cancercelllines.org</a>).</p>

<a href="news/2023-07-20-progenetix-as-SIB-ELIXIR-resource/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      July 03, 2023
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Short tandem repeat mutations regulate gene expression in colorectal cancer</h3>
    
<h5 class="title_news">Exploring STR patterns and their relation to expression changes in cancer</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="max-a-verbiest-oxana-lundstrom-feifei-xia-michael-baudis-tugce-bilgin-sonay-maria-anisimova">Max A Verbiest, Oxana Lundström, Feifei Xia, Michael Baudis, Tugce Bilgin Sonay, Maria Anisimova<a class="headerlink" href="#max-a-verbiest-oxana-lundstrom-feifei-xia-michael-baudis-tugce-bilgin-sonay-maria-anisimova" title="Permanent link">&para;</a></h4>
<h5 id="doi-httpsdoiorg10110120231129569189">doi: <a href="https://doi.org/10.1101/2023.11.29.569189">https://doi.org/10.1101/2023.11.29.569189</a><a class="headerlink" href="#doi-httpsdoiorg10110120231129569189" title="Permanent link">&para;</a></h5>
<p><img alt="biorXiv logo" src="/img/logo_biorXiv.jpg" style="float: right; margin-left: 20px; margin-bottom: 10px; margin-top: -10px;" />
Short tandem repeat (STR) mutations are prevalent in colorectal cancer (CRC), especially in tumours with the microsatellite instability (MSI) phenotype. While STR length variations are known to regulate gene expression under physiological conditions, the functional impact of STR mutations in CRC remains unclear. Here, we integrate STR mutation data with clinical information and gene expression levels to study the gene regulatory effects of STR mutations in CRC. We confirm that STR mutability in CRC highly depends on the MSI status, repeat unit size, and repeat length. Furthermore, we present a set of 1244 putative expression STRs (eSTRs) for which the STR length is associated with gene expression levels in CRC tumours. The length of 73 eSTRs is associated with expression levels of cancer-related genes, nine of which are CRC-specific genes. We show that linear models describing eSTR-gene expression relationships allow for predictions of gene expression changes in response to eSTR mutations. Moreover, we found an increased mutability of eSTRs in MSI tumours. Our evidence of gene regulatory roles for eSTRs in CRC highlights a mostly overlooked way through which tumours may modulate their phenotypes. The increased mutability of eSTRs in MSI tumours may be an early indication that eSTR mutations can confer a selective advantage to tumours. Future extensions of our findings into larger cohorts could uncover new STR-based targets in the treatment of cancer.</p>
<a href="publications/2023-11-29-repeats/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      June 05, 2023
      
    </span>
</aside>

    
    
    

<h3 class="title_news">ELIXIR All Hands Dublin</h3>
    
<h5 class="title_news">Baudisgroup presentations at the AHM 2023 in Dublin</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="rahel-hangjia-michael-for-the-group">Rahel, Hangjia &amp; Michael for the group<a class="headerlink" href="#rahel-hangjia-michael-for-the-group" title="Permanent link">&para;</a></h4>
<p>At the ELIXIR All Hands Meeting 2023 in Dublin our group presented several posters
about our resources and work in standards development.</p>
<a href="presentations/2020-2024/2023-06-05-ELIXIR-AHM/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      May 17, 2023
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Phenopacket-tools: Building and validating GA4GH Phenopackets</h3>
    
<h5 class="title_news">Bioinformatics tools and examples for working with the Phenopackets standard</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h5 id="danis-d-jacobsen-job-wagner-ah-groza-t-beckwith-ma-rekerle-l-carmody-lc-reese-j-hegde-h-ladewig-ms-seitz-b-munoz-torres-m-harris-nl-rambla-j-baudis-m-mungall-cj-haendel-ma-robinson-pn-2023-phenopacket-tools-building-and-validating-ga4gh-phenopackets-plos-one-18e0285433">Danis D, Jacobsen JOB, Wagner AH, Groza T, Beckwith MA, Rekerle L, Carmody LC, Reese J, Hegde H, Ladewig MS, Seitz B, Munoz-Torres M, Harris NL, Rambla J, Baudis M, Mungall CJ, Haendel MA, Robinson PN. (2023) <strong>Phenopacket-tools: Building and validating GA4GH Phenopackets.</strong> <em>PLoS One.</em> 18:e0285433.<a class="headerlink" href="#danis-d-jacobsen-job-wagner-ah-groza-t-beckwith-ma-rekerle-l-carmody-lc-reese-j-hegde-h-ladewig-ms-seitz-b-munoz-torres-m-harris-nl-rambla-j-baudis-m-mungall-cj-haendel-ma-robinson-pn-2023-phenopacket-tools-building-and-validating-ga4gh-phenopackets-plos-one-18e0285433" title="Permanent link">&para;</a></h5>
<p><strong>Abstract</strong> The Global Alliance for Genomics and Health (GA4GH) is a standards-setting organization that is developing a suite of coordinated standards for genomics. The GA4GH Phenopacket Schema is a standard for sharing disease and phenotype information that characterizes an individual person or biosample. The Phenopacket Schema is flexible and can represent clinical data for any kind of human disease including rare disease, complex disease, and cancer. It also allows consortia or databases to apply additional constraints to ensure uniform data collection for specific goals. We present phenopacket-tools, an open-source Java library and command-line application for construction, conversion, and validation of phenopackets. Phenopacket-tools simplifies construction of phenopackets by providing concise builders, programmatic shortcuts, and predefined building blocks (ontology classes) for concepts such as anatomical organs, age of onset, biospecimen type, and clinical modifiers.
<a href="publications/2023-05-17-phenopackets-tools/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      March 24, 2023
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Theoretical Cytogenetics and Oncogenomics</h3>
    
<h5 class="title_news">DMLS Tandem Talks</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h4>
<p>In this short presentation Michael provides an overview of the group's work
in cancer genomics resources, data analysis and standard development, including
the involvement in the Global Alliance for Genomics and Health GA4GH.</p>
<ul>
<li>Cancer Genome Profiles</li>
<li>Oncogenomic Data Resources</li>
<li>Bioinformatics Methods</li>
<li>Data Exchange Standards for Genomics and Personalized Health</li>
</ul>
<a href="presentations/2020-2024/2023-03-24-Michael-Baudis__Baudisgroup-portfolio__DMLS-Tandem-Talks/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      February 15, 2023
      
    </span>
</aside>

    
    
    

<h3 class="title_news">ZHAW Visitors for Cancercelllines Hackathon</h3>
    
<h5 class="title_news">Implementing INODE-driven literature collections</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <p>Fore some months <a href="http://baudisgroup.org">our group</a> has worked with members
of <a href="https://www.zhaw.ch/en/about-us/person/stog/">Kurt Stockinger's team</a> from
the ZHAW on a cancer cell lines use case for the 
<a href="https://www.inode-project.eu/inode">INODE</a> project. In the last 2 days we had a
site visit for a first implementation of the use case specific system on 
<a href="http://cancercelllines.org">cancercelllines.org</a>. More information to follow - and
thanks to Ellery &amp; Dimitris for the great work!</p>
<p><img alt="" src="/img/2023-02-14-BG-ZHAW-1024.png" /></p>

<a href="news/2023-02-15-ZHAW-coding-visit/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      December 28, 2022
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Candidate targets of copy number deletion events across 17 cancer types</h3>
    
<h5 class="title_news">Identifying cancer related genes against the background of somatic CNV events</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="huang-q-and-baudis-m">Huang Q and Baudis M<a class="headerlink" href="#huang-q-and-baudis-m" title="Permanent link">&para;</a></h4>
<h5 id="doi-103389fgene20221017657">doi: <a href="https://www.frontiersin.org/articles/10.3389/fgene.2022.1017657/full">10.3389/fgene.2022.1017657</a><a class="headerlink" href="#doi-103389fgene20221017657" title="Permanent link">&para;</a></h5>
<h5 id="previous-biorxiv-first-2022-06-29-doiorg10110120220629498080">previous bioRxiv (first )2022-06-29), <a href="https://doi.org/10.1101/2022.06.29.498080">doi.org/10.1101/2022.06.29.498080</a><a class="headerlink" href="#previous-biorxiv-first-2022-06-29-doiorg10110120220629498080" title="Permanent link">&para;</a></h5>
<p><strong>Abstract</strong> Genome variation is the direct cause of cancer and driver of its clonal evolution. While the impact of many point mutations can be evaluated through their modification of individual genomic elements, even a single copy number aberration (CNA) may encompass hundreds of genes and therefore pose challenges to untangle potentially complex functional effects. However, consistent, recurring and disease-specific patterns in the genome-wide CNA landscape imply that particular CNA may promote cancer-type-specific characteristics. Discerning essential cancer-promoting alterations from the inherent co-dependency in CNA would improve the understanding of mechanisms of CNA and provide new insights into cancer biology and potential therapeutic targets.
<a href="publications/2022-12-28-signatures-in-17-cancer-types/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      December 15, 2022
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Genomic Resource Built with GA4GH Standards</h3>
    
<h5 class="title_news">EORTC PAMM Firenze</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h4>
<p>This brief presentation introduces the Progenetix resource, the Gobal Alliance for
Genomics and Health as a developers of standards for data sharing in biomedical genomics
as well as the use of Progenetix in GA4GH standards development.</p>
<a href="presentations/2020-2024/2022-12-15-Michael-Baudis__Genomic-resource-built-with-GA4GH-standards__EORTC-PAMM-Firenze/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      November 16, 2022
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Beacon v2 - Onboarding Strategies & Feature Examples</h3>
    
<h5 class="title_news">Beacon Sessions at GA4GH Connect</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h4>
<p>The Beacon Sessions at GA4GH Connect November 2022 targeted the migration of existing and
implementation of new <strong>v2</strong> Beacons, with emphasis on the "how to get there easily" rather
than on all Beacon v2 features.
<a href="presentations/2020-2024/2022-11-16-Michael-Baudis__Beacon-Migration-and-Features__GA4GH-Connect/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      October 20, 2022
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Genomics Data Federation through Global Alliance for Genomics and Health Standards: Development and Implementation of the GA4GH Beacon Protocol</h3>
    
<h5 class="title_news">Seminar Yonsei University Medical School Seoul</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h4>
<p>In this Seoul meeting presentation Michael introduces the Global Alliance for Genomics and Healt and its involvement in Genomics standards development, followed by a discussion of the Beacon protocol and the role of the Progenetix resouce in its development. 
<a href="presentations/2020-2024/2022-10-20-Michael-Baudis__Genomics-Data-Federation-through-Global-Alliance-for-Genomics-and-Health-Standards--Development-and-Implementation-of-the-GA4GH-Beacon-Protocol__Korea-genomics-seminar/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      September 22, 2022
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Beacon v2 - Feature-rich Implementation of the Genomic Data Discovery Protocol</h3>
    
<h5 class="title_news">GA4GH 2022 Plenary Barcelona</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h4>
<p>The “Beacon” protocol - developed with support from ELIXIR, the European bioinformatics infrastructure organization, as a standard of the Global Alliance for Genomics and Health (GA4GH) - represents an emerging standard for an “Internet for Genomics”. While the initial version of the protocol served as a widely adopted test bed for the sharing of genomic variants over federated query systems connecting hundreds of internationally distributed resources, the version 2 of the protocol provides a framework for extended, metadata-rich query and response options in both public and restricted federated access scenarios.
<a href="presentations/2020-2024/2022-09-22-Michael-Baudis__Feature-rich-Beacon-v2-Implementation__GA4GH-plenary-poster/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      August 26, 2022
      
    </span>
</aside>

    
    
    

<h3 class="title_news">GA4GH Phenopackets: A Practical Introduction</h3>
    
<h5 class="title_news">Phenopackets v2 introduction with practical examples</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <p><img alt="" src="/img/logo_AdvancedGenetics.png" style="float: right; width: 200px; margin-top: -35px;" /></p>
<h4 id="ladewig-ms-jacobsen-jo-wagner-ah-danis-d-kassaby-be-gargano-m-groza-t-baudis-m-steinhaus-r-seelow-d-bechrakis-ne-mungall-cj-schofield-pn-elemento-o-smith-l-mcmurry-ja-munoz-torres-m-haendel-ma-and-robinson-pn">Ladewig MS, Jacobsen JO, Wagner AH, Danis D, Kassaby BE, Gargano M, Groza T, Baudis M, Steinhaus R, Seelow D, Bechrakis NE, Mungall CJ, Schofield PN, Elemento O, Smith L, McMurry JA, Munoz-Torres M, Haendel MA and Robinson PN<a class="headerlink" href="#ladewig-ms-jacobsen-jo-wagner-ah-danis-d-kassaby-be-gargano-m-groza-t-baudis-m-steinhaus-r-seelow-d-bechrakis-ne-mungall-cj-schofield-pn-elemento-o-smith-l-mcmurry-ja-munoz-torres-m-haendel-ma-and-robinson-pn" title="Permanent link">&para;</a></h4>
<h5 id="advanced-genetics-2022-2200016-link">Advanced Genetics 2022, 2200016. <a href="https://doi.org/10.1002/ggn2.202200016">LINK</a><a class="headerlink" href="#advanced-genetics-2022-2200016-link" title="Permanent link">&para;</a></h5>
<p><strong>Abstract</strong> The Global Alliance for Genomics and Health (GA4GH) is developing a suite of coordinated standards for genomics for healthcare. The Phenopacket is a new GA4GH standard for sharing disease and phenotype information that characterizes an individual person, linking that individual to detailed phenotypic descriptions, genetic information, diagnoses, and treatments. A detailed example is presented that illustrates how to use the schema to represent the clinical course of a patient with retinoblastoma, including demographic information, the clinical diagnosis, phenotypic features and clinical measurements, an examination of the extirpated tumor, therapies, and the results of genomic analysis. The Phenopacket Schema, together with other GA4GH data and technical standards, will enable data exchange and provide a foundation for the computational analysis of disease and phenotype information to improve our ability to diagnose and conduct research on all types of disorders, including cancer and rare diseases.</p>
<p>The Phenopacket software is available at <a href="https://github.com/phenopackets/">github.com/phenopackets/</a>.</p>
<a href="publications/2022-08-26-Phenopackets-practical-introduction-AdvGen/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      August 25, 2022
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Beacon - Ethical & Legal Aspects of a Genomic Data Discovery Protocol</h3>
    
<h5 class="title_news">DSI Ethics Project Pitch</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h4>
<p>Here Michael provides a very brief presentation about the GA4GH <em>Beacon</em>
protocol, especially as a target for projects discussing the ethical
implications of genome data discovery &amp; sharing as well as the relevant legal
frameworks, with emphasis on the Swiss context.
<a href="presentations/2020-2024/2022-08-25-Michael-Baudis__Beacon%2C-Ethical-%26-Legal-Aspects-of-a-Genomic-Data-Discovery-Protocol__UZH-DSI-Ethics/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      July 31, 2022
      
    </span>
</aside>

    
    
    

<h3 class="title_news">A cancer genomics resource built on GA4GH standards</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="rahel-paloots-michael-baudis">Rahel Paloots, Michael Baudis<a class="headerlink" href="#rahel-paloots-michael-baudis" title="Permanent link">&para;</a></h4>
<h5 id="cgc-st-louis-2022">CGC St Louis 2022<a class="headerlink" href="#cgc-st-louis-2022" title="Permanent link">&para;</a></h5>
<p>Progenetix is a cancer genomics resource that includes genomic profiling data as well as biomedical annotations and provenance data for cancer studies. The main goal of the Progenetix database is to provide easy, open access for research studies and clinical diagnostics. To facilitate sharing of genomic data, Progenetix complies with and contributes to GA4GH and Beacon data standards. Beacon, developed with the support from ELXIR (the European bioinformatics infrastructure organization), started out as protocol to share genomic variants over federated queries.</p>

<a href="presentations/2020-2024/2022-07-31-Rahel_CGC/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      June 27, 2022
      
    </span>
</aside>

    
    
    

<h3 class="title_news">A cancer genomics reference resource powered by GA4GH standards</h3>
    
<h5 class="title_news">Roche Data Science Seminar</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h4>
<p>The presentation reports about the Progenetix cancer genomics resource and its role
in the GA4GH ecosystem &amp; the Beacon genomics API development process.</p>
<a href="presentations/2020-2024/2022-06-27-Michael-Baudis__Progenetix-and-BeaconPlus__Roche-data-science-seminar/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      June 15, 2022
      
    </span>
</aside>

    
    
    

<h3 class="title_news">The GA4GH Phenopacket schema defines a computable representation of clinical data</h3>
    
<h5 class="title_news">Phenopackets v2 publication</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <p><img alt="Cell Genomics logo" src="/img/nature-biotechnology-logo.svg" style="float: right; width: 200px; margin-top: -35px;" /></p>
<h4 id="jacobsen-job-baudis-m-baynam-gs-beckmann-js-beltran-s-buske-oj-callahan-tj-chute-cg-courtot-m-danis-d-elemento-o-essenwanger-a-freimuth-rr-haendel-ma-robinson-pn-the-gaghpmc">Jacobsen JOB, Baudis M, Baynam GS, Beckmann JS, Beltran S, Buske OJ, Callahan TJ, Chute CG, Courtot M, Danis D, Elemento O, Essenwanger A, Freimuth RR, ... , Haendel MA, Robinson PN, The GAGHPMC.<a class="headerlink" href="#jacobsen-job-baudis-m-baynam-gs-beckmann-js-beltran-s-buske-oj-callahan-tj-chute-cg-courtot-m-danis-d-elemento-o-essenwanger-a-freimuth-rr-haendel-ma-robinson-pn-the-gaghpmc" title="Permanent link">&para;</a></h4>
<h5 id="nature-biotechnology-202240817-820-link-pmid35705716">Nature Biotechnology. 2022;40:817-820. <a href="https://rdcu.be/cPLaA">LINK</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/35705716/">PMID:35705716</a><a class="headerlink" href="#nature-biotechnology-202240817-820-link-pmid35705716" title="Permanent link">&para;</a></h5>
<p><strong>Abstract</strong> Despite great strides made in the development and wide acceptance of standards for exchanging structured information about genomic variants, progress in standards for computational phenotype analysis for translational genomics has lagged behind. Phenotypic features (signs, symptoms, laboratory and imaging findings, results of physiological tests, etc.) are of high clinical importance, yet exchanging them in conjunction with genomic variation information is often overlooked or even neglected.
<a href="publications/2022-06-15-Phenopackets-paper-NatBiotech/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      June 13, 2022
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Implementation of the GA4GH Beacon protocol for discovery and sharing of genomic copy number variation data</h3>
    
<h5 class="title_news">ESHG Vienna 2022</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h4>
<p><strong>Background &amp; Objectives</strong> Genomic copy number variations (CNV) are a major contributor to inter-individual genomic variation, can be causative events in rare diseases, but especially represent the majority of the mutational landscape in the most malignancies. While specific CNV events and some recurring patterns have contributed to the identification of individual cancer drivers and the recognition of cancer  subtypes, the complexity of genomic CNV patterns requires large amounts of well-defined genomic profiles for statistically meaningful analyses. At the other end of the spectrum, in the area of rare disease genomics the potential pathogenicity of individual CNV events requires validation against a vast set of disease-related and reference genomic profiles and annotations.</p>

<a href="presentations/2020-2024/2022-06-13-Michael_ESHG/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      June 07, 2022
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Progenetix & BeaconPlus - An open cancer genomics resource on a stack of Beacon code...</h3>
    
<h5 class="title_news">ELIXIR All Hands Amsterdam 2022</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h4>
<p>Here Michael provides some overview of the multi-year trajectory of the Beacon API
development, and how BeaconPlus &amp; Progenetix have been utilized for "implementation
driven design".</p>
<a href="presentations/2020-2024/2022-06-07-Michael-Baudis__Progenetix-and-BeaconPlus__ELIXIR-All-Hands/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      March 17, 2022
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Beacon v2 and Beacon networks: A "lingua franca" for federated data discovery in biomedical genomics, and beyond</h3>
    
<h5 class="title_news">Beacon v2 publication</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="rambla-j-baudis-m-ariosa-r-beck-t-fromont-la-navarro-a-paloots-r-rueda-m-saunders-g-singh-b-spalding-jd">Rambla J, Baudis M, Ariosa R, Beck T, Fromont LA, Navarro A, Paloots R, Rueda M, Saunders G, Singh B, Spalding JD.<a class="headerlink" href="#rambla-j-baudis-m-ariosa-r-beck-t-fromont-la-navarro-a-paloots-r-rueda-m-saunders-g-singh-b-spalding-jd" title="Permanent link">&para;</a></h4>
<h5 id="human-mutation-2022-mar-17-pmid35297548">Human Mutation. 2022 Mar 17. <a href="https://onlinelibrary.wiley.com/doi/10.1002/humu.24369">PMID:35297548</a><a class="headerlink" href="#human-mutation-2022-mar-17-pmid35297548" title="Permanent link">&para;</a></h5>
<p><strong>Abstract</strong> Beacon is a basic data discovery protocol issued by the Global Alliance for Genomics and Health (GA4GH). The main goal addressed by version 1 of the Beacon protocol was to test the feasibility of broadly sharing human genomic data, through providing simple "yes" or "no" responses to queries about the presence of a given variant in datasets hosted by Beacon providers.</p>

<a href="publications/2022-03-17-Beacon_paper/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      March 15, 2022
      
    </span>
</aside>

    
    
    

<h3 class="title_news">hCNV Implementation Studies Old and New</h3>
    
<h5 class="title_news">ELIXIR Human Data Communities</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h4>
<p>This presentation provided an overview about the hCNV community, implementation studies
and ongoing work, e.g. interaction with the GA4GH VRS standard group and Beacon development.</p>
<a href="presentations/2020-2024/2022-03-15-Michael-Baudis__hCNV-implementation-studies__ELIXIR-Human-Data-Communities/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      January 12, 2022
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Technical, legal and ethics aspects of genomic data sharing</h3>
    
<h5 class="title_news">Genomes | Privacy | Laws | Society - DSI Ethics Seminar</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h4>
<p>The presentation introduces the need for sharing and federated discovery of
genome data in the contexts of personalized health and genomic researchand some
of teh current developments in international standards and practices in the
area.
<a href="presentations/2020-2024/2022-01-12-Michael-Baudis__Technical%2C-legal-and-ethics-aspects-of-genomic-data-sharing-__UZH-DSI-Ethics/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      November 30, 2021
      
    </span>
</aside>

    
    
    

<h3 class="title_news">The GA4GH Phenopacket schema: A computable representation of clinical data for precision medicine</h3>
    
<h5 class="title_news">Phenopackets v2 preprint</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="jacobsen-job-baudis-m-baynam-gs-beckmann-js-beltran-s-callahan-tj-chute-cg-courtot-m-danis-d-elemento-o-freimuth-rr-haendel-ma-robinson-pn">Jacobsen JOB, Baudis M, Baynam GS, Beckmann JS, Beltran S, Callahan TJ, Chute CG, Courtot M, Danis D, Elemento O, Freimuth RR, ..., Haendel MA, Robinson PN.<a class="headerlink" href="#jacobsen-job-baudis-m-baynam-gs-beckmann-js-beltran-s-callahan-tj-chute-cg-courtot-m-danis-d-elemento-o-freimuth-rr-haendel-ma-robinson-pn" title="Permanent link">&para;</a></h4>
<h5 id="medrxiv-2021112721266944-doi1011012021112721266944">medRxiv, 2021.11.27.21266944. <a href="https://doi.org/10.1101/2021.11.27.21266944">doi:10.1101/2021.11.27.21266944</a><a class="headerlink" href="#medrxiv-2021112721266944-doi1011012021112721266944" title="Permanent link">&para;</a></h5>
<p><strong>Abstract</strong> Despite great strides in the development and wide acceptance of standards for exchanging structured information about genomic variants, there is no corresponding standard for exchanging phenotypic data, and this has impeded the sharing of phenotypic information for computational analysis. Here, we introduce the Global Alliance for Genomics and Health (GA4GH) Phenopacket schema, which supports exchange of computable longitudinal case-level phenotypic information for diagnosis and research of all types of disease including Mendelian and complex genetic diseases, cancer, and infectious diseases.
<a href="publications/2021-11-30-preprint-phenopackets-v2/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      November 10, 2021
      
    </span>
</aside>

    
    
    

<h3 class="title_news">The GA4GH Variation Representation Specification (VRS): a Computational Framework for the Precise Representation and Federated Identification of Molecular Variation.</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="alex-h-wagner-lawrence-babb-gil-alterovitz-michael-baudis-matthew-brush-daniel-l-cameron-melissa-cline-malachi-griffith-obi-l-griffith-melissa-konopko-heidi-l-rehm-andrew-d-yates-robert-r-freimuth-reece-k-hart">Alex H. Wagner, Lawrence Babb, Gil Alterovitz, Michael Baudis, Matthew Brush, Daniel L. Cameron, Melissa Cline , Malachi Griffith, Obi L. Griffith, ..., Melissa Konopko, Heidi L. Rehm, Andrew D. Yates, Robert R. Freimuth, Reece K. Hart<a class="headerlink" href="#alex-h-wagner-lawrence-babb-gil-alterovitz-michael-baudis-matthew-brush-daniel-l-cameron-melissa-cline-malachi-griffith-obi-l-griffith-melissa-konopko-heidi-l-rehm-andrew-d-yates-robert-r-freimuth-reece-k-hart" title="Permanent link">&para;</a></h4>
<h5 id="wagner-alex-h-et-al-cell-genomics-volume-1-issue-2-100027-doi101016jxgen2021100027">Wagner, Alex H. et al. Cell Genomics, Volume 1, Issue 2, 100027 <a href="https://doi.org/10.1016/j.xgen.2021.100027">doi:10.1016/j.xgen.2021.100027</a><a class="headerlink" href="#wagner-alex-h-et-al-cell-genomics-volume-1-issue-2-100027-doi101016jxgen2021100027" title="Permanent link">&para;</a></h5>
<h5 id="biorxiv-version-202120210115426843-2021-01-15">bioRxiv. version 20212021.01.15.426843. (2021-01-15)<a class="headerlink" href="#biorxiv-version-202120210115426843-2021-01-15" title="Permanent link">&para;</a></h5>
<h4 id="note">Note<a class="headerlink" href="#note" title="Permanent link">&para;</a></h4>
<p><img alt="Cell Genomics logo" src="/img/logo-cell-genomics.png" style="float: right; width: 150px; margin-top: -50px;" />This article was published as part of a special GA4GH edition of <a href="https://www.cell.com/cell-genomics/issue?pii=S2666-979X(21)X0003-1">Cell Genomics</a>.</p>
<p><strong>Abstract</strong> Maximizing the personal, public, research, and clinical value of genomic information will require the reliable exchange of genetic variation data. We report here the Variation Representation Specification (VRS, pronounced “verse”), an extensible framework for the computable representation of variation that complements contemporary human-readable and flat file standards for genomic variation representation. VRS provides semantically precise representations of variation and leverages this design to enable federated identification of biomolecular variation with globally consistent and unique computed identifiers.
<a href="publications/2021-11-10-Cell-Genomics-GA4GH-VRS/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      November 10, 2021
      
    </span>
</aside>

    
    
    

<h3 class="title_news">International federation of genomic medicine databases using GA4GH standards</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="adrian-thorogood-heidi-l-rehm-peter-goodhand-angela-jh-page-yann-joly-michael-baudis-jordi-rambla-arcadi-navarro-tommi-h-nyronen-mikael-linden-edward-s-dove-marc-fiume-michael-brudno-melissa-s-cline-ewan-birney">Adrian Thorogood, Heidi L. Rehm, Peter Goodhand, Angela J.H. Page, Yann Joly, Michael Baudis, Jordi Rambla, Arcadi Navarro, Tommi H. Nyronen, Mikael Linden, Edward S. Dove, Marc Fiume, Michael Brudno, Melissa S. Cline, Ewan Birney<a class="headerlink" href="#adrian-thorogood-heidi-l-rehm-peter-goodhand-angela-jh-page-yann-joly-michael-baudis-jordi-rambla-arcadi-navarro-tommi-h-nyronen-mikael-linden-edward-s-dove-marc-fiume-michael-brudno-melissa-s-cline-ewan-birney" title="Permanent link">&para;</a></h4>
<h5 id="thorogood-adrian-et-al-cell-genomics-volume-1-issue-2-100032-doi101016jxgen2021100032">Thorogood, Adrian et al. Cell Genomics, Volume 1, Issue 2, 100032 <a href="https://doi.org/10.1016/j.xgen.2021.100032">doi:10.1016/j.xgen.2021.100032</a><a class="headerlink" href="#thorogood-adrian-et-al-cell-genomics-volume-1-issue-2-100032-doi101016jxgen2021100032" title="Permanent link">&para;</a></h5>
<h4 id="note">Note<a class="headerlink" href="#note" title="Permanent link">&para;</a></h4>
<p><img alt="Cell Genomics logo" src="/img/logo-cell-genomics.png" style="float: right; width: 150px; margin-top: -50px;" />This article was published as part of a special GA4GH edition of <a href="https://www.cell.com/cell-genomics/issue?pii=S2666-979X(21)X0003-1">Cell Genomics</a>.</p>
<p><strong>Abstract</strong> We promote a shared vision and guide for how and when to federate genomic and health-related data sharing, enabling connections and insights across independent, secure databases. The GA4GH encourages a federated approach wherein data providers have the mandate and resources to share, but where data cannot move for legal or technical reasons. We recommend a federated approach to connect national genomics initiatives into a global network and precision medicine resource.</p>
<a href="publications/2021-11-10-Cell-Genomics-GA4GH-federation-paper/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      November 10, 2021
      
    </span>
</aside>

    
    
    

<h3 class="title_news">GA4GH: International policies and standards for data sharing across genomic research and healthcare</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="heidi-l-rehm-angela-jh-page-lindsay-smith-jeremy-b-adams-gil-alterovitz-lawrence-j-babb-maxmillian-p-barkley-michael-baudis-michael-js-beauvais-tim-beck-jacques-s-beckmann-sergi-beltran-david-bernick-alexander-bernier-james-k-bonfield-tiffany-f-boughtwood-guillaume-bourque-sarion-r-bowers-anthony-j-brookes-michael-brudno-matthew-h-brush-david-bujold-tony-burdett-orion-j-buske-moran-n-cabili-daniel-l-cameron-robert-j-carroll-esmeralda-casas-silva-debyani-chakravarty-bimal-p-chaudhari-shu-hui-chen-j-michael-cherry-justina-chung-melissa-cline-hayley-l-clissold-robert-m-cook-deegan-melanie-courtot-peter-goodhand-kathryn-north-ewan-birney">Heidi L. Rehm, Angela J.H. Page, Lindsay Smith, Jeremy B. Adams, Gil Alterovitz, Lawrence J. Babb, Maxmillian P. Barkley, Michael Baudis, Michael J.S. Beauvais, Tim Beck, Jacques S. Beckmann, Sergi Beltran, David Bernick, Alexander Bernier, James K. Bonfield, Tiffany F. Boughtwood, Guillaume Bourque, Sarion R. Bowers, Anthony J. Brookes, Michael Brudno, Matthew H. Brush, David Bujold, Tony Burdett, Orion J. Buske, Moran N. Cabili , Daniel L. Cameron, Robert J. Carroll, Esmeralda Casas-Silva, Debyani Chakravarty, Bimal P. Chaudhari, Shu Hui Chen, J. Michael Cherry, Justina Chung, Melissa Cline, Hayley L. Clissold, Robert M. Cook-Deegan, Mélanie Courtot, ..., Peter Goodhand, Kathryn North, Ewan Birney<a class="headerlink" href="#heidi-l-rehm-angela-jh-page-lindsay-smith-jeremy-b-adams-gil-alterovitz-lawrence-j-babb-maxmillian-p-barkley-michael-baudis-michael-js-beauvais-tim-beck-jacques-s-beckmann-sergi-beltran-david-bernick-alexander-bernier-james-k-bonfield-tiffany-f-boughtwood-guillaume-bourque-sarion-r-bowers-anthony-j-brookes-michael-brudno-matthew-h-brush-david-bujold-tony-burdett-orion-j-buske-moran-n-cabili-daniel-l-cameron-robert-j-carroll-esmeralda-casas-silva-debyani-chakravarty-bimal-p-chaudhari-shu-hui-chen-j-michael-cherry-justina-chung-melissa-cline-hayley-l-clissold-robert-m-cook-deegan-melanie-courtot-peter-goodhand-kathryn-north-ewan-birney" title="Permanent link">&para;</a></h4>
<h5 id="rehm-heidi-l-et-al-cell-genomics-volume-1-issue-2-100029-doi101016jxgen2021100029">Rehm, Heidi L. et al. Cell Genomics, Volume 1, Issue 2, 100029 <a href="https://doi.org/10.1016/j.xgen.2021.100029">doi:10.1016/j.xgen.2021.100029</a><a class="headerlink" href="#rehm-heidi-l-et-al-cell-genomics-volume-1-issue-2-100029-doi101016jxgen2021100029" title="Permanent link">&para;</a></h5>
<h4 id="note">Note<a class="headerlink" href="#note" title="Permanent link">&para;</a></h4>
<p>This article was published as part of a special GA4GH edition of <a href="https://www.cell.com/cell-genomics/issue?pii=S2666-979X(21)X0003-1">Cell Genomics</a>.</p>
<p><strong>Abstract</strong> The Global Alliance for Genomics and Health (GA4GH) aims to accelerate biomedical advances by enabling the responsible sharing of clinical and genomic data through both harmonized data aggregation and federated approaches. The decreasing cost of genomic sequencing (along with other genome-wide molecular assays) and increasing evidence of its clinical utility will soon drive the generation of sequence data from tens of millions of humans, with increasing levels of diversity. In this perspective, we present the GA4GH strategies for addressing the major challenges of this data revolution.
<a href="publications/2021-11-10-Cell-Genomics-GA4GH-marker-paper/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      September 22, 2021
      
    </span>
</aside>

    
    
    

<h3 class="title_news">A cancer genomics resource built around and driving GA4GH standards</h3>
    
<h5 class="title_news">GRIC sponsored workshop with the Swiss Institute of Bioinformatics</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h4>
<p>The Progenetix oncogenomics resource provides sample-specific cancer genome profiling data and biomedical annotations as well as provenance data for cancer studies. With more than 100k genomic copy number number (CNV) profiles from over 700 cancer types, Progenetix empowers comparative analyses beyond individual studies and diagnostic concepts.</p>

<a href="presentations/2020-2024/2021-09-22-Michael-Baudis-Progenetix%2C-Beacon-and-GA4GH-Novartis/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      September 13, 2021
      
    </span>
</aside>

    
    
    

<h3 class="title_news">A Standardized Format for Federated Genomic Data Exchange</h3>
    
<h5 class="title_news">The GA4GH Beacon Protocol Presented at BC<sup>2</sup> Basel 2021<br/>Session "Federating computational analyses with GA4GH standards"</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h4>
<p><img alt="BC2 logo" src="http://baudisgroup.org/img/logo_bc2.svg" style="float: right; width: 200px; margin-top: -120px;" />During the "Federating computational analyses with GA4GH standards" workshop at BC<sup>2</sup> 2021 Michael presented history and the current status of the Beacon project, as well as its integration with specific data resources and analysis initiatives.</p>
<a href="presentations/2020-2024/2021-09-13-Michael-Baudis__Beacon__BC2-2021-GA4GH-Session/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      August 31, 2021
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Cancer genomics reference resource and toolkit around GA4GH standards</h3>
    
<h5 class="title_news">ESHG 2021</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="q-huang-b-gao-r-paloots-p-carrio-cordo-z-yang-m-baudis">Q. Huang, B. Gao, R. Paloots, P. Carrio-Cordo, Z. Yang, M. Baudis<a class="headerlink" href="#q-huang-b-gao-r-paloots-p-carrio-cordo-z-yang-m-baudis" title="Permanent link">&para;</a></h4>
<p><img alt="ESHG Logo" src="/img/eshg_logo_260x208.png" style="float: right; width: 130px; margin-top: -65px;" />This poster presentation at the European Society of Human Genetics meeting 2021 discusses the integration and development of GA4GH standards by the Progenetix oncogenomics resource.</p>

<a href="presentations/2020-2024/2021-08-31-Michael-Baudis__Progenetix-GA4GH-standards__ESHG-2021/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      August 03, 2021
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Progenetix - An open reference resource for copy number vatiation data in cancer</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h3 id="qingyao-huang">Qingyao Huang<a class="headerlink" href="#qingyao-huang" title="Permanent link">&para;</a></h3>
<p><img src="/img/2021-08-03_CGC-session-info.png" style="margin-left: auto; margin-right:auto" /></p>

<a href="presentations/2020-2024/2021-08-03-Qingyao-Huang__Progenetix-oncogenomic-resource__CGC-talk/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      June 30, 2021
      
    </span>
</aside>

    
    
    

<h3 class="title_news">The Progenetix oncogenomic resource in 2021</h3>
    
<h5 class="title_news">Article describing the current content & technical status of progenetix.org</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="qingyao-huang-paula-carrio-cordo-bo-gao-rahel-paloots-michael-baudis">Qingyao Huang, Paula Carrio Cordo, Bo Gao, Rahel Paloots, Michael Baudis<a class="headerlink" href="#qingyao-huang-paula-carrio-cordo-bo-gao-rahel-paloots-michael-baudis" title="Permanent link">&para;</a></h4>
<h5 id="database-oxford-2021-jul-172021baab043">Database (Oxford). 2021 Jul 17;2021:baab043.<a class="headerlink" href="#database-oxford-2021-jul-172021baab043" title="Permanent link">&para;</a></h5>
<ul>
<li>doi: 10.1093/database/baab043.</li>
<li>PMID: 34272855</li>
<li>PMCID: PMC8285936.</li>
<li>bioRxiv. doi: <a href="https://doi.org/10.1101/2021.02.15.428237">doi.org/10.1101/2021.02.15.428237</a></li>
</ul>
<p><img alt="DATABASE logo" src="/img/logo-database-journal-300x150.png" style="float: right; width: 150px; margin-top: -100px;" /></p>
<p><strong>Abstract</strong> In cancer, copy number aberrations (CNAs) represent a type of nearly ubiquitous and frequently extensive structural genome variations. To disentangle the molecular mechanisms underlying tumorigenesis as well as identify and characterize molecular subtypes, the comparative and meta-analysis of large genomic variant collections can be of immense importance. Over the last decades, cancer genomic profiling projects have resulted in a large amount of somatic genome variation profiles, however segregated in a multitude of individual studies and datasets. The Progenetix project, initiated in 2001, curates individual cancer CNA profiles and associated metadata from published oncogenomic studies and data repositories with the aim to empower integrative analyses spanning all different cancer biologies.
<a href="publications/2021-06-30-publication-Progenetix-2021/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      June 11, 2021
      
    </span>
</aside>

    
    
    

<h3 class="title_news">hCNV Community and Implementation Studies</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<h4 id="elixir-all-hands-2021-human-data-day"><a href="https://elixirallhands.eventscase.com/EN/meeting">ELIXIR All Hands 2021 Human Data Day</a><a class="headerlink" href="#elixir-all-hands-2021-human-data-day" title="Permanent link">&para;</a></h4>
<p>At the Human Data Day Michael presents a very brief overview of the ending and
upcoming ELIXIR hCNV implementation studies.</p>

<a href="presentations/2020-2024/2021-06-11-Michael-Baudis__hCNV-implementation-studies__ELIXIR-Human-Data-Day/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      June 08, 2021
      
    </span>
</aside>

    
    
    

<h3 class="title_news">hCNV data and the Progenetix Beacon</h3>
    
<h5 class="title_news">Presentation at ELIXIR All Hands 2021</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<h4 id="elixir-all-hands-2021"><a href="https://elixirallhands.eventscase.com/EN/meeting">ELIXIR All Hands 2021</a><a class="headerlink" href="#elixir-all-hands-2021" title="Permanent link">&para;</a></h4>
<p>This presentation gives a brief overview of the use of the Progenetix resource
to test and implement a genomics reference resource using the emerging Beacon v2
protocol.</p>
<a href="presentations/2020-2024/2021-06-08-Michael-Baudis__hCNV-Progenetix-Beacon__ELIXIR-All-Hands/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      May 13, 2021
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Signatures of Discriminative CNA in 31 Cancer Subtypes</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h3 id="bo-gao-and-michael-baudis-2021">Bo Gao and Michael Baudis (2021)<a class="headerlink" href="#bo-gao-and-michael-baudis-2021" title="Permanent link">&para;</a></h3>
<h4 id="published-at-frontiers-in-genetics-2021-05-13">Published at <em>Frontiers in Genetics</em>, 2021-05-13<a class="headerlink" href="#published-at-frontiers-in-genetics-2021-05-13" title="Permanent link">&para;</a></h4>
<h4 id="abstract">Abstract<a class="headerlink" href="#abstract" title="Permanent link">&para;</a></h4>
<p>Copy number aberrations (CNA) are one of the most important classes of genomic mutations relatedto oncogenetic effects. In the past three decades, a vast amount of CNA data has been generated bymolecular-cytogenetic and genome sequencing based methods. While this data has been instrumentalin the identification of cancer-related genes and promoted research into the relation between CNA andhisto-pathologically defined cancer types, the heterogeneity of source data and derived CNV profilespose great challenges for data integration and comparative analysis. Furthermore, a majority of exist-ing studies have been focused on the association of CNA to pre-selected ”driver” genes with limitedapplication to rare drivers and other genomic elements.</p>

<a href="publications/2021-04-15-publication-Bo-classifiers/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      April 20, 2021
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Progenetix, Beacon and GA4GH at RDA</h3>
    
<h5 class="title_news">Research Data Alliance - RDA Virtual Plenary 17</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="concepts-status-history-outlook">Concepts | Status | History | Outlook<a class="headerlink" href="#concepts-status-history-outlook" title="Permanent link">&para;</a></h4>
<h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<h4 id="research-data-alliance-rda-virtual-plenary-17"><a href="https://rda-idcc.junolive.co/RDAVP17">Research Data Alliance - RDA Virtual Plenary 17</a><a class="headerlink" href="#research-data-alliance-rda-virtual-plenary-17" title="Permanent link">&para;</a></h4>
<p>This seminar gives an overview of current state of the Progenetix Beacon project
and the overall connection to the Global Alliance for Genomics and Health (GA4GH).</p>
<a href="presentations/2020-2024/2021-04-20-Michael-Baudis__ELIXIR-Beacon__RDA/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      March 29, 2021
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Implementing GA4GH Standards to Drive an Open Oncogenomics Resource</h3>
    
<h5 class="title_news">Research Seminar Kinderspital Zürich - Neuroonkologie</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<h4 id="seminar-neurooncology-childrens-hospital-zurich">Seminar Neurooncology Childrens Hospital Zürich<a class="headerlink" href="#seminar-neurooncology-childrens-hospital-zurich" title="Permanent link">&para;</a></h4>
<p>This seminar gives an overview of the history &amp; current state of the Progenetix resource, it's role in Beacon API development and the overall connection to the Global Alliance for Genomics and Health (GA4GH).</p>
<a href="presentations/2020-2024/2021-03-29-Michael-Baudis__CNVs%2C-Progenetix%2C-Beacon-and-GA4GH__Kinderspital-Neuroonkologie/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      March 25, 2021
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Discovering copy number variation across multiple cancer types</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h3 id="qingyao-huang">Qingyao Huang<a class="headerlink" href="#qingyao-huang" title="Permanent link">&para;</a></h3>
<p><em>Abstract</em></p>
<p>Genomic variations are direct cause of tumor formation and accomplice in its continuous evolution. While point mutations can be pinpointed to a targeted genetic element, copy number variations (CNVs) involve copy number gain or loss of a large DNA segment which often covers hundreds of genetic elements in one event.
<a href="presentations/2020-2024/2021-03-25-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      March 24, 2021
      
    </span>
</aside>

    
    
    

<h3 class="title_news">EACR conference - The Progenetix Oncogenomic Resource</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    
<a href="presentations/2020-2024/2021-03-24-Qingyao-Huang__Progenetix-Update__EACR-conference-poster/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      March 02, 2021
      
    </span>
</aside>

    
    
    

<h3 class="title_news">GA4GH Connect - Beacon v2 and SchemaBlocks</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="ga4gh-connect-2020"><a href="https://broadinstitute.swoogo.com/ga4gh-connect-2021/?i=sNK7RoeeqhyJE1ueSnn5lz4F9RRfnnIU">GA4GH Connect 2020</a><a class="headerlink" href="#ga4gh-connect-2020" title="Permanent link">&para;</a></h4>
<h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<h5 id="beacon-v2-structural-variants-slides">Beacon v2 Structural Variants <a href="http://baudisgroup.org/pdf/2021-03-02___Michael-Baudis__Beacon-structural-variants-queries.pdf">[slides]</a><a class="headerlink" href="#beacon-v2-structural-variants-slides" title="Permanent link">&para;</a></h5>
<h5 id="schemablocks-sb-slides">SchemaBlocks {S}[B] <a href="http://baudisgroup.org/pdf/2021-03-04___Michael-Baudis__SchemaBocks__GA4GH-Connect-2021-slides.pdf">[slides]</a><a class="headerlink" href="#schemablocks-sb-slides" title="Permanent link">&para;</a></h5>

<a href="presentations/2020-2024/2021-03-02-ga4gh-connect/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      March 01, 2021
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Copy number variant heterogeneity among cancer types reflects inconsistent concordance with diagnostic classifications</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="paula-carrio-cordo-and-michael-baudis">Paula Carrio Cordo and Michael Baudis<a class="headerlink" href="#paula-carrio-cordo-and-michael-baudis" title="Permanent link">&para;</a></h4>
<h5 id="biorxiv-doi-doiorg10110120210301433348">bioRxiv. doi: <a href="https://doi.org/10.1101/2021.03.01.433348">doi.org/10.1101/2021.03.01.433348</a><a class="headerlink" href="#biorxiv-doi-doiorg10110120210301433348" title="Permanent link">&para;</a></h5>
<p>This article explores the correlation between subsets of cancer entities, grouped by their somatic CNV patterns, and levels of diagnostic classification systems.</p>

<a href="publications/2021-03-01-preprint-Copy-number-variant-heterogeneity/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      December 07, 2020
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Genomic data and Privacy</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<h4 id="ethz-lecture">ETHZ Lecture<a class="headerlink" href="#ethz-lecture" title="Permanent link">&para;</a></h4>
<p>The understanding of the impact of individual inherited and somatic genome variants on phenotypes and diseases requires a thorough understanding of the occurrence of such variants amongst populations in general and carriers of the phenotypes and diseases in particular. This information can only be provided through the inclusion of data from a multitude of genome resources  in variant evaluation efforts, including such from outside (international) jurisdictions. However, opening such resources carries the inherent risk of breaching privacy, particularly through re-identification of individuals or their relatives and potentially through the exposure of individual genome-related personal information including  phenotypic and "performance" prediction and relative disease risk.</p>

<a href="presentations/2020-2024/2020-12-07-Michael-Baudis__Genomic-data-and-Privacy__ETHZ-551-1299-00L-HS20/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      November 04, 2020
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Beacon v2 – Towards flexible use and clinical applications for a reference genomic data protocol</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="sphn-webinar"><a href="https://sphn.ch/seminar-training/beacon-project/">SPHN Webinar</a><a class="headerlink" href="#sphn-webinar" title="Permanent link">&para;</a></h4>
<h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<p>Genomic “Beacons” provide discovery services for genomic data using the Beacon API developed as a key driver project of the Global Alliance for Genomics and Health (GA4GH). The Beacon protocol itself defines an open standard for genomics data discovery and provides a framework for web services responding to queries against genomic data collections, for instance from population based or disease specific genome repositories.
<a href="presentations/2020-2024/2020-11-04-Michael-Baudis__Beaconv2__SPHN-webinar/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      October 30, 2020
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Update of Progenetix Oncogenomics Resource</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="research-progress-report-dmls-university-of-zurich"><a href="https://www.mls.uzh.ch/en/seminars.html">Research Progress Report, DMLS, University of Zurich</a><a class="headerlink" href="#research-progress-report-dmls-university-of-zurich" title="Permanent link">&para;</a></h4>
<h3 id="qingyao-huang">Qingyao Huang<a class="headerlink" href="#qingyao-huang" title="Permanent link">&para;</a></h3>
<p>Copy number aberration (CNA) is frequently observed in cancer genomes. Meta-analysis of genomic variations helps to disentangle the multiplex molecular mechanism underlying tumorigenesis as well as identify and characterize molecular subtypes. Over the years, cancer genomic research have resulted in a considerable amount of data segregated by studies. The Progenetix project (<a href="http://www.progenetix.org">www.progenetix.org</a>), initiated in 2001, aims to systematize the published cancer genomic profiles and provide accurate annotation to facilitate integrative analysis.
<a href="presentations/2020-2024/2020-10-30-Qingyao-Huang__Progenetix-Update__Progress-report-DMLS-presentation/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      October 26, 2020
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Welcome to Ziying</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <p>Today <a href="/group/Ziying_Yang/">Ziying Yang</a> arrived as a new member of the <em>baudisgroup</em>. </p>
<p>Welcome Ziying!</p>

<a href="news/2020-10-26-Ziying-Yang-welcome/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      September 30, 2020
      
    </span>
</aside>

    
    
    

<h3 class="title_news">GA4GH Beacon v2 at GA4GH Plenary</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h2 id="ga4gh-beacon-v2-evolving-reference-standard-for-genomic-data-exchange">GA4GH Beacon v2 - Evolving Reference Standard for Genomic Data Exchange<a class="headerlink" href="#ga4gh-beacon-v2-evolving-reference-standard-for-genomic-data-exchange" title="Permanent link">&para;</a></h2>
<h4 id="ga4gh-8th-plenary"><a href="https://broadinstitute.swoogo.com/ga4gh-8th-plenary/455678">GA4GH 8th Plenary</a><a class="headerlink" href="#ga4gh-8th-plenary" title="Permanent link">&para;</a></h4>
<h3 id="gary-saunders-jordi-rambla-de-argila-anthony-brookes-juha-tornroos-and-michael-baudis">Gary Saunders, Jordi Rambla de Argila, Anthony Brookes, Juha Törnroos and Michael Baudis<a class="headerlink" href="#gary-saunders-jordi-rambla-de-argila-anthony-brookes-juha-tornroos-and-michael-baudis" title="Permanent link">&para;</a></h3>
<h5 id="for-the-elixir-beacon-project-ga4gh-discovery-work-stream-and-the-international-network-of-beacon-api-developers">For the ELIXIR Beacon project, GA4GH Discovery work stream and the international network of Beacon API developers<a class="headerlink" href="#for-the-elixir-beacon-project-ga4gh-discovery-work-stream-and-the-international-network-of-beacon-api-developers" title="Permanent link">&para;</a></h5>
<p>The Beacon driver project was one of the earliest initiatives of the Global Alliance for Genomics and Health with the Beacon v1.0 API as first approved GA4GH standard.
Version 2 of the protocol is slated to provide fundamental changes, towards a Internet of Genomics foundational standard:
* requests beyond genomic variants ("filters")
* payload responses, secured through open AAI
* aligning w/ GA4GH standards (Phenopackets, VRS, DUO...) through SchemaBlocks <a href="https://schemablocks.org">{S}[B]</a>
* Working with international partners on deployment of advanced implementations</p>
<a href="presentations/2020-2024/2020-09-30-Michael-Baudis__Beaconv2__GA4GH2020-poster/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      September 30, 2020
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Progenetix - A cancer genomics reference resource around GA4GH standards</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="ga4gh-8th-plenary"><a href="https://broadinstitute.swoogo.com/ga4gh-8th-plenary/455678">GA4GH 8th Plenary</a><a class="headerlink" href="#ga4gh-8th-plenary" title="Permanent link">&para;</a></h4>
<h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<p>The Progenetix oncogenomics resource provides sample-specific cancer
genome profiling data and biomedical annotations as well as provenance data from cancer studies. Especially through currently 113322 curated genomic copy number number (CNV) profiles from 1600 individual studies representing over 500 cancer types (NCIt), Progenetix empowers aggregate and comparative analyses which vastly exceed individual studies or single diagnostic concepts.
<a href="presentations/2020-2024/2020-09-30-Michael-Baudis__Progenetix-and-Beacon__GA4GH2020-poster/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      September 02, 2020
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Cancer Data - ELIXIR::GA4GH: Advancing genomics resources through standards and ontologies</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="eccb2020"><a href="https://eccb2020.info/ntbew02-elixirga4gh-advancing-genomics-through-expedited-data-access-enabled-by-standards-and-ontologies/">ECCB2020</a><a class="headerlink" href="#eccb2020" title="Permanent link">&para;</a></h4>
<h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<h4 id="additional-links">Additional Links<a class="headerlink" href="#additional-links" title="Permanent link">&para;</a></h4>
<ul>
<li><a href="http://beacon-project.io">Beacon Website</a></li>
<li><a href="http://next.progenetix.org/">Beacon+ in progenetix-next</a></li>
<li><a href="https://github.com/progenetix/bycon">Bycon Project</a></li>
</ul>
<a href="presentations/2020-2024/2020-09-02-Michael-Baudis__cancer-data-use-case-ECCB2020/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      August 10, 2020
      
    </span>
</aside>

    
    
    

<h3 class="title_news">The Ubiquitin Ligase TRIP12 Limits PARP1 Trapping and Constrains PARP Inhibitor Efficiency</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="marco-gatti-ralph-imhof-qingyao-huang-michael-baudis-matthias-altmeyer">Marco Gatti, Ralph Imhof, Qingyao Huang, Michael Baudis, Matthias Altmeyer<a class="headerlink" href="#marco-gatti-ralph-imhof-qingyao-huang-michael-baudis-matthias-altmeyer" title="Permanent link">&para;</a></h4>
<h5 id="cell-rep-2020-aug-4-doi-101016jcelrep2020107985">Cell Rep. 2020 Aug 4 DOI: <a href="https://doi.org/10.1016/j.celrep.2020.107985">10.1016/j.celrep.2020.107985</a><a class="headerlink" href="#cell-rep-2020-aug-4-doi-101016jcelrep2020107985" title="Permanent link">&para;</a></h5>
<p><strong>Abstract</strong> PARP inhibitors (PARPi) cause synthetic lethality in BRCA-deficient tumors. Whether specific vulnerabilities to PARPi exist beyond BRCA mutations and related defects in homology-directed repair (HDR) is not well understood. Here, we identify the ubiquitin E3 ligase TRIP12 as negative regulator of PARPi sensitivity.
<a href="publications/2020-08-04-publication-trip12-parp1-study-CellRep/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      June 30, 2020
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Beacon v2 - Towards Flexible Use and Clinical Applications for a Reference Genomic Data Sharing Protocol</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="personalized-health-technologies-2020"><a href="https://www.personalizedhealth.nexus.ethz.ch">Personalized Health Technologies 2020</a><a class="headerlink" href="#personalized-health-technologies-2020" title="Permanent link">&para;</a></h4>
<h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<p>Beacons provide discovery services for genomic data using the Beacon API developed under
the leadership of ELIXIR, as a key driver project of the Global Alliance for Genomics and
Health (GA4GH). The Beacon protocol itself defines an open standard for genomics data
discovery. It provides a framework for public web services responding to queries against
genomic data collections, for instance from population based or disease specific genome
repositories. Sites offering beacons can scale through aggregation in "Beacon Networks",
which distribute single genome queries among a potentially large number of international
beacons and assemble their responses.
<a href="presentations/2020-2024/2020-06-30-Michael-Baudis__Beacon-evolution__NEXUS-Personalized-Health-2020/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      June 10, 2020
      
    </span>
</aside>

    
    
    

<h3 class="title_news">ELIXIR All Hands - Beacon Evolution</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="elixir-all-hands-2020-beacon-workshop">ELIXIR All Hands 2020 - Beacon Workshop<a class="headerlink" href="#elixir-all-hands-2020-beacon-workshop" title="Permanent link">&para;</a></h4>
<h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<p>This presentation covers some of Beacon's
origins, features and directions.</p>
<h4 id="links">Links<a class="headerlink" href="#links" title="Permanent link">&para;</a></h4>
<ul>
<li><a href="http://baudisgroup.org/pdf/2020-06-10___Michael-Baudis__Beacon-evolution__ELIXIR-All-Hands-slides.pdf">Presentation slides</a></li>
<li><a href="http://beacon-project.io">Beacon Website</a></li>
</ul>
<a href="presentations/2020-2024/2020-06-10-Michael-Baudis__Beacon-evolution__ELIXIR-All-Hands/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      May 14, 2020
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Oncology Informatics: Status Quo and Outlook - Review</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="paul-martin-putora-michael-baudis-beth-m-beadle-issam-el-naqa-frank-a-giordano-and-nils-h-nicolay">Paul Martin Putora, Michael Baudis, Beth M. Beadle, Issam El Naqa, Frank A. Giordano and Nils H. Nicolay<a class="headerlink" href="#paul-martin-putora-michael-baudis-beth-m-beadle-issam-el-naqa-frank-a-giordano-and-nils-h-nicolay" title="Permanent link">&para;</a></h4>
<h5 id="oncology-2020-05-14-doi-101159000507586-review">Oncology, 2020-05-14. <a href="https://doi.org/10.1159/000507586">DOI 10.1159/000507586</a> (Review)<a class="headerlink" href="#oncology-2020-05-14-doi-101159000507586-review" title="Permanent link">&para;</a></h5>
<p><strong>Abstract</strong> Oncology has undergone rapid progress, with emerging developments in areas including cancer stem cells, molecularly targeted therapies, genomic analyses, and individually tai- lored immunotherapy. These advances have expanded the tools available in the fight against cancer. Some of these have seen broad media coverage resulting in justified public attention. However, these achievements have only been possible due to rapid developments in the expanding field of biomedical informatics and information technology (IT).
<a href="publications/2020-05-14-publication-oncology-informatics-review/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      May 06, 2020
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Swissnex SF: Laura & Michael Baudis - Life & Family</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <p>Originally planned for their stays at <a href="https://www.berkeley.edu">UCB</a> and <a href="http://lbl.gov">LBNL</a>, Laura &amp; Michael were interviewed by Tabea Stoeckel from <a href="https://www.swissnexsanfrancisco.org">swissnex San Francisco</a> about their stay in the Bay Area and their research &amp; family life as internationally active scientists.</p>

<a href="news/2020-05-06-lbaudis-mbaudis-swissnex-nextrends/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      May 06, 2020
      
    </span>
</aside>

    
    
    

<h3 class="title_news">Minimum Error Calibration and Normalization for Genomic Copy Number Analysis</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="bo-gao-and-michael-baudis-2020">Bo Gao and Michael Baudis (2020)<a class="headerlink" href="#bo-gao-and-michael-baudis-2020" title="Permanent link">&para;</a></h4>
<h5 id="biorxiv-2019-07-31-doi-101101720854">bioRxiv, 2019-07-31. <a href="https://doi.org/10.1101/720854">DOI 10.1101/720854</a><a class="headerlink" href="#biorxiv-2019-07-31-doi-101101720854" title="Permanent link">&para;</a></h5>
<h5 id="genomics-volume-112-issue-5-september-2020-pages-3331-3341-accepted-2020-05-06-doiorg101016jygeno202005008">Genomics, Volume 112, Issue 5, September 2020, Pages 3331-3341, accepted 2020-05-06 <a href="https://doi.org/10.1016/j.ygeno.2020.05.008">doi.org/10.1016/j.ygeno.2020.05.008</a>.<a class="headerlink" href="#genomics-volume-112-issue-5-september-2020-pages-3331-3341-accepted-2020-05-06-doiorg101016jygeno202005008" title="Permanent link">&para;</a></h5>
<h5 id="background">Background<a class="headerlink" href="#background" title="Permanent link">&para;</a></h5>
<p>Copy number variations (CNV) are regional deviations from the normal autosomal bi-allelic DNA content. While germline CNVs are a major contributor to genomic syndromes and inherited diseases, the majority of cancers accumulate extensive "somatic" CNV (sCNV or CNA) during the process of oncogenetic transformation and progression. While specific sCNV have closely been associated with tumorigenesis, intriguingly many neoplasias exhibit recurrent sCNV patterns beyond the involvement of a few cancer driver genes.
<a href="publications/2020-05-07-publication-Mecan4CNA-Genomics/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      April 20, 2020
      
    </span>
</aside>

    
    
    

<h3 class="title_news">SchemaBlocks and GA4GH TASC</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <p>A brief presentation about SchemaBlocks concepts and its possible integration into the new GA4GH TASC effort.</p>
<h4 id="additional-links">Additional Links<a class="headerlink" href="#additional-links" title="Permanent link">&para;</a></h4>
<ul>
<li><a href="http://schemablocks.org">schemablocks</a></li>
<li><a href="http://ga4gh.org">GA4GH</a></li>
</ul>
<a href="presentations/2020-2024/2020-04-20-Michael-Baudis__GA4GH%20-and-SchemaBlocks-for-TASC__presentation/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      April 03, 2020
      
    </span>
</aside>

    
    
    

<h3 class="title_news">A harmonized meta-knowledgebase of clinical interpretations of somatic genomic variants in cancer</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="alex-h-wagner-brian-walsh-georgia-mayfield-david-tamborero-dmitriy-sonkin-kilannin-krysiak-jordi-deu-pons-ryan-p-duren-jianjiong-gao-julie-mcmurry-sara-patterson-catherine-del-vecchio-fitz-beth-a-pitel-nuria-lopez-bigas-mark-lawler-jeremy-goecks-malachi-griffith-obi-l-griffith-adam-a-margolin-variant-interpretation-for-cancer-consortium">Alex H. Wagner, Brian Walsh, Georgia Mayfield, David Tamborero, Dmitriy Sonkin, Kilannin Krysiak, Jordi Deu-Pons, Ryan P. Duren, Jianjiong Gao, Julie McMurry, Sara Patterson, Catherine del Vecchio Fitz, Beth A. Pitel, ..., Nuria Lopez-Bigas, Mark Lawler, Jeremy Goecks, Malachi Griffith, Obi L. Griffith, Adam A. Margolin &amp; Variant Interpretation for Cancer Consortium<a class="headerlink" href="#alex-h-wagner-brian-walsh-georgia-mayfield-david-tamborero-dmitriy-sonkin-kilannin-krysiak-jordi-deu-pons-ryan-p-duren-jianjiong-gao-julie-mcmurry-sara-patterson-catherine-del-vecchio-fitz-beth-a-pitel-nuria-lopez-bigas-mark-lawler-jeremy-goecks-malachi-griffith-obi-l-griffith-adam-a-margolin-variant-interpretation-for-cancer-consortium" title="Permanent link">&para;</a></h4>
<h5 id="nature-genetics-volume-52-pages-448457-2020">Nature Genetics volume 52, pages 448–457 (2020)<a class="headerlink" href="#nature-genetics-volume-52-pages-448457-2020" title="Permanent link">&para;</a></h5>
<p>Precision oncology relies on accurate discovery and interpretation of genomic variants, enabling individualized diagnosis, prognosis and therapy selection. We found that six prominent somatic cancer variant knowledgebases were highly disparate in content, structure and supporting primary literature, impeding consensus when evaluating variants and their relevance in a clinical setting. We developed a framework for harmonizing variant interpretations to produce a meta-knowledgebase of 12,856 aggregate interpretations.
<a href="publications/2020-04-03-publication-Wagner-vicc-NatGen/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  

              </article>
            </div>
          
          
  <script>var tabs=__md_get("__tabs");if(Array.isArray(tabs))e:for(var set of document.querySelectorAll(".tabbed-set")){var labels=set.querySelector(".tabbed-labels");for(var tab of tabs)for(var label of labels.getElementsByTagName("label"))if(label.innerText.trim()===tab){var input=document.getElementById(label.htmlFor);input.checked=!0;continue e}}</script>

<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      &copy; Copyright 2007 - 2025 Michael Baudis
    </div>
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
        
<div class="md-social">
  
    
    
    
    
      
      
    
    <a href="https://github.com/baudisgroup" target="_blank" rel="noopener" title="github.com" class="md-social__link">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><!--! Font Awesome Free 7.1.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M173.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6m-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3m44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9M252.8 8C114.1 8 8 113.3 8 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C436.2 457.8 504 362.9 504 252 504 113.3 391.5 8 252.8 8M105.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1m-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7m32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1m-11.4-14.7c-1.6 1-1.6 3.6 0 5.9s4.3 3.3 5.6 2.3c1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2"/></svg>
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    
      
      
      <script id="__config" type="application/json">{"annotate": null, "base": ".", "features": ["content.tabs.link", "search.highlight", "search.share", "navigation.instant", "navigation.tracking", "navigation.sections", "toc.integrate"], "search": "assets/javascripts/workers/search.2c215733.min.js", "tags": null, "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}, "version": null}</script>
    
    
      <script src="assets/javascripts/bundle.79ae519e.min.js"></script>
      
    
  </body>
</html>