
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
        <meta name="description" content="Computational Oncogenomics Group @ UZH & SIB">
      
      
        <meta name="author" content="Michael Baudis">
      
      
      <link rel="icon" href="../img/favicon.ico">
      <meta name="generator" content="mkdocs-1.2.3, mkdocs-material-8.1.8">
    
    
      
        <title>Baudisgroup Publications - baudisgroup @ UZH & SIB</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.6e60f8b8.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.e6a45f82.min.css">
        
          
          
          <meta name="theme-color" content="#02a6f2">
        
      
    
    
    
      
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,400i,700%7CRoboto+Mono&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../css/theme_overrides.css">
    
    <script>__md_scope=new URL("..",location),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="" data-md-color-primary="light-blue" data-md-color-accent="indigo">
  
    
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="baudisgroup @ UZH &amp; SIB" class="md-header__button md-logo" aria-label="baudisgroup @ UZH & SIB" data-md-component="logo">
      
  <img src="../img/site_logo.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            baudisgroup @ UZH & SIB
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Baudisgroup Publications
            
          </span>
        </div>
      </div>
    </div>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"/></svg>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
          <a href="javascript:void(0)" class="md-search__icon md-icon" aria-label="Share" data-clipboard data-clipboard-text="" data-md-component="search-share" tabindex="-1">
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M18 16.08c-.76 0-1.44.3-1.96.77L8.91 12.7c.05-.23.09-.46.09-.7 0-.24-.04-.47-.09-.7l7.05-4.11c.54.5 1.25.81 2.04.81a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3c0 .24.04.47.09.7L8.04 9.81C7.5 9.31 6.79 9 6 9a3 3 0 0 0-3 3 3 3 0 0 0 3 3c.79 0 1.5-.31 2.04-.81l7.12 4.15c-.05.21-.08.43-.08.66 0 1.61 1.31 2.91 2.92 2.91 1.61 0 2.92-1.3 2.92-2.91A2.92 2.92 0 0 0 18 16.08z"/></svg>
          </a>
        
        <button type="reset" class="md-search__icon md-icon" aria-label="Clear" tabindex="-1">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/baudisgroup/baudisgroup.github.io/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 480 512"><path d="M186.1 328.7c0 20.9-10.9 55.1-36.7 55.1s-36.7-34.2-36.7-55.1 10.9-55.1 36.7-55.1 36.7 34.2 36.7 55.1zM480 278.2c0 31.9-3.2 65.7-17.5 95-37.9 76.6-142.1 74.8-216.7 74.8-75.8 0-186.2 2.7-225.6-74.8-14.6-29-20.2-63.1-20.2-95 0-41.9 13.9-81.5 41.5-113.6-5.2-15.8-7.7-32.4-7.7-48.8 0-21.5 4.9-32.3 14.6-51.8 45.3 0 74.3 9 108.8 36 29-6.9 58.8-10 88.7-10 27 0 54.2 2.9 80.4 9.2 34-26.7 63-35.2 107.8-35.2 9.8 19.5 14.6 30.3 14.6 51.8 0 16.4-2.6 32.7-7.7 48.2 27.5 32.4 39 72.3 39 114.2zm-64.3 50.5c0-43.9-26.7-82.6-73.5-82.6-18.9 0-37 3.4-56 6-14.9 2.3-29.8 3.2-45.1 3.2-15.2 0-30.1-.9-45.1-3.2-18.7-2.6-37-6-56-6-46.8 0-73.5 38.7-73.5 82.6 0 87.8 80.4 101.3 150.4 101.3h48.2c70.3 0 150.6-13.4 150.6-101.3zm-82.6-55.1c-25.8 0-36.7 34.2-36.7 55.1s10.9 55.1 36.7 55.1 36.7-34.2 36.7-55.1-10.9-55.1-36.7-55.1z"/></svg>
  </div>
  <div class="md-source__repository">
    baudisgroup.github.io
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  

<nav class="md-nav md-nav--primary md-nav--integrated" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="baudisgroup @ UZH &amp; SIB" class="md-nav__button md-logo" aria-label="baudisgroup @ UZH & SIB" data-md-component="logo">
      
  <img src="../img/site_logo.png" alt="logo">

    </a>
    baudisgroup @ UZH & SIB
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/baudisgroup/baudisgroup.github.io/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 480 512"><path d="M186.1 328.7c0 20.9-10.9 55.1-36.7 55.1s-36.7-34.2-36.7-55.1 10.9-55.1 36.7-55.1 36.7 34.2 36.7 55.1zM480 278.2c0 31.9-3.2 65.7-17.5 95-37.9 76.6-142.1 74.8-216.7 74.8-75.8 0-186.2 2.7-225.6-74.8-14.6-29-20.2-63.1-20.2-95 0-41.9 13.9-81.5 41.5-113.6-5.2-15.8-7.7-32.4-7.7-48.8 0-21.5 4.9-32.3 14.6-51.8 45.3 0 74.3 9 108.8 36 29-6.9 58.8-10 88.7-10 27 0 54.2 2.9 80.4 9.2 34-26.7 63-35.2 107.8-35.2 9.8 19.5 14.6 30.3 14.6 51.8 0 16.4-2.6 32.7-7.7 48.2 27.5 32.4 39 72.3 39 114.2zm-64.3 50.5c0-43.9-26.7-82.6-73.5-82.6-18.9 0-37 3.4-56 6-14.9 2.3-29.8 3.2-45.1 3.2-15.2 0-30.1-.9-45.1-3.2-18.7-2.6-37-6-56-6-46.8 0-73.5 38.7-73.5 82.6 0 87.8 80.4 101.3 150.4 101.3h48.2c70.3 0 150.6-13.4 150.6-101.3zm-82.6-55.1c-25.8 0-36.7 34.2-36.7 55.1s10.9 55.1 36.7 55.1 36.7-34.2 36.7-55.1-10.9-55.1-36.7-55.1z"/></svg>
  </div>
  <div class="md-source__repository">
    baudisgroup.github.io
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        Baudisgroup Home
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../group" class="md-nav__link">
        Group
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="." class="md-nav__link">
        Publications
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../presentations" class="md-nav__link">
        Presentations
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="https://schemablocks.org" class="md-nav__link">
        SchemaBlocks {S}[B] &#8599;
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="https://beacon-project.io" class="md-nav__link">
        Beacon Project &#8599;
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="https://www.mls.uzh.ch/en/research/baudis.html" class="md-nav__link">
        Michael Baudis @ UZH &#8599;
      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
          
          <div class="md-content" data-md-component="content">
            <article class="md-content__inner md-typeset">
              
  <h1 id="baudisgroup-publications">Baudisgroup Publications<a class="headerlink" href="#baudisgroup-publications" title="Permanent link">&para;</a></h1>
<p>A list of publication can also be retrieved through <a href="https://europepmc.org/search?query=0000-0002-9903-4248">EuropePMC</a>.</p>
  

    
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
    
  
      
      
        
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  
      
      
        
          
      
    
  

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      July 01, 2022
      
    </span>
</aside>
    <h3 class="title_news">Candidate targets of copy number deletion events across 17 cancer types</h3>

<h5 class="title_news">Identifying cancer related genes against the background of somatic CNV events</h5>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="huang-q-and-baudis-m">Huang Q and Baudis M<a class="headerlink" href="#huang-q-and-baudis-m" title="Permanent link">&para;</a></h4>
<h5 id="biorxiv-doiorg10110120220629498080">bioRxiv, <a href="https://doi.org/10.1101/2022.06.29.498080">doi.org/10.1101/2022.06.29.498080</a><a class="headerlink" href="#biorxiv-doiorg10110120220629498080" title="Permanent link">&para;</a></h5>

<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      June 15, 2022
      
    </span>
</aside>
    <h3 class="title_news">The GA4GH Phenopacket schema defines a computable representation of clinical data</h3>

<h5 class="title_news">Phenopackets v2 publication</h5>

<!-- Use a hidden p tag to provide a preview -->
<p><img alt="Cell Genomics logo" src="/img/nature-biotechnology-logo.svg" style="float: right; width: 200px; margin-top: -5px;" /></p>
<h4 id="jacobsen-job-baudis-m-baynam-gs-beckmann-js-beltran-s-buske-oj-callahan-tj-chute-cg-courtot-m-danis-d-elemento-o-essenwanger-a-freimuth-rr-gargano-ma-groza-t-hamosh-a-harris-nl-kaliyaperumal-r-lloyd-kck-khalifa-a-krawitz-pm-kohler-s-laraway-bj-lehvaslaiho-h-matalonga-l-mcmurry-ja-metke-jimenez-a-mungall-cj-munoz-torres-mc-ogishima-s-papakonstantinou-a-piscia-d-pontikos-n-queralt-rosinach-n-roos-m-sass-j-schofield-pn-seelow-d-siapos-a-smedley-d-smith-ld-steinhaus-r-sundaramurthi-jc-swietlik-em-thun-s-vasilevsky-na-wagner-ah-warner-jl-weiland-c-axton-m-babb-l-boerkoel-cf-chaudhari-bp-chin-h-l-dumontier-m-gazzaz-n-hansen-dp-hochheiser-h-kinsler-va-lochmuller-h-mankovich-ar-saunders-gi-sergouniotis-pi-thompson-r-zankl-a-haendel-ma-robinson-pn-the-gaghpmc">Jacobsen JOB, Baudis M, Baynam GS, Beckmann JS, Beltran S, Buske OJ, Callahan TJ, Chute CG, Courtot M, Danis D, Elemento O, Essenwanger A, Freimuth RR, Gargano MA, Groza T, Hamosh A, Harris NL, Kaliyaperumal R, Lloyd KCK, Khalifa A, Krawitz PM, Köhler S
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      March 17, 2022
      
    </span>
</aside>
    <h3 class="title_news">Beacon v2 and Beacon networks: A "lingua franca" for federated data discovery in biomedical genomics, and beyond</h3>

<h5 class="title_news">Beacon v2 publication</h5>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="rambla-j-baudis-m-ariosa-r-beck-t-fromont-la-navarro-a-paloots-r-rueda-m-saunders-g-singh-b-spalding-jd">Rambla J, Baudis M, Ariosa R, Beck T, Fromont LA, Navarro A, Paloots R, Rueda M, Saunders G, Singh B, Spalding JD.<a class="headerlink" href="#rambla-j-baudis-m-ariosa-r-beck-t-fromont-la-navarro-a-paloots-r-rueda-m-saunders-g-singh-b-spalding-jd" title="Permanent link">&para;</a></h4>
<h5 id="human-mutation-2022-mar-17-pmid35297548">Human Mutation. 2022 Mar 17. <a href="https://onlinelibrary.wiley.com/doi/10.1002/humu.24369">PMID:35297548</a><a class="headerlink" href="#human-mutation-2022-mar-17-pmid35297548" title="Permanent link">&para;</a></h5>
<p><strong>Abstract</strong> Beacon is a basic data discovery protocol issued by the Global Alliance for Genomics and Health (GA4GH). The main goal addressed by version 1 of the Beacon protocol was to test the feasibility of broadly sharing human genomic data, through providing simple "yes" or "no" responses to queries about the presence of a given variant in datasets hosted by Beacon providers.</p>

<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      November 30, 2021
      
    </span>
</aside>
    <h3 class="title_news">The GA4GH Phenopacket schema: A computable representation of clinical data for precision medicine</h3>

<h5 class="title_news">Phenopackets v2 preprint</h5>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="jacobsen-job-baudis-m-baynam-gs-beckmann-js-beltran-s-callahan-tj-chute-cg-courtot-m-danis-d-elemento-o-freimuth-rr-gargano-ma-groza-t-hamosh-a-harris-nl-kaliyaperumal-r-khalifa-a-krawitz-pm-koehler-s-laraway-bj-lehvaeslaiho-h-lloyd-kc-matalonga-l-mcmurry-ja-metke-jimenez-a-mungall-cj-munoz-torres-mc-ogishima-s-papakonstantinou-a-piscia-d-pontikos-n-queralt-rosinach-n-roos-m-schofield-pn-siapos-a-smedley-d-smith-ld-steinhaus-r-sundaramurthi-jc-swietlik-em-thun-s-vasilevsky-na-wagner-ah-warner-jl-weiland-c-haendel-ma-robinson-pn">Jacobsen JOB, Baudis M, Baynam GS, Beckmann JS, Beltran S, Callahan TJ, Chute CG, Courtot M, Danis D, Elemento O, Freimuth RR, Gargano MA, Groza T, Hamosh A, Harris NL, Kaliyaperumal R, Khalifa A, Krawitz PM, Koehler S
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      November 10, 2021
      
    </span>
</aside>
    <h3 class="title_news">The GA4GH Variation Representation Specification (VRS): a Computational Framework for the Precise Representation and Federated Identification of Molecular Variation.</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="alex-h-wagner-lawrence-babb-gil-alterovitz-michael-baudis-matthew-brush-daniel-l-cameron-melissa-cline-malachi-griffith-obi-l-griffith-sarah-e-hunt-david-kreda-jennifer-m-lee-stephanie-li-javier-lopez-eric-moyer-tristan-nelson-ronak-y-patel-kevin-riehle-peter-n-robinson-shawn-rynearson-helen-schuilenburg-kirill-tsukanov-brian-walsh-melissa-konopko-heidi-l-rehm-andrew-d-yates-robert-r-freimuth-reece-k-hart">Alex H. Wagner, Lawrence Babb, Gil Alterovitz, Michael Baudis, Matthew Brush, Daniel L. Cameron, Melissa Cline 
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      November 10, 2021
      
    </span>
</aside>
    <h3 class="title_news">GA4GH - International policies and standards</h3>

<!-- Use a hidden p tag to provide a preview -->
<h3 id="international-federation-of-genomic-medicine-databases-using-ga4gh-standards">International federation of genomic medicine databases using GA4GH standards<a class="headerlink" href="#international-federation-of-genomic-medicine-databases-using-ga4gh-standards" title="Permanent link">&para;</a></h3>
<h4 id="adrian-thorogood-heidi-l-rehm-peter-goodhand-angela-jh-page-yann-joly-michael-baudis-jordi-rambla-arcadi-navarro-tommi-h-nyronen-mikael-linden-edward-s-dove-marc-fiume-michael-brudno-melissa-s-cline-ewan-birney">Adrian Thorogood, Heidi L. Rehm, Peter Goodhand, Angela J.H. Page, Yann Joly, Michael Baudis, Jordi Rambla, Arcadi Navarro, Tommi H. Nyronen, Mikael Linden, Edward S. Dove, Marc Fiume, Michael Brudno, Melissa S. Cline, Ewan Birney<a class="headerlink" href="#adrian-thorogood-heidi-l-rehm-peter-goodhand-angela-jh-page-yann-joly-michael-baudis-jordi-rambla-arcadi-navarro-tommi-h-nyronen-mikael-linden-edward-s-dove-marc-fiume-michael-brudno-melissa-s-cline-ewan-birney" title="Permanent link">&para;</a></h4>
<h5 id="thorogood-adrian-et-al-cell-genomics-volume-1-issue-2-100032-doi101016jxgen2021100032">Thorogood, Adrian et al. Cell Genomics, Volume 1, Issue 2, 100032 <a href="https://doi.org/10.1016/j.xgen.2021.100032">doi:10.1016/j.xgen.2021.100032</a><a class="headerlink" href="#thorogood-adrian-et-al-cell-genomics-volume-1-issue-2-100032-doi101016jxgen2021100032" title="Permanent link">&para;</a></h5>
<h4 id="note">Note<a class="headerlink" href="#note" title="Permanent link">&para;</a></h4>
<p><img alt="Cell Genomics logo" src="/img/logo-cell-genomics.png" style="float: right; width: 150px; margin-top: -50px;" />This article was published as part of a special GA4GH edition of <a href="https://www.cell.com/cell-genomics/issue?pii=S2666-979X(21)X0003-1">Cell Genomics</a>.</p>

<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      November 10, 2021
      
    </span>
</aside>
    <h3 class="title_news">GA4GH: International policies and standards for data sharing across genomic research and healthcare</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="heidi-l-rehm-angela-jh-page-lindsay-smith-jeremy-b-adams-gil-alterovitz-lawrence-j-babb-maxmillian-p-barkley-michael-baudis-michael-js-beauvais-tim-beck-jacques-s-beckmann-sergi-beltran-david-bernick-alexander-bernier-james-k-bonfield-tiffany-f-boughtwood-guillaume-bourque-sarion-r-bowers-anthony-j-brookes-michael-brudno-matthew-h-brush-david-bujold-tony-burdett-orion-j-buske-moran-n-cabili-daniel-l-cameron-robert-j-carroll-esmeralda-casas-silva-debyani-chakravarty-bimal-p-chaudhari-shu-hui-chen-j-michael-cherry-justina-chung-melissa-cline-hayley-l-clissold-robert-m-cook-deegan-melanie-courtot-fiona-cunningham-miro-cupak-robert-m-davies-danielle-denisko-megan-j-doerr-lena-i-dolman-edward-s-dove-l-jonathan-dursi-stephanie-om-dyke-james-a-eddy-karen-eilbeck-kyle-p-ellrott-susan-fairley-khalid-a-fakhro-helen-v-firth-michael-s-fitzsimons-marc-fiume-paul-flicek-ian-m-fore-mallory-a-freeberg-robert-r-freimuth-lauren-a-fromont-jonathan-fuerth-clara-l-gaff-weiniu-gan-elena-m-ghanaim-david-glazer-robert-c-green-malachi-griffith-obi-l-griffith-robert-l-grossman-tudor-groza-jaime-m-guidry-auvil-roderic-guigo-dipayan-gupta-melissa-a-haendel-ada-hamosh-david-p-hansen-reece-k-hart-dean-mitchell-hartley-david-haussler-rachele-m-hendricks-sturrup-calvin-wl-ho-ashley-e-hobb-michael-m-hoffman-oliver-m-hofmann-petr-holub-jacob-shujui-hsu-jean-pierre-hubaux-sarah-e-hunt-ammar-husami-julius-o-jacobsen-saumya-s-jamuar-elizabeth-l-janes-francis-jeanson-aina-jene-amber-l-johns-yann-joly-steven-jm-jones-alexander-kanitz-kazuto-kato-thomas-m-keane-kristina-kekesi-lafrance-jerome-kelleher-giselle-kerry-seik-soon-khor-bartha-m-knoppers-melissa-a-konopko-kenjiro-kosaki-martin-kuba-jonathan-lawson-rasko-leinonen-stephanie-li-michael-f-lin-mikael-linden-xianglin-liu-isuru-udara-liyanage-javier-lopez-anneke-m-lucassen-michael-lukowski-alice-l-mann-john-marshall-michele-mattioni-alejandro-metke-jimenez-anna-middleton-richard-j-milne-fruzsina-molnar-gabor-nicola-mulder-monica-c-munoz-torres-rishi-nag-hidewaki-nakagawa-jamal-nasir-arcadi-navarro-tristan-h-nelson-ania-niewielska-amy-nisselle-jeffrey-niu-tommi-h-nyronen-brian-d-oconnor-sabine-oesterle-soichi-ogishima-vivian-ota-wang-laura-ad-paglione-emilio-palumbo-helen-e-parkinson-anthony-a-philippakis-angel-d-pizarro-andreas-prlic-jordi-rambla-augusto-rendon-renee-a-rider-peter-n-robinson-kurt-w-rodarmer-laura-lyman-rodriguez-alan-f-rubin-manuel-rueda-gregory-a-rushton-rosalyn-s-ryan-gary-i-saunders-helen-schuilenburg-torsten-schwede-serena-scollen-alexander-senf-nathan-c-sheffield-neerjah-skantharajah-albert-v-smith-heidi-j-sofia-dylan-spalding-amanda-b-spurdle-zornitza-stark-lincoln-d-stein-makoto-suematsu-patrick-tan-jonathan-a-tedds-alastair-a-thomson-adrian-thorogood-timothy-l-tickle-katsushi-tokunaga-juha-tornroos-david-torrents-sean-upchurch-alfonso-valencia-roman-valls-guimera-jessica-vamathevan-susheel-varma-danya-f-vears-coby-viner-craig-voisin-alex-h-wagner-susan-e-wallace-brian-p-walsh-marc-s-williams-eva-c-winkler-barbara-j-wold-grant-m-wood-j-patrick-woolley-chisato-yamasaki-andrew-d-yates-christina-k-yung-lyndon-j-zass-ksenia-zaytseva-junjun-zhang-peter-goodhand-kathryn-north-ewan-birney">Heidi L. Rehm, Angela J.H. Page, Lindsay Smith, Jeremy B. Adams, Gil Alterovitz, Lawrence J. Babb, Maxmillian P. Barkley, Michael Baudis, Michael J.S. Beauvais, Tim Beck, Jacques S. Beckmann, Sergi Beltran, David Bernick, Alexander Bernier, James K. Bonfield, Tiffany F. Boughtwood, Guillaume Bourque, Sarion R. Bowers, Anthony J. Brookes, Michael Brudno, Matthew H. Brush, David Bujold, Tony Burdett, Orion J. Buske, Moran N. Cabili
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      June 30, 2021
      
    </span>
</aside>
    <h3 class="title_news">The Progenetix oncogenomic resource in 2021</h3>

<h5 class="title_news">Article describing the current content & technical status of progenetix.org</h5>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="qingyao-huang-paula-carrio-cordo-bo-gao-rahel-paloots-michael-baudis">Qingyao Huang, Paula Carrio Cordo, Bo Gao, Rahel Paloots, Michael Baudis<a class="headerlink" href="#qingyao-huang-paula-carrio-cordo-bo-gao-rahel-paloots-michael-baudis" title="Permanent link">&para;</a></h4>
<h5 id="database-oxford-2021-jul-172021baab043">Database (Oxford). 2021 Jul 17;2021:baab043.<a class="headerlink" href="#database-oxford-2021-jul-172021baab043" title="Permanent link">&para;</a></h5>
<ul>
<li>doi: 10.1093/database/baab043.</li>
<li>PMID: 34272855</li>
<li>PMCID: PMC8285936.</li>
<li>bioRxiv. doi: <a href="https://doi.org/10.1101/2021.02.15.428237">doi.org/10.1101/2021.02.15.428237</a></li>
</ul>
<p><img alt="DATABASE logo" src="/img/logo-database-journal-300x150.png" style="float: right; width: 150px; margin-top: -100px;" />This article provides an overview of recent changes and additions to the Progenetix database and the services provided through the resource.</p>

<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      May 13, 2021
      
    </span>
</aside>
    <h3 class="title_news">Signatures of Discriminative CNA in 31 Cancer Subtypes</h3>

<!-- Use a hidden p tag to provide a preview -->
<h3 id="bo-gao-and-michael-baudis-2021">Bo Gao and Michael Baudis (2021)<a class="headerlink" href="#bo-gao-and-michael-baudis-2021" title="Permanent link">&para;</a></h3>
<h4 id="published-at-frontiers-in-genetics-2021-05-13">Published at <em>Frontiers in Genetics</em>, 2021-05-13<a class="headerlink" href="#published-at-frontiers-in-genetics-2021-05-13" title="Permanent link">&para;</a></h4>
<h4 id="abstract">Abstract<a class="headerlink" href="#abstract" title="Permanent link">&para;</a></h4>
<p>Copy number aberrations (CNA) are one of the most important classes of genomic mutations relatedto oncogenetic effects. In the past three decades, a vast amount of CNA data has been generated bymolecular-cytogenetic and genome sequencing based methods. While this data has been instrumentalin the identification of cancer-related genes and promoted research into the relation between CNA andhisto-pathologically defined cancer types, the heterogeneity of source data and derived CNV profilespose great challenges for data integration and comparative analysis. Furthermore, a majority of exist-ing studies have been focused on the association of CNA to pre-selected ”driver” genes with limitedapplication to rare drivers and other genomic elements.</p>

<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      March 01, 2021
      
    </span>
</aside>
    <h3 class="title_news">Copy number variant heterogeneity among cancer types reflects inconsistent concordance with diagnostic classifications</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="paula-carrio-cordo-and-michael-baudis">Paula Carrio Cordo and Michael Baudis<a class="headerlink" href="#paula-carrio-cordo-and-michael-baudis" title="Permanent link">&para;</a></h4>
<h5 id="biorxiv-doi-doiorg10110120210301433348">bioRxiv. doi: <a href="https://doi.org/10.1101/2021.03.01.433348">doi.org/10.1101/2021.03.01.433348</a><a class="headerlink" href="#biorxiv-doi-doiorg10110120210301433348" title="Permanent link">&para;</a></h5>
<p>This article explores the correlation between subsets of cancer entities, grouped by their somatic CNV patterns, and levels of diagnostic classification systems.</p>

<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      August 10, 2020
      
    </span>
</aside>
    <h3 class="title_news">The Ubiquitin Ligase TRIP12 Limits PARP1 Trapping and Constrains PARP Inhibitor Efficiency</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="marco-gatti-ralph-imhof-qingyao-huang-michael-baudis-matthias-altmeyer">Marco Gatti, Ralph Imhof, Qingyao Huang, Michael Baudis, Matthias Altmeyer<a class="headerlink" href="#marco-gatti-ralph-imhof-qingyao-huang-michael-baudis-matthias-altmeyer" title="Permanent link">&para;</a></h4>
<h5 id="cell-rep-2020-aug-4-doi-101016jcelrep2020107985">Cell Rep. 2020 Aug 4 DOI: <a href="https://doi.org/10.1016/j.celrep.2020.107985">10.1016/j.celrep.2020.107985</a><a class="headerlink" href="#cell-rep-2020-aug-4-doi-101016jcelrep2020107985" title="Permanent link">&para;</a></h5>

<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      May 14, 2020
      
    </span>
</aside>
    <h3 class="title_news">Oncology Informatics: Status Quo and Outlook - Review</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="paul-martin-putora-michael-baudis-beth-m-beadle-issam-el-naqa-frank-a-giordano-and-nils-h-nicolay">Paul Martin Putora, Michael Baudis, Beth M. Beadle, Issam El Naqa, Frank A. Giordano and Nils H. Nicolay<a class="headerlink" href="#paul-martin-putora-michael-baudis-beth-m-beadle-issam-el-naqa-frank-a-giordano-and-nils-h-nicolay" title="Permanent link">&para;</a></h4>
<h5 id="oncology-2020-05-14-doi-101159000507586-review">Oncology, 2020-05-14. <a href="https://doi.org/10.1159/000507586">DOI 10.1159/000507586</a> (Review)<a class="headerlink" href="#oncology-2020-05-14-doi-101159000507586-review" title="Permanent link">&para;</a></h5>

<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      May 06, 2020
      
    </span>
</aside>
    <h3 class="title_news">Minimum Error Calibration and Normalization for Genomic Copy Number Analysis</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="bo-gao-and-michael-baudis-2020">Bo Gao and Michael Baudis (2020)<a class="headerlink" href="#bo-gao-and-michael-baudis-2020" title="Permanent link">&para;</a></h4>
<h5 id="biorxiv-2019-07-31-doi-101101720854">bioRxiv, 2019-07-31. <a href="https://doi.org/10.1101/720854">DOI 10.1101/720854</a><a class="headerlink" href="#biorxiv-2019-07-31-doi-101101720854" title="Permanent link">&para;</a></h5>
<h5 id="genomics-volume-112-issue-5-september-2020-pages-3331-3341-accepted-2020-05-06-doiorg101016jygeno202005008">Genomics, Volume 112, Issue 5, September 2020, Pages 3331-3341, accepted 2020-05-06 <a href="https://doi.org/10.1016/j.ygeno.2020.05.008">doi.org/10.1016/j.ygeno.2020.05.008</a>.<a class="headerlink" href="#genomics-volume-112-issue-5-september-2020-pages-3331-3341-accepted-2020-05-06-doiorg101016jygeno202005008" title="Permanent link">&para;</a></h5>

<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      April 03, 2020
      
    </span>
</aside>
    <h3 class="title_news">A harmonized meta-knowledgebase of clinical interpretations of somatic genomic variants in cancer</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="alex-h-wagner-brian-walsh-georgia-mayfield-david-tamborero-dmitriy-sonkin-kilannin-krysiak-jordi-deu-pons-ryan-p-duren-jianjiong-gao-julie-mcmurry-sara-patterson-catherine-del-vecchio-fitz-beth-a-pitel-ozman-u-sezerman-kyle-ellrott-jeremy-l-warner-damian-t-rieke-tero-aittokallio-ethan-cerami-deborah-i-ritter-lynn-m-schriml-robert-r-freimuth-melissa-haendel-gordana-raca-subha-madhavan-michael-baudis-jacques-s-beckmann-rodrigo-dienstmann-debyani-chakravarty-xuan-shirley-li-susan-mockus-olivier-elemento-nikolaus-schultz-nuria-lopez-bigas-mark-lawler-jeremy-goecks-malachi-griffith-obi-l-griffith-adam-a-margolin-variant-interpretation-for-cancer-consortium">Alex H. Wagner, Brian Walsh, Georgia Mayfield, David Tamborero, Dmitriy Sonkin, Kilannin Krysiak, Jordi Deu-Pons, Ryan P. Duren, Jianjiong Gao, Julie McMurry, Sara Patterson, Catherine del Vecchio Fitz, Beth A. Pitel, Ozman U. Sezerman, Kyle Ellrott, Jeremy L. Warner, Damian T. Rieke, Tero Aittokallio, Ethan Cerami, Deborah I. Ritter, Lynn M. Schriml, Robert R. Freimuth, Melissa Haendel, Gordana Raca, Subha Madhavan, Michael Baudis, Jacques S. Beckmann
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      April 02, 2020
      
    </span>
</aside>
    <h3 class="title_news">Geographic assessment of cancer genome profiling studies</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="paula-carrio-cordo-elise-acheson-qingyao-huang-and-michael-baudis-2020">Paula Carrio Cordo, Elise Acheson, Qingyao Huang and Michael Baudis (2020)<a class="headerlink" href="#paula-carrio-cordo-elise-acheson-qingyao-huang-and-michael-baudis-2020" title="Permanent link">&para;</a></h4>
<h5 id="database-volume-2020-2020-baaa009-doiorg101093databasebaaa009">DATABASE, Volume 2020, 2020, baaa009, <a href="https://doi.org/10.1093/database/baaa009">doi.org/10.1093/database/baaa009</a><a class="headerlink" href="#database-volume-2020-2020-baaa009-doiorg101093databasebaaa009" title="Permanent link">&para;</a></h5>

<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      March 16, 2020
      
    </span>
</aside>
    <h3 class="title_news">Enabling population assignment from cancer genomes with SNP2pop</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="huang-q-and-baudis-m-2020">Huang Q and Baudis M. (2020)<a class="headerlink" href="#huang-q-and-baudis-m-2020" title="Permanent link">&para;</a></h4>
<h5 id="sci-rep-10-4846-2020-doiorg101038s41598-020-61854-x">Sci Rep 10, 4846 (2020). <a href="https://doi.org/10.1038/s41598-020-61854-x">doi.org/10.1038/s41598-020-61854-x</a><a class="headerlink" href="#sci-rep-10-4846-2020-doiorg101038s41598-020-61854-x" title="Permanent link">&para;</a></h5>

<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      January 11, 2020
      
    </span>
</aside>
    <h3 class="title_news">Geographic assessment of cancer genome profiling studies</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="paula-carrio-cordo-elise-acheson-qingyao-huang-and-michael-baudis-2020">Paula Carrio Cordo, Elise Acheson, Qingyao Huang and Michael Baudis (2020)<a class="headerlink" href="#paula-carrio-cordo-elise-acheson-qingyao-huang-and-michael-baudis-2020" title="Permanent link">&para;</a></h4>
<h5 id="biorxiv-2020-11-01-doi-101101827683">bioRxiv, 2020-11-01. <a href="https://doi.org/10.1101/827683">DOI 10.1101/827683</a><a class="headerlink" href="#biorxiv-2020-11-01-doi-101101827683" title="Permanent link">&para;</a></h5>

<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      August 27, 2019
      
    </span>
</aside>
    <h3 class="title_news">Leveraging European infrastructures to access 1 million human genomes by 2022</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="gary-saunders-michael-baudis-regina-becker-sergi-beltran-christophe-beroud-ewan-birney-cath-brooksbank-sren-brunak-marc-van-den-bulcke-rachel-drysdale-salvador-capella-gutierrez-paul-flicek-francesco-florindi-peter-goodhand-ivo-gut-jaap-heringa-petr-holub-jef-hooyberghs-nick-juty-thomas-m-keane-jan-o-korbel-ilkka-lappalainen-brane-leskosek-gert-matthijs-michaela-th-mayrhofer-andres-metspalu-steven-newhouse-tommi-nyronen-angela-page-bengt-persson-aarno-palotie-helen-parkinson-jordi-rambla-david-salgado-erik-steinfelder-morris-a-swertz-susheel-varma-niklas-blomberg-and-serena-scollen">Gary Saunders, Michael Baudis, Regina Becker, Sergi Beltran, Christophe Béroud, Ewan Birney, Cath Brooksbank, Søren Brunak, Marc Van den Bulcke, Rachel Drysdale, Salvador Capella-Gutierrez, Paul Flicek, Francesco Florindi, Peter Goodhand, Ivo Gut, Jaap Heringa, Petr Holub, Jef Hooyberghs, Nick Juty, Thomas M. Keane, Jan O. Korbel, Ilkka Lappalainen, Brane Leskosek, Gert Matthijs, Michaela Th. Mayrhofer, Andres Metspalu, Steven Newhouse, Tommi Nyrönen, Angela Page, Bengt Persson, Aarno Palotie, Helen Parkinson, Jordi Rambla, David Salgado, Erik Steinfelder, Morris A. Swertz, Susheel Varma, Niklas Blomberg, and Serena Scollen<a class="headerlink" href="#gary-saunders-michael-baudis-regina-becker-sergi-beltran-christophe-beroud-ewan-birney-cath-brooksbank-sren-brunak-marc-van-den-bulcke-rachel-drysdale-salvador-capella-gutierrez-paul-flicek-francesco-florindi-peter-goodhand-ivo-gut-jaap-heringa-petr-holub-jef-hooyberghs-nick-juty-thomas-m-keane-jan-o-korbel-ilkka-lappalainen-brane-leskosek-gert-matthijs-michaela-th-mayrhofer-andres-metspalu-steven-newhouse-tommi-nyronen-angela-page-bengt-persson-aarno-palotie-helen-parkinson-jordi-rambla-david-salgado-erik-steinfelder-morris-a-swertz-susheel-varma-niklas-blomberg-and-serena-scollen" title="Permanent link">&para;</a></h4>
<h5 id="nature-reviews-genetics-volume-20-pages693701-2019">Nature Reviews Genetics volume 20, pages693–701 (2019)<a class="headerlink" href="#nature-reviews-genetics-volume-20-pages693701-2019" title="Permanent link">&para;</a></h5>
<!--  CONTENT  -->

<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      July 31, 2019
      
    </span>
</aside>
    <h3 class="title_news">Minimum Error Calibration and Normalization for Genomic Copy Number Analysis</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="bo-gao-and-michael-baudis-2019">Bo Gao and Michael Baudis (2019)<a class="headerlink" href="#bo-gao-and-michael-baudis-2019" title="Permanent link">&para;</a></h4>
<h5 id="biorxiv-2019-07-31-doi-101101720854">bioRxiv, 2019-07-31. <a href="https://doi.org/10.1101/720854">DOI 10.1101/720854</a><a class="headerlink" href="#biorxiv-2019-07-31-doi-101101720854" title="Permanent link">&para;</a></h5>

<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      January 23, 2019
      
    </span>
</aside>
    <h3 class="title_news">Federated discovery and sharing of genomic data using Beacons</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="miroslav-cupak-stephen-keenan-jordi-rambla-sabela-de-la-torre-stephanie-dyke-anthony-brookes-knox-carey-david-lloyd-peter-goodhand-maximilian-haeussler-michael-baudis-heinz-stockinger-lena-dolman-ilkka-lappalainen-juha-tornroos-mikael-linden-john-spalding-saif-ur-rehman-angela-page-paul-flicek-susheel-varma-gary-saunders-serena-scollen-stephen-sherry-david-haussler-beacon-project-team">Miroslav Cupak , Stephen Keenan , Jordi Rambla , Sabela de la Torre , Stephanie Dyke , Anthony Brookes , Knox Carey , David Lloyd , Peter Goodhand , Maximilian Haeussler , Michael Baudis , Heinz Stockinger , Lena Dolman , Ilkka Lappalainen , Juha Törnroos , Mikael Linden , John Spalding , Saif Ur-Rehman , Angela Page , Paul Flicek , Susheel Varma , Gary Saunders , Serena Scollen , Stephen Sherry , David Haussler , Beacon Project Team<a class="headerlink" href="#miroslav-cupak-stephen-keenan-jordi-rambla-sabela-de-la-torre-stephanie-dyke-anthony-brookes-knox-carey-david-lloyd-peter-goodhand-maximilian-haeussler-michael-baudis-heinz-stockinger-lena-dolman-ilkka-lappalainen-juha-tornroos-mikael-linden-john-spalding-saif-ur-rehman-angela-page-paul-flicek-susheel-varma-gary-saunders-serena-scollen-stephen-sherry-david-haussler-beacon-project-team" title="Permanent link">&para;</a></h4>
<h5 id="nat-biotechnol-2019-accepted-2019-01-23">Nat Biotechnol (2019), accepted 2019-01-23<a class="headerlink" href="#nat-biotechnol-2019-accepted-2019-01-23" title="Permanent link">&para;</a></h5>
<!--  CONTENT  -->

<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      January 19, 2019
      
    </span>
</aside>
    <h3 class="title_news">DNA copy number imbalances in primary cutaneous lymphomas (PCL)</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="gug-g-huang-q-chiticariu-e-solovan-c-and-baudis-m-2019">Gug G, Huang Q, Chiticariu E, Solovan C and Baudis M  (2019)<a class="headerlink" href="#gug-g-huang-q-chiticariu-e-solovan-c-and-baudis-m-2019" title="Permanent link">&para;</a></h4>
<h5 id="jeadv-2019-01-19-doiorg101111jdv15442">JEADV, 2019-01-19. <a href="https://doi.org/10.1111/jdv.15442">doi.org/10.1111/jdv.15442</a><a class="headerlink" href="#jeadv-2019-01-19-doiorg101111jdv15442" title="Permanent link">&para;</a></h5>

<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      January 14, 2019
      
    </span>
</aside>
    <h3 class="title_news">Enabling population assignment from cancer genomes with SNP2pop</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="huang-q-and-baudis-m-2019">Huang Q and Baudis M. (2019)<a class="headerlink" href="#huang-q-and-baudis-m-2019" title="Permanent link">&para;</a></h4>
<h5 id="biorxiv-2019-01-14-doiorg101101368647-first-version-2018-07-14">bioRxiv, 2019-01-14. <a href="https://doi.org/10.1101/368647">doi.org/10.1101/368647</a> (first version 2018-07-14)<a class="headerlink" href="#biorxiv-2019-01-14-doiorg101101368647-first-version-2018-07-14" title="Permanent link">&para;</a></h5>

<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      August 02, 2018
      
    </span>
</aside>
    <h3 class="title_news">Registered access: authorizing data access</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="dyke-som-linden-m-lappalainen-i-de-argila-jr-carey-k-lloyd-d-spalding-jd-cabili-mn-kerry-g-foreman-j-cutts-t-shabani-m-rodriguez-ll-haeussler-m-walsh-b-jiang-x-wang-s-perrett-d-boughtwood-t-matern-a-brookes-aj-cupak-m-fiume-m-pandya-r-tulchinsky-i-scollen-s-tornroos-j-das-s-evans-ac-malin-ba-beck-s-brenner-se-nyronen-t-blomberg-n-firth-hv-hurles-m-philippakis-aa-ratsch-g-brudno-m-boycott-km-rehm-hl-baudis-m-sherry-st-kato-k-knoppers-bm-baker-d-and-flicek-p">Dyke SOM, Linden M, Lappalainen I, De Argila JR, Carey K, Lloyd D, Spalding JD, Cabili MN, Kerry G, Foreman J, Cutts T, Shabani M, Rodriguez LL, Haeussler M, Walsh B, Jiang X, Wang S, Perrett D, Boughtwood T, Matern A, Brookes AJ, Cupak M, Fiume M, Pandya R, Tulchinsky I, Scollen S, Törnroos J, Das S, Evans AC, Malin BA, Beck S, Brenner SE, Nyrönen T, Blomberg N, Firth HV, Hurles M, Philippakis AA, Rätsch G, Brudno M, Boycott KM, Rehm HL, Baudis M, Sherry ST, Kato K, Knoppers BM, Baker D, and Flicek P<a class="headerlink" href="#dyke-som-linden-m-lappalainen-i-de-argila-jr-carey-k-lloyd-d-spalding-jd-cabili-mn-kerry-g-foreman-j-cutts-t-shabani-m-rodriguez-ll-haeussler-m-walsh-b-jiang-x-wang-s-perrett-d-boughtwood-t-matern-a-brookes-aj-cupak-m-fiume-m-pandya-r-tulchinsky-i-scollen-s-tornroos-j-das-s-evans-ac-malin-ba-beck-s-brenner-se-nyronen-t-blomberg-n-firth-hv-hurles-m-philippakis-aa-ratsch-g-brudno-m-boycott-km-rehm-hl-baudis-m-sherry-st-kato-k-knoppers-bm-baker-d-and-flicek-p" title="Permanent link">&para;</a></h4>
<h5 id="european-journal-of-human-genetics-2018">European Journal of Human Genetics (2018)<a class="headerlink" href="#european-journal-of-human-genetics-2018" title="Permanent link">&para;</a></h5>
<!--  CONTENT  -->

<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      July 31, 2018
      
    </span>
</aside>
    <h3 class="title_news">Mountains and Chasms - Surveying the Oncogenomic Publication Landscape</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="carrio-cordo-p-and-baudis-m-2018">Carrio Cordo P and Baudis M. (2018)<a class="headerlink" href="#carrio-cordo-p-and-baudis-m-2018" title="Permanent link">&para;</a></h4>
<h5 id="preprints-2018-2018070618-doi-1020944preprints2018070618v1">Preprints 2018, 2018070618 (doi: 10.20944/preprints201807.0618.v1).<a class="headerlink" href="#preprints-2018-2018070618-doi-1020944preprints2018070618v1" title="Permanent link">&para;</a></h5>
<h5 id="oncology-2018-online-oct-26"><em>Oncology</em> (2018; online Oct 26)<a class="headerlink" href="#oncology-2018-online-oct-26" title="Permanent link">&para;</a></h5>

<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      July 14, 2018
      
    </span>
</aside>
    <h3 class="title_news">Population assignment from cancer genome profiling data</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="huang-q-and-baudis-m-2018">Huang Q and Baudis M. (2018)<a class="headerlink" href="#huang-q-and-baudis-m-2018" title="Permanent link">&para;</a></h4>
<h5 id="biorxiv-2018-07-14-doi101101368647">bioRxiv, 2018-07-14. doi:10.1101/368647<a class="headerlink" href="#biorxiv-2018-07-14-doi101101368647" title="Permanent link">&para;</a></h5>

<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      July 12, 2018
      
    </span>
</aside>
    <h3 class="title_news">Meta-knowledgebase of cancer genomic variants</h3>

<!-- Use a hidden p tag to provide a preview -->
<h3 id="a-harmonized-meta-knowledgebase-of-clinical-interpretations-of-cancer-genomic-variants">A harmonized meta-knowledgebase of clinical interpretations of cancer genomic variants<a class="headerlink" href="#a-harmonized-meta-knowledgebase-of-clinical-interpretations-of-cancer-genomic-variants" title="Permanent link">&para;</a></h3>
<h4 id="wagner-ah-walsh-b-mayfield-g-tamborero-d-sonkin-d-krysiak-k-deu-pons-j-duren-r-gao-j-mcmurry-j-patterson-s-del-vecchio-fitz-c-sezerman-ou-warner-j-rieke-dt-aittokallio-t-cerami-e-ritter-d-schriml-lm-haendel-m-raca-g-madhavan-s-baudis-m-beckmann-js-dienstmann-r-chakravarty-d-li-xs-mockus-sm-elemento-o-schultz-n-lopez-bigas-n-lawler-m-goecks-j-griffith-m-griffith-ol-and-margolin-a">Wagner AH, Walsh B, Mayfield G, Tamborero D, Sonkin D, Krysiak K, Deu Pons J, Duren R, Gao J, McMurry J, Patterson S, Del Vecchio Fitz C, Sezerman OU, Warner J, Rieke DT, Aittokallio T, Cerami E, Ritter D, Schriml LM, Haendel M, Raca G, Madhavan S, Baudis M, Beckmann JS, Dienstmann R, Chakravarty D, Li XS, Mockus SM, Elemento O, Schultz N, Lopez-Bigas N, Lawler M, Goecks J, Griffith M, Griffith OL, and Margolin A<a class="headerlink" href="#wagner-ah-walsh-b-mayfield-g-tamborero-d-sonkin-d-krysiak-k-deu-pons-j-duren-r-gao-j-mcmurry-j-patterson-s-del-vecchio-fitz-c-sezerman-ou-warner-j-rieke-dt-aittokallio-t-cerami-e-ritter-d-schriml-lm-haendel-m-raca-g-madhavan-s-baudis-m-beckmann-js-dienstmann-r-chakravarty-d-li-xs-mockus-sm-elemento-o-schultz-n-lopez-bigas-n-lawler-m-goecks-j-griffith-m-griffith-ol-and-margolin-a" title="Permanent link">&para;</a></h4>
<h5 id="biorxiv-doi101101366856">bioRxiv. doi:10.1101/366856<a class="headerlink" href="#biorxiv-doi101101366856" title="Permanent link">&para;</a></h5>
<!--  CONTENT  -->


<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      May 24, 2018
      
    </span>
</aside>
    <h3 class="title_news">KLF10 in Cervical Cancer</h3>

<!-- Use a hidden p tag to provide a preview -->
<h3 id="kruppel-like-factor-10-participates-in-cervical-cancer-immunoediting-through-transcriptional-regulation-of-pregnancy-specific-beta-1-glycoproteins">Krüppel-Like Factor 10 participates in cervical cancer immunoediting through transcriptional regulation of Pregnancy-Specific Beta-1 Glycoproteins.<a class="headerlink" href="#kruppel-like-factor-10-participates-in-cervical-cancer-immunoediting-through-transcriptional-regulation-of-pregnancy-specific-beta-1-glycoproteins" title="Permanent link">&para;</a></h3>
<h4 id="marrero-rodriguez-d-taniguchi-ponciano-k-subramaniam-m-hawse-jr-pitel-ks-arreola-de-la-cruz-h-huerta-padilla-v-ponce-navarrete-g-figueroa-corona-mdp-gomez-virgilio-l-martinez-cuevas-ti-mendoza-rodriguez-m-rodriguez-esquivel-m-romero-morelos-p-ramirez-salcedo-j-baudis-m-meraz-rios-m-jimenez-vega-f-salcedo-m">Marrero-Rodríguez D, Taniguchi-Ponciano K, Subramaniam M, Hawse JR, Pitel KS, Arreola-De la Cruz H, Huerta-Padilla V, Ponce-Navarrete G, Figueroa-Corona MDP, Gomez-Virgilio L, Martinez-Cuevas TI, Mendoza-Rodriguez M, Rodriguez-Esquivel M, Romero-Morelos P, Ramirez-Salcedo J, Baudis M, Meraz-Rios M, Jimenez-Vega F, Salcedo M.<a class="headerlink" href="#marrero-rodriguez-d-taniguchi-ponciano-k-subramaniam-m-hawse-jr-pitel-ks-arreola-de-la-cruz-h-huerta-padilla-v-ponce-navarrete-g-figueroa-corona-mdp-gomez-virgilio-l-martinez-cuevas-ti-mendoza-rodriguez-m-rodriguez-esquivel-m-romero-morelos-p-ramirez-salcedo-j-baudis-m-meraz-rios-m-jimenez-vega-f-salcedo-m" title="Permanent link">&para;</a></h4>

<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      March 14, 2018
      
    </span>
</aside>
    <h3 class="title_news">segment_liftover...</h3>

<!-- Use a hidden p tag to provide a preview -->
<h3 id="segment_liftover-a-python-tool-to-convert-segments-between-genome-assemblies">segment_liftover : a Python tool to convert segments between genome assemblies.<a class="headerlink" href="#segment_liftover-a-python-tool-to-convert-segments-between-genome-assemblies" title="Permanent link">&para;</a></h3>
<h4 id="gao-b-huang-q-and-baudis-m">Gao B, Huang Q and Baudis M<a class="headerlink" href="#gao-b-huang-q-and-baudis-m" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> The process of assembling a species’ reference genome may be performed in a number of iterations, with subsequent genome assemblies differing in the coordinates of mapped elements. The conversion of genome coordinates between different assemblies is required for many integrative and comparative studies. While currently a number of bioinformatics tools are available to accomplish this task, most of them are tailored towards the conversion of single genome coordinates. When converting the boundary positions of segments spanning larger genome regions, segments may be mapped into smaller sub-segments if the original segment’s continuity is disrupted in the target assembly. Such a conversion may lead to a relevant degree of data loss in some circumstances such as copy number variation (CNV) analysis, where the quantitative representation of a genomic region takes precedence over base-specific accuracy. segment_liftover aims at continuity-preserving remapping of genome segments between assemblies and provides features such as approximate locus conversion, automated batch processing and comprehensive logging to facilitate processing of datasets containing large numbers of structural genome variation data.</p>
<h5 id="online">Online<a class="headerlink" href="#online" title="Permanent link">&para;</a></h5>
<ul>
<li><a href="https://f1000research.com/articles/7-319/v2"><em>F1000</em></a></li>
<li><a href="https://www.biorxiv.org/content/early/2018/03/01/274084">bioRxiv</a></li>
</ul>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      October 01, 2017
      
    </span>
</aside>
    <h3 class="title_news">Integrated Molecular...</h3>

<!-- Use a hidden p tag to provide a preview -->
<h3 id="integrated-molecular-meta-analysis-of-1000-pediatric-high-grade-and-diffuse-intrinsic-pontine-glioma">Integrated Molecular Meta-Analysis of 1,000 Pediatric High-Grade and Diffuse Intrinsic Pontine Glioma.<a class="headerlink" href="#integrated-molecular-meta-analysis-of-1000-pediatric-high-grade-and-diffuse-intrinsic-pontine-glioma" title="Permanent link">&para;</a></h3>
<h4 id="mackay-a-burford-a-carvalho-d-izquierdo-e-fazal-salom-j-taylor-kr-bjerke-l-clarke-m-vinci-m-nandhabalan-m-temelso-s-popov-s-molinari-v-raman-p-waanders-aj-han-hj-gupta-s-marshall-l-zacharoulis-s-vaidya-s-mandeville-hc-bridges-lr-martin-aj-al-sarraj-s-chandler-c-ng-hk-li-x-mu-k-trabelsi-s-brahim-dh-kisljakov-an-konovalov-dm-moore-as-carcaboso-am-sunol-m-de-torres-c-cruz-o-mora-j-shats-li-stavale-jn-bidinotto-lt-reis-rm-entz-werle-n-farrell-m-cryan-j-crimmins-d-caird-j-pears-j-monje-m-debily-ma-castel-d-grill-j-hawkins-c-nikbakht-h-jabado-n-baker-sj-pfister-sm-jones-dtw-fouladi-m-von-bueren-ao-baudis-m-resnick-a-jones-c">Mackay A, Burford A, Carvalho D, Izquierdo E, Fazal-Salom J, Taylor KR, Bjerke L, Clarke M, Vinci M, Nandhabalan M, Temelso S, Popov S, Molinari V, Raman P, Waanders AJ, Han HJ, Gupta S, Marshall L, Zacharoulis S, Vaidya S, Mandeville HC, Bridges LR, Martin AJ, Al-Sarraj S, Chandler C, Ng HK, Li X, Mu K, Trabelsi S, Brahim DH, Kisljakov AN, Konovalov DM, Moore AS, Carcaboso AM, Sunol M, de Torres C, Cruz O, Mora J, Shats LI, Stavale JN, Bidinotto LT, Reis RM, Entz-Werle N, Farrell M, Cryan J, Crimmins D, Caird J, Pears J, Monje M, Debily MA, Castel D, Grill J, Hawkins C, Nikbakht H, Jabado N, Baker SJ, Pfister SM, Jones DTW, Fouladi M, von Bueren AO, Baudis M, Resnick A, Jones C.<a class="headerlink" href="#mackay-a-burford-a-carvalho-d-izquierdo-e-fazal-salom-j-taylor-kr-bjerke-l-clarke-m-vinci-m-nandhabalan-m-temelso-s-popov-s-molinari-v-raman-p-waanders-aj-han-hj-gupta-s-marshall-l-zacharoulis-s-vaidya-s-mandeville-hc-bridges-lr-martin-aj-al-sarraj-s-chandler-c-ng-hk-li-x-mu-k-trabelsi-s-brahim-dh-kisljakov-an-konovalov-dm-moore-as-carcaboso-am-sunol-m-de-torres-c-cruz-o-mora-j-shats-li-stavale-jn-bidinotto-lt-reis-rm-entz-werle-n-farrell-m-cryan-j-crimmins-d-caird-j-pears-j-monje-m-debily-ma-castel-d-grill-j-hawkins-c-nikbakht-h-jabado-n-baker-sj-pfister-sm-jones-dtw-fouladi-m-von-bueren-ao-baudis-m-resnick-a-jones-c" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> We collated data from 157 unpublished cases of pediatric high-grade glioma and diffuse intrinsic pontine glioma and 20 publicly available datasets in an integrated analysis of &gt;1,000 cases. We identified co-segregating mutations in histone-mutant subgroups including loss of FBXW7 in H3.3G34R/V, TOP3A rearrangements in H3.3K27M, and BCOR mutations in H3.1K27M. Histone wild-type subgroups are refined by the presence of key oncogenic events or methylation profiles more closely resembling lower-grade tumors. Genomic aberrations increase with age, highlighting the infant population as biologically and clinically distinct. Uncommon pathway dysregulation is seen in small subsets of tumors, further defining the molecular diversity of the disease, opening up avenues for biological study and providing a basis for functionally defined future treatment stratification.</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      October 01, 2016
      
    </span>
</aside>
    <h3 class="title_news">CNARA: reliability...</h3>

<!-- Use a hidden p tag to provide a preview -->
<h3 id="cnara-reliability-assessment-for-genomic-copy-number-profiles">CNARA: reliability assessment for genomic copy number profiles.<a class="headerlink" href="#cnara-reliability-assessment-for-genomic-copy-number-profiles" title="Permanent link">&para;</a></h3>
<h4 id="ai-n-cai-h-solovan-c-baudis-m">Ai N, Cai H, Solovan C, Baudis M.<a class="headerlink" href="#ai-n-cai-h-solovan-c-baudis-m" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> DNA copy number profiles from microarray and sequencing experiments sometimes contain wave artefacts which may be introduced during sample preparation and cannot be removed completely by existing preprocessing methods. Besides, large derivative log ratio spread (DLRS) of the probes correlating with poor DNA quality is sometimes observed in genome screening experiments and may lead to unreliable copy number profiles. Depending on the extent of these artefacts and the resulting misidentification of copy number alterations/variations (CNA/CNV), it may be desirable to exclude such samples from analyses or to adapt the downstream data analysis strategy accordingly.Here, we propose a method to distinguish reliable genomic copy number profiles from those containing heavy wave artefacts and/or large DLRS. We define four features that adequately summarize the copy number profiles for reliability assessment, and train a classifier on a dataset of 1522 copy number profiles from various microarray platforms. The method can be applied to predict the reliability of copy number profiles irrespective of the underlying microarray platform and may be adapted for those sequencing platforms from which copy number estimates could be computed as a piecewise constant signal. Further details can be found at <a href="https://github.com/baudisgroup/CNARA">github.com/baudisgroup/CNARA</a> .We have developed a method for the assessment of genomic copy number profiling data, and suggest to apply the method in addition to and after other state-of-the-art noise correction and quality control procedures. CNARA could be instrumental in improving the assessment of data used for genomic data mining experiments and support the reliable functional attribution of copy number aberrations especially in cancer research.</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      September 01, 2016
      
    </span>
</aside>
    <h3 class="title_news">PKCα and HMGB1...</h3>

<!-- Use a hidden p tag to provide a preview -->
<h3 id="pkc-and-hmgb1-antagonistically-control-hydrogen-peroxide-induced-poly-adp-ribose-formation">PKCα and HMGB1 antagonistically control hydrogen peroxide-induced poly-ADP-ribose formation.<a class="headerlink" href="#pkc-and-hmgb1-antagonistically-control-hydrogen-peroxide-induced-poly-adp-ribose-formation" title="Permanent link">&para;</a></h3>
<h4 id="andersson-a-bluwstein-a-kumar-n-teloni-f-traenkle-j-baudis-m-altmeyer-m-hottiger-mo">Andersson A, Bluwstein A, Kumar N, Teloni F, Traenkle J, Baudis M, Altmeyer M, Hottiger MO.<a class="headerlink" href="#andersson-a-bluwstein-a-kumar-n-teloni-f-traenkle-j-baudis-m-altmeyer-m-hottiger-mo" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> Harmful oxidation of proteins, lipids and nucleic acids is observed when reactive oxygen species (ROS) are produced excessively and/or the antioxidant capacity is reduced, causing 'oxidative stress'. Nuclear poly-ADP-ribose (PAR) formation is thought to be induced in response to oxidative DNA damage and to promote cell death under sustained oxidative stress conditions. However, what exactly triggers PAR induction in response to oxidative stress is incompletely understood. Using reverse phase protein array (RPPA) and in-depth analysis of key stress signaling components, we observed that PAR formation induced by H2O2 was mediated by the PLC/IP3R/Ca(2+)/PKCα signaling axis. Mechanistically, H2O2-induced PAR formation correlated with Ca(2+)-dependent DNA damage, which, however, was PKCα-independent. In contrast, PAR formation was completely lost upon knockdown of PKCα, suggesting that DNA damage alone was not sufficient for inducing PAR formation, but required a PKCα-dependent process. Intriguingly, the loss of PAR formation observed upon PKCα depletion was overcome when the chromatin structure-modifying protein HMGB1 was co-depleted with PKCα, suggesting that activation and nuclear translocation of PKCα releases the inhibitory effect of HMGB1 on PAR formation. Together, these results identify PKCα and HMGB1 as important co-regulators involved in H2O2-induced PAR formation, a finding that may have important relevance for oxidative stress-associated pathophysiological conditions.</p>
<h5 id="links">Links<a class="headerlink" href="#links" title="Permanent link">&para;</a></h5>
<ul>
<li><a href="http://europepmc.org/abstract/MED/27198223"><em>EPMC</em></a></li>
<li><a href="http://internal.baudisgroup.org/public/pdf/27198223.pdf"><em>PDF</em></a></li>
</ul>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      January 01, 2016
      
    </span>
</aside>
    <h3 class="title_news">The SIB Swiss...</h3>

<!-- Use a hidden p tag to provide a preview -->
<h3 id="the-sib-swiss-institute-of-bioinformatics-resources-focus-on-curated-databases">The SIB Swiss Institute of Bioinformatics' resources: focus on curated databases.<a class="headerlink" href="#the-sib-swiss-institute-of-bioinformatics-resources-focus-on-curated-databases" title="Permanent link">&para;</a></h3>
<h4 id="sib-swiss-institute-of-bioinformatics-members">SIB Swiss Institute of Bioinformatics Members.<a class="headerlink" href="#sib-swiss-institute-of-bioinformatics-members" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> The SIB Swiss Institute of Bioinformatics (<a href="http://www.isb-sib.ch">www.isb-sib.ch</a>) provides world-class bioinformatics databases, software tools, services and training to the international life science community in academia and industry. These solutions allow life scientists to turn the exponentially growing amount of data into knowledge. Here, we provide an overview of SIB's resources and competence areas, with a strong focus on curated databases and SIB's most popular and widely used resources. In particular, SIB's Bioinformatics resource portal ExPASy features over 150 resources, including UniProtKB/Swiss-Prot, ENZYME, PROSITE, neXtProt, STRING, UniCarbKB, SugarBindDB, SwissRegulon, EPD, arrayMap, Bgee, SWISS-MODEL Repository, OMA, OrthoDB and other databases, which are briefly described in this article.</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      September 01, 2015
      
    </span>
</aside>
    <h3 class="title_news">A Biobank Supporting Rare Disease Research In Dermatopathology. Our Experience In Establishing A Biobank.</h3>

<!-- Use a hidden p tag to provide a preview -->
<h3 id="beleut-m-seclaman-e-baudis-m-nicula-a-and-solovan-c">Beleut M, Seclaman E, Baudis M, Nicula A, and Solovan C<a class="headerlink" href="#beleut-m-seclaman-e-baudis-m-nicula-a-and-solovan-c" title="Permanent link">&para;</a></h3>
<h4 id="rojced-2-202-206-2015">RoJCED 2, 202-206 (2015)<a class="headerlink" href="#rojced-2-202-206-2015" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> Biobanks of human patient sample tissues and blood fractions are increasingly recognized as major assets in disease research. We aim to identify DNA copy number and gene expression aberrations typical of different cutaneous pathologies. Another goal is the identification of circulating biomarkers both as prognostic, therapy- responsive and/or therapy-monitoring factors and as disease classifiers and subclassifiers.</p>

<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      January 01, 2015
      
    </span>
</aside>
    <h3 class="title_news">Genomic instability...</h3>

<!-- Use a hidden p tag to provide a preview -->
<h3 id="genomic-instability-of-osteosarcoma-cell-lines-in-culture-impact-on-the-prediction-of-metastasis-relevant-genes">Genomic instability of osteosarcoma cell lines in culture: impact on the prediction of metastasis relevant genes.<a class="headerlink" href="#genomic-instability-of-osteosarcoma-cell-lines-in-culture-impact-on-the-prediction-of-metastasis-relevant-genes" title="Permanent link">&para;</a></h3>
<h4 id="muff-r-rath-p-ram-kumar-rm-husmann-k-born-w-baudis-m-fuchs-b">Muff R, Rath P, Ram Kumar RM, Husmann K, Born W, Baudis M, Fuchs B.<a class="headerlink" href="#muff-r-rath-p-ram-kumar-rm-husmann-k-born-w-baudis-m-fuchs-b" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> Osteosarcoma is a rare but highly malignant cancer of the bone. As a consequence, the number of established cell lines used for experimental in vitro and in vivo osteosarcoma research is limited and the value of these cell lines relies on their stability during culture. Here we investigated the stability in gene expression by microarray analysis and array genomic hybridization of three low metastatic cell lines and derivatives thereof with increased metastatic potential using cells of different passages.The osteosarcoma cell lines showed altered gene expression during in vitro culture, and it was more pronounced in two metastatic cell lines compared to the respective parental cells. Chromosomal instability contributed in part to the altered gene expression in SAOS and LM5 cells with low and high metastatic potential. To identify metastasis-relevant genes in a background of passage-dependent altered gene expression, genes involved in "Pathways in cancer" that were consistently regulated under all passage comparisons were evaluated. Genes belonging to "Hedgehog signaling pathway" and "Wnt signaling pathway" were significantly up-regulated, and IHH, WNT10B and TCF7 were found up-regulated in all three metastatic compared to the parental cell lines.Considerable instability during culture in terms of gene expression and chromosomal aberrations was observed in osteosarcoma cell lines. The use of cells from different passages and a search for genes consistently regulated in early and late passages allows the analysis of metastasis-relevant genes despite the observed instability in gene expression in osteosarcoma cell lines during culture.</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      January 01, 2015
      
    </span>
</aside>
    <h3 class="title_news">arrayMap 2014: an...</h3>

<!-- Use a hidden p tag to provide a preview -->
<h3 id="arraymap-2014-an-updated-cancer-genome-resource">arrayMap 2014: an updated cancer genome resource.<a class="headerlink" href="#arraymap-2014-an-updated-cancer-genome-resource" title="Permanent link">&para;</a></h3>
<h4 id="cai-h-gupta-s-rath-p-ai-n-baudis-m">Cai H, Gupta S, Rath P, Ai N, Baudis M.<a class="headerlink" href="#cai-h-gupta-s-rath-p-ai-n-baudis-m" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> Somatic copy number aberrations (CNA) represent a mutation type encountered in the majority of cancer genomes. Here, we present the 2014 edition of arrayMap (<a href="http://www.arraymap.org">www.arraymap.org</a>), a publicly accessible collection of pre-processed oncogenomic array data sets and CNA profiles, representing a vast range of human malignancies. Since the initial release, we have enhanced this resource both in content and especially with regard to data mining support. The 2014 release of arrayMap contains more than 64,000 genomic array data sets, representing about 250 tumor diagnoses. Data sets included in arrayMap have been assembled from public repositories as well as additional resources, and integrated by applying custom processing pipelines. Online tools have been upgraded for a more flexible array data visualization, including options for processing user provided, non-public data sets. Data integration has been improved by mapping to multiple editions of the human reference genome, with the majority of the data now being available for the UCSC hg18 as well as GRCh37 versions. The large amount of tumor CNA data in arrayMap can be freely downloaded by users to promote data mining projects, and to explore special events such as chromothripsis-like genome patterns.</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      January 01, 2014
      
    </span>
</aside>
    <h3 class="title_news">Biopsying parapsoria...</h3>

<!-- Use a hidden p tag to provide a preview -->
<h3 id="biopsying-parapsoriasis-quo-vadis-are-morphological-stains-enough-or-are-ancillary-tests-needed">Biopsying parapsoriasis: quo vadis? Are morphological stains enough or are ancillary tests needed?<a class="headerlink" href="#biopsying-parapsoriasis-quo-vadis-are-morphological-stains-enough-or-are-ancillary-tests-needed" title="Permanent link">&para;</a></h3>
<h4 id="baderca-f-chiticariu-e-baudis-m-solovan-c">Baderca F, Chiticariu E, Baudis M, Solovan C.<a class="headerlink" href="#baderca-f-chiticariu-e-baudis-m-solovan-c" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> BACKGROUND: Parapsoriasis represents a group of cutaneous disorders that shows variable clinical aspects somehow resembling to psoriasis, how is reflecting by its name. It was first named by Brocq, in 1902, as an entity with three components: pityriasis lichenoides, small plaque parapsoriasis and large plaque parapsoriasis. Nowadays, under the name of parapsoriasis are included only the last two categories, that are considered disorders characterized by the presence of a mononuclear infiltrate in the dermis, composed of T-cells. Until now, there were not established pathognomonic histopathological features to diagnose parapsoriasis. AIM: The aim of the study was to investigate the epidemiological and morphological data of parapsoriasis cases diagnosed at Emergency City Hospital, Timisoara, Romania for a period of 12 years. MATERIALS AND METHODS: The study had two parts; one was retrospective and another one prospective. For the retrospective part, we searched 210111 patient files recorded in our Pathology Service for a period of 11 years, from January 2002 to December 2012. The slides were searched from the archive and re-read by two individual pathologists. For prospective part of the study, we reviewed 11815 histological slides read between January and June 2013. After inspection of the recorded files, the pathologists noted, were available, the localization and number of the lesions, together with symptoms. The biopsied specimens were initially processed with routine histological technique, the archive slides being stained with Hematoxylin and Eosin. While reading the slides, the pathologists paid attention to the architecture of the epidermis, the presence of epidermotropism and interface dermatitis, type of the dermal infiltrate and its distributions. CONCLUSIONS: In the present study, we emphasized the histopathological aspects of parapsoriasis in order to create a basic line that could help in the establishment of a uniformly accepted definition of parapsoriasis on histopathological grounds.</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      January 01, 2014
      
    </span>
</aside>
    <h3 class="title_news">Chromothripsis-like...</h3>

<!-- Use a hidden p tag to provide a preview -->
<h3 id="chromothripsis-like-patterns-are-recurring-but-heterogeneously-distributed-features-in-a-survey-of-22347-cancer-genome-screens">Chromothripsis-like patterns are recurring but heterogeneously distributed features in a survey of 22,347 cancer genome screens.<a class="headerlink" href="#chromothripsis-like-patterns-are-recurring-but-heterogeneously-distributed-features-in-a-survey-of-22347-cancer-genome-screens" title="Permanent link">&para;</a></h3>
<h4 id="cai-h-kumar-n-bagheri-hc-von-mering-c-robinson-md-baudis-m">Cai H, Kumar N, Bagheri HC, von Mering C, Robinson MD, Baudis M.<a class="headerlink" href="#cai-h-kumar-n-bagheri-hc-von-mering-c-robinson-md-baudis-m" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> BACKGROUND: Chromothripsis is a recently discovered phenomenon of genomic rearrangement, possibly arising during a single genome-shattering event. This could provide an alternative paradigm in cancer development, replacing the gradual accumulation of genomic changes with a "one-off" catastrophic event. However, the term has been used with varying operational definitions, with the minimal consensus being a large number of locally clustered copy number aberrations. The mechanisms underlying these chromothripsis-like patterns (CTLP) and their specific impact on tumorigenesis are still poorly understood. RESULTS: Here, we identified CTLP in 918 cancer samples, from a dataset of more than 22,000 oncogenomic arrays covering 132 cancer types. Fragmentation hotspots were found to be located on chromosome 8, 11, 12 and 17. Among the various cancer types, soft-tissue tumors exhibited particularly high CTLP frequencies. Genomic context analysis revealed that CTLP rearrangements frequently occurred in genomes that additionally harbored multiple copy number aberrations (CNAs). An investigation into the affected chromosomal regions showed a large proportion of arm-level pulverization and telomere related events, which would be compatible to a number of underlying mechanisms. We also report evidence that these genomic events may be correlated with patient age, stage and survival rate. CONCLUSIONS: Through a large-scale analysis of oncogenomic array data sets, this study characterized features associated with genomic aberrations patterns, compatible to the spectrum of "chromothripsis"-definitions as previously used. While quantifying clustered genomic copy number aberrations in cancer samples, our data indicates an underlying biological heterogeneity behind these chromothripsis-like patterns, beyond a well defined "chromthripsis" phenomenon.</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      January 01, 2014
      
    </span>
</aside>
    <h3 class="title_news">Progenetix: 12 years...</h3>

<!-- Use a hidden p tag to provide a preview -->
<h3 id="progenetix-12-years-of-oncogenomic-data-curation">Progenetix: 12 years of oncogenomic data curation.<a class="headerlink" href="#progenetix-12-years-of-oncogenomic-data-curation" title="Permanent link">&para;</a></h3>
<h4 id="cai-h-kumar-n-ai-n-gupta-s-rath-p-baudis-m">Cai H, Kumar N, Ai N, Gupta S, Rath P, Baudis M.<a class="headerlink" href="#cai-h-kumar-n-ai-n-gupta-s-rath-p-baudis-m" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> DNA copy number aberrations (CNAs) can be found in the majority of cancer genomes and are crucial for understanding the potential mechanisms underlying tumor initiation and progression. Since the first release in 2001, the Progenetix project (<a href="http://www.progenetix.org">www.progenetix.org</a>) has provided a reference resource dedicated to provide the most comprehensive collection of genome-wide CNA profiles. Reflecting the application of comparative genomic hybridization techniques to tens of thousands of cancer genomes, over the past 12 years our data curation efforts have resulted in a more than 60-fold increase in the number of cancer samples presented through Progenetix. In addition, new data exploration tools and visualization options have been added. In particular, the gene-specific CNA frequency analysis should facilitate the assignment of cancer genes to related cancer types. In addition, the new user file processing interface allows users to take advantage of the online tools, including various data representation options for proprietary data pre-publication. In this update article, we report recent improvements of the database in terms of content, user interface and online tools.</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      December 01, 2013
      
    </span>
</aside>
    <h3 class="title_news">SIL1 mutations and...</h3>

<!-- Use a hidden p tag to provide a preview -->
<h3 id="sil1-mutations-and-clinical-spectrum-in-patients-with-marinesco-sjogren-syndrome">SIL1 mutations and clinical spectrum in patients with Marinesco-Sjogren syndrome.<a class="headerlink" href="#sil1-mutations-and-clinical-spectrum-in-patients-with-marinesco-sjogren-syndrome" title="Permanent link">&para;</a></h3>
<h4 id="krieger-m-roos-a-stendel-c-claeys-kg-sonmez-fm-baudis-m-bauer-p-bornemann-a-de-goede-c-dufke-a-finkel-rs-goebel-hh-haussler-m-kingston-h-kirschner-j-medne-l-muschke-p-rivier-f-rudnik-schoneborn-s-spengler-s-inzana-f-stanzial-f-benedicenti-f-synofzik-m-lia-taratuto-a-pirra-l-tay-sk-topaloglu-h-uyanik-g-wand-d-williams-d-zerres-k-weis-j-senderek-j">Krieger M, Roos A, Stendel C, Claeys KG, Sonmez FM, Baudis M, Bauer P, Bornemann A, de Goede C, Dufke A, Finkel RS, Goebel HH, Häussler M, Kingston H, Kirschner J, Medne L, Muschke P, Rivier F, Rudnik-Schöneborn S, Spengler S, Inzana F, Stanzial F, Benedicenti F, Synofzik M, Lia Taratuto A, Pirra L, Tay SK, Topaloglu H, Uyanik G, Wand D, Williams D, Zerres K, Weis J, Senderek J.<a class="headerlink" href="#krieger-m-roos-a-stendel-c-claeys-kg-sonmez-fm-baudis-m-bauer-p-bornemann-a-de-goede-c-dufke-a-finkel-rs-goebel-hh-haussler-m-kingston-h-kirschner-j-medne-l-muschke-p-rivier-f-rudnik-schoneborn-s-spengler-s-inzana-f-stanzial-f-benedicenti-f-synofzik-m-lia-taratuto-a-pirra-l-tay-sk-topaloglu-h-uyanik-g-wand-d-williams-d-zerres-k-weis-j-senderek-j" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> Marinesco-Sjögren syndrome is a rare autosomal recessive multisystem disorder featuring cerebellar ataxia, early-onset cataracts, chronic myopathy, variable intellectual disability and delayed motor development. More recently, mutations in the SIL1 gene, which encodes an endoplasmic reticulum resident co-chaperone, were identified as the main cause of Marinesco-Sjögren syndrome. Here we describe the results of SIL1 mutation analysis in 62 patients presenting with early-onset ataxia, cataracts and myopathy or combinations of at least two of these. We obtained a mutation detection rate of 60% (15/25) among patients with the characteristic Marinesco-Sjögren syndrome triad (ataxia, cataracts, myopathy) whereas the detection rate in the group of patients with more variable phenotypic presentation was below 3% (1/37). We report 16 unrelated families with a total of 19 different SIL1 mutations. Among these mutations are 15 previously unreported changes, including single- and multi-exon deletions. Based on data from our screening cohort and data compiled from the literature we found that SIL1 mutations are invariably associated with the combination of a cerebellar syndrome and chronic myopathy. Cataracts were observed in all patients beyond the age of 7 years, but might be missing in infants. Six patients with SIL1 mutations had no intellectual disability, extending the known wide range of cognitive capabilities in Marinesco-Sjögren syndrome to include normal intelligence. Modestly constant features were somatic growth retardation, skeletal abnormalities and pyramidal tract signs. Examination of mutant SIL1 expression in cultured patient lymphoblasts suggested that SIL1 mutations result in severely reduced SIL1 protein levels irrespective of the type and position of mutations. Our data broaden the SIL1 mutation spectrum and confirm that SIL1 is the major Marinesco-Sjögren syndrome gene. SIL1 patients usually present with the characteristic triad but cataracts might be missing in young children. As cognitive impairment is not obligatory, patients without intellectual disability but a Marinesco-Sjögren syndrome-compatible phenotype should receive SIL1 mutation analysis. Despite allelic heterogeneity and many families with private mutations, the phenotype related to SIL1 mutations is relatively homogenous. Based on SIL1 expression studies we speculate that this may arise from a uniform effect of different mutations on protein expression.</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      August 01, 2013
      
    </span>
</aside>
    <h3 class="title_news">Recurrent loss of...</h3>

<!-- Use a hidden p tag to provide a preview -->
<h3 id="recurrent-loss-of-heterozygosity-in-1p36-associated-with-tnfrsf14-mutations-in-irf4-translocation-negative-pediatric-follicular-lymphomas">Recurrent loss of heterozygosity in 1p36 associated with TNFRSF14 mutations in IRF4 translocation negative pediatric follicular lymphomas.<a class="headerlink" href="#recurrent-loss-of-heterozygosity-in-1p36-associated-with-tnfrsf14-mutations-in-irf4-translocation-negative-pediatric-follicular-lymphomas" title="Permanent link">&para;</a></h3>
<h4 id="martin-guerrero-i-salaverria-i-burkhardt-b-szczepanowski-m-baudis-m-bens-s-de-leval-l-garcia-orad-a-horn-h-lisfeld-j-pellissery-s-klapper-w-oschlies-i-siebert-r">Martin-Guerrero I, Salaverria I, Burkhardt B, Szczepanowski M, Baudis M, Bens S, de Leval L, Garcia-Orad A, Horn H, Lisfeld J, Pellissery S, Klapper W, Oschlies I, Siebert R.<a class="headerlink" href="#martin-guerrero-i-salaverria-i-burkhardt-b-szczepanowski-m-baudis-m-bens-s-de-leval-l-garcia-orad-a-horn-h-lisfeld-j-pellissery-s-klapper-w-oschlies-i-siebert-r" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> Pediatric follicular lymphoma is a rare disease that differs genetically and clinically from its adult counterpart. With the exception of pediatric follicular lymphoma with IRF4-translocation, the genetic events associated with these lymphomas have not yet been defined. We applied array-comparative genomic hybridization and molecular inversion probe assay analyses to formalin-fixed paraffin-embedded tissues from 18 patients aged 18 years and under with IRF4 translocation negative follicular lymphoma. All evaluable cases lacked t(14;18). Only 6 of 16 evaluable cases displayed chromosomal imbalances with gains or amplifications of 6pter-p24.3 (including IRF4) and deletion and copy number neutral-loss of heterozygosity in 1p36 (including TNFRSF14) being most frequent. Sequencing of TNFRSF14 located in the minimal region of loss in 1p36.32 showed nine mutations in 7 cases from our series. Two subsets of pediatric follicular lymphoma were delineated according to the presence of molecular alterations, one with genomic aberrations associated with higher grade and/or diffuse large B-cell lymphoma component and more widespread disease, and another one lacking genetic alterations associated with more limited disease.</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      February 01, 2013
      
    </span>
</aside>
    <h3 class="title_news">High resolution copy...</h3>

<!-- Use a hidden p tag to provide a preview -->
<h3 id="high-resolution-copy-number-analysis-of-irf4-translocation-positive-diffuse-large-b-cell-and-follicular-lymphomas">High resolution copy number analysis of IRF4 translocation-positive diffuse large B-cell and follicular lymphomas.<a class="headerlink" href="#high-resolution-copy-number-analysis-of-irf4-translocation-positive-diffuse-large-b-cell-and-follicular-lymphomas" title="Permanent link">&para;</a></h3>
<h4 id="salaverria-i-martin-guerrero-i-burkhardt-b-kreuz-m-zenz-t-oschlies-i-arnold-n-baudis-m-bens-s-garcia-orad-a-lisfeld-j-schwaenen-c-szczepanowski-m-wessendorf-s-pfreundschuh-m-trumper-l-klapper-w-siebert-r">Salaverria I, Martin-Guerrero I, Burkhardt B, Kreuz M, Zenz T, Oschlies I, Arnold N, Baudis M, Bens S, García-Orad A, Lisfeld J, Schwaenen C, Szczepanowski M, Wessendorf S, Pfreundschuh M, Trümper L, Klapper W, Siebert R.<a class="headerlink" href="#salaverria-i-martin-guerrero-i-burkhardt-b-kreuz-m-zenz-t-oschlies-i-arnold-n-baudis-m-bens-s-garcia-orad-a-lisfeld-j-schwaenen-c-szczepanowski-m-wessendorf-s-pfreundschuh-m-trumper-l-klapper-w-siebert-r" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> Translocations affecting chromosome subband 6p25.3 containing the IRF4 gene have been recently described as characteristic alterations in a molecularly distinct subset of germinal center B-cell-derived lymphomas. Secondary changes have yet only been described in few of these lymphomas. Here, we performed array-comparative genomic hybridization and molecular inversion probe microarray analyses on DNA from 12 formalin-fixed paraffin-embedded and two fresh-frozen IRF4 translocation-positive lymphomas, which together with the previously published data on nine cases allowed the extension of copy number analyses to a total of 23 of these lymphomas. All except one case carried chromosomal imbalances, most frequently gains in Xq28, 11q22.3-qter, and 7q32.1-qter and losses in 6q13-16.1, 15q14-22.31, and 17p. No recurrent copy-neutral losses of heterozygosity were observed. TP53 point mutations were detected in three of six cases with loss of 17p. Overall this study unravels a recurrent pattern of secondary genetic alterations in IRF4 translocation-positive lymphomas.</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      January 01, 2013
      
    </span>
</aside>
    <h3 class="title_news">PKC signaling...</h3>

<!-- Use a hidden p tag to provide a preview -->
<h3 id="pkc-signaling-prevents-irradiation-induced-apoptosis-of-primary-human-fibroblasts">PKC signaling prevents irradiation-induced apoptosis of primary human fibroblasts.<a class="headerlink" href="#pkc-signaling-prevents-irradiation-induced-apoptosis-of-primary-human-fibroblasts" title="Permanent link">&para;</a></h3>
<h4 id="bluwstein-a-kumar-n-leger-k-traenkle-j-oostrum-jv-rehrauer-h-baudis-m-hottiger-mo">Bluwstein A, Kumar N, Léger K, Traenkle J, Oostrum Jv, Rehrauer H, Baudis M, Hottiger MO.<a class="headerlink" href="#bluwstein-a-kumar-n-leger-k-traenkle-j-oostrum-jv-rehrauer-h-baudis-m-hottiger-mo" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> Primary cells respond to irradiation by activation of the DNA damage response and cell cycle arrest, which eventually leads to senescence or apoptosis. It is not clear in detail which signaling pathways or networks regulate the induction of either apoptosis or senescence. Primary human fibroblasts are able to withstand high doses of irradiation and to prevent irradiation-induced apoptosis. However, the underlying regulatory basis for this phenotype is not well understood. Here, a kinetic network analysis based on reverse phase protein arrays (RPPAs) in combination with extensive western blot and cell culture analyses was employed to decipher the cytoplasmic and nuclear signaling networks and to identify possible antiapoptotic pathways. This analysis identified activation of known DNA damage response pathways (e.g., phosphorylation of MKK3/6, p38, MK2, Hsp27, p53 and Chk1) as well as of prosurvival (e.g., MEK-ERK, cAMP response element-binding protein (CREB), protein kinase C (PKC)) and antiapoptotic markers (e.g., Bad, Bcl-2). Interestingly, PKC family members were activated early upon irradiation, suggesting a regulatory function in the ionizing radiation (IR) response of these cells. Inhibition or downregulation of PKC in primary human fibroblasts caused IR-dependent downregulation of the identified prosurvival (CREB phosphorylation) and antiapoptotic (Bad phosphorylation, Bcl-2) markers and thus lead to a proliferation stop and to apoptosis. Taken together, our analysis suggests that cytoplasmic PKC signaling conditions IR-stressed MRC-5 and IMR-90 cells to prevent irradiation-induced apoptosis. These findings contribute to the understanding of the cellular and nuclear IR response and may thus eventually improve the efficacy of radiotherapy and help overcome tumor radioresistance.</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      November 01, 2012
      
    </span>
</aside>
    <h3 class="title_news">Molecular karyotypin...</h3>

<!-- Use a hidden p tag to provide a preview -->
<h3 id="molecular-karyotyping-as-a-relevant-diagnostic-tool-in-children-with-growth-retardation-with-silver-russell-features">Molecular karyotyping as a relevant diagnostic tool in children with growth retardation with Silver-Russell features.<a class="headerlink" href="#molecular-karyotyping-as-a-relevant-diagnostic-tool-in-children-with-growth-retardation-with-silver-russell-features" title="Permanent link">&para;</a></h3>
<h4 id="spengler-s-begemann-m-ortiz-bruchle-n-baudis-m-denecke-b-kroisel-pm-oehl-jaschkowitz-b-schulze-b-raabe-meyer-g-spaich-c-blumel-p-jauch-a-moog-u-zerres-k-eggermann-t">Spengler S, Begemann M, Ortiz Brüchle N, Baudis M, Denecke B, Kroisel PM, Oehl-Jaschkowitz B, Schulze B, Raabe-Meyer G, Spaich C, Blümel P, Jauch A, Moog U, Zerres K, Eggermann T.<a class="headerlink" href="#spengler-s-begemann-m-ortiz-bruchle-n-baudis-m-denecke-b-kroisel-pm-oehl-jaschkowitz-b-schulze-b-raabe-meyer-g-spaich-c-blumel-p-jauch-a-moog-u-zerres-k-eggermann-t" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> OBJECTIVE: To determine the contribution of submicroscopic chromosomal imbalances to the etiology of Silver-Russell syndrome (SRS) and SRS-like phenotypes. STUDY DESIGN: We performed molecular karyotyping in 41 patients with SRS or SRS-like features without known chromosome 7 and 11 defects using the Affymetrix SNP Array 6.0 system (Affymetrix, High Wycombe, United Kingdom). RESULTS: In 8 patients, pathogenic copy number variations with sizes ranging from 672 kb to 9.158 Mb were identified. The deletions in 1q21, 15q26, 17p13, and 22q11 were associated with known microdeletion syndromes with overlapping features with SRS. The duplications in 22q13 and Xq25q27 represent unique novel copy number variations but have an obvious influence on the phenotype. In 5 additional patients, the pathogenetic relevance of the detected variants remained unclear. CONCLUSION: Pathogenic submicroscopic imbalances were detectable in a significant proportion of patients with short stature and features reminiscent of SRS. Therefore, molecular karyotyping should be implemented in routine diagnostics for growth-retarded patients with even slight dysmorphisms suggestive for SRS.</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      October 01, 2012
      
    </span>
</aside>
    <h3 class="title_news">2p21 Deletions in...</h3>

<!-- Use a hidden p tag to provide a preview -->
<h3 id="2p21-deletions-in-hypotonia-cystinuria-syndrome">2p21 Deletions in hypotonia-cystinuria syndrome.<a class="headerlink" href="#2p21-deletions-in-hypotonia-cystinuria-syndrome" title="Permanent link">&para;</a></h3>
<h4 id="eggermann-t-spengler-s-venghaus-a-denecke-b-zerres-k-baudis-m-ensenauer-r">Eggermann T, Spengler S, Venghaus A, Denecke B, Zerres K, Baudis M, Ensenauer R.<a class="headerlink" href="#eggermann-t-spengler-s-venghaus-a-denecke-b-zerres-k-baudis-m-ensenauer-r" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> The significant role of the SLC3A1 gene in the aetiology of cystinuria is meanwhile well established and more than 130 point mutations have been reported. With the reports on genomic deletions including at least both SLC3A1 and the neighboured PREPL gene the spectrum of cystinuria mutations and of clinical symptoms could recently be enlarged: patients homozygous for these deletions suffer from a general neonatal hypotonia and growth retardation in addition to cystinuria. The hypotonia in these hypotonia-cystinuria (HCS) patients has been attributed to the total loss of the PREPL protein. Here we report on the clinical course and molecular findings in a HCS patient compound heterozygote for a new deletion in 2p21 and a previously reported deletion, both identified by molecular karyotyping. The diagnostic workup in this patient illustrates the need for a careful clinical examination in context with powerful molecular genetic tools in patients with unusual phenotypes. The identification of unique genomic alterations and their interpretation serves as a prerequisite for the individual counselling of patients and their families. In diagnostic strategies to identify the molecular basis of both cystinuria and hypotonia 2p21 deletions should be considered as the molecular basis of the phenotype.</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      September 01, 2012
      
    </span>
</aside>
    <h3 class="title_news">DNA copy number...</h3>

<!-- Use a hidden p tag to provide a preview -->
<h3 id="dna-copy-number-alterations-in-central-primitive-neuroectodermal-tumors-and-tumors-of-the-pineal-region-an-international-individual-patient-data-meta-analysis">DNA copy number alterations in central primitive neuroectodermal tumors and tumors of the pineal region: an international individual patient data meta-analysis.<a class="headerlink" href="#dna-copy-number-alterations-in-central-primitive-neuroectodermal-tumors-and-tumors-of-the-pineal-region-an-international-individual-patient-data-meta-analysis" title="Permanent link">&para;</a></h3>
<h4 id="von-bueren-ao-gerss-j-hagel-c-cai-h-remke-m-hasselblatt-m-feuerstein-bg-pernet-s-delattre-o-korshunov-a-rutkowski-s-pfister-sm-baudis-m">von Bueren AO, Gerss J, Hagel C, Cai H, Remke M, Hasselblatt M, Feuerstein BG, Pernet S, Delattre O, Korshunov A, Rutkowski S, Pfister SM, Baudis M.<a class="headerlink" href="#von-bueren-ao-gerss-j-hagel-c-cai-h-remke-m-hasselblatt-m-feuerstein-bg-pernet-s-delattre-o-korshunov-a-rutkowski-s-pfister-sm-baudis-m" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> Little is known about frequency, association with clinical characteristics, and prognostic impact of DNA copy number alterations (CNA) on survival in central primitive neuroectodermal tumors (CNS-PNET) and tumors of the pineal region. Searches of MEDLINE, Pubmed, and EMBASE--after the original description of comparative genomic hybridization in 1992 and July 2010--identified 15 case series of patients with CNS-PNET and tumors of the pineal region whose tumors were investigated for genome-wide CNA. One additional case study was identified from contact with experts. Individual patient data were extracted from publications or obtained from investigators, and CNAs were converted to a digitized format suitable for data mining and subgroup identification. Summary profiles for genomic imbalances were generated from case-specific data. Overall survival (OS) was estimated using the Kaplan-Meier method, and by univariable and multivariable Cox regression models. In their overall CNA profiles, low grade tumors of the pineal region clearly diverged from CNS-PNET and pineoblastoma. At a median follow-up of 89 months, 7-year OS rates of CNS-PNET, pineoblastoma, and low grade tumors of the pineal region were 22.9 ± 6, 0 ± 0, and 87.5 ± 12 %, respectively. Multivariable analysis revealed that histology (CNS-PNET), age (≤2.5 years), and possibly recurrent CNAs were associated with unfavorable OS. DNA copy number profiling suggests a close relationship between CNS-PNET and pineoblastoma. Low grade tumors of the pineal region differed from CNS-PNET and pineoblastoma. Due to their high biological and clinical variability, a coordinated prospective validation in future studies is necessary to establish robust risk factors.</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      September 01, 2012
      
    </span>
</aside>
    <h3 class="title_news">Improved multiplex...</h3>

<!-- Use a hidden p tag to provide a preview -->
<h3 id="improved-multiplex-ligation-dependent-probe-amplification-analysis-identifies-a-deleterious-pms2-allele-generated-by-recombination-with-crossover-between-pms2-and-pms2cl">Improved multiplex ligation-dependent probe amplification analysis identifies a deleterious PMS2 allele generated by recombination with crossover between PMS2 and PMS2CL.<a class="headerlink" href="#improved-multiplex-ligation-dependent-probe-amplification-analysis-identifies-a-deleterious-pms2-allele-generated-by-recombination-with-crossover-between-pms2-and-pms2cl" title="Permanent link">&para;</a></h3>
<h4 id="wernstedt-a-valtorta-e-armelao-f-togni-r-girlando-s-baudis-m-heinimann-k-messiaen-l-staehli-n-zschocke-j-marra-g-wimmer-k">Wernstedt A, Valtorta E, Armelao F, Togni R, Girlando S, Baudis M, Heinimann K, Messiaen L, Staehli N, Zschocke J, Marra G, Wimmer K.<a class="headerlink" href="#wernstedt-a-valtorta-e-armelao-f-togni-r-girlando-s-baudis-m-heinimann-k-messiaen-l-staehli-n-zschocke-j-marra-g-wimmer-k" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> Heterozygous PMS2 germline mutations are associated with Lynch syndrome. Up to one third of these mutations are genomic deletions. Their detection is complicated by a pseudogene (PMS2CL), which--owing to extensive interparalog sequence exchange--closely resembles PMS2 downstream of exon 12. A recently redesigned multiplex ligation-dependent probe amplification (MLPA) assay identifies PMS2 copy number alterations with improved reliability when used with reference DNAs containing equal numbers of PMS2- and PMS2CL-specific sequences. We selected eight such reference samples--all publicly available--and used them with this assay to study 13 patients with PMS2-defective colorectal tumors. Three presented deleterious alterations: an Alu-mediated exon deletion; a 125-kb deletion encompassing PMS2 and four additional genes (two with tumor-suppressing functions); and a novel deleterious hybrid PMS2 allele produced by recombination with crossover between PMS2 and PMS2CL, with the breakpoint in intron 10 (the most 5' breakpoint of its kind reported thus far). We discuss mechanisms that might generate this allele in different chromosomal configurations (and their diagnostic implications) and describe an allele-specific PCR assay that facilitates its detection. Our data indicate that the redesigned PMS2 MLPA assay is a valid first-line option. In our series, it identified roughly a quarter of all PMS2 mutations.</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      January 01, 2012
      
    </span>
</aside>
    <h3 class="title_news">Integrative...</h3>

<!-- Use a hidden p tag to provide a preview -->
<h3 id="integrative-genome-wide-expression-profiling-identifies-three-distinct-molecular-subgroups-of-renal-cell-carcinoma-with-different-patient-outcome">Integrative genome-wide expression profiling identifies three distinct molecular subgroups of renal cell carcinoma with different patient outcome.<a class="headerlink" href="#integrative-genome-wide-expression-profiling-identifies-three-distinct-molecular-subgroups-of-renal-cell-carcinoma-with-different-patient-outcome" title="Permanent link">&para;</a></h3>
<h4 id="beleut-m-zimmermann-p-baudis-m-bruni-n-buhlmann-p-laule-o-luu-vd-gruissem-w-schraml-p-moch-h">Beleut M, Zimmermann P, Baudis M, Bruni N, Bühlmann P, Laule O, Luu VD, Gruissem W, Schraml P, Moch H.<a class="headerlink" href="#beleut-m-zimmermann-p-baudis-m-bruni-n-buhlmann-p-laule-o-luu-vd-gruissem-w-schraml-p-moch-h" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> BACKGROUND: Renal cell carcinoma (RCC) is characterized by a number of diverse molecular aberrations that differ among individuals. Recent approaches to molecularly classify RCC were based on clinical, pathological as well as on single molecular parameters. As a consequence, gene expression patterns reflecting the sum of genetic aberrations in individual tumors may not have been recognized. In an attempt to uncover such molecular features in RCC, we used a novel, unbiased and integrative approach. METHODS: We integrated gene expression data from 97 primary RCC of different pathologic parameters, 15 RCC metastases as well as 34 cancer cell lines for two-way nonsupervised hierarchical clustering using gene groups suggested by the PANTHER Classification System. We depicted the genomic landscape of the resulted tumor groups by means of Single Nuclear Polymorphism (SNP) technology. Finally, the achieved results were immunohistochemically analyzed using a tissue microarray (TMA) composed of 254 RCC. RESULTS: We found robust, genome wide expression signatures, which split RCC into three distinct molecular subgroups. These groups remained stable even if randomly selected gene sets were clustered. Notably, the pattern obtained from RCC cell lines was clearly distinguishable from that of primary tumors. SNP array analysis demonstrated differing frequencies of chromosomal copy number alterations among RCC subgroups. TMA analysis with group-specific markers showed a prognostic significance of the different groups. CONCLUSION: We propose the existence of characteristic and histologically independent genome-wide expression outputs in RCC with potential biological and clinical relevance.</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      January 01, 2012
      
    </span>
</aside>
    <h3 class="title_news">Losses at chromosome...</h3>

<!-- Use a hidden p tag to provide a preview -->
<h3 id="losses-at-chromosome-4q-are-associated-with-poor-survival-in-operable-ductal-pancreatic-adenocarcinoma">Losses at chromosome 4q are associated with poor survival in operable ductal pancreatic adenocarcinoma.<a class="headerlink" href="#losses-at-chromosome-4q-are-associated-with-poor-survival-in-operable-ductal-pancreatic-adenocarcinoma" title="Permanent link">&para;</a></h3>
<h4 id="luebke-am-baudis-m-matthaei-h-vashist-yk-verde-pe-hosch-sb-erbersdobler-a-klein-ca-izbicki-jr-knoefel-wt-stoecklein-nh">Luebke AM, Baudis M, Matthaei H, Vashist YK, Verde PE, Hosch SB, Erbersdobler A, Klein CA, Izbicki JR, Knoefel WT, Stoecklein NH.<a class="headerlink" href="#luebke-am-baudis-m-matthaei-h-vashist-yk-verde-pe-hosch-sb-erbersdobler-a-klein-ca-izbicki-jr-knoefel-wt-stoecklein-nh" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> Here we tested the prognostic impact of genomic alterations in operable localized pancreatic ductal adenocarcinoma (PDAC). Fifty-two formalin-fixed and paraffin-embedded primary PDAC were laser micro-dissected and were investigated by comparative genomic hybridization after whole genome amplification using an adapter-linker PCR. Chromosomal gains and losses were correlated to clinico-pathological parameters and clinical follow-up data. The most frequent aberration was loss on chromosome 17p (65%) while the most frequent gains were detected at 2q (41%) and 8q (41%), respectively. The concomitant occurrence of losses at 9p and 17p was found to be statistically significant. Higher rates of chromosomal losses were associated with a more advanced primary tumor stage and losses at 9p and 18q were significantly associated with presence of lymphatic metastasis (chi-square: p = 0.03, p = 0.05, respectively). Deletions on chromosome 4 were of prognostic significance for overall survival and tumor recurrence (Cox-multivariate analysis: p = 0.026 and p = 0.021, respectively). In conclusion our data suggest the common alterations at chromosome 8q, 9p, 17p and 18q as well as the prognostic relevant deletions on chromosome 4q as relevant for PDAC progression. Our comprehensive data from 52 PDAC should provide a basis for future studies with a higher resolution to discover the relevant genes located within the chromosomal aberrations identified.</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      January 01, 2012
      
    </span>
</aside>
    <h3 class="title_news">Specific genomic...</h3>

<!-- Use a hidden p tag to provide a preview -->
<h3 id="specific-genomic-regions-are-differentially-affected-by-copy-number-alterations-across-distinct-cancer-types-in-aggregated-cytogenetic-data">Specific genomic regions are differentially affected by copy number alterations across distinct cancer types, in aggregated cytogenetic data.<a class="headerlink" href="#specific-genomic-regions-are-differentially-affected-by-copy-number-alterations-across-distinct-cancer-types-in-aggregated-cytogenetic-data" title="Permanent link">&para;</a></h3>
<h4 id="kumar-n-cai-h-von-mering-c-baudis-m">Kumar N, Cai H, von Mering C, Baudis M.<a class="headerlink" href="#kumar-n-cai-h-von-mering-c-baudis-m" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> BACKGROUND: Regional genomic copy number alterations (CNA) are observed in the vast majority of cancers. Besides specifically targeting well-known, canonical oncogenes, CNAs may also play more subtle roles in terms of modulating genetic potential and broad gene expression patterns of developing tumors. Any significant differences in the overall CNA patterns between different cancer types may thus point towards specific biological mechanisms acting in those cancers. In addition, differences among CNA profiles may prove valuable for cancer classifications beyond existing annotation systems. PRINCIPAL FINDINGS: We have analyzed molecular-cytogenetic data from 25579 tumors samples, which were classified into 160 cancer types according to the International Classification of Disease (ICD) coding system. When correcting for differences in the overall CNA frequencies between cancer types, related cancers were often found to cluster together according to similarities in their CNA profiles. Based on a randomization approach, distance measures from the cluster dendrograms were used to identify those specific genomic regions that contributed significantly to this signal. This approach identified 43 non-neutral genomic regions whose propensity for the occurrence of copy number alterations varied with the type of cancer at hand. Only a subset of these identified loci overlapped with previously implied, highly recurrent (hot-spot) cytogenetic imbalance regions. CONCLUSIONS: Thus, for many genomic regions, a simple null-hypothesis of independence between cancer type and relative copy number alteration frequency can be rejected. Since a subset of these regions display relatively low overall CNA frequencies, they may point towards second-tier genomic targets that are adaptively relevant but not necessarily essential for cancer development.</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      January 01, 2012
      
    </span>
</aside>
    <h3 class="title_news">arrayMap: a...</h3>

<!-- Use a hidden p tag to provide a preview -->
<h3 id="arraymap-a-reference-resource-for-genomic-copy-number-imbalances-in-human-malignancies">arrayMap: a reference resource for genomic copy number imbalances in human malignancies.<a class="headerlink" href="#arraymap-a-reference-resource-for-genomic-copy-number-imbalances-in-human-malignancies" title="Permanent link">&para;</a></h3>
<h4 id="cai-h-kumar-n-baudis-m">Cai H, Kumar N, Baudis M.<a class="headerlink" href="#cai-h-kumar-n-baudis-m" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> BACKGROUND: The delineation of genomic copy number abnormalities (CNAs) from cancer samples has been instrumental for identification of tumor suppressor genes and oncogenes and proven useful for clinical marker detection. An increasing number of projects have mapped CNAs using high-resolution microarray based techniques. So far, no single resource does provide a global collection of readily accessible oncogenomic array data. METHODOLOGY/PRINCIPAL FINDINGS: We here present arrayMap, a curated reference database and bioinformatics resource targeting copy number profiling data in human cancer. The arrayMap database provides a platform for meta-analysis and systems level data integration of high-resolution oncogenomic CNA data. To date, the resource incorporates more than 40,000 arrays in 224 cancer types extracted from several resources, including the NCBI's Gene Expression Omnibus (GEO), EBI's ArrayExpress (AE), The Cancer Genome Atlas (TCGA), publication supplements and direct submissions. For the majority of the included datasets, probe level and integrated visualization facilitate gene level and genome wide data review. Results from multi-case selections can be connected to downstream data analysis and visualization tools. CONCLUSIONS/SIGNIFICANCE: To our knowledge, currently no data source provides an extensive collection of high resolution oncogenomic CNA data which readily could be used for genomic feature mining, across a representative range of cancer entities. arrayMap represents our effort for providing a long term platform for oncogenomic CNA data independent of specific platform considerations or specific project dependence. The online database can be accessed at http//<a href="http://www.arraymap.org">www.arraymap.org</a>.</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  

            </article>
          </div>
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      &copy; Copyright 2007 - 2022 Michael Baudis
    </div>
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
        <div class="md-social">
  
    
    
      
      
    
    <a href="https://github.com/baudisgroup" target="_blank" rel="noopener" title="github.com" class="md-social__link">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    <script id="__config" type="application/json">{"base": "..", "features": ["content.tabs.link", "search.highlight", "search.share", "navigation.instant", "navigation.tracking", "navigation.sections", "toc.integrate"], "translations": {"clipboard.copy": "Copy to clipboard", "clipboard.copied": "Copied to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.placeholder": "Type to start searching", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.term.missing": "Missing", "select.version.title": "Select version"}, "search": "../assets/javascripts/workers/search.22074ed6.min.js"}</script>
    
    
      <script src="../assets/javascripts/bundle.960e086b.min.js"></script>
      
    
  </body>
</html>