
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
        <meta name="description" content="Computational Oncogenomics Group @ UZH & SIB">
      
      
        <meta name="author" content="Michael Baudis">
      
      
      <link rel="icon" href="../img/favicon.ico">
      <meta name="generator" content="mkdocs-1.2.3, mkdocs-material-8.1.8">
    
    
      
        <title>Presentations - baudisgroup @ UZH & SIB</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.6e60f8b8.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.e6a45f82.min.css">
        
          
          
          <meta name="theme-color" content="#02a6f2">
        
      
    
    
    
      
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,400i,700%7CRoboto+Mono&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../css/theme_overrides.css">
    
    <script>__md_scope=new URL("..",location),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="" data-md-color-primary="light-blue" data-md-color-accent="indigo">
  
    
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="baudisgroup @ UZH &amp; SIB" class="md-header__button md-logo" aria-label="baudisgroup @ UZH & SIB" data-md-component="logo">
      
  <img src="../img/site_logo.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            baudisgroup @ UZH & SIB
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Presentations
            
          </span>
        </div>
      </div>
    </div>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"/></svg>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
          <a href="javascript:void(0)" class="md-search__icon md-icon" aria-label="Share" data-clipboard data-clipboard-text="" data-md-component="search-share" tabindex="-1">
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M18 16.08c-.76 0-1.44.3-1.96.77L8.91 12.7c.05-.23.09-.46.09-.7 0-.24-.04-.47-.09-.7l7.05-4.11c.54.5 1.25.81 2.04.81a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3c0 .24.04.47.09.7L8.04 9.81C7.5 9.31 6.79 9 6 9a3 3 0 0 0-3 3 3 3 0 0 0 3 3c.79 0 1.5-.31 2.04-.81l7.12 4.15c-.05.21-.08.43-.08.66 0 1.61 1.31 2.91 2.92 2.91 1.61 0 2.92-1.3 2.92-2.91A2.92 2.92 0 0 0 18 16.08z"/></svg>
          </a>
        
        <button type="reset" class="md-search__icon md-icon" aria-label="Clear" tabindex="-1">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/baudisgroup/baudisgroup.github.io/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 480 512"><path d="M186.1 328.7c0 20.9-10.9 55.1-36.7 55.1s-36.7-34.2-36.7-55.1 10.9-55.1 36.7-55.1 36.7 34.2 36.7 55.1zM480 278.2c0 31.9-3.2 65.7-17.5 95-37.9 76.6-142.1 74.8-216.7 74.8-75.8 0-186.2 2.7-225.6-74.8-14.6-29-20.2-63.1-20.2-95 0-41.9 13.9-81.5 41.5-113.6-5.2-15.8-7.7-32.4-7.7-48.8 0-21.5 4.9-32.3 14.6-51.8 45.3 0 74.3 9 108.8 36 29-6.9 58.8-10 88.7-10 27 0 54.2 2.9 80.4 9.2 34-26.7 63-35.2 107.8-35.2 9.8 19.5 14.6 30.3 14.6 51.8 0 16.4-2.6 32.7-7.7 48.2 27.5 32.4 39 72.3 39 114.2zm-64.3 50.5c0-43.9-26.7-82.6-73.5-82.6-18.9 0-37 3.4-56 6-14.9 2.3-29.8 3.2-45.1 3.2-15.2 0-30.1-.9-45.1-3.2-18.7-2.6-37-6-56-6-46.8 0-73.5 38.7-73.5 82.6 0 87.8 80.4 101.3 150.4 101.3h48.2c70.3 0 150.6-13.4 150.6-101.3zm-82.6-55.1c-25.8 0-36.7 34.2-36.7 55.1s10.9 55.1 36.7 55.1 36.7-34.2 36.7-55.1-10.9-55.1-36.7-55.1z"/></svg>
  </div>
  <div class="md-source__repository">
    baudisgroup.github.io
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  

<nav class="md-nav md-nav--primary md-nav--integrated" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="baudisgroup @ UZH &amp; SIB" class="md-nav__button md-logo" aria-label="baudisgroup @ UZH & SIB" data-md-component="logo">
      
  <img src="../img/site_logo.png" alt="logo">

    </a>
    baudisgroup @ UZH & SIB
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/baudisgroup/baudisgroup.github.io/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 480 512"><path d="M186.1 328.7c0 20.9-10.9 55.1-36.7 55.1s-36.7-34.2-36.7-55.1 10.9-55.1 36.7-55.1 36.7 34.2 36.7 55.1zM480 278.2c0 31.9-3.2 65.7-17.5 95-37.9 76.6-142.1 74.8-216.7 74.8-75.8 0-186.2 2.7-225.6-74.8-14.6-29-20.2-63.1-20.2-95 0-41.9 13.9-81.5 41.5-113.6-5.2-15.8-7.7-32.4-7.7-48.8 0-21.5 4.9-32.3 14.6-51.8 45.3 0 74.3 9 108.8 36 29-6.9 58.8-10 88.7-10 27 0 54.2 2.9 80.4 9.2 34-26.7 63-35.2 107.8-35.2 9.8 19.5 14.6 30.3 14.6 51.8 0 16.4-2.6 32.7-7.7 48.2 27.5 32.4 39 72.3 39 114.2zm-64.3 50.5c0-43.9-26.7-82.6-73.5-82.6-18.9 0-37 3.4-56 6-14.9 2.3-29.8 3.2-45.1 3.2-15.2 0-30.1-.9-45.1-3.2-18.7-2.6-37-6-56-6-46.8 0-73.5 38.7-73.5 82.6 0 87.8 80.4 101.3 150.4 101.3h48.2c70.3 0 150.6-13.4 150.6-101.3zm-82.6-55.1c-25.8 0-36.7 34.2-36.7 55.1s10.9 55.1 36.7 55.1 36.7-34.2 36.7-55.1-10.9-55.1-36.7-55.1z"/></svg>
  </div>
  <div class="md-source__repository">
    baudisgroup.github.io
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        Baudisgroup Home
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../group" class="md-nav__link">
        Group
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../publications" class="md-nav__link">
        Publications
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="." class="md-nav__link">
        Presentations
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="https://schemablocks.org" class="md-nav__link">
        SchemaBlocks {S}[B] &#8599;
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="https://beacon-project.io" class="md-nav__link">
        Beacon Project &#8599;
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="https://www.mls.uzh.ch/en/research/baudis.html" class="md-nav__link">
        Michael Baudis @ UZH &#8599;
      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
          
          <div class="md-content" data-md-component="content">
            <article class="md-content__inner md-typeset">
              
  <h1 id="presentations">Presentations<a class="headerlink" href="#presentations" title="Permanent link">&para;</a></h1>
<p>The shows a probably incomplete following list of presentations (posters, talks, abstracts) from group members.
Usually, PDF versions of the slides or posters are linked from the details pages.</p>
  

  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      July 31, 2022
      
    </span>
</aside>
    <h3 class="title_news">A cancer genomics resource built on GA4GH standards</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="rahel-paloots-michael-baudis">Rahel Paloots, Michael Baudis<a class="headerlink" href="#rahel-paloots-michael-baudis" title="Permanent link">&para;</a></h4>
<h5 id="cgc-st-louis-2022">CGC St Louis 2022<a class="headerlink" href="#cgc-st-louis-2022" title="Permanent link">&para;</a></h5>
<p>Progenetix is a cancer genomics resource that includes genomic profiling data as well as biomedical annotations and provenance data for cancer studies. The main goal of the Progenetix database is to provide easy, open access for research studies and clinical diagnostics. To facilitate sharing of genomic data, Progenetix complies with and contributes to GA4GH and Beacon data standards. Beacon, developed with the support from ELXIR (the European bioinformatics infrastructure organization), started out as protocol to share genomic variants over federated queries.</p>

<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      June 27, 2022
      
    </span>
</aside>
    <h3 class="title_news">Progenetix, BeaconPlus & GA4GH</h3>

<!-- Use a hidden p tag to provide a preview -->
<h5 id="a-cancer-genomics-reference-resource-powered-by-ga4gh-standards">A cancer genomics reference resource powered by GA4GH standards<a class="headerlink" href="#a-cancer-genomics-reference-resource-powered-by-ga4gh-standards" title="Permanent link">&para;</a></h5>
<h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<h4 id="roche-data-science-seminar">Roche Data Science Seminar<a class="headerlink" href="#roche-data-science-seminar" title="Permanent link">&para;</a></h4>
<p>The presentation reports about the Progenetix cancer genomics resource and its role
in the GA4GH ecosystem &amp; the Beacon genomics API development process.</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      June 13, 2022
      
    </span>
</aside>
    <h3 class="title_news">Implementation of the GA4GH Beacon protocol for discovery and sharing of genomic copy number variation data</h3>

<!-- Use a hidden p tag to provide a preview -->
<h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<h4 id="eshg-vienna-2022">ESHG Vienna 2022<a class="headerlink" href="#eshg-vienna-2022" title="Permanent link">&para;</a></h4>
<p><strong>Background &amp; Objectives</strong> Genomic copy number variations (CNV) are a major contributor to inter-individual genomic variation, can be causative events in rare diseases, but especially represent the majority of the mutational landscape in the most malignancies. While specific CNV events and some recurring patterns have contributed to the identification of individual cancer drivers and the recognition of cancer  subtypes, the complexity of genomic CNV patterns requires large amounts of well-defined genomic profiles for statistically meaningful analyses. At the other end of the spectrum, in the area of rare disease genomics the potential pathogenicity of individual CNV events requires validation against a vast set of disease-related and reference genomic profiles and annotations.</p>

<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      June 07, 2022
      
    </span>
</aside>
    <h3 class="title_news">Progenetix & BeaconPlus</h3>

<h5 class="title_news">An open cancer genomics resource on a stack of Beacon code...</h5>

<!-- Use a hidden p tag to provide a preview -->
<h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<h4 id="elixir-all-hands-amsterdam-2022">ELIXIR All Hands Amsterdam 2022<a class="headerlink" href="#elixir-all-hands-amsterdam-2022" title="Permanent link">&para;</a></h4>
<p>Here Michael provides some overview of the multi-year trajectory of the Beacon API
development, and how BeaconPlus &amp; Progenetix have been utilized for "implementation
driven design".</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      March 02, 2021
      
    </span>
</aside>
    <h3 class="title_news">GA4GH Connect - Beacon v2 and SchemaBlocks</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="ga4gh-connect-2020"><a href="https://broadinstitute.swoogo.com/ga4gh-connect-2021/?i=sNK7RoeeqhyJE1ueSnn5lz4F9RRfnnIU">GA4GH Connect 2020</a><a class="headerlink" href="#ga4gh-connect-2020" title="Permanent link">&para;</a></h4>
<h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<h5 id="beacon-v2-structural-variants-slides">Beacon v2 Structural Variants <a href="http://info.baudisgroup.org/pdf/2021-03-02___Michael-Baudis__Beacon-structural-variants-queries.pdf">[slides]</a><a class="headerlink" href="#beacon-v2-structural-variants-slides" title="Permanent link">&para;</a></h5>
<h5 id="schemablocks-sb-slides">SchemaBlocks {S}[B] <a href="http://info.baudisgroup.org/pdf/2021-03-04___Michael-Baudis__SchemaBocks__GA4GH-Connect-2021-slides.pdf">[slides]</a><a class="headerlink" href="#schemablocks-sb-slides" title="Permanent link">&para;</a></h5>

<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      December 07, 2020
      
    </span>
</aside>
    <h3 class="title_news">Genomic data and Privacy</h3>

<!-- Use a hidden p tag to provide a preview -->
<h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<h4 id="ethz-lecture">ETHZ Lecture<a class="headerlink" href="#ethz-lecture" title="Permanent link">&para;</a></h4>
<p>The understanding of the impact of individual inherited and somatic genome variants on phenotypes and diseases requires a thorough understanding of the occurrence of such variants amongst populations in general and carriers of the phenotypes and diseases in particular. This information can only be provided through the inclusion of data from a multitude of genome resources  in variant evaluation efforts, including such from outside (international) jurisdictions. However, opening such resources carries the inherent risk of breaching privacy, particularly through re-identification of individuals or their relatives and potentially through the exposure of individual genome-related personal information including  phenotypic and "performance" prediction and relative disease risk.</p>

<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      November 04, 2020
      
    </span>
</aside>
    <h3 class="title_news">Beacon v2 – Towards flexible use and clinical applications for a reference genomic data protocol</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="sphn-webinar"><a href="https://sphn.ch/seminar-training/beacon-project/">SPHN Webinar</a><a class="headerlink" href="#sphn-webinar" title="Permanent link">&para;</a></h4>
<h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<p>Genomic “Beacons” provide discovery services for genomic data using the Beacon API developed as a key driver project of the Global Alliance for Genomics and Health (GA4GH). The Beacon protocol itself defines an open standard for genomics data discovery and provides a framework for web services responding to queries against genomic data collections, for instance from population based or disease specific genome repositories. Sites offering beacons can scale through aggregation in “Beacon Networks” which distribute queries among a potentially large number of beacons and assemble their responses.</p>
<p>As part of ELIXIR’s Beacon 2019–21 project work has advanced on a re-designed Beacon protocol, with the aim to provide a maximum of flexibility while closely adhering to data and security standards promoted by the international research community, aligned with the GA4GH ecosystem. Beacon v2 features have been designed with view to clinical and biomedical research, e.g. for variant evidence and annotation data or the retrieval of diagnostic and other clinical data when used in secure environments. Recently, the efficient use of the Beacon protocol for non-human genome data has been demonstrated for COVID-19 repositories, showcasing the flexibility of the protocol.</p>
<p>Here I will provide an overview about the history, current status future directions of the Beacon protocol, including details about the Beacon+ service on top of the progenetix.org cancer genomics resource.</p>
<h4 id="additional-links">Additional Links<a class="headerlink" href="#additional-links" title="Permanent link">&para;</a></h4>
<ul>
<li><a href="http://beacon-project.io">Beacon Website</a></li>
</ul>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      October 30, 2020
      
    </span>
</aside>
    <h3 class="title_news">Update of Progenetix Oncogenomics Resource</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="research-progress-report-dmls-university-of-zurich"><a href="https://www.mls.uzh.ch/en/seminars.html">Research Progress Report, DMLS, University of Zurich</a><a class="headerlink" href="#research-progress-report-dmls-university-of-zurich" title="Permanent link">&para;</a></h4>
<h3 id="qingyao-huang">Qingyao Huang<a class="headerlink" href="#qingyao-huang" title="Permanent link">&para;</a></h3>
<p>Copy number aberration (CNA) is frequently observed in cancer genomes. Meta-analysis of genomic variations helps to disentangle the multiplex molecular mechanism underlying tumorigenesis as well as identify and characterize molecular subtypes. Over the years, cancer genomic research have resulted in a considerable amount of data segregated by studies. The Progenetix project (<a href="http://www.progenetix.org">www.progenetix.org</a>), initiated in 2001, aims to systematize the published cancer genomic profiles and provide accurate annotation to facilitate integrative analysis. Since the last update of the Progenetix database in 2013, the field of genomics research and cancer research has seen significant advancement in terms of molecular genetics technology, annotation/reference refinement, data standard harmonization and data gathering in an increasingly structured and systematic manner. Continuous data integration, curation and maintenance have been dedicated to provide the most comprehensive representation of the cancer genome profiling data. </p>
<p>Since last release, there have been significant expansion on sample number, cancer type diversity, publications, study cohorts and technology platforms, improvements on data quality, ontology representation, as well as stuctural changes in terms of database schema, and data access including web interface. In this presentation, we detail the 2020 update of the Progenetix in six aspects: 1. Introduction, 2 New meta-data features with domain-specific mapping, 3 Data sources with sample expansion and process pipeline, 4 Data standard and object model, 5 Adopted Beacon protocol and associated features, 6 New web interface and features. </p>
<h4 id="additional-links">Additional Links<a class="headerlink" href="#additional-links" title="Permanent link">&para;</a></h4>
<ul>
<li><a href="https://progenetix.org/storage-ext/2020-10-30___Qingyao__DMLS_progress_report_recording.mp4">Video Recording of Presentation</a></li>
<li><a href="http://progenetix.org">Progenetix Website</a></li>
<li><a href="http://beacon-project.io">Beacon Website</a></li>
</ul>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      September 30, 2020
      
    </span>
</aside>
    <h3 class="title_news">GA4GH Beacon v2 at GA4GH Plenary</h3>

<!-- Use a hidden p tag to provide a preview -->
<h2 id="ga4gh-beacon-v2-evolving-reference-standard-for-genomic-data-exchange">GA4GH Beacon v2 - Evolving Reference Standard for Genomic Data Exchange<a class="headerlink" href="#ga4gh-beacon-v2-evolving-reference-standard-for-genomic-data-exchange" title="Permanent link">&para;</a></h2>
<h4 id="ga4gh-8th-plenary"><a href="https://broadinstitute.swoogo.com/ga4gh-8th-plenary/455678">GA4GH 8th Plenary</a><a class="headerlink" href="#ga4gh-8th-plenary" title="Permanent link">&para;</a></h4>
<h3 id="gary-saunders-jordi-rambla-de-argila-anthony-brookes-juha-tornroos-and-michael-baudis">Gary Saunders, Jordi Rambla de Argila, Anthony Brookes, Juha Törnroos and Michael Baudis<a class="headerlink" href="#gary-saunders-jordi-rambla-de-argila-anthony-brookes-juha-tornroos-and-michael-baudis" title="Permanent link">&para;</a></h3>
<h5 id="for-the-elixir-beacon-project-ga4gh-discovery-work-stream-and-the-international-network-of-beacon-api-developers">For the ELIXIR Beacon project, GA4GH Discovery work stream and the international network of Beacon API developers<a class="headerlink" href="#for-the-elixir-beacon-project-ga4gh-discovery-work-stream-and-the-international-network-of-beacon-api-developers" title="Permanent link">&para;</a></h5>
<p>The Beacon driver project was one of the earliest initiatives of the Global Alliance for Genomics and Health with the Beacon v1.0 API as first approved GA4GH standard.
Version 2 of the protocol is slated to provide fundamental changes, towards a Internet of Genomics foundational standard:
* requests beyond genomic variants ("filters")
* payload responses, secured through open AAI
* aligning w/ GA4GH standards (Phenopackets, VRS, DUO...) through SchemaBlocks <a href="https://schemablocks.org">{S}[B]</a>
* Working with international partners on deployment of advanced implementations</p>
<h4 id="additional-links">Additional Links<a class="headerlink" href="#additional-links" title="Permanent link">&para;</a></h4>
<ul>
<li><a href="http://beacon-project.io">Beacon Website</a></li>
<li><a href="https://schemablocks.org">SchemaBlocks {S}[B]</a></li>
<li><a href="https://progenetix.org/beaconplus-instances/beaconplus/">Beacon+ in progenetix-next</a></li>
<li><a href="https://github.com/progenetix/bycon">Bycon Project</a></li>
</ul>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      September 30, 2020
      
    </span>
</aside>
    <h3 class="title_news">Progenetix - A cancer genomics reference resource around GA4GH standards</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="ga4gh-8th-plenary"><a href="https://broadinstitute.swoogo.com/ga4gh-8th-plenary/455678">GA4GH 8th Plenary</a><a class="headerlink" href="#ga4gh-8th-plenary" title="Permanent link">&para;</a></h4>
<h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<p>The Progenetix oncogenomics resource provides sample-specific cancer
genome profiling data and biomedical annotations as well as provenance data from cancer studies. Especially through currently 113322 curated genomic copy number number (CNV) profiles from 1600 individual studies representing over 500 cancer types (NCIt), Progenetix empowers aggregate and comparative analyses which vastly exceed individual studies or single diagnostic concepts.</p>
<p>Progenetix has been widely used in research studies, clinical diagnostics and recently in the development of data protocols and standards with the Global Alliance for Genomics and Health (GA4GH) and the European bioinformatics initiative ELIXIR. Specifically, the Beacon+ service, implemented on top of the Progenetix datasets, has been instrumental in developing and testing advanced features of the Beacon protocol such as structural variant queries, "handover" data delivery and the implementation of queries for phenoptypic and diagnostic parameters using CURIE based "filters". During development of GA4GH metadata concepts and schemas - which have influenced emerging standards such as the Phenopackets format - cancer specific annotations from the Progenetix collection have served to identify conceptual requirements and to develop domain-specific mappings (e.g. ICD-O to NCIt translations). The resource's focus on structural genome variants has been instrumental in addressing their specific requirements in GA4GH schema development and the Beacon protocol.</p>
<p>Here, we will demonstrate how an open genomic reference resource has been built around GA4GH standards and how it is being used to support ongoing and future developments in GA4GH standard development and ELIXIR implementation studies.</p>
<p>progenetix.org</p>
<h4 id="additional-links">Additional Links<a class="headerlink" href="#additional-links" title="Permanent link">&para;</a></h4>
<ul>
<li><a href="http://beacon-project.io">Beacon Website</a></li>
<li><a href="https://progenetix.org/beacon-plus/search">Beacon+ in progenetix-next</a></li>
<li><a href="https://github.com/progenetix/bycon">Bycon Project</a></li>
</ul>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      September 02, 2020
      
    </span>
</aside>
    <h3 class="title_news">Cancer Data - ELIXIR::GA4GH: Advancing genomics resources through standards and ontologies</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="eccb2020"><a href="https://eccb2020.info/ntbew02-elixirga4gh-advancing-genomics-through-expedited-data-access-enabled-by-standards-and-ontologies/">ECCB2020</a><a class="headerlink" href="#eccb2020" title="Permanent link">&para;</a></h4>
<h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<h4 id="additional-links">Additional Links<a class="headerlink" href="#additional-links" title="Permanent link">&para;</a></h4>
<ul>
<li><a href="http://beacon-project.io">Beacon Website</a></li>
<li><a href="http://next.progenetix.org/">Beacon+ in progenetix-next</a></li>
<li><a href="https://github.com/progenetix/bycon">Bycon Project</a></li>
</ul>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      June 30, 2020
      
    </span>
</aside>
    <h3 class="title_news">Beacon v2 - Flexible Use and Clinical Applications</h3>

<!-- Use a hidden p tag to provide a preview -->
<h2 id="beacon-v2-towards-flexible-use-and-clinical-applications-for-a-reference-genomic-data-sharing-protocol">Beacon v2 - Towards Flexible Use and Clinical Applications for a Reference Genomic Data Sharing Protocol<a class="headerlink" href="#beacon-v2-towards-flexible-use-and-clinical-applications-for-a-reference-genomic-data-sharing-protocol" title="Permanent link">&para;</a></h2>
<h4 id="personalized-health-technologies-2020"><a href="https://www.personalizedhealth.nexus.ethz.ch">Personalized Health Technologies 2020</a><a class="headerlink" href="#personalized-health-technologies-2020" title="Permanent link">&para;</a></h4>
<h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<p>Beacons provide discovery services for genomic data using the Beacon API developed under
the leadership of ELIXIR, as a key driver project of the Global Alliance for Genomics and
Health (GA4GH). The Beacon protocol itself defines an open standard for genomics data
discovery. It provides a framework for public web services responding to queries against
genomic data collections, for instance from population based or disease specific genome
repositories. Sites offering beacons can scale through aggregation in "Beacon Networks",
which distribute single genome queries among a potentially large number of international
beacons and assemble their responses.</p>
<p>As part of ELIXIR’s Beacon 2019-21 project work has started on a radically re-designed
Beacon protocol, with the aim to provide a maximum of flexibility while closely adhering
to data and security standards promoted by the international research community, and
especially as part of projects in the GA4GH ecosystem. Upcoming features have been designed
with view to biomedical applications, e.g. for variant evidence and annotation data or the
retrieval of phenotype and other clinical data when used in secure (network) environments.</p>
<p>Recently, the efficient use of the Beacon protocol for non-human genome data has been
demonstrated for COVID-19 repositories, showcasing the flexibility of the protocol.</p>
<p>Here I will provide an overview about the history, current status future directions of the
Beacon protocol, including details about the Beacon+ service on top of the <a href="http://progenetix.org">progenetix.org</a>
cancer genomics resource.</p>
<h4 id="additional-links">Additional Links<a class="headerlink" href="#additional-links" title="Permanent link">&para;</a></h4>
<ul>
<li><a href="http://beacon-project.io">Beacon Website</a></li>
<li><a href="http://beacon.progenetix.org">Beacon+</a></li>
<li><a href="http://beacon.progenetix.org">Bycon Project</a></li>
</ul>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      June 10, 2020
      
    </span>
</aside>
    <h3 class="title_news">ELIXIR All Hands - Beacon Evolution</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="elixir-all-hands-2020-beacon-workshop">ELIXIR All Hands 2020 - Beacon Workshop<a class="headerlink" href="#elixir-all-hands-2020-beacon-workshop" title="Permanent link">&para;</a></h4>
<h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<p>This presentation covers some of Beacon's
origins, features and directions.</p>
<h4 id="links">Links<a class="headerlink" href="#links" title="Permanent link">&para;</a></h4>
<ul>
<li><a href="http://info.baudisgroup.org/pdf/2020-06-10___Michael-Baudis__Beacon-evolution__ELIXIR-All-Hands-slides.pdf">Presentation slides</a></li>
<li><a href="http://beacon-project.io">Beacon Website</a></li>
</ul>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      April 20, 2020
      
    </span>
</aside>
    <h3 class="title_news">SchemaBlocks and GA4GH TASC</h3>

<!-- Use a hidden p tag to provide a preview -->
<p>A brief presentation about SchemaBlocks concepts and its possible integration into the new GA4GH TASC effort.</p>
<h4 id="additional-links">Additional Links<a class="headerlink" href="#additional-links" title="Permanent link">&para;</a></h4>
<ul>
<li><a href="http://schemablocks.org">schemablocks</a></li>
<li><a href="http://ga4gh.org">GA4GH</a></li>
</ul>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      March 19, 2020
      
    </span>
</aside>
    <h3 class="title_news">SWISSNEX SF Lunch Seminar - Data Mining in Genomics</h3>

<!-- Use a hidden p tag to provide a preview -->
<h3 id="genomic-research-and-personalised-health-strategies">Genomic Research and Personalised Health Strategies<a class="headerlink" href="#genomic-research-and-personalised-health-strategies" title="Permanent link">&para;</a></h3>
<h4 id="resources-standards-protocols-tools-discourse">Resources | Standards | Protocols | Tools | Discourse<a class="headerlink" href="#resources-standards-protocols-tools-discourse" title="Permanent link">&para;</a></h4>
<p>These are the <a href="/pdf/2020-03-19___Swissnex-SF-Michael-Baudis-Biodata.pdf">slides</a>
of a short presentation, given (virtually, since COVID-19) for a
<a href="https://www.swissnexsanfrancisco.org">SWISSNEX San Francisco</a> lunch meeting.</p>
<h4 id="additional-links">Additional Links<a class="headerlink" href="#additional-links" title="Permanent link">&para;</a></h4>
<ul>
<li><a href="https://progenetix.org">Progenetix</a></li>
<li><a href="http://schemablocks.org">schemablocks</a></li>
<li><a href="http://beacon.progenetix.org">Beacon+</a></li>
<li><a href="http://ga4gh.org">GA4GH</a></li>
<li><a href="http://beacon-project.io">ELIXIR Beacon Project</a></li>
</ul>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      February 14, 2020
      
    </span>
</aside>
    <h3 class="title_news">BBOP Presentation - Baudisgroup Projects & Interests</h3>

<!-- Use a hidden p tag to provide a preview -->
<h5 id="links">Links<a class="headerlink" href="#links" title="Permanent link">&para;</a></h5>
<ul>
<li><a href="https://progenetix.org">Progenetix</a>
     <a href="https://info.progenetix.org/doc/+generated-doc-API-api/">Progenetix API</a></li>
<li><a href="http://schemablocks.org">schemablocks</a></li>
<li><a href="http://beacon.progenetix.org">Beacon+</a></li>
<li><a href="http://beacon-project.io">ELIXIR Beacon Project</a></li>
<li><a href="http://ga4gh-discovery.github.io">GA4GH :: Discovery</a></li>
</ul>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      February 06, 2020
      
    </span>
</aside>
    <h3 class="title_news">ELIXIR Beacon Project - Networking Resources Across and Beyond ELIXIR Human Data Communities</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="elixir-open-day-wellcome-trust-genome-campus-hinxton"><a href="https://elixir-europe.org/events/elixir-open-day">ELIXIR Open Day</a> - Wellcome Trust Genome Campus Hinxton<a class="headerlink" href="#elixir-open-day-wellcome-trust-genome-campus-hinxton" title="Permanent link">&para;</a></h4>
<h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<p>In this presentation I introduce the Beacon project and provide
my opinions about its future trajectory, and especially its role
in driving the alignment of ELIXIR and GA4GH projects in related
to (human) genome data sharing.</p>
<h5 id="links">Links<a class="headerlink" href="#links" title="Permanent link">&para;</a></h5>
<ul>
<li><a href="http://info.baudisgroup.org/pdf/2020-02-06___Michael-Baudis__Beacon-Networks__ELIXIR-Day-2020.pdf">Presentation slides</a></li>
<li><a href="http://beacon-project.io">ELIXIR Beacon Project</a></li>
<li><a href="http://beacon.progenetix.org">Beacon+</a></li>
<li><a href="http://schemablocks.org">schemablocks</a></li>
<li><a href="http://ga4gh-discovery.github.io">GA4GH :: Discovery</a></li>
</ul>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      January 28, 2020
      
    </span>
</aside>
    <h3 class="title_news">GA4GH SchemaBlocks for Human Cell Atlas</h3>

<!-- Use a hidden p tag to provide a preview -->
<p>This is a presentation of the SchemaBlocks initiative and the overall GA4GH
context, for the Human Cell Atlas project, given by Michael at one of their
teleconferences.</p>
<h4 id="additional-links">Additional Links<a class="headerlink" href="#additional-links" title="Permanent link">&para;</a></h4>
<ul>
<li><a href="http://schemablocks.org">schemablocks</a></li>
<li><a href="http://ga4gh.org">GA4GH</a></li>
<li><a href="https://www.humancellatlas.org">Human Cell Atlas</a></li>
</ul>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      November 08, 2019
      
    </span>
</aside>
    <h3 class="title_news">Talk at St. Gallen Radiation Oncology - Bioinformatics and Data Exchange</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="3rd-st-gallen-radiation-oncology-informatics-meeting">3rd St. Gallen Radiation Oncology Informatics Meeting<a class="headerlink" href="#3rd-st-gallen-radiation-oncology-informatics-meeting" title="Permanent link">&para;</a></h4>
<h2 id="bioinformatics-and-data-exchange-for-genomics-in-an-international-context">Bioinformatics and data exchange for genomics in an international context<a class="headerlink" href="#bioinformatics-and-data-exchange-for-genomics-in-an-international-context" title="Permanent link">&para;</a></h2>
<h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<p>The presentation at the St. Gallen meeting introduced the audience to the 
group's research and resources, and how this is connected to the different
national &amp; international data standards and sharing initiatives.</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      October 08, 2019
      
    </span>
</aside>
    <h3 class="title_news">Talk at AMED Tokyo -  Cancer Genomics and Standards</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="mini-symposium-about-cnv-and-data-standards-at-amed-japan-tokyo">Mini-Symposium about CNV and Data Standards at <a href="https://www.amed.go.jp/en/index.html">AMED Japan, Tokyo</a><a class="headerlink" href="#mini-symposium-about-cnv-and-data-standards-at-amed-japan-tokyo" title="Permanent link">&para;</a></h4>
<h2 id="cancer-genomics-and-implementation-of-data-driven-standards-for-genomic-data-exchange">Cancer Genomics and Implementation of Data Driven Standards for Genomic Data Exchange<a class="headerlink" href="#cancer-genomics-and-implementation-of-data-driven-standards-for-genomic-data-exchange" title="Permanent link">&para;</a></h2>
<h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<p>At this meeting, several Japanese participants presented their research and 
results, with a focus on Copy Number Variants and other structural genome 
variations.</p>
<p>Michael presented the overarching work of the group, from curating &amp; analyzing 
cancer CNV datasets towards the projects (Beacon, Schemablocks, h-CNV) to enable 
sharing of these and other types of genomic data.</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      September 10, 2019
      
    </span>
</aside>
    <h3 class="title_news">Minimum Error Calibration and Normalization for Genomic Copy Number Analysis</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="bc2-2019-basel">BC2 2019, Basel<a class="headerlink" href="#bc2-2019-basel" title="Permanent link">&para;</a></h4>
<h2 id="minimum-error-calibration-and-normalization-for-genomic-copy-number-analysis">Minimum Error Calibration and Normalization for Genomic Copy Number Analysis<a class="headerlink" href="#minimum-error-calibration-and-normalization-for-genomic-copy-number-analysis" title="Permanent link">&para;</a></h2>
<h3 id="bo-gao">Bo Gao<a class="headerlink" href="#bo-gao" title="Permanent link">&para;</a></h3>
<h4 id="abstract">Abstract<a class="headerlink" href="#abstract" title="Permanent link">&para;</a></h4>
<p><strong>Background</strong>:<br />
Copy number variations (CNV) are regional deviations from thenormal autosomal bi-allelic DNA content. While germline CNVs are a majorcontributor to genomic syndromes and inherited diseases, the majority of cancersaccumulate extensive ”somatic” CNV (sCNV or CNA) during the process ofoncogenetic transformation and progression. While specific sCNV have closelybeen associated with tumorigenesis, intriguingly many neoplasias exhibitrecurrent sCNV patterns beyond the involvement of a few cancer driver genes.Currently, CNV profiles of tumor samples are generated using genomicmicro-arrays or high-throughput DNA sequencing. Regardless of the underlyingtechnology, genomic copy number data is derived from the relative assessmentand integration of multiple signals, with the data generation process being proneto contamination from several sources. Estimated copy number values have noabsolute and linear correlation to their corresponding DNA levels, and the extentof deviation differs between sample profiles which poses a great challenge fordata integration and comparison in large scale genome analysis.</p>
<p><strong>Results</strong>:<br />
In this study, we present a novel method named Minimum ErrorCalibration and Normalization of Copy Numbers Analysis (Mecan4CNA). Foreach sCNV profile,Mecan4CNAreduces the noise level, calculates valuesrepresenting the normal DNA copies (baseline) and the change of one copy (leveldistance), and finally normalizes all values. Experiments ofMecan4CNAonsimulated data showed an overall accuracy of 93% and 91% in determining thebaseline and level distance, respectively. Comparison of baseline and level distanceestimation with existing methods and karyotyping data on the NCI-60 tumor cellline produced coherent results. To estimate the method’s impact on downstreamanalyses we performed GISTIC analyses on original andMecan4CNAdata fromthe Cancer Genome Atlas (TCGA) where the normalized data showed prominentimprovements of both sensitivity and specificity in detecting focal regions.</p>
<p><strong>Conclusions</strong>: 
In general,Mecan4CNAprovides an advanced method for CNAdata normalization especially in research involving data of high volume andheterogeneous quality. With its informative output and visualization, it can alsofacilitate analysis of individual CNA profiles.Mecan4CNAis freely available as aPython package and through Github.</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      September 10, 2019
      
    </span>
</aside>
    <h3 class="title_news">Cancer cell lines in focus: somatic copy number & germline variation</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="bc2-2019-basel">BC2 2019, Basel<a class="headerlink" href="#bc2-2019-basel" title="Permanent link">&para;</a></h4>
<h3 id="qingyao-huang">Qingyao Huang<a class="headerlink" href="#qingyao-huang" title="Permanent link">&para;</a></h3>
<h4 id="abstract">Abstract<a class="headerlink" href="#abstract" title="Permanent link">&para;</a></h4>
<p><strong>Background</strong>:<br />
Human cell lines are convenient model systems in cancer research, for validation of proposed molecular mechanisms as well as to evaluate potential therapeutic approaches, e.g. through high- throughput screening of potential anti-tumour compounds against cancer cell line panels. However, conclusions about biological pathways or pharmacological potential depend on a close molecular relation between the cancer type represented and the cell line model used for analyses.  
The validity of cell line based observations depends therefore A) on the correct identification and lineage assignment, and B) on the conservation of cancer specific molecular alterations during cell line propagation. However, according to ICLAC (International Cell Line Authentication Committee) a large proportion of cell lines have been found to be misidentified.</p>
<p><strong>Methods</strong>:
Our study is based on 3675 genomic copy number variation (CNV) profiling data from arrayMap resource, representing 1539 distinct cell lines as identified in the Cellosaurus database. In addition, manually curated ontology terms based on retrieved metadata are available, as defined by NCI Thesaurus (NCIt) and International Classification of Diseases-Ontology (ICD-O). For germline analysis, we performed genome-wide phasing and imputation to integrate samples from various array platforms. Then, we estimated kinship and built phylogenetic trees based on germline similarity.</p>
<p><strong>Results</strong>:<br />
We confirmed the close relatedness within cell lines as well as between cell lines of similar ethnic origin. For somatic variation, we clustered CNV patterns and evaluated the consistency between cell lines and the primary tutor sample data of the same origin.</p>
<p><strong>Conclusions</strong>: 
We analysed both somatic variation and germline signature within the same cell lines, to evaluate dynamic genome changes related to cell line propagation, as well as to determine instances of mis- assignment based on strong genomic profile divergence. As a research output, we provide the germline lineage tree and somatic CNV clustering for available cancer cell lines as an online resource.</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      September 03, 2019
      
    </span>
</aside>
    <h3 class="title_news">UniFi Seminar - Structural Genome Variants in Cancer</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="seminar-universita-degli-studi-firenze-dipartimento-di-biologia">Seminar Universita degli Studi Firenze - Dipartimento di Biologia<a class="headerlink" href="#seminar-universita-degli-studi-firenze-dipartimento-di-biologia" title="Permanent link">&para;</a></h4>
<h2 id="structural-genome-variants-in-cancer-research-resources-standards">Structural Genome Variants in Cancer: Research, resources standards<a class="headerlink" href="#structural-genome-variants-in-cancer-research-resources-standards" title="Permanent link">&para;</a></h2>
<h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<h5 id="abstract">Abstract<a class="headerlink" href="#abstract" title="Permanent link">&para;</a></h5>
<p>Genomic copy number variations are major contributors to malignant transformation and progression and constitute - at least in their quantitative extension - the largest contributors to genomic mutation landscapes, in the majority of cancer types. Such mutations occur in the vast majority of tumors as somatic genome alterations (sCNV) during clonal development and expansion and are promoted by a variety of mechanisms leading to extended or focal changes in the number of genomic segments. </p>
<p>However, while sCNV based alterations of many canonical cancer related genes has been shown, the impact of the widespread, extensive sCNV patterns is poorly understood, not the least due to the limited amount of accessible whole-genome sCNV datasets and the lack of appropriate data standards and exchange formats.</p>
<p>With the Progenetix and arrayMap repositories our group provides 2 of the largest resources for pre-processed CNV data in cancer, mostly based on the curation and re-analysis of data from public repositories and individual publications. As members of the Global Alliance for Genomics and Health (GA4GH) and participants in the ELIXIR h-CNV community and Beacon project, we utilise our resources to design, implement and test variant annotation and storage formats, as well as federated genome variation discovery tools such as the “Beacon” API, with focus on CNV data.</p>
<p>The presentation will focus on our work for sCNV resources and tools, and also provide some introduction into GA4GH and ELIXIR, with focus of our specific areas of interest. </p>
<ul>
<li><a href="/pdf/2019-09-03___Michael-Baudis__Structural-Genome-Variants-in-Cancer__Seminar-UniFirenze.pdf">[slides]</a></li>
</ul>
<h5 id="links">Links<a class="headerlink" href="#links" title="Permanent link">&para;</a></h5>
<ul>
<li><a href="http://progenetix.org">progenetix.org</a></li>
<li><a href="http://arraymap.org">arraymap.org</a></li>
<li><a href="http://beacon-project.io">ELIXIR Beacon Project</a></li>
<li><a href="http://beacon.progenetix.org">Beacon+</a></li>
<li><a href="http://schemablocks.org">schemablocks</a></li>
</ul>
<p><img alt="" src="/img/2019-09-03-seminar-Firenze-Michael-flyer.png" /></p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      June 17, 2019
      
    </span>
</aside>
    <h3 class="title_news">ELIXIR All Hands - Beacon Introduction</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="elixir-all-hands-lisbon-2019">ELIXIR All Hands - Lisbon 2019<a class="headerlink" href="#elixir-all-hands-lisbon-2019" title="Permanent link">&para;</a></h4>
<h2 id="introduction-to-beacon">Introduction to Beacon<a class="headerlink" href="#introduction-to-beacon" title="Permanent link">&para;</a></h2>
<h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<p>This presentation was the opener for the ELIXIR Beacon session, and introduces to current developments and especially the interactions between GA4GH :: Discovery and ELIXIR Beacon.</p>
<h5 id="links">Links<a class="headerlink" href="#links" title="Permanent link">&para;</a></h5>
<ul>
<li><a href="http://beacon-project.io">ELIXIR Beacon Project</a></li>
<li><a href="http://beacon.progenetix.org">Beacon+</a></li>
<li><a href="http://schemablocks.org">schemablocks</a></li>
<li><a href="http://ga4gh-discovery.github.io">GA4GH :: Discovery</a></li>
</ul>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      June 14, 2019
      
    </span>
</aside>
    <h3 class="title_news">HGVS 2019 - Development of Standards for Genomic Data Exchange</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="human-genome-variation-society-gothenburg-2019">Human Genome Variation Society - Gothenburg 2019<a class="headerlink" href="#human-genome-variation-society-gothenburg-2019" title="Permanent link">&para;</a></h4>
<h2 id="implementation-driven-development-of-standards-for-genomic-data-exchange-from-cancer-genome-data-collections">Implementation Driven Development of Standards for Genomic Data Exchange from Cancer Genome Data Collections<a class="headerlink" href="#implementation-driven-development-of-standards-for-genomic-data-exchange-from-cancer-genome-data-collections" title="Permanent link">&para;</a></h2>
<h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<h5 id="abstract">Abstract<a class="headerlink" href="#abstract" title="Permanent link">&para;</a></h5>
<p>Cancers are genomic diseases, arising from the clonal propagation of somatic mutation events, with a limited contribution from inherited genomic variants. Genomic copy number variations are major contributors to malignant transformation and progression and constitute - at least in their quantitative extension - the largest contributors to genomic mutation landscapes, in the majority of cancer types. However, while sCNV based alterations of many canonical cancer related genes has been shown, the impact of the widespread, extensive sCNV patterns is poorly understood, not the least due to the limited amount of accessible whole-genome sCNV datasets and the lack of appropriate data standards and exchange formats.
With the Progenetix and arrayMap repositories our group provides 2 of the largest resources for pre-processed CNV data in cancer, mostly based on the curation and re-analysis of data from public repositories and individual publications. As members of the Global Alliance for Genomics and Health (GA4GH) and participants in the ELIXIR h-CNV community and Beacon project, we utilise our resources to design, implement and test variant annotation and storage formats, as well as federated genome variation discovery tools such as the “Beacon” API, with focus on CNV data.</p>
<h5 id="links">Links<a class="headerlink" href="#links" title="Permanent link">&para;</a></h5>
<ul>
<li><a href="http://progenetix.org">progenetix.org</a></li>
<li><a href="http://arraymap.org">arraymap.org</a></li>
<li><a href="http://beacon-project.io">ELIXIR Beacon Project</a></li>
<li><a href="http://beacon.progenetix.org">Beacon+</a></li>
<li><a href="http://schemablocks.org">schemablocks</a></li>
</ul>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      January 25, 2019
      
    </span>
</aside>
    <h3 class="title_news">2019 01 25 talk RD basel michael</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="rd-data-intelligence-leaders-forum-basel">R&amp;D Data Intelligence Leaders Forum Basel<a class="headerlink" href="#rd-data-intelligence-leaders-forum-basel" title="Permanent link">&para;</a></h4>
<h2 id="connecting-the-silos-genomic-data-standards-resources-and-the-global-alliance-for-genomics-and-health">Connecting the silos - Genomic Data Standards, Resources and the Global Alliance for Genomics and Health<a class="headerlink" href="#connecting-the-silos-genomic-data-standards-resources-and-the-global-alliance-for-genomics-and-health" title="Permanent link">&para;</a></h2>
<h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<h5 id="abstract">Abstract<a class="headerlink" href="#abstract" title="Permanent link">&para;</a></h5>
<p>This presentation discusses the need for data sharing in genomics, provides information about the Global Alliance for Genomics and Health (GA4GH), and shows some of our group's contributions, especially regarding Beacon development.</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      September 17, 2018
      
    </span>
</aside>
    <h3 class="title_news">2018 09 18 BIO390 Michael Baudis Introduction to Bioinformatics</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="uzh-bio390-introduction-to-bioinformatics">UZH BIO390 "Introduction to Bioinformatics"<a class="headerlink" href="#uzh-bio390-introduction-to-bioinformatics" title="Permanent link">&para;</a></h4>
<h2 id="bioinformatics-introduction">Bioinformatics - Introduction<a class="headerlink" href="#bioinformatics-introduction" title="Permanent link">&para;</a></h2>
<h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<h5 id="abstract">Abstract<a class="headerlink" href="#abstract" title="Permanent link">&para;</a></h5>
<p>First lecture in the UZH BIO390 "Introduction to Bioinformatics" series, introducing concepts and scope of bioinformatics as a field - 2018 version.</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      June 29, 2018
      
    </span>
</aside>
    <h3 class="title_news">Qingyao - Institute Progress Report</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="imls-progress-report">IMLS Progress Report<a class="headerlink" href="#imls-progress-report" title="Permanent link">&para;</a></h4>
<h2 id="towards-understanding-population-effect-on-cancer">Towards understanding population effect on cancer<a class="headerlink" href="#towards-understanding-population-effect-on-cancer" title="Permanent link">&para;</a></h2>
<h3 id="qingyao-huang">Qingyao Huang<a class="headerlink" href="#qingyao-huang" title="Permanent link">&para;</a></h3>

<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      June 06, 2018
      
    </span>
</aside>
    <h3 class="title_news">2018 06 06 talk ELIXIR Beacon2018 michael</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="elixir-all-hands-berlin"><a href="https://www.elixir-europe.org/events/elixir-all-hands-2018">ELIXIR All Hands</a>, Berlin<a class="headerlink" href="#elixir-all-hands-berlin" title="Permanent link">&para;</a></h4>
<h2 id="the-elixir-beacon-in-2018-a-driver-project-of-ga4gh">The ELIXIR Beacon in 2018: A driver project of GA4GH<a class="headerlink" href="#the-elixir-beacon-in-2018-a-driver-project-of-ga4gh" title="Permanent link">&para;</a></h2>
<h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<h5 id="abstract">Abstract<a class="headerlink" href="#abstract" title="Permanent link">&para;</a></h5>
<p>The core mission of the Global Alliance for Genomics and Health is to "...enable genomic data sharing for the benefit of human health". One of the instruments to enact this mission is the selection and support of driver projects, which address particular scientific, technical, regulatory or security related aspects of federated access to human genomes and related metadata.</p>
<p>The "Beacon" project had been initiated at the first GA4GH plenary in 2014, to test the willingness of genome resource providers to allow web-based queries in the most simple format, against aggregated genome variant data. While this particular model had limited practical utility, it established the foundation for automated queries against world-wide data resources, using a unified protocol. A secondary aspect of the Beacon project was the, intentional, challenge of existing notions of data security and privacy protection, including possible re-identification attacks and risk mitigation scenarios.</p>
<p>Now continued through the ELIXIR-Beacon GA4GH driver project, extensions to the Beacon protocol will provide real-world utility for genome resource access under a number of different usage scenarios. With its support for developing the Beacon protocol and distributing it throughout its members, ELIXIR is providing one of the first, widespread implementations of a GA4GH related products. Additionally, the upcoming version of the Beacon protocol is expected to be one of the first official standards which underwent the GA4GH product approval process, facilitating widespread international adoption.</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      June 04, 2018
      
    </span>
</aside>
    <h3 class="title_news">ELIXIR All Hands 2018 - segment_liftover</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="abstract-elixir-all-hands-2018">Abstract: <a href="https://www.elixir-europe.org/events/elixir-all-hands-2018">ELIXIR All Hands 2018</a><a class="headerlink" href="#abstract-elixir-all-hands-2018" title="Permanent link">&para;</a></h4>
<h2 id="segment_liftover-a-python-tool-to-convert-segments-between-genome-assemblies">segment_liftover : a Python tool to convert segments between genome assemblies<a class="headerlink" href="#segment_liftover-a-python-tool-to-convert-segments-between-genome-assemblies" title="Permanent link">&para;</a></h2>
<h3 id="bo-gao-qingyao-huang-and-michael-baudis">Bo Gao, Qingyao Huang and Michael Baudis<a class="headerlink" href="#bo-gao-qingyao-huang-and-michael-baudis" title="Permanent link">&para;</a></h3>
<p>The process of assembling a species’ reference genome may be performed in a number of iterations, with subsequent genome assemblies differing in the coordinates of mapped elements. The conversion of genome coordinates between different assemblies is required for many integrative and comparative studies. While currently a number of bioinformatics tools are available to accomplish this task, most of them are tailored towards the conversion of single genome coordinates. When converting the boundary positions of segments spanning larger genome regions, segments may be mapped into smaller sub-segments if the original segment’s continuity is disrupted in the target assembly. Such a conversion may lead to a relevant degree of data loss in some circumstances such as copy number variation (CNV) analysis, where the quantitative representation of a genomic region takes precedence over base-specific accuracy. segment_liftover aims at continuity-preserving remapping of genome segments between assemblies and provides features such as approximate locus conversion, automated batch processing and comprehensive logging to facilitate processing of datasets containing large numbers of structural genome variation data.</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      June 04, 2018
      
    </span>
</aside>
    <h3 class="title_news">ELIXIR All Hands 2018 - Beacon</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="abstract-elixir-all-hands-2018">Abstract: <a href="https://www.elixir-europe.org/events/elixir-all-hands-2018">ELIXIR All Hands 2018</a><a class="headerlink" href="#abstract-elixir-all-hands-2018" title="Permanent link">&para;</a></h4>
<h2 id="elixir-beacon-a-driver-project-for-the-global-alliance-for-genomics-and-health">ELIXIR Beacon - A Driver Project for the Global Alliance for Genomics and Health<a class="headerlink" href="#elixir-beacon-a-driver-project-for-the-global-alliance-for-genomics-and-health" title="Permanent link">&para;</a></h2>
<h3 id="michael-baudis-for-the-elixir-beacon-project">Michael Baudis for the ELIXIR Beacon Project<a class="headerlink" href="#michael-baudis-for-the-elixir-beacon-project" title="Permanent link">&para;</a></h3>
<p>The core mission of the Global Alliance for Genomics and Health is to "...enable genomic data sharing for the benefit of human health". One of the instruments to enact this mission is the selection and support of driver projects, which address particular scientific, technical, regulatory or security related aspects of federated access to human genomes and related metadata.</p>
<p>The "Beacon" project had been initiated at the first GA4GH plenary in 2014, to test the willingness of genome resource providers to allow web-based queries in the most simple format, against aggregated genome variant data. While this particular model had limited practical utility, it established the foundation for automated queries against world-wide data resources, using a unified protocol. A secondary aspect of the Beacon project was the, intentional, challenge of existing notions of data security and privacy protection, including possible re-identification attacks and risk mitigation scenarios.</p>
<p>Now continued through the ELIXIR-Beacon GA4GH driver project, extensions to the Beacon protocol will provide real-world utility for genome resource access under a number of different usage scenarios. With its support for developing the Beacon protocol and distributing it throughout its members, ELIXIR is providing one of the first, widespread implementations of a GA4GH related products. Additionally, the upcoming version of the Beacon protocol is expected to be one of the first official standards which underwent the GA4GH product approval process, facilitating widespread international adoption.</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      June 04, 2018
      
    </span>
</aside>
    <h3 class="title_news">ELIXIR All Hands 2018 - Biocuration</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="abstract-elixir-all-hands-2018">Abstract: <a href="https://www.elixir-europe.org/events/elixir-all-hands-2018">ELIXIR All Hands 2018</a><a class="headerlink" href="#abstract-elixir-all-hands-2018" title="Permanent link">&para;</a></h4>
<h2 id="biocuration-for-cancer-genome-databases-arraymap-and-progenetix">Biocuration for cancer genome databases: arrayMap and Progenetix<a class="headerlink" href="#biocuration-for-cancer-genome-databases-arraymap-and-progenetix" title="Permanent link">&para;</a></h2>
<h3 id="paula-carrio-cordo-and-michael-baudis">Paula Carrio Cordo and Michael Baudis<a class="headerlink" href="#paula-carrio-cordo-and-michael-baudis" title="Permanent link">&para;</a></h3>
<p>Screening for somatic mutations in cancer has become integral to diagnostic and target evaluation for personalized therapeutic approaches. <a href="http://arraymap.org"><em>arrayMap</em></a> is a curated oncogenomic resource, focusing on copy number aberration (CNA) profiles derived from genomic arrays. The information has been processed from data accessed through NCBI’s Gene Expression Omnibus (GEO), EBI’s ArrayExpress, and, importantly, through targeted mining of publication data. Whereas this database is based on raw probe data sets, the parental project, <a href="http://progenetix.org"><em>Progenetix</em></a>, allows for genome variant analysis from additional sources and serves as metadata reference.</p>
<p><em>arrayMap</em> underwent improvements to facilitate meta-analysis of cancer related genome data and clinical use, such as the adoption of a hierarchical schema representation and expansion of the data. The resulting, comprehensive resource consisting of <em>Progenetix</em> and arrayMap contains information for more than 400 ICD-O entities and 63'000 genomic array profiles. For interoperability of clinical data efforts are done inn data standardization. Existing standards such as ICD-O have proven to give a good description of a cancer entity based on the morphology and topography of the sample, but of interested is to move into ontology codes that provide hierarchical reference terminology such as NCIt neoplasm core.</p>
<p>For the integrative analysis of distributed data resources necessary for precision oncology research, translation between different standards used in local and external resources is needed, using ontology mappings to allow for different vocabularies. Our work contributes to cross-resource mapping of cancer classifications, providing both suitable input for mapping services as well as testing new paradigms for data integration in biomedical settings. However, further curation of vocabularies is needed, for instance using mapping concepts beyond single point-to-point assertions.</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      June 04, 2018
      
    </span>
</aside>
    <h3 class="title_news">ELIXIR All Hands 2018 - arrayMap</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="abstract-elixir-all-hands-2018">Abstract: <a href="https://www.elixir-europe.org/events/elixir-all-hands-2018">ELIXIR All Hands 2018</a><a class="headerlink" href="#abstract-elixir-all-hands-2018" title="Permanent link">&para;</a></h4>
<h2 id="update-on-arraymap-cancer-genome-resource">Update on arrayMap Cancer Genome Resource<a class="headerlink" href="#update-on-arraymap-cancer-genome-resource" title="Permanent link">&para;</a></h2>
<h3 id="qingyao-huang-and-michael-baudis">Qingyao Huang and Michael Baudis<a class="headerlink" href="#qingyao-huang-and-michael-baudis" title="Permanent link">&para;</a></h3>
<p>arrayMap is a cancer-related genome profile database, curated from public data repositories. The resources’s data processing pipelines handle the homogeneous conversion of available raw data from various genomic array platforms (e.g. cCGH, SNP) and represents pre-computed copy number variation (CNV) profiles, allowing the evaluation of target gene involvement as well as the comparative analysis of whole-genome CNV patterns, e.g. for assessing between-sample CNV heterogeneity and associate sets of similar patterns with metadata qualifiers such as diagnostic classifications. In addition to the representation of cancer-related samples, we also plan to generate copy number profiles from non-cancer samples, in an effort to represent population-related CNV patterns.</p>
<p>Last year, we updated the arrayMap database with new samples from NCBI’s Gene Expression Omnibus (GEO: 24k), EBI’s ArrayExpress (15k) and the Cancer Genome Atlas (TCGA: 22k). The genome version of all genomics data has been lifted to the latest GRCh38 with our batch-processing segment_liftover tool. CNVs are assessed, evaluated and visualized for all these cancer-related genomics single profiles as well as with summary histograms on selected sample groups (e.g. cancer entities, experimental series). </p>
<p>In the newest edition of the database, we also introduced allele-specific representation of SNP-array based data, allowing for better evaluation of segmental copy number status, the detection of coy number neutral LOH regions, as well as providing support for ploidy estimation. Furthermore, we performed a pilot study aimed at data-driven assessment of the population background for these samples taking 1000 Genomes project as reference.</p>
<p>The updated data repository is accessible through the arraymap.org website as well as through the <a href="http://beacon.progenetix.org/">beacon+ API</a>. Software tools are available as Docker image "baudisgroup/tum2pop" on <a href="https://hub.docker.com/">Docker Hub</a> and provided through the <a href="http://github.com/baudisgroup/">"baudisgroup" organisation on Github</a>. </p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      June 04, 2018
      
    </span>
</aside>
    <h3 class="title_news">ELIXIR All Hands 2018 - Cell Line profiling</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="abstract-elixir-all-hands-2018">Abstract: <a href="https://www.elixir-europe.org/events/elixir-all-hands-2018">ELIXIR All Hands 2018</a><a class="headerlink" href="#abstract-elixir-all-hands-2018" title="Permanent link">&para;</a></h4>
<h2 id="genomic-profiles-of-cancer-cell-lines-a-systematic-review">Genomic profiles of cancer cell lines - A systematic review<a class="headerlink" href="#genomic-profiles-of-cancer-cell-lines-a-systematic-review" title="Permanent link">&para;</a></h2>
<h3 id="rahel-paloots-qingyao-huang-paula-carrio-cordo-and-michael-baudis">Rahel Paloots, Qingyao Huang, Paula Carrio Cordo and Michael Baudis<a class="headerlink" href="#rahel-paloots-qingyao-huang-paula-carrio-cordo-and-michael-baudis" title="Permanent link">&para;</a></h3>
<p>Human cell lines are convenient model systems in cancer research, for validation of proposed molecular mechanisms as well as to evaluate potential therapeutic approaches, e.g. through high-throughput screening of potential antitumor compounds against cancer cell line panels. Thousands of established cell lines are available from commercial or academic providers, covering a wide selection of cancer types. However, conclusions about biological pathways or pharmacological potential depend on a close molecular relation between the cancer type represented and the cell line model used for analyses.</p>
<p>The validity of cell line based observations depends therefore A) on the correct identification and lineage assignment, and B) on the conservation of cancer specific molecular alterations during cell line propagation. However, according to ICLAC (International Cell Line Authentication Committee), in a text-based survey of ~32,000 publications for 488 cell lines, a large proportion of cell lines was found to be misidentified.</p>
<p>In the current study, we perform a data-driven assessment of heterogeneity and similarities in-between cell lines, as well as a comparison of cell lines with primary tumor samples of same tumor type. Our study is based on the curation of genomic copy number profiling and metadata data from 3675 genome profiling experiments from our <a href="http://arraymap.org">arrayMap resource</a>, representing 1539 distinct cell lines as identified in the Cellosaurus database.</p>
<p>In our analysis, we compare the genomic heterogeneity of "idem" cell lines, to establish measures for dynamic genome changes related to cell line propagation, as well as to determine instances of mis-assignment based on strong genomic profile divergence. Copy number profiles of cell lines will be compared to such from matched primary tumors, to establish a measure for genomic similarity in these model systems compared to the target tissues. For the evaluation of our findings, corresponding publications as well as related resource entries (GEO, Cellosaurus) are reviewed for possible reasons for misidentification or technical peculiarities.</p>
<p>Finally, we will provide reference copy number variation profiles for cancer cell lines as a resource linked to the Cellosaurus cell line database, to reduce the amount of erroneous cell line use in future research studies, and to provide additional information for the interpretation of data derived from these model systems.</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      February 15, 2018
      
    </span>
</aside>
    <h3 class="title_news">2018 02 15 talk DPPH michael</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="dpph-data-protection-in-personalized-health-2018-lausanne"><a href="https://dpph18.epfl.ch">DPPH - Data Protection in Personalized Health 2018</a>, Lausanne<a class="headerlink" href="#dpph-data-protection-in-personalized-health-2018-lausanne" title="Permanent link">&para;</a></h4>
<h2 id="genome-beacons-for-data-discovery-technical-advances-in-a-challenging-environment">Genome Beacons for Data Discovery - Technical advances in a challenging environment<a class="headerlink" href="#genome-beacons-for-data-discovery-technical-advances-in-a-challenging-environment" title="Permanent link">&para;</a></h2>
<h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<h5 id="abstract">Abstract<a class="headerlink" href="#abstract" title="Permanent link">&para;</a></h5>
<p>The core mission of the Global Alliance for Genomics and Health is to "...enable genomic data sharing for the benefit of human health". One of the instruments to enact this mission is the selection and support of driver projects, which address particular scientific, technical, regulatory or security related aspects of federated access to human genomes and related metadata.
The "Beacon" project had been initiated at the first GA4GH plenary in 2014, to test the willingness of genome resource providers to allow web-based queries in the most simple format, against aggregated genome variant data. While this particular model had limited practical utility, it established the foundation for automated queries against world-wide data resources, using a unified protocol. A secondary aspect of the Beacon project was the, intentional, challenge of existing notions of data security and privacy protection, including possible re-identification attacks and risk mitigation scenarios.
Now continued through the ELIXIR-Beacon GA4GH driver project, extensions to the Beacon protocol will provide real-world utility for genome resource access under a number of different usage scenarios, some of which will be addressed in this presentation.</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      December 07, 2017
      
    </span>
</aside>
    <h3 class="title_news">2017 12 07 talk timisoara michael</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="special-seminar-at-university-of-timisoara-dermatology">Special seminar at University of Timisoara Dermatology<a class="headerlink" href="#special-seminar-at-university-of-timisoara-dermatology" title="Permanent link">&para;</a></h4>
<h2 id="cancer-genome-data-analysis-in-personalised-medicine">Cancer Genome (Data) Analysis in Personalised Medicine<a class="headerlink" href="#cancer-genome-data-analysis-in-personalised-medicine" title="Permanent link">&para;</a></h2>
<h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<h5 id="abstract">Abstract<a class="headerlink" href="#abstract" title="Permanent link">&para;</a></h5>
<p>General presentation about challenges and possibilities of cancer genome screening for medical applications.</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      December 01, 2017
      
    </span>
</aside>
    <h3 class="title_news">2017 12 01 talk lszgs michael</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="uzh-cancer-biology-course-research-with-clinical-samples">UZH Cancer Biology course "Research with clinical samples<a class="headerlink" href="#uzh-cancer-biology-course-research-with-clinical-samples" title="Permanent link">&para;</a></h4>
<h2 id="personalised-medicine-genome-variation-data-formats-resources-sharing-privacy">Personalised Medicine - Genome Variation / Data Formats / Resources / Sharing / Privacy<a class="headerlink" href="#personalised-medicine-genome-variation-data-formats-resources-sharing-privacy" title="Permanent link">&para;</a></h2>
<h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<h5 id="abstract">Abstract<a class="headerlink" href="#abstract" title="Permanent link">&para;</a></h5>
<p>Personalised Medicine - Genome Variation / Data Formats / Resources / Sharing / Privacy, presented for PhD program in Cancer Biology</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      October 15, 2017
      
    </span>
</aside>
    <h3 class="title_news">2017 10 15 talk ga4gh michael</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="ga4gh-2017-plenary-workstream-day">GA4GH 2017 Plenary / Workstream Day<a class="headerlink" href="#ga4gh-2017-plenary-workstream-day" title="Permanent link">&para;</a></h4>
<h2 id="developing-beacons-for-data-discovery">Developing Beacons for Data Discovery<a class="headerlink" href="#developing-beacons-for-data-discovery" title="Permanent link">&para;</a></h2>
<h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      September 19, 2017
      
    </span>
</aside>
    <h3 class="title_news">2017 09 19 talk michael</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="uzh-bio390-introduction-to-bioinformatics">UZH BIO390 "Introduction to Bioinformatics"<a class="headerlink" href="#uzh-bio390-introduction-to-bioinformatics" title="Permanent link">&para;</a></h4>
<h2 id="bioinformatics-introduction">Bioinformatics - Introduction<a class="headerlink" href="#bioinformatics-introduction" title="Permanent link">&para;</a></h2>
<h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<h5 id="abstract">Abstract<a class="headerlink" href="#abstract" title="Permanent link">&para;</a></h5>
<p>First lecture in the UZH BIO390 "Introduction to Bioinformatics" series, introducing concepts and scope of bioinformatics as a field.</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      September 13, 2017
      
    </span>
</aside>
    <h3 class="title_news">2017 09 13 talk bc2 michael</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="bc2-2017-basel-computational-biology-conference">[BC]2 2017 - <a href="https://www.bc2.ch/2017/">Basel Computational Biology Conference</a><a class="headerlink" href="#bc2-2017-basel-computational-biology-conference" title="Permanent link">&para;</a></h4>
<h2 id="advancing-the-global-alliance-for-genomics-and-health-data-schemas-through-data-driven-implementations">Advancing the Global Alliance for Genomics and Health data schemas through data-driven implementations<a class="headerlink" href="#advancing-the-global-alliance-for-genomics-and-health-data-schemas-through-data-driven-implementations" title="Permanent link">&para;</a></h2>
<h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<h5 id="abstract">Abstract<a class="headerlink" href="#abstract" title="Permanent link">&para;</a></h5>
<p>The core mission of the Global Alliance for Genomics and Health (GA4GH) is to advance the utility of biomedical data through facilitating of federated analyses, across geographic, regulatory and technical boundaries. A central component is the development of standardised data schemas and application programming interfaces (APIs), for molecular screening data as well as biological and technical metadata.</p>
<p>In addition to direct schema development, demonstration projects explore strategies and limitations of federated data access and attempt to engage active participants from different areas of biomedical research and clinical practice as well as experts from areas such as computer and network security, law and ethics. One of the main demonstrators has been the “Beacon” project, which through a lightweight “beacon-network” registry and API allows the querying for specific genome sequence variants at resources from all over the world.</p>
<p>Early on, the main focus of GA4GH was on demonstrating functionality and to engage participants through through low-complexity scenarios. However, increased utility of GA4GH schemas requires representation of common biological, health specific and technical metadata.</p>
<p>The Metadata Task Team (MTT) of the GA4GH Data Working Group (DWG) has been working on defining the part of the schema representation which not primarily addresses primary or derived “omics” (e.g. sequences, variants) data. Core principles of schema development have been the restriction of schema complexity, the use of standardised data formats (e.g. ISO time), and the prominent use of ontologies as externally maintained value descriptors.</p>
<p>An important benchmark for a given data schema is the successful projection of reference datasets. However, so far the mapping of rich metadata attributes and structures into the GA4GH schema has not been explored for large, real world datasets or reference repositories.</p>
<p>As part of an ELIXIR “human data implementation study”, we have started to implement datasets as schema-compatible resources, and to explore linking these to external reference services for biomedical ontologies. At the core of this project, we are using the arraymap.org reference database for molecular genomic profiling data in cancer, which contains data from oncogenomic copy number analyses and curated metadata. The study has been directed at testing the GA4GH object model, with a focus on metadata, and at using this experience to guide schema development efforts of the GA4GH MTT. </p>
<p>Additionally, as part of a parallel ELIXIR study, our GA4GH conform representation arrayMap data tests forward looking developments of genomic "Beacons", with focus on structural genome variants, the inclusion of metadata parameters for data queries, and the development of query interfaces and informative Beacon response formats.</p>
<p>At the present time, our test implementation represents &gt;55’000 genomic call sets and allows the direct use of variant and metadata queries against a set of database collections implemented in MongoDB. A first API implementation provides responses to Beacon-style queries, e.g. the frequency of a variant for a tumor type specified through an ontology term.</p>
<p>Ongoing efforts aim to promote rapid evolution of the GA4GH metadata schema for a variety of data types and proposed use cases, such as geolocation data, metagenomic applications and temporal relations. On the implementation side, next steps include the integration of ontology services for data query and ingestion scenarios, as well as the extension of the demonstrator API for rich metadata query scenarios. An important part of current and future developments will be the use of additional data resources for demonstration projects.</p>
<p>Example datasets as well as tools and documentation are made available through the <a href="http://github.com/progenetix/">“arraymap2ga4gh” project on Github</a>.</p>
<h5 id="highlights">Highlights<a class="headerlink" href="#highlights" title="Permanent link">&para;</a></h5>
<p>This presentation will be aimed at reporting the current status of the GA4GH schema development with focus on the lessons learned from the data implementation studies, as well as the challenges from design and implementation of data concepts centred around modern object based biodata data standards with heavy dependency on ontologies. We hope to excite the audience about the ongoing developments at GA4GH, with emphasis on increased engagement in data sharing activities.</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      August 29, 2017
      
    </span>
</aside>
    <h3 class="title_news">2017 03 28 talk biocuration stanford michael</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="seminar-at-sdu-odense">Seminar at <a href="http://www.sdu.dk/en/">SDU Odense</a><a class="headerlink" href="#seminar-at-sdu-odense" title="Permanent link">&para;</a></h4>
<h2 id="somatic-cnv-databases-as-resources-for-cancer-research-ga4gh-schema-development">Somatic CNV databases as resources for cancer research &amp;&amp; GA4GH schema development<a class="headerlink" href="#somatic-cnv-databases-as-resources-for-cancer-research-ga4gh-schema-development" title="Permanent link">&para;</a></h2>
<h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      August 29, 2017
      
    </span>
</aside>
    <h3 class="title_news">2017 08 29 talk odense michael</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="seminar-at-sdu-odense">Seminar at <a href="http://www.sdu.dk/en/">SDU Odense</a><a class="headerlink" href="#seminar-at-sdu-odense" title="Permanent link">&para;</a></h4>
<h2 id="somatic-cnv-databases-as-resources-for-cancer-research-ga4gh-schema-development">Somatic CNV databases as resources for cancer research &amp;&amp; GA4GH schema development<a class="headerlink" href="#somatic-cnv-databases-as-resources-for-cancer-research-ga4gh-schema-development" title="Permanent link">&para;</a></h2>
<h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      March 22, 2017
      
    </span>
</aside>
    <h3 class="title_news">2017 03 22 talk elixir2017 michael</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="elixir-all-hands-rome">ELIXIR All Hands, Rome<a class="headerlink" href="#elixir-all-hands-rome" title="Permanent link">&para;</a></h4>
<h2 id="ga4gh-metadata-beyond">GA4GH Metadata &amp; Beyond<a class="headerlink" href="#ga4gh-metadata-beyond" title="Permanent link">&para;</a></h2>
<h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      February 21, 2017
      
    </span>
</aside>
    <h3 class="title_news">Personalized medicine in cancer</h3>

<h5 class="title_news">Seminar - Stanford Center for Genomics & Personalized Medicine</h5>

<!-- Use a hidden p tag to provide a preview -->
<h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<h5 id="abstract">Abstract<a class="headerlink" href="#abstract" title="Permanent link">&para;</a></h5>
<p>Some reasoning for the need of genome profiling data in cancer and reference samples, and our own work on structural aberration data in cancer as well as contributions to the projects and standard definitions of the Global Alliance for Genomics and Health</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      February 01, 2017
      
    </span>
</aside>
    <h3 class="title_news">2017 02 01 poster bo</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="brupbacher-meeting"><a href="http://www.brupbacher-foundation.org">Brupbacher Meeting</a><a class="headerlink" href="#brupbacher-meeting" title="Permanent link">&para;</a></h4>
<h2 id="implementing-data-models-for-the-global-alliance-for-genomics-and-health">Implementing Data Models for the Global Alliance for Genomics and Health<a class="headerlink" href="#implementing-data-models-for-the-global-alliance-for-genomics-and-health" title="Permanent link">&para;</a></h2>
<h3 id="bo-gao">Bo Gao<a class="headerlink" href="#bo-gao" title="Permanent link">&para;</a></h3>
<h5 id="abstract">Abstract<a class="headerlink" href="#abstract" title="Permanent link">&para;</a></h5>
<p>Poster about our implementation studies for testing and developing data schemas and resources for the <a href="http://ga4gh.org">Global Alliance for Genomics and Health</a>, based on data from our <a href="http://arraymap.org">arrayMap</a> resource</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      January 26, 2017
      
    </span>
</aside>
    <h3 class="title_news">2017 01 26 talk RD basel michael</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="rd-data-intelligence-leaders-forum-basel">R&amp;D Data Intelligence Leaders Forum Basel<a class="headerlink" href="#rd-data-intelligence-leaders-forum-basel" title="Permanent link">&para;</a></h4>
<h2 id="genome-data-access-application-to-personalized-health-and-cancer-research">Genome data access - Application to personalized health and cancer research<a class="headerlink" href="#genome-data-access-application-to-personalized-health-and-cancer-research" title="Permanent link">&para;</a></h2>
<h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<h5 id="abstract">Abstract<a class="headerlink" href="#abstract" title="Permanent link">&para;</a></h5>
<ul>
<li>Discussing genome data access, with focus ion cancer, GA4GH and own contributions</li>
</ul>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      December 19, 2016
      
    </span>
</aside>
    <h3 class="title_news">2016 12 19 talk antrittsvorlesung michael</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="antrittsvorlesung-universitat-zurich-inaugural-lecture-at-university-of-zurich">Antrittsvorlesung Universität Zürich - Inaugural Lecture at University of Zurich<a class="headerlink" href="#antrittsvorlesung-universitat-zurich-inaugural-lecture-at-university-of-zurich" title="Permanent link">&para;</a></h4>
<h2 id="cancer-genome-data-analysis">Cancer Genome (Data) Analysis<a class="headerlink" href="#cancer-genome-data-analysis" title="Permanent link">&para;</a></h2>
<h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<h5 id="abstract">Abstract<a class="headerlink" href="#abstract" title="Permanent link">&para;</a></h5>
<ul>
<li>General lecture introducing the "Why"s of cancer genome analysis and some specific problems, as well as some of our own contributions</li>
</ul>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      December 02, 2016
      
    </span>
</aside>
    <h3 class="title_news">2016 12 02 talk lszgs michael</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="lszgs-cancer-biology-course-2016">LSZGS Cancer Biology course 2016<a class="headerlink" href="#lszgs-cancer-biology-course-2016" title="Permanent link">&para;</a></h4>
<h2 id="personalized-medicine-in-cancer">Personalized medicine in cancer<a class="headerlink" href="#personalized-medicine-in-cancer" title="Permanent link">&para;</a></h2>
<h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<h5 id="abstract">Abstract<a class="headerlink" href="#abstract" title="Permanent link">&para;</a></h5>
<p>Lecture as part of the <a href="http://www.cnz.uzh.ch/en/phdprogram.html">LSZGS Cancer Biology PhD program</a>, touching some aspects of cancer genome data analysis (why, amount and cost of genome sequencing, genome variation, privacy &amp; data sharing ...</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      April 14, 2016
      
    </span>
</aside>
    <h3 class="title_news">2016 04 14 talk biocuration2016 michael</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="biocuration-2016-geneva">Biocuration 2016, Geneva<a class="headerlink" href="#biocuration-2016-geneva" title="Permanent link">&para;</a></h4>
<h2 id="harvesting-cancer-genome-data">Harvesting Cancer Genome Data<a class="headerlink" href="#harvesting-cancer-genome-data" title="Permanent link">&para;</a></h2>
<h3 id="michael-baudis">Michael Baudis<a class="headerlink" href="#michael-baudis" title="Permanent link">&para;</a></h3>
<h5 id="abstract">Abstract<a class="headerlink" href="#abstract" title="Permanent link">&para;</a></h5>
<p>While the analysis of cancer genomes using high-throughput technologies has generated tens of thousands of oncogenomic profiles, meta-analyses of datasets is greatly inhibited through limited data access, technical fragmentation and a multitude of raw data and annotation formats.</p>
<p>For the arrayMap cancer genome resource, our group collects, re-processes and annotates cancer and associated reference data from genomic array experiments, retrieved from public repositories (e.g. NCBI GEO, EBI ArrayExpress) as well as from publication supplements and through direct requests to the primary producers. So far, our resources provide pre-formatted data for more than 50'000 cancer genome profiling experiments, derived from 340 array platforms and representing more than 700 original publications.</p>
<p>Here, I will present some aspects of the shifting landscape of cancer genome data production and publication, an overview about the data accessible through our resources, and an outlook into developments especially with regard of work performed in the context of the Global Alliance for Genomics and Heath.</p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0zm-3.5 4.5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25H3.25zM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25V9.5z"/></svg>
      </span>
      March 17, 2016
      
    </span>
</aside>
    <h3 class="title_news">2016 03 17 poster saumya</h3>

<!-- Use a hidden p tag to provide a preview -->
<h4 id="vib-conference-applied-bioinformatics-in-life-sciences">VIB Conference "Applied Bioinformatics in Life Sciences"<a class="headerlink" href="#vib-conference-applied-bioinformatics-in-life-sciences" title="Permanent link">&para;</a></h4>
<h2 id="gene-based-analysis-of-focal-copy-number-aberration-patterns-in-cancer-genomes">Gene-based analysis of focal copy number aberration patterns in cancer genomes<a class="headerlink" href="#gene-based-analysis-of-focal-copy-number-aberration-patterns-in-cancer-genomes" title="Permanent link">&para;</a></h2>
<h3 id="saumya-gupta">Saumya Gupta<a class="headerlink" href="#saumya-gupta" title="Permanent link">&para;</a></h3>
<h5 id="abstract">Abstract<a class="headerlink" href="#abstract" title="Permanent link">&para;</a></h5>
<p>Poster about the project to evaluate focal copy number variants in cancer, based on data from <a href="http://arraymap.org">arraymap.org</a></p>
<a href="./">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill-rule="evenodd" d="M13.22 19.03a.75.75 0 0 0 1.06 0l6.25-6.25a.75.75 0 0 0 0-1.06l-6.25-6.25a.75.75 0 1 0-1.06 1.06l4.97 4.97H3.75a.75.75 0 0 0 0 1.5h14.44l-4.97 4.97a.75.75 0 0 0 0 1.06z"/></svg>
  </span>
  Continue reading
</a>
  

            </article>
          </div>
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      &copy; Copyright 2007 - 2022 Michael Baudis
    </div>
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
        <div class="md-social">
  
    
    
      
      
    
    <a href="https://github.com/baudisgroup" target="_blank" rel="noopener" title="github.com" class="md-social__link">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    <script id="__config" type="application/json">{"base": "..", "features": ["content.tabs.link", "search.highlight", "search.share", "navigation.instant", "navigation.tracking", "navigation.sections", "toc.integrate"], "translations": {"clipboard.copy": "Copy to clipboard", "clipboard.copied": "Copied to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.placeholder": "Type to start searching", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.term.missing": "Missing", "select.version.title": "Select version"}, "search": "../assets/javascripts/workers/search.22074ed6.min.js"}</script>
    
    
      <script src="../assets/javascripts/bundle.960e086b.min.js"></script>
      
    
  </body>
</html>