
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
        <meta name="description" content="Computational Oncogenomics Group @ UZH & SIB">
      
      
        <meta name="author" content="Michael Baudis">
      
      
      
      
      
        
      
      
      <link rel="icon" href="../img/favicon.ico">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-9.7.3">
    
    
      
        <title>Baudisgroup Publications - baudisgroup @ UZH & SIB</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.484c7ddc.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.ab4e12ef.min.css">
      
      
  
  
    
      
      
    
  
  
  <style>:root{--md-admonition-icon--example:url('data:image/svg+xml;charset=utf-8,%3Csvg%20xmlns%3D%22http%3A//www.w3.org/2000/svg%22%20viewBox%3D%220%200%2024%2024%22%3E%3Cpath%20d%3D%22M3.604%203.089A.75.75%200%200%201%204%203.75V8.5h.75a.75.75%200%200%201%200%201.5h-3a.75.75%200%201%201%200-1.5h.75V5.151l-.334.223a.75.75%200%200%201-.832-1.248l1.5-1a.75.75%200%200%201%20.77-.037M8.75%205.5a.75.75%200%200%200%200%201.5h11.5a.75.75%200%200%200%200-1.5zm0%206a.75.75%200%200%200%200%201.5h11.5a.75.75%200%200%200%200-1.5zm0%206a.75.75%200%200%200%200%201.5h11.5a.75.75%200%200%200%200-1.5zM5.5%2015.75c0-.704-.271-1.286-.72-1.686a2.3%202.3%200%200%200-1.53-.564c-.535%200-1.094.178-1.53.565-.449.399-.72.982-.72%201.685a.75.75%200%200%200%201.5%200c0-.296.104-.464.217-.564A.8.8%200%200%201%203.25%2015c.215%200%20.406.072.533.185.113.101.217.268.217.565%200%20.332-.069.48-.21.657-.092.113-.216.24-.403.419l-.147.14c-.152.144-.33.313-.52.504l-1.5%201.5a.75.75%200%200%200-.22.53v.25c0%20.414.336.75.75.75H5A.75.75%200%200%200%205%2019H3.31l.47-.47c.176-.176.333-.324.48-.465l.165-.156a6%206%200%200%200%20.536-.566c.358-.447.539-.925.539-1.593%22/%3E%3C/svg%3E');}</style>



    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../css/theme_overrides.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="white">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="baudisgroup @ UZH &amp; SIB" class="md-header__button md-logo" aria-label="baudisgroup @ UZH & SIB" data-md-component="logo">
      
  <img src="../img/site_logo.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            baudisgroup @ UZH & SIB
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Baudisgroup Publications
            
          </span>
        </div>
      </div>
    </div>
    
      
    
    
    
    
      
      
        <label class="md-header__button md-icon" for="__search">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        </label>
        <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
          <a href="javascript:void(0)" class="md-search__icon md-icon" title="Share" aria-label="Share" data-clipboard data-clipboard-text="" data-md-component="search-share" tabindex="-1">
            
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M18 16.08c-.76 0-1.44.3-1.96.77L8.91 12.7c.05-.23.09-.46.09-.7s-.04-.47-.09-.7l7.05-4.11c.54.5 1.25.81 2.04.81a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3c0 .24.04.47.09.7L8.04 9.81C7.5 9.31 6.79 9 6 9a3 3 0 0 0-3 3 3 3 0 0 0 3 3c.79 0 1.5-.31 2.04-.81l7.12 4.15c-.05.21-.08.43-.08.66 0 1.61 1.31 2.91 2.92 2.91s2.92-1.3 2.92-2.91A2.92 2.92 0 0 0 18 16.08"/></svg>
          </a>
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" tabindex="0" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
      
    
    
      <div class="md-header__source">
        <a href="https://github.com/baudisgroup/baudisgroup.github.io" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><!--! Font Awesome Free 7.1.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M202.1 328.7c0 20.9-10.9 55.1-36.7 55.1s-36.7-34.2-36.7-55.1 10.9-55.1 36.7-55.1 36.7 34.2 36.7 55.1M496 278.2c0 31.9-3.2 65.7-17.5 95-37.9 76.6-142.1 74.8-216.7 74.8-75.8 0-186.2 2.7-225.6-74.8-14.6-29-20.2-63.1-20.2-95 0-41.9 13.9-81.5 41.5-113.6-5.2-15.8-7.7-32.4-7.7-48.8 0-21.5 4.9-32.3 14.6-51.8 45.3 0 74.3 9 108.8 36 29-6.9 58.8-10 88.7-10 27 0 54.2 2.9 80.4 9.2 34-26.7 63-35.2 107.8-35.2 9.8 19.5 14.6 30.3 14.6 51.8 0 16.4-2.6 32.7-7.7 48.2 27.5 32.4 39 72.3 39 114.2m-64.3 50.5c0-43.9-26.7-82.6-73.5-82.6-18.9 0-37 3.4-56 6-14.9 2.3-29.8 3.2-45.1 3.2-15.2 0-30.1-.9-45.1-3.2-18.7-2.6-37-6-56-6-46.8 0-73.5 38.7-73.5 82.6 0 87.8 80.4 101.3 150.4 101.3h48.2c70.3 0 150.6-13.4 150.6-101.3m-82.6-55.1c-25.8 0-36.7 34.2-36.7 55.1s10.9 55.1 36.7 55.1 36.7-34.2 36.7-55.1-10.9-55.1-36.7-55.1"/></svg>
  </div>
  <div class="md-source__repository">
    baudisgroup.github.io
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    



  

<nav class="md-nav md-nav--primary md-nav--integrated" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="baudisgroup @ UZH &amp; SIB" class="md-nav__button md-logo" aria-label="baudisgroup @ UZH & SIB" data-md-component="logo">
      
  <img src="../img/site_logo.png" alt="logo">

    </a>
    baudisgroup @ UZH & SIB
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/baudisgroup/baudisgroup.github.io" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><!--! Font Awesome Free 7.1.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M202.1 328.7c0 20.9-10.9 55.1-36.7 55.1s-36.7-34.2-36.7-55.1 10.9-55.1 36.7-55.1 36.7 34.2 36.7 55.1M496 278.2c0 31.9-3.2 65.7-17.5 95-37.9 76.6-142.1 74.8-216.7 74.8-75.8 0-186.2 2.7-225.6-74.8-14.6-29-20.2-63.1-20.2-95 0-41.9 13.9-81.5 41.5-113.6-5.2-15.8-7.7-32.4-7.7-48.8 0-21.5 4.9-32.3 14.6-51.8 45.3 0 74.3 9 108.8 36 29-6.9 58.8-10 88.7-10 27 0 54.2 2.9 80.4 9.2 34-26.7 63-35.2 107.8-35.2 9.8 19.5 14.6 30.3 14.6 51.8 0 16.4-2.6 32.7-7.7 48.2 27.5 32.4 39 72.3 39 114.2m-64.3 50.5c0-43.9-26.7-82.6-73.5-82.6-18.9 0-37 3.4-56 6-14.9 2.3-29.8 3.2-45.1 3.2-15.2 0-30.1-.9-45.1-3.2-18.7-2.6-37-6-56-6-46.8 0-73.5 38.7-73.5 82.6 0 87.8 80.4 101.3 150.4 101.3h48.2c70.3 0 150.6-13.4 150.6-101.3m-82.6-55.1c-25.8 0-36.7 34.2-36.7 55.1s10.9 55.1 36.7 55.1 36.7-34.2 36.7-55.1-10.9-55.1-36.7-55.1"/></svg>
  </div>
  <div class="md-source__repository">
    baudisgroup.github.io
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Baudisgroup Home
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../address" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Address
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../group" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Group Members
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="." class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Publications
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../presentations" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Presentations
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../recipes" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Recipes
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../projects-and-positions/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Projects and Open Positions
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
      
        
        
      
    
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_8" >
        
          
          <label class="md-nav__link" for="__nav_8" id="__nav_8_label" tabindex="">
            
  
  
  <span class="md-ellipsis">
    
  
    Teaching and Seminars
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_8_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_8">
            <span class="md-nav__icon md-icon"></span>
            
  
    Teaching and Seminars
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="https://compbiozurich.org/teaching/UZH-BIO390/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    BIO390 Lectures &#8599;
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="https://compbiozurich.org/teaching/UZH-BIO392/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    BIO392 Block Course &#8599;
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="https://compbiozurich.org/news/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    ZH Bioinfo Seminars &#8599;
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="http://progenetix.org" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Progenetix &#8599;
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="http://cancercelllines.org" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Cancercelllines &#8599;
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="https://compbiozurich.org" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    CompbioZurich &#8599;
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="https://schemablocks.org" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    SchemaBlocks {S}[B] &#8599;
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="https://beacon-project.io" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Beacon Project &#8599;
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="https://www.mls.uzh.ch/en/research/baudis.html" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Michael Baudis @ UZH &#8599;
  

    
  </span>
  
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
          
          
            <div class="md-content" data-md-component="content">
              
              <article class="md-content__inner md-typeset">
                

  <h1 id="baudisgroup-publications">Baudisgroup Publications<a class="headerlink" href="#baudisgroup-publications" title="Permanent link">&para;</a></h1>
<p>A list of publication can also be retrieved through <a href="https://europepmc.org/search?query=0000-0002-9903-4248">EuropePMC</a>.</p>
 
  
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
        
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      January 15, 2026
      
    </span>
</aside>

<h3 class="title_news">CNAttention: an attention-based deep multiple-instanc method for uncovering copy number aberratio signatures across cancers</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="ziying-yang-and-michael-baudis">Ziying Yang and Michael Baudis<a class="headerlink" href="#ziying-yang-and-michael-baudis" title="Permanent link">&para;</a></h4>
<h5 id="briefings-in-bioinformatics-2026-271-bbaf696-2026-01-15">Briefings in Bioinformatics, 2026, 27(1), bbaf696, 2026-01-15<a class="headerlink" href="#briefings-in-bioinformatics-2026-271-bbaf696-2026-01-15" title="Permanent link">&para;</a></h5>
<ul>
<li>doi: <a href="https://doi.org/10.1093/bib/bbaf696">10.1093/bib/bbaf696</a></li>
</ul>
<p><strong>Abstract</strong>: Somatic copy number aberrations (CNAs) represent a distinct class of genomic mutations associated with oncogenetic effects. Over
the past three decades, significant volumes of CNA data have been generated through molecular-cytogenetic and genome sequencing-
based techniques. These data have been pivotal in identifying cancer-related genes and advancing research on the relationship between
CNAs and histopathologically defined cancer types. However, comprehensive studies of CNA landscapes and disease parameters are
challenging due to the vast diagnostic and genomic heterogeneity encountered in ”pan-cancer” approaches. In this study, we introduce
CNAttention, an attention-based deep multiple instance learning method designed to comprehensively analyze CNAs across different
cancers and uncover specific CNA patterns within integrated gene-level CNA profiles of 30 cancer types. CNAttention effectively learns
CNA features unique to each cancer type and generates CNA signatures for 30 cancer types using attention mechanisms, highlighting
the distinctiveness of their CNA landscapes. CNAttention demonstrates high accuracy and exhibits stable performance even with
the incorporation of external datasets or parameter adjustments, underscoring its effectiveness in tumor identification. Expanding
these signatures to cancer classification trees reveals common patterns not only among physiologically related cancer types but
also among clinico-pathologically distant types, such as different cancers originating from neural crest derived cells. Additionally,
detected signatures also uncover genomic heterogeneity in individual cancer types, for instance in brain lower grade glioma. Additional
experiments with classification models underscore the efficacy of these signatures in representing various cancer types and their
potential utility in clinical diagnosis.</p>
<a href="2026-01-15-CNAttention__Briefings-in-Bioinformatics/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      September 26, 2025
      
    </span>
</aside>

<h3 class="title_news">CNAdjust: Enhancing CNA Calling Accuracy through Systematic Baseline Adjustment</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="hangjia-zhao-and-michael-baudis">Hangjia Zhao and Michael Baudis<a class="headerlink" href="#hangjia-zhao-and-michael-baudis" title="Permanent link">&para;</a></h4>
<h5 id="frontiers-in-genetics-2025-sep-26">Frontiers in Genetics. 2025 Sep 26.<a class="headerlink" href="#frontiers-in-genetics-2025-sep-26" title="Permanent link">&para;</a></h5>
<ul>
<li>doi: <a href="https://doi.org/10.3389/fgene.2025.1674138">10.3389/fgene.2025.1674138</a></li>
</ul>
<p><strong>Abstract</strong>: Accurate determination of the genomic copy number baseline is crucial for identifying copy number alterations (CNAs) in cancer, yet it remains a significant challenge in tumors with complex karyotypes. To address this, we present CNAdjust, an integrated method to systematically detect and correct baseline inaccuracies in CNA data. CNAdjust employs a Bayesian framework that integrates cohort-specific CNA frequency priors with a data-driven plausibility score, ensuring that adjusted calls align with both biological cohort patterns and study-specific data. 
<a href="2025-09-10-CNAdjust/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      July 16, 2025
      
    </span>
</aside>

<h3 class="title_news">pgxRpi: an R/Bioconductor package for user-friendly access to the Beacon v2 API</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="hangjia-zhao-and-michael-baudis">Hangjia Zhao and Michael Baudis<a class="headerlink" href="#hangjia-zhao-and-michael-baudis" title="Permanent link">&para;</a></h4>
<h5 id="bioinformatics-advances-oxford-2025-july-16">Bioinformatics Advances (Oxford). 2025 July 16.<a class="headerlink" href="#bioinformatics-advances-oxford-2025-july-16" title="Permanent link">&para;</a></h5>
<ul>
<li>doi: <a href="https://doi.org/10.1093/bioadv/vbaf172">10.1093/bioadv/vbaf172</a></li>
<li>PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40761325/">40761325</a></li>
</ul>
<p><strong>Abstract</strong>: The Beacon v2 specification, established by the Global Alliance for Genomics and Health (GA4GH), consists of a standardized framework and data models for genomic and phenotypic data discovery. By enabling secure, federated data sharing, it fosters interoperability across genomic resources. Progenetix, a reference implementation of Beacon v2, exemplifies its potential for large-scale genomic data integration, offering open access to genomic mutation data across diverse cancer types. Here we present pgxRpi, an open-source R/Bioconductor package that provides a streamlined interface to the Progenetix Beacon v2 REST API, 
<a href="2025-03-23-pgxRpi/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      May 22, 2025
      
    </span>
</aside>

<h3 class="title_news">Multicancer analyses of short tandem repeat variations reveal shared gene regulatory mechanisms</h3>
    
<h5 class="title_news">Repeat lengths association with the expression of nearby genes (eSTRs) in CRC, STAD and UCEC tumors</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="feifei-xia-max-verbiest-oxana-lundstrom-tugce-bilgin-sonay-michael-baudis-and-maria-anisimova">Feifei Xia, Max Verbiest, Oxana Lundström, Tugce Bilgin Sonay, Michael Baudis and Maria Anisimova<a class="headerlink" href="#feifei-xia-max-verbiest-oxana-lundstrom-tugce-bilgin-sonay-michael-baudis-and-maria-anisimova" title="Permanent link">&para;</a></h4>
<h5 id="briefings-in-bioinformatics-doi-httpsdoiorg101093bibbbaf219">Briefings in Bioinformatics, doi: <a href="https://doi.org/10.1093/bib/bbaf219">https://doi.org/10.1093/bib/bbaf219</a><a class="headerlink" href="#briefings-in-bioinformatics-doi-httpsdoiorg101093bibbbaf219" title="Permanent link">&para;</a></h5>
<h5 id="biorxiv-preprint-2025-01-06-doi-httpsdoiorg10110120250106629343">BiorXiv preprint 2025-01-06, doi: <a href="https://doi.org/10.1101/2025.01.06.629343">https://doi.org/10.1101/2025.01.06.629343</a><a class="headerlink" href="#biorxiv-preprint-2025-01-06-doi-httpsdoiorg10110120250106629343" title="Permanent link">&para;</a></h5>
<p><img alt="OUP logo" src="/img/logo-oxford-academic.svg" style="width: 180px; float: right; margin-left: 20px; margin-bottom: 10px; margin-top: -10px;" />
<img alt="biorXiv logo" src="/img/logo_biorXiv.jpg" style="float: right; margin-left: 20px; margin-bottom: 10px; margin-top: -10px;" />
<em>Abstract</em> Short tandem repeats (STRs) have been reported to influence gene expression across various human tissues. While STR variations are enriched in colorectal, stomach, and endometrial cancers, particularly in microsatellite instable tumors, their functional effects and regulatory mechanisms on gene expression remain poorly understood across these cancer types. Here, we leverage whole-exome sequencing and gene expression data to identify STRs for which repeat lengths are associated with the expression of nearby genes (eSTRs) in colorectal, stomach, and endometrial tumors. 
<a href="2025-05-22-multicancer-repeats/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      November 14, 2024
      
    </span>
</aside>

<h3 class="title_news">A systematic benchmark of copy number variation detection tools for high density SNP genotyping arrays</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="mn-van-baardwijk-lsem-heijnen-h-zhao-m-baudis-and-ap-stubbs">M.N. van Baardwijk, L.S.E.M. Heijnen, H. Zhao, M. Baudis and A.P. Stubbs<a class="headerlink" href="#mn-van-baardwijk-lsem-heijnen-h-zhao-m-baudis-and-ap-stubbs" title="Permanent link">&para;</a></h4>
<h5 id="genomics-elsevier-2024-nov-14">Genomics (Elsevier). 2024 Nov 14.<a class="headerlink" href="#genomics-elsevier-2024-nov-14" title="Permanent link">&para;</a></h5>
<ul>
<li>doi: <a href="https://doi.org/10.1016/j.ygeno.2024.110962">10.1016/j.ygeno.2024.110962</a></li>
<li>PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/39547585/">39547585</a></li>
</ul>
<p><img alt="elsevier genomics logo" src="/img/logo-elsevier-genomics.gif" style="float: right; margin: 0px 0px 20px 20px;" />
<strong>Abstract</strong> Copy Number Variations (CNVs) are crucial in various diseases, especially cancer, but detecting them accurately from SNP genotyping arrays remains challenging. Therefore, this study benchmarked five CNV detection tools-PennCNV, QuantiSNP, iPattern, EnsembleCNV, and R-GADA-using SNP array and WGS data from 2002 individuals of the DRAGEN re-analysis of the 1000 Genomes project. 
<a href="2024-11-14-benchmark/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      November 06, 2024
      
    </span>
</aside>

<h3 class="title_news">Copy number variation heterogeneity reveals biological inconsistency in hierarchical cancer classifications</h3>
    
<h5 class="title_news">Research Article</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="ziying-yang-paula-carrio-cordo-and-michael-baudis">Ziying Yang, Paula Carrio-Cordo and Michael Baudis<a class="headerlink" href="#ziying-yang-paula-carrio-cordo-and-michael-baudis" title="Permanent link">&para;</a></h4>
<h5 id="molecular-cytogenetics-spring-nature-doi-101186s13039-024-00692-2">Molecular Cytogenetics (Spring Nature). <a href="https://doi.org/10.1186/s13039-024-00692-2">doi: 10.1186/s13039-024-00692-2</a><a class="headerlink" href="#molecular-cytogenetics-spring-nature-doi-101186s13039-024-00692-2" title="Permanent link">&para;</a></h5>
<p><strong>Abstract</strong>: Cancers are heterogeneous diseases with unifying features of abnormal and consuming cell growth, where the deregulation of normal cellular functions is initiated by the accumulation of genomic mutations in cells of - potentially - any organ. At diagnosis malignancies typically present with patterns of somatic genome variants on diverse levels of heterogeneity. Among the different types of genomic alterations, copy number variants (CNV) represent a distinct, near-ubiquitous class of structural variants. Cancer classifications are foundational for patient care and oncology research. Terminologies such as the National Cancer Institute Thesaurus provide large sets of hierarchical cancer classification vocabularies and promote data interoperability and ontology-driven computational analysis. To find out how categorical classifications correspond to genomic observations, we conducted a meta-analysis of inter-sample genomic heterogeneity for classification hierarchies on CNV profiles from 97,142 individual samples across 512 cancer entities, and evaluated recurring CNV signatures across diagnostic subsets. Our results highlight specific biological mechanisms across cancer entities with the potential for improvement of patient stratification and future enhancement of cancer classification systems and provide some indications for cooperative genomic events across distinct clinical entities.</p>
<a href="2024-11-06-CNV-heterogeneity/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      May 15, 2024
      
    </span>
</aside>

<h3 class="title_news">In Search of the Perfect Model: How Cance Cell Lines Relate to Native Cancers</h3>
    
<h5 class="title_news">bioRxiv preprint</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="rahel-paloots-ziying-yang-and-michael-baudis">Rahel Paloots , Ziying Yang and Michael Baudis<a class="headerlink" href="#rahel-paloots-ziying-yang-and-michael-baudis" title="Permanent link">&para;</a></h4>
<h5 id="bioarxiv-preprint-2024-05-15-httpsdoiorg10110120240515594310">bioarXiv preprint (2024-05-15): <a href="https://doi.org/10.1101/2024.05.15.594310">https://doi.org/10.1101/2024.05.15.594310</a><a class="headerlink" href="#bioarxiv-preprint-2024-05-15-httpsdoiorg10110120240515594310" title="Permanent link">&para;</a></h5>
<p><img alt="biorXiv logo" src="/img/logo_biorXiv.jpg" style="float: right; margin-left: 20px; margin-bottom: 10px; margin-top: -10px;" />
<strong>Abstract</strong>: Cancer cell lines are frequently used in biological and transla-
tional research to study cellular mechanisms and explore treat-
ment options. However, cancer cell lines may display mutational
profiles divergent from native cancers or may be misidentified
or contaminated. We explored how similar cancer cell lines are
to native cancers to find the most suitable representations for
the corresponding diseases by utilising large collections of copy
number variation (CNV) profiles and applied machine learning
(ML) algorithms to predict cell line classifications.
<a href="2024-05-15-cell-lines-cancer-heterogeneity/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      March 27, 2024
      
    </span>
</aside>

<h3 class="title_news">cancercelllines.org - a Novel Resource for Genomic Variants in Cancer Cell Lines</h3>
    
<h5 class="title_news">DATABASE Article</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="rahel-paloots-and-michael-baudis">Rahel Paloots and Michael Baudis<a class="headerlink" href="#rahel-paloots-and-michael-baudis" title="Permanent link">&para;</a></h4>
<h5 id="database-oxford-2024-apr-302024baae030-doi-101093databasebaae030">Database (Oxford). 2024 Apr 30:2024:baae030.  <a href="https://doi.org/10.1093/database/baae030">doi: 10.1093/database/baae030</a><a class="headerlink" href="#database-oxford-2024-apr-302024baae030-doi-101093databasebaae030" title="Permanent link">&para;</a></h5>
<h5 id="bioarxiv-preprint-2023-12-13-httpsdoiorg10110120231212571281">bioarXiv preprint (2023-12-13): <a href="https://doi.org/10.1101/2023.12.12.571281">https://doi.org/10.1101/2023.12.12.571281</a><a class="headerlink" href="#bioarxiv-preprint-2023-12-13-httpsdoiorg10110120231212571281" title="Permanent link">&para;</a></h5>
<p><img alt="DATABASE logo" src="/img/logo-database-journal-300x150.png" style="float: right; width: 150px; margin-top: -100px;" />
<strong>Abstract</strong>: Cancer cell lines are an important component in biological and medical research, enabling studies of cellular mechanisms as well as the development and testing of pharmaceuticals. Genomic alterations in cancer cell lines are widely studied as models for oncogenetic events and are represented in a wide range of primary resources. We have created a comprehensive, curated knowledge resource - cancercelllines.org - with the aim to enable easy access to genomic profiling data in cancer cell lines, curated from a variety of resources and integrating both copy number and single nucleotide variants (SNVs) data. We have gathered over 5,600 copy number profiles as well as SNV annotations for 16,000 cell lines and provide this data with mappings to the GRCh38 reference genome. Both genomic variations and associated curated metadata can be queried through the GA4GH Beacon v2 API and a graphical user interface with extensive data retrieval enabled using GA4GH data schemas under a permissive licensing scheme.</p>
<p><strong>Availability and Implementation</strong>: Our resource is publicly available on the web at <a href="https://cancercelllines.org">cancercelllines.org</a>.</p>
<a href="2024-03-27-cancercelllines.org/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      March 16, 2024
      
    </span>
</aside>

<h3 class="title_news">Data-Driven Information Extraction and Enrichment of Molecular Profiling Data for Cancer Cell Lines</h3>
    
<h5 class="title_news">Literature-derived annotations as entry point for data exploration</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="ellery-smith-rahel-paloots-dimitris-giagkos-michael-baudis-and-kurt-stockinger">Ellery Smith, Rahel Paloots, Dimitris Giagkos, Michael Baudis and Kurt Stockinger<a class="headerlink" href="#ellery-smith-rahel-paloots-dimitris-giagkos-michael-baudis-and-kurt-stockinger" title="Permanent link">&para;</a></h4>
<h5 id="bioinformatics-advances-vbae045-doiorg101093bioadvvbae045">Bioinformatics Advances, vbae045, <a href="https://doi.org/10.1093/bioadv/vbae045">doi.org/10.1093/bioadv/vbae045</a><a class="headerlink" href="#bioinformatics-advances-vbae045-doiorg101093bioadvvbae045" title="Permanent link">&para;</a></h5>
<h5 id="previous-arxiv-preprint-2023-07-03-httpsdoiorg1048550arxiv230700933">Previous arXiv preprint (2023-07-03): <a href="https://doi.org/10.48550/arXiv.2307.00933">https://doi.org/10.48550/arXiv.2307.00933</a><a class="headerlink" href="#previous-arxiv-preprint-2023-07-03-httpsdoiorg1048550arxiv230700933" title="Permanent link">&para;</a></h5>
<p><img alt="OUP logo" src="/img/logo-oxford-academic.svg" style="width: 180px; float: right; margin-left: 20px; margin-bottom: 10px; margin-top: -10px;" />
<img alt="arXiv logo" src="/img/logo_arXiv.png" style="float: right; margin-left: 20px; margin-bottom: 10px; margin-top: -10px;" />
<strong>Motivation</strong>: With the proliferation of research means and computational methodologies, published biomedical literature
is growing exponentially in numbers and volume (Lubowitz et al., 2021). As a consequence, in the fields of biological,
medical and clinical research, domain experts have to sift through massive amounts of scientific text to find relevant
information. However, this process is extremely tedious and slow to be performed by humans. Hence, novel computational
information extraction and correlation mechanisms are required to boost meaningful knowledge extraction. Results: In
this work, we present the design, implementation and application of a novel data extraction and exploration system. This
system extracts deep semantic relations between textual entities from scientific literature to enrich existing structured
clinical data in the domain of cancer cell lines. We introduce a new public data exploration portal, which enables automatic
linking of genomic copy number variants plots with ranked, related entities such as affected genes. Each relation is
accompanied by literature-derived evidences, allowing for deep, yet rapid, literature search, using existing structured
data as a springboard.</p>
<p><strong>Availability and Implementation</strong>: Our system is publicly available on the web at <a href="https://cancercelllines.org">cancercelllines.org</a>.</p>
<p><strong>Contact</strong>: The authors can be contacted at <a href="&#109;&#97;&#105;&#108;&#116;&#111;&#58;&#101;&#108;&#108;&#101;&#114;&#121;&#46;&#115;&#109;&#105;&#116;&#104;&#64;&#122;&#104;&#97;&#119;&#46;&#99;&#104;">&#101;&#108;&#108;&#101;&#114;&#121;&#46;&#115;&#109;&#105;&#116;&#104;&#64;&#122;&#104;&#97;&#119;&#46;&#99;&#104;</a> or <a href="&#109;&#97;&#105;&#108;&#116;&#111;&#58;&#114;&#97;&#104;&#101;&#108;&#46;&#112;&#97;&#108;&#111;&#111;&#116;&#115;&#64;&#117;&#122;&#104;&#46;&#99;&#104;">&#114;&#97;&#104;&#101;&#108;&#46;&#112;&#97;&#108;&#111;&#111;&#116;&#115;&#64;&#117;&#122;&#104;&#46;&#99;&#104;</a>.</p>
<a href="2023-07-03-zhaw/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      March 01, 2024
      
    </span>
</aside>

<h3 class="title_news">Twelve quick tips for deploying a Beacon</h3>
    
<h5 class="title_news">Some hints for Beacon developers & implementers</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="lauren-a-fromont-mauricio-moldes-michael-baudis-anthony-j-brookes-arcadi-navarro-and-jordi-rambla">Lauren A Fromont, Mauricio Moldes, Michael Baudis, Anthony J Brookes, Arcadi Navarro and Jordi Rambla<a class="headerlink" href="#lauren-a-fromont-mauricio-moldes-michael-baudis-anthony-j-brookes-arcadi-navarro-and-jordi-rambla" title="Permanent link">&para;</a></h4>
<h5 id="plos-comput-biol-2024-mar-1203e1011817">PLoS Comput Biol. 2024 Mar 1;20(3):e1011817.<a class="headerlink" href="#plos-comput-biol-2024-mar-1203e1011817" title="Permanent link">&para;</a></h5>
<ul>
<li>doi: <a href="https://doi.org/10.1371/journal.pcbi.1011817">10.1371/journal.pcbi.1011817</a>.</li>
<li>PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/38427629/">38427629</a></li>
</ul>
<p><strong>Introduction</strong>: In the age of data-driven biomedical research and clinical practice, the sharing of genomic and clinical data for health research and personalized medicine has become an important contribu- tor to improved diagnosis and treatment. From the data owner’s perspective, potential benefits include improved treatments, personalization of healthcare practice, and more effective con- trol of disease proliferation. However, the requirement for high levels of data security to pro- tect sensitive information presents a barrier to data discovery and sharing.</p>
<p>Beacon is designed to enable the benefits of data discovery while minimizing the associated risks...</p>
<a href="2024-03-01-Beacon-Tips/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      December 28, 2023
      
    </span>
</aside>

<h3 class="title_news">labelSeg: segment annotation for tumor copy number alteration profiles</h3>
    
<h5 class="title_news">A tool to assign relative SCNA levels to segments</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="hangjia-zhao-and-michael-baudis">Hangjia Zhao and Michael Baudis<a class="headerlink" href="#hangjia-zhao-and-michael-baudis" title="Permanent link">&para;</a></h4>
<h5 id="briefings-in-bioinformatics-oxford-2024-jan-312024bbad541">Briefings in Bioinformatics (Oxford). 2024 Jan 31;2024:bbad541.<a class="headerlink" href="#briefings-in-bioinformatics-oxford-2024-jan-312024bbad541" title="Permanent link">&para;</a></h5>
<ul>
<li>doi: <a href="https://doi.org/10.1093/bib/bbad541">10.1093/bib/bbad541</a></li>
<li>PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/38300514/">38300514</a></li>
<li>bioRxiv. doi: <a href="https://doi.org/10.1101/2023.05.17.541097">10.1101/2023.05.17.541097</a></li>
</ul>
<p><strong>Abstract</strong> Somatic copy number alterations (SCNAs) are a predominant type of oncogenomic alterations that affect a large proportion of the genome in the majority of cancer samples. Current technologies allow high-throughput measurement of such copy number aberrations, generating results consisting of frequently large sets of SCNA segments. However, the automated annotation and integration of such data are particularly challenging because the measured signals reflect biased, relative copy number ratios. In this study, we introduce labelSeg, an algorithm designed for rapid and accurate annotation of CNA segments, with the aim of enhancing the interpretation of tumor SCNA profiles. 
<a href="2023-12-28-labelseg/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      July 03, 2023
      
    </span>
</aside>

<h3 class="title_news">Short tandem repeat mutations regulate gene expression in colorectal cancer</h3>
    
<h5 class="title_news">Exploring STR patterns and their relation to expression changes in cancer</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="max-a-verbiest-oxana-lundstrom-feifei-xia-michael-baudis-tugce-bilgin-sonay-maria-anisimova">Max A Verbiest, Oxana Lundström, Feifei Xia, Michael Baudis, Tugce Bilgin Sonay, Maria Anisimova<a class="headerlink" href="#max-a-verbiest-oxana-lundstrom-feifei-xia-michael-baudis-tugce-bilgin-sonay-maria-anisimova" title="Permanent link">&para;</a></h4>
<h5 id="doi-httpsdoiorg10110120231129569189">doi: <a href="https://doi.org/10.1101/2023.11.29.569189">https://doi.org/10.1101/2023.11.29.569189</a><a class="headerlink" href="#doi-httpsdoiorg10110120231129569189" title="Permanent link">&para;</a></h5>
<p><img alt="biorXiv logo" src="/img/logo_biorXiv.jpg" style="float: right; margin-left: 20px; margin-bottom: 10px; margin-top: -10px;" />
Short tandem repeat (STR) mutations are prevalent in colorectal cancer (CRC), especially in tumours with the microsatellite instability (MSI) phenotype. While STR length variations are known to regulate gene expression under physiological conditions, the functional impact of STR mutations in CRC remains unclear. Here, we integrate STR mutation data with clinical information and gene expression levels to study the gene regulatory effects of STR mutations in CRC. We confirm that STR mutability in CRC highly depends on the MSI status, repeat unit size, and repeat length. Furthermore, we present a set of 1244 putative expression STRs (eSTRs) for which the STR length is associated with gene expression levels in CRC tumours. The length of 73 eSTRs is associated with expression levels of cancer-related genes, nine of which are CRC-specific genes. We show that linear models describing eSTR-gene expression relationships allow for predictions of gene expression changes in response to eSTR mutations. Moreover, we found an increased mutability of eSTRs in MSI tumours. Our evidence of gene regulatory roles for eSTRs in CRC highlights a mostly overlooked way through which tumours may modulate their phenotypes. The increased mutability of eSTRs in MSI tumours may be an early indication that eSTR mutations can confer a selective advantage to tumours. Future extensions of our findings into larger cohorts could uncover new STR-based targets in the treatment of cancer.</p>
<a href="2023-11-29-repeats/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      May 17, 2023
      
    </span>
</aside>

<h3 class="title_news">Phenopacket-tools: Building and validating GA4GH Phenopackets</h3>
    
<h5 class="title_news">Bioinformatics tools and examples for working with the Phenopackets standard</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h5 id="danis-d-jacobsen-job-wagner-ah-groza-t-beckwith-ma-rekerle-l-carmody-lc-reese-j-hegde-h-ladewig-ms-seitz-b-munoz-torres-m-harris-nl-rambla-j-baudis-m-mungall-cj-haendel-ma-robinson-pn-2023-phenopacket-tools-building-and-validating-ga4gh-phenopackets-plos-one-18e0285433">Danis D, Jacobsen JOB, Wagner AH, Groza T, Beckwith MA, Rekerle L, Carmody LC, Reese J, Hegde H, Ladewig MS, Seitz B, Munoz-Torres M, Harris NL, Rambla J, Baudis M, Mungall CJ, Haendel MA, Robinson PN. (2023) <strong>Phenopacket-tools: Building and validating GA4GH Phenopackets.</strong> <em>PLoS One.</em> 18:e0285433.<a class="headerlink" href="#danis-d-jacobsen-job-wagner-ah-groza-t-beckwith-ma-rekerle-l-carmody-lc-reese-j-hegde-h-ladewig-ms-seitz-b-munoz-torres-m-harris-nl-rambla-j-baudis-m-mungall-cj-haendel-ma-robinson-pn-2023-phenopacket-tools-building-and-validating-ga4gh-phenopackets-plos-one-18e0285433" title="Permanent link">&para;</a></h5>
<p><strong>Abstract</strong> The Global Alliance for Genomics and Health (GA4GH) is a standards-setting organization that is developing a suite of coordinated standards for genomics. The GA4GH Phenopacket Schema is a standard for sharing disease and phenotype information that characterizes an individual person or biosample. The Phenopacket Schema is flexible and can represent clinical data for any kind of human disease including rare disease, complex disease, and cancer. It also allows consortia or databases to apply additional constraints to ensure uniform data collection for specific goals. We present phenopacket-tools, an open-source Java library and command-line application for construction, conversion, and validation of phenopackets. Phenopacket-tools simplifies construction of phenopackets by providing concise builders, programmatic shortcuts, and predefined building blocks (ontology classes) for concepts such as anatomical organs, age of onset, biospecimen type, and clinical modifiers.
<a href="2023-05-17-phenopackets-tools/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      December 28, 2022
      
    </span>
</aside>

<h3 class="title_news">Candidate targets of copy number deletion events across 17 cancer types</h3>
    
<h5 class="title_news">Identifying cancer related genes against the background of somatic CNV events</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="huang-q-and-baudis-m">Huang Q and Baudis M<a class="headerlink" href="#huang-q-and-baudis-m" title="Permanent link">&para;</a></h4>
<h5 id="doi-103389fgene20221017657">doi: <a href="https://www.frontiersin.org/articles/10.3389/fgene.2022.1017657/full">10.3389/fgene.2022.1017657</a><a class="headerlink" href="#doi-103389fgene20221017657" title="Permanent link">&para;</a></h5>
<h5 id="previous-biorxiv-first-2022-06-29-doiorg10110120220629498080">previous bioRxiv (first )2022-06-29), <a href="https://doi.org/10.1101/2022.06.29.498080">doi.org/10.1101/2022.06.29.498080</a><a class="headerlink" href="#previous-biorxiv-first-2022-06-29-doiorg10110120220629498080" title="Permanent link">&para;</a></h5>
<p><strong>Abstract</strong> Genome variation is the direct cause of cancer and driver of its clonal evolution. While the impact of many point mutations can be evaluated through their modification of individual genomic elements, even a single copy number aberration (CNA) may encompass hundreds of genes and therefore pose challenges to untangle potentially complex functional effects. However, consistent, recurring and disease-specific patterns in the genome-wide CNA landscape imply that particular CNA may promote cancer-type-specific characteristics. Discerning essential cancer-promoting alterations from the inherent co-dependency in CNA would improve the understanding of mechanisms of CNA and provide new insights into cancer biology and potential therapeutic targets.
<a href="2022-12-28-signatures-in-17-cancer-types/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      August 26, 2022
      
    </span>
</aside>

<h3 class="title_news">GA4GH Phenopackets: A Practical Introduction</h3>
    
<h5 class="title_news">Phenopackets v2 introduction with practical examples</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <p><img alt="" src="/img/logo_AdvancedGenetics.png" style="float: right; width: 200px; margin-top: -35px;" /></p>
<h4 id="ladewig-ms-jacobsen-jo-wagner-ah-danis-d-kassaby-be-gargano-m-groza-t-baudis-m-steinhaus-r-seelow-d-bechrakis-ne-mungall-cj-schofield-pn-elemento-o-smith-l-mcmurry-ja-munoz-torres-m-haendel-ma-and-robinson-pn">Ladewig MS, Jacobsen JO, Wagner AH, Danis D, Kassaby BE, Gargano M, Groza T, Baudis M, Steinhaus R, Seelow D, Bechrakis NE, Mungall CJ, Schofield PN, Elemento O, Smith L, McMurry JA, Munoz-Torres M, Haendel MA and Robinson PN<a class="headerlink" href="#ladewig-ms-jacobsen-jo-wagner-ah-danis-d-kassaby-be-gargano-m-groza-t-baudis-m-steinhaus-r-seelow-d-bechrakis-ne-mungall-cj-schofield-pn-elemento-o-smith-l-mcmurry-ja-munoz-torres-m-haendel-ma-and-robinson-pn" title="Permanent link">&para;</a></h4>
<h5 id="advanced-genetics-2022-2200016-link">Advanced Genetics 2022, 2200016. <a href="https://doi.org/10.1002/ggn2.202200016">LINK</a><a class="headerlink" href="#advanced-genetics-2022-2200016-link" title="Permanent link">&para;</a></h5>
<p><strong>Abstract</strong> The Global Alliance for Genomics and Health (GA4GH) is developing a suite of coordinated standards for genomics for healthcare. The Phenopacket is a new GA4GH standard for sharing disease and phenotype information that characterizes an individual person, linking that individual to detailed phenotypic descriptions, genetic information, diagnoses, and treatments. A detailed example is presented that illustrates how to use the schema to represent the clinical course of a patient with retinoblastoma, including demographic information, the clinical diagnosis, phenotypic features and clinical measurements, an examination of the extirpated tumor, therapies, and the results of genomic analysis. The Phenopacket Schema, together with other GA4GH data and technical standards, will enable data exchange and provide a foundation for the computational analysis of disease and phenotype information to improve our ability to diagnose and conduct research on all types of disorders, including cancer and rare diseases.</p>
<p>The Phenopacket software is available at <a href="https://github.com/phenopackets/">github.com/phenopackets/</a>.</p>
<a href="2022-08-26-Phenopackets-practical-introduction-AdvGen/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      June 15, 2022
      
    </span>
</aside>

<h3 class="title_news">The GA4GH Phenopacket schema defines a computable representation of clinical data</h3>
    
<h5 class="title_news">Phenopackets v2 publication</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <p><img alt="Cell Genomics logo" src="/img/nature-biotechnology-logo.svg" style="float: right; width: 200px; margin-top: -35px;" /></p>
<h4 id="jacobsen-job-baudis-m-baynam-gs-beckmann-js-beltran-s-buske-oj-callahan-tj-chute-cg-courtot-m-danis-d-elemento-o-essenwanger-a-freimuth-rr-haendel-ma-robinson-pn-the-gaghpmc">Jacobsen JOB, Baudis M, Baynam GS, Beckmann JS, Beltran S, Buske OJ, Callahan TJ, Chute CG, Courtot M, Danis D, Elemento O, Essenwanger A, Freimuth RR, ... , Haendel MA, Robinson PN, The GAGHPMC.<a class="headerlink" href="#jacobsen-job-baudis-m-baynam-gs-beckmann-js-beltran-s-buske-oj-callahan-tj-chute-cg-courtot-m-danis-d-elemento-o-essenwanger-a-freimuth-rr-haendel-ma-robinson-pn-the-gaghpmc" title="Permanent link">&para;</a></h4>
<h5 id="nature-biotechnology-202240817-820-link-pmid35705716">Nature Biotechnology. 2022;40:817-820. <a href="https://rdcu.be/cPLaA">LINK</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/35705716/">PMID:35705716</a><a class="headerlink" href="#nature-biotechnology-202240817-820-link-pmid35705716" title="Permanent link">&para;</a></h5>
<p><strong>Abstract</strong> Despite great strides made in the development and wide acceptance of standards for exchanging structured information about genomic variants, progress in standards for computational phenotype analysis for translational genomics has lagged behind. Phenotypic features (signs, symptoms, laboratory and imaging findings, results of physiological tests, etc.) are of high clinical importance, yet exchanging them in conjunction with genomic variation information is often overlooked or even neglected.
<a href="2022-06-15-Phenopackets-paper-NatBiotech/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      March 17, 2022
      
    </span>
</aside>

<h3 class="title_news">Beacon v2 and Beacon networks: A "lingua franca" for federated data discovery in biomedical genomics, and beyond</h3>
    
<h5 class="title_news">Beacon v2 publication</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="rambla-j-baudis-m-ariosa-r-beck-t-fromont-la-navarro-a-paloots-r-rueda-m-saunders-g-singh-b-spalding-jd">Rambla J, Baudis M, Ariosa R, Beck T, Fromont LA, Navarro A, Paloots R, Rueda M, Saunders G, Singh B, Spalding JD.<a class="headerlink" href="#rambla-j-baudis-m-ariosa-r-beck-t-fromont-la-navarro-a-paloots-r-rueda-m-saunders-g-singh-b-spalding-jd" title="Permanent link">&para;</a></h4>
<h5 id="human-mutation-2022-mar-17-pmid35297548">Human Mutation. 2022 Mar 17. <a href="https://onlinelibrary.wiley.com/doi/10.1002/humu.24369">PMID:35297548</a><a class="headerlink" href="#human-mutation-2022-mar-17-pmid35297548" title="Permanent link">&para;</a></h5>
<p><strong>Abstract</strong> Beacon is a basic data discovery protocol issued by the Global Alliance for Genomics and Health (GA4GH). The main goal addressed by version 1 of the Beacon protocol was to test the feasibility of broadly sharing human genomic data, through providing simple "yes" or "no" responses to queries about the presence of a given variant in datasets hosted by Beacon providers.</p>

<a href="2022-03-17-Beacon_paper/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      November 30, 2021
      
    </span>
</aside>

<h3 class="title_news">The GA4GH Phenopacket schema: A computable representation of clinical data for precision medicine</h3>
    
<h5 class="title_news">Phenopackets v2 preprint</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="jacobsen-job-baudis-m-baynam-gs-beckmann-js-beltran-s-callahan-tj-chute-cg-courtot-m-danis-d-elemento-o-freimuth-rr-haendel-ma-robinson-pn">Jacobsen JOB, Baudis M, Baynam GS, Beckmann JS, Beltran S, Callahan TJ, Chute CG, Courtot M, Danis D, Elemento O, Freimuth RR, ..., Haendel MA, Robinson PN.<a class="headerlink" href="#jacobsen-job-baudis-m-baynam-gs-beckmann-js-beltran-s-callahan-tj-chute-cg-courtot-m-danis-d-elemento-o-freimuth-rr-haendel-ma-robinson-pn" title="Permanent link">&para;</a></h4>
<h5 id="medrxiv-2021112721266944-doi1011012021112721266944">medRxiv, 2021.11.27.21266944. <a href="https://doi.org/10.1101/2021.11.27.21266944">doi:10.1101/2021.11.27.21266944</a><a class="headerlink" href="#medrxiv-2021112721266944-doi1011012021112721266944" title="Permanent link">&para;</a></h5>
<p><strong>Abstract</strong> Despite great strides in the development and wide acceptance of standards for exchanging structured information about genomic variants, there is no corresponding standard for exchanging phenotypic data, and this has impeded the sharing of phenotypic information for computational analysis. Here, we introduce the Global Alliance for Genomics and Health (GA4GH) Phenopacket schema, which supports exchange of computable longitudinal case-level phenotypic information for diagnosis and research of all types of disease including Mendelian and complex genetic diseases, cancer, and infectious diseases.
<a href="2021-11-30-preprint-phenopackets-v2/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      November 10, 2021
      
    </span>
</aside>

<h3 class="title_news">The GA4GH Variation Representation Specification (VRS): a Computational Framework for the Precise Representation and Federated Identification of Molecular Variation.</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="alex-h-wagner-lawrence-babb-gil-alterovitz-michael-baudis-matthew-brush-daniel-l-cameron-melissa-cline-malachi-griffith-obi-l-griffith-melissa-konopko-heidi-l-rehm-andrew-d-yates-robert-r-freimuth-reece-k-hart">Alex H. Wagner, Lawrence Babb, Gil Alterovitz, Michael Baudis, Matthew Brush, Daniel L. Cameron, Melissa Cline , Malachi Griffith, Obi L. Griffith, ..., Melissa Konopko, Heidi L. Rehm, Andrew D. Yates, Robert R. Freimuth, Reece K. Hart<a class="headerlink" href="#alex-h-wagner-lawrence-babb-gil-alterovitz-michael-baudis-matthew-brush-daniel-l-cameron-melissa-cline-malachi-griffith-obi-l-griffith-melissa-konopko-heidi-l-rehm-andrew-d-yates-robert-r-freimuth-reece-k-hart" title="Permanent link">&para;</a></h4>
<h5 id="wagner-alex-h-et-al-cell-genomics-volume-1-issue-2-100027-doi101016jxgen2021100027">Wagner, Alex H. et al. Cell Genomics, Volume 1, Issue 2, 100027 <a href="https://doi.org/10.1016/j.xgen.2021.100027">doi:10.1016/j.xgen.2021.100027</a><a class="headerlink" href="#wagner-alex-h-et-al-cell-genomics-volume-1-issue-2-100027-doi101016jxgen2021100027" title="Permanent link">&para;</a></h5>
<h5 id="biorxiv-version-202120210115426843-2021-01-15">bioRxiv. version 20212021.01.15.426843. (2021-01-15)<a class="headerlink" href="#biorxiv-version-202120210115426843-2021-01-15" title="Permanent link">&para;</a></h5>
<h4 id="note">Note<a class="headerlink" href="#note" title="Permanent link">&para;</a></h4>
<p><img alt="Cell Genomics logo" src="/img/logo-cell-genomics.png" style="float: right; width: 150px; margin-top: -50px;" />This article was published as part of a special GA4GH edition of <a href="https://www.cell.com/cell-genomics/issue?pii=S2666-979X(21)X0003-1">Cell Genomics</a>.</p>
<p><strong>Abstract</strong> Maximizing the personal, public, research, and clinical value of genomic information will require the reliable exchange of genetic variation data. We report here the Variation Representation Specification (VRS, pronounced “verse”), an extensible framework for the computable representation of variation that complements contemporary human-readable and flat file standards for genomic variation representation. VRS provides semantically precise representations of variation and leverages this design to enable federated identification of biomolecular variation with globally consistent and unique computed identifiers.
<a href="2021-11-10-Cell-Genomics-GA4GH-VRS/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      November 10, 2021
      
    </span>
</aside>

<h3 class="title_news">International federation of genomic medicine databases using GA4GH standards</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="adrian-thorogood-heidi-l-rehm-peter-goodhand-angela-jh-page-yann-joly-michael-baudis-jordi-rambla-arcadi-navarro-tommi-h-nyronen-mikael-linden-edward-s-dove-marc-fiume-michael-brudno-melissa-s-cline-ewan-birney">Adrian Thorogood, Heidi L. Rehm, Peter Goodhand, Angela J.H. Page, Yann Joly, Michael Baudis, Jordi Rambla, Arcadi Navarro, Tommi H. Nyronen, Mikael Linden, Edward S. Dove, Marc Fiume, Michael Brudno, Melissa S. Cline, Ewan Birney<a class="headerlink" href="#adrian-thorogood-heidi-l-rehm-peter-goodhand-angela-jh-page-yann-joly-michael-baudis-jordi-rambla-arcadi-navarro-tommi-h-nyronen-mikael-linden-edward-s-dove-marc-fiume-michael-brudno-melissa-s-cline-ewan-birney" title="Permanent link">&para;</a></h4>
<h5 id="thorogood-adrian-et-al-cell-genomics-volume-1-issue-2-100032-doi101016jxgen2021100032">Thorogood, Adrian et al. Cell Genomics, Volume 1, Issue 2, 100032 <a href="https://doi.org/10.1016/j.xgen.2021.100032">doi:10.1016/j.xgen.2021.100032</a><a class="headerlink" href="#thorogood-adrian-et-al-cell-genomics-volume-1-issue-2-100032-doi101016jxgen2021100032" title="Permanent link">&para;</a></h5>
<h4 id="note">Note<a class="headerlink" href="#note" title="Permanent link">&para;</a></h4>
<p><img alt="Cell Genomics logo" src="/img/logo-cell-genomics.png" style="float: right; width: 150px; margin-top: -50px;" />This article was published as part of a special GA4GH edition of <a href="https://www.cell.com/cell-genomics/issue?pii=S2666-979X(21)X0003-1">Cell Genomics</a>.</p>
<p><strong>Abstract</strong> We promote a shared vision and guide for how and when to federate genomic and health-related data sharing, enabling connections and insights across independent, secure databases. The GA4GH encourages a federated approach wherein data providers have the mandate and resources to share, but where data cannot move for legal or technical reasons. We recommend a federated approach to connect national genomics initiatives into a global network and precision medicine resource.</p>
<a href="2021-11-10-Cell-Genomics-GA4GH-federation-paper/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      November 10, 2021
      
    </span>
</aside>

<h3 class="title_news">GA4GH: International policies and standards for data sharing across genomic research and healthcare</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="heidi-l-rehm-angela-jh-page-lindsay-smith-jeremy-b-adams-gil-alterovitz-lawrence-j-babb-maxmillian-p-barkley-michael-baudis-michael-js-beauvais-tim-beck-jacques-s-beckmann-sergi-beltran-david-bernick-alexander-bernier-james-k-bonfield-tiffany-f-boughtwood-guillaume-bourque-sarion-r-bowers-anthony-j-brookes-michael-brudno-matthew-h-brush-david-bujold-tony-burdett-orion-j-buske-moran-n-cabili-daniel-l-cameron-robert-j-carroll-esmeralda-casas-silva-debyani-chakravarty-bimal-p-chaudhari-shu-hui-chen-j-michael-cherry-justina-chung-melissa-cline-hayley-l-clissold-robert-m-cook-deegan-melanie-courtot-peter-goodhand-kathryn-north-ewan-birney">Heidi L. Rehm, Angela J.H. Page, Lindsay Smith, Jeremy B. Adams, Gil Alterovitz, Lawrence J. Babb, Maxmillian P. Barkley, Michael Baudis, Michael J.S. Beauvais, Tim Beck, Jacques S. Beckmann, Sergi Beltran, David Bernick, Alexander Bernier, James K. Bonfield, Tiffany F. Boughtwood, Guillaume Bourque, Sarion R. Bowers, Anthony J. Brookes, Michael Brudno, Matthew H. Brush, David Bujold, Tony Burdett, Orion J. Buske, Moran N. Cabili , Daniel L. Cameron, Robert J. Carroll, Esmeralda Casas-Silva, Debyani Chakravarty, Bimal P. Chaudhari, Shu Hui Chen, J. Michael Cherry, Justina Chung, Melissa Cline, Hayley L. Clissold, Robert M. Cook-Deegan, Mélanie Courtot, ..., Peter Goodhand, Kathryn North, Ewan Birney<a class="headerlink" href="#heidi-l-rehm-angela-jh-page-lindsay-smith-jeremy-b-adams-gil-alterovitz-lawrence-j-babb-maxmillian-p-barkley-michael-baudis-michael-js-beauvais-tim-beck-jacques-s-beckmann-sergi-beltran-david-bernick-alexander-bernier-james-k-bonfield-tiffany-f-boughtwood-guillaume-bourque-sarion-r-bowers-anthony-j-brookes-michael-brudno-matthew-h-brush-david-bujold-tony-burdett-orion-j-buske-moran-n-cabili-daniel-l-cameron-robert-j-carroll-esmeralda-casas-silva-debyani-chakravarty-bimal-p-chaudhari-shu-hui-chen-j-michael-cherry-justina-chung-melissa-cline-hayley-l-clissold-robert-m-cook-deegan-melanie-courtot-peter-goodhand-kathryn-north-ewan-birney" title="Permanent link">&para;</a></h4>
<h5 id="rehm-heidi-l-et-al-cell-genomics-volume-1-issue-2-100029-doi101016jxgen2021100029">Rehm, Heidi L. et al. Cell Genomics, Volume 1, Issue 2, 100029 <a href="https://doi.org/10.1016/j.xgen.2021.100029">doi:10.1016/j.xgen.2021.100029</a><a class="headerlink" href="#rehm-heidi-l-et-al-cell-genomics-volume-1-issue-2-100029-doi101016jxgen2021100029" title="Permanent link">&para;</a></h5>
<h4 id="note">Note<a class="headerlink" href="#note" title="Permanent link">&para;</a></h4>
<p>This article was published as part of a special GA4GH edition of <a href="https://www.cell.com/cell-genomics/issue?pii=S2666-979X(21)X0003-1">Cell Genomics</a>.</p>
<p><strong>Abstract</strong> The Global Alliance for Genomics and Health (GA4GH) aims to accelerate biomedical advances by enabling the responsible sharing of clinical and genomic data through both harmonized data aggregation and federated approaches. The decreasing cost of genomic sequencing (along with other genome-wide molecular assays) and increasing evidence of its clinical utility will soon drive the generation of sequence data from tens of millions of humans, with increasing levels of diversity. In this perspective, we present the GA4GH strategies for addressing the major challenges of this data revolution.
<a href="2021-11-10-Cell-Genomics-GA4GH-marker-paper/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      June 30, 2021
      
    </span>
</aside>

<h3 class="title_news">The Progenetix oncogenomic resource in 2021</h3>
    
<h5 class="title_news">Article describing the current content & technical status of progenetix.org</h5>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="qingyao-huang-paula-carrio-cordo-bo-gao-rahel-paloots-michael-baudis">Qingyao Huang, Paula Carrio Cordo, Bo Gao, Rahel Paloots, Michael Baudis<a class="headerlink" href="#qingyao-huang-paula-carrio-cordo-bo-gao-rahel-paloots-michael-baudis" title="Permanent link">&para;</a></h4>
<h5 id="database-oxford-2021-jul-172021baab043">Database (Oxford). 2021 Jul 17;2021:baab043.<a class="headerlink" href="#database-oxford-2021-jul-172021baab043" title="Permanent link">&para;</a></h5>
<ul>
<li>doi: 10.1093/database/baab043.</li>
<li>PMID: 34272855</li>
<li>PMCID: PMC8285936.</li>
<li>bioRxiv. doi: <a href="https://doi.org/10.1101/2021.02.15.428237">doi.org/10.1101/2021.02.15.428237</a></li>
</ul>
<p><img alt="DATABASE logo" src="/img/logo-database-journal-300x150.png" style="float: right; width: 150px; margin-top: -100px;" /></p>
<p><strong>Abstract</strong> In cancer, copy number aberrations (CNAs) represent a type of nearly ubiquitous and frequently extensive structural genome variations. To disentangle the molecular mechanisms underlying tumorigenesis as well as identify and characterize molecular subtypes, the comparative and meta-analysis of large genomic variant collections can be of immense importance. Over the last decades, cancer genomic profiling projects have resulted in a large amount of somatic genome variation profiles, however segregated in a multitude of individual studies and datasets. The Progenetix project, initiated in 2001, curates individual cancer CNA profiles and associated metadata from published oncogenomic studies and data repositories with the aim to empower integrative analyses spanning all different cancer biologies.
<a href="2021-06-30-publication-Progenetix-2021/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      May 13, 2021
      
    </span>
</aside>

<h3 class="title_news">Signatures of Discriminative CNA in 31 Cancer Subtypes</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h3 id="bo-gao-and-michael-baudis-2021">Bo Gao and Michael Baudis (2021)<a class="headerlink" href="#bo-gao-and-michael-baudis-2021" title="Permanent link">&para;</a></h3>
<h4 id="published-at-frontiers-in-genetics-2021-05-13">Published at <em>Frontiers in Genetics</em>, 2021-05-13<a class="headerlink" href="#published-at-frontiers-in-genetics-2021-05-13" title="Permanent link">&para;</a></h4>
<h4 id="abstract">Abstract<a class="headerlink" href="#abstract" title="Permanent link">&para;</a></h4>
<p>Copy number aberrations (CNA) are one of the most important classes of genomic mutations relatedto oncogenetic effects. In the past three decades, a vast amount of CNA data has been generated bymolecular-cytogenetic and genome sequencing based methods. While this data has been instrumentalin the identification of cancer-related genes and promoted research into the relation between CNA andhisto-pathologically defined cancer types, the heterogeneity of source data and derived CNV profilespose great challenges for data integration and comparative analysis. Furthermore, a majority of exist-ing studies have been focused on the association of CNA to pre-selected ”driver” genes with limitedapplication to rare drivers and other genomic elements.</p>

<a href="2021-04-15-publication-Bo-classifiers/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      March 01, 2021
      
    </span>
</aside>

<h3 class="title_news">Copy number variant heterogeneity among cancer types reflects inconsistent concordance with diagnostic classifications</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="paula-carrio-cordo-and-michael-baudis">Paula Carrio Cordo and Michael Baudis<a class="headerlink" href="#paula-carrio-cordo-and-michael-baudis" title="Permanent link">&para;</a></h4>
<h5 id="biorxiv-doi-doiorg10110120210301433348">bioRxiv. doi: <a href="https://doi.org/10.1101/2021.03.01.433348">doi.org/10.1101/2021.03.01.433348</a><a class="headerlink" href="#biorxiv-doi-doiorg10110120210301433348" title="Permanent link">&para;</a></h5>
<p>This article explores the correlation between subsets of cancer entities, grouped by their somatic CNV patterns, and levels of diagnostic classification systems.</p>

<a href="2021-03-01-preprint-Copy-number-variant-heterogeneity/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      August 10, 2020
      
    </span>
</aside>

<h3 class="title_news">The Ubiquitin Ligase TRIP12 Limits PARP1 Trapping and Constrains PARP Inhibitor Efficiency</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="marco-gatti-ralph-imhof-qingyao-huang-michael-baudis-matthias-altmeyer">Marco Gatti, Ralph Imhof, Qingyao Huang, Michael Baudis, Matthias Altmeyer<a class="headerlink" href="#marco-gatti-ralph-imhof-qingyao-huang-michael-baudis-matthias-altmeyer" title="Permanent link">&para;</a></h4>
<h5 id="cell-rep-2020-aug-4-doi-101016jcelrep2020107985">Cell Rep. 2020 Aug 4 DOI: <a href="https://doi.org/10.1016/j.celrep.2020.107985">10.1016/j.celrep.2020.107985</a><a class="headerlink" href="#cell-rep-2020-aug-4-doi-101016jcelrep2020107985" title="Permanent link">&para;</a></h5>
<p><strong>Abstract</strong> PARP inhibitors (PARPi) cause synthetic lethality in BRCA-deficient tumors. Whether specific vulnerabilities to PARPi exist beyond BRCA mutations and related defects in homology-directed repair (HDR) is not well understood. Here, we identify the ubiquitin E3 ligase TRIP12 as negative regulator of PARPi sensitivity.
<a href="2020-08-04-publication-trip12-parp1-study-CellRep/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      May 14, 2020
      
    </span>
</aside>

<h3 class="title_news">Oncology Informatics: Status Quo and Outlook - Review</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="paul-martin-putora-michael-baudis-beth-m-beadle-issam-el-naqa-frank-a-giordano-and-nils-h-nicolay">Paul Martin Putora, Michael Baudis, Beth M. Beadle, Issam El Naqa, Frank A. Giordano and Nils H. Nicolay<a class="headerlink" href="#paul-martin-putora-michael-baudis-beth-m-beadle-issam-el-naqa-frank-a-giordano-and-nils-h-nicolay" title="Permanent link">&para;</a></h4>
<h5 id="oncology-2020-05-14-doi-101159000507586-review">Oncology, 2020-05-14. <a href="https://doi.org/10.1159/000507586">DOI 10.1159/000507586</a> (Review)<a class="headerlink" href="#oncology-2020-05-14-doi-101159000507586-review" title="Permanent link">&para;</a></h5>
<p><strong>Abstract</strong> Oncology has undergone rapid progress, with emerging developments in areas including cancer stem cells, molecularly targeted therapies, genomic analyses, and individually tai- lored immunotherapy. These advances have expanded the tools available in the fight against cancer. Some of these have seen broad media coverage resulting in justified public attention. However, these achievements have only been possible due to rapid developments in the expanding field of biomedical informatics and information technology (IT).
<a href="2020-05-14-publication-oncology-informatics-review/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      May 06, 2020
      
    </span>
</aside>

<h3 class="title_news">Minimum Error Calibration and Normalization for Genomic Copy Number Analysis</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="bo-gao-and-michael-baudis-2020">Bo Gao and Michael Baudis (2020)<a class="headerlink" href="#bo-gao-and-michael-baudis-2020" title="Permanent link">&para;</a></h4>
<h5 id="biorxiv-2019-07-31-doi-101101720854">bioRxiv, 2019-07-31. <a href="https://doi.org/10.1101/720854">DOI 10.1101/720854</a><a class="headerlink" href="#biorxiv-2019-07-31-doi-101101720854" title="Permanent link">&para;</a></h5>
<h5 id="genomics-volume-112-issue-5-september-2020-pages-3331-3341-accepted-2020-05-06-doiorg101016jygeno202005008">Genomics, Volume 112, Issue 5, September 2020, Pages 3331-3341, accepted 2020-05-06 <a href="https://doi.org/10.1016/j.ygeno.2020.05.008">doi.org/10.1016/j.ygeno.2020.05.008</a>.<a class="headerlink" href="#genomics-volume-112-issue-5-september-2020-pages-3331-3341-accepted-2020-05-06-doiorg101016jygeno202005008" title="Permanent link">&para;</a></h5>
<h5 id="background">Background<a class="headerlink" href="#background" title="Permanent link">&para;</a></h5>
<p>Copy number variations (CNV) are regional deviations from the normal autosomal bi-allelic DNA content. While germline CNVs are a major contributor to genomic syndromes and inherited diseases, the majority of cancers accumulate extensive "somatic" CNV (sCNV or CNA) during the process of oncogenetic transformation and progression. While specific sCNV have closely been associated with tumorigenesis, intriguingly many neoplasias exhibit recurrent sCNV patterns beyond the involvement of a few cancer driver genes.
<a href="2020-05-07-publication-Mecan4CNA-Genomics/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      April 03, 2020
      
    </span>
</aside>

<h3 class="title_news">A harmonized meta-knowledgebase of clinical interpretations of somatic genomic variants in cancer</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="alex-h-wagner-brian-walsh-georgia-mayfield-david-tamborero-dmitriy-sonkin-kilannin-krysiak-jordi-deu-pons-ryan-p-duren-jianjiong-gao-julie-mcmurry-sara-patterson-catherine-del-vecchio-fitz-beth-a-pitel-nuria-lopez-bigas-mark-lawler-jeremy-goecks-malachi-griffith-obi-l-griffith-adam-a-margolin-variant-interpretation-for-cancer-consortium">Alex H. Wagner, Brian Walsh, Georgia Mayfield, David Tamborero, Dmitriy Sonkin, Kilannin Krysiak, Jordi Deu-Pons, Ryan P. Duren, Jianjiong Gao, Julie McMurry, Sara Patterson, Catherine del Vecchio Fitz, Beth A. Pitel, ..., Nuria Lopez-Bigas, Mark Lawler, Jeremy Goecks, Malachi Griffith, Obi L. Griffith, Adam A. Margolin &amp; Variant Interpretation for Cancer Consortium<a class="headerlink" href="#alex-h-wagner-brian-walsh-georgia-mayfield-david-tamborero-dmitriy-sonkin-kilannin-krysiak-jordi-deu-pons-ryan-p-duren-jianjiong-gao-julie-mcmurry-sara-patterson-catherine-del-vecchio-fitz-beth-a-pitel-nuria-lopez-bigas-mark-lawler-jeremy-goecks-malachi-griffith-obi-l-griffith-adam-a-margolin-variant-interpretation-for-cancer-consortium" title="Permanent link">&para;</a></h4>
<h5 id="nature-genetics-volume-52-pages-448457-2020">Nature Genetics volume 52, pages 448–457 (2020)<a class="headerlink" href="#nature-genetics-volume-52-pages-448457-2020" title="Permanent link">&para;</a></h5>
<p>Precision oncology relies on accurate discovery and interpretation of genomic variants, enabling individualized diagnosis, prognosis and therapy selection. We found that six prominent somatic cancer variant knowledgebases were highly disparate in content, structure and supporting primary literature, impeding consensus when evaluating variants and their relevance in a clinical setting. We developed a framework for harmonizing variant interpretations to produce a meta-knowledgebase of 12,856 aggregate interpretations.
<a href="2020-04-03-publication-Wagner-vicc-NatGen/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      April 02, 2020
      
    </span>
</aside>

<h3 class="title_news">Geographic assessment of cancer genome profiling studies</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="paula-carrio-cordo-elise-acheson-qingyao-huang-and-michael-baudis-2020">Paula Carrio Cordo, Elise Acheson, Qingyao Huang and Michael Baudis (2020)<a class="headerlink" href="#paula-carrio-cordo-elise-acheson-qingyao-huang-and-michael-baudis-2020" title="Permanent link">&para;</a></h4>
<h5 id="database-volume-2020-2020-baaa009-doiorg101093databasebaaa009">DATABASE, Volume 2020, 2020, baaa009, <a href="https://doi.org/10.1093/database/baaa009">doi.org/10.1093/database/baaa009</a><a class="headerlink" href="#database-volume-2020-2020-baaa009-doiorg101093databasebaaa009" title="Permanent link">&para;</a></h5>
<h5 id="biorxiv-preprint-2020-01-11-doi-101101827683">bioRxiv preprint, 2020-01-11. <a href="https://doi.org/10.1101/827683">DOI 10.1101/827683</a><a class="headerlink" href="#biorxiv-preprint-2020-01-11-doi-101101827683" title="Permanent link">&para;</a></h5>
<p><strong>Abstract</strong> Cancers arise from the accumulation of somatic genome mutations, which can be influenced by inherited genomic variants and external factors such as environmental or lifestyle-related exposure. Due to the heterogeneity of cancers, precise information about the genomic composition of germline and malignant tissues has to be correlated with morphological, clinical and extrinsic features to advance medical knowledge and treatment options. With global differences in cancer frequencies and disease types, geographic data is of importance to understand the interplay between genetic ancestry and environmental influence in cancer incidence, progression and treatment outcome.
<a href="2020-04-02-publication-geography-assesment-DATABASE/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      March 16, 2020
      
    </span>
</aside>

<h3 class="title_news">Enabling population assignment from cancer genomes with SNP2pop</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="huang-q-and-baudis-m-2020">Huang Q and Baudis M. (2020)<a class="headerlink" href="#huang-q-and-baudis-m-2020" title="Permanent link">&para;</a></h4>
<h5 id="sci-rep-10-4846-2020-doiorg101038s41598-020-61854-x">Sci Rep 10, 4846 (2020). <a href="https://doi.org/10.1038/s41598-020-61854-x">doi.org/10.1038/s41598-020-61854-x</a><a class="headerlink" href="#sci-rep-10-4846-2020-doiorg101038s41598-020-61854-x" title="Permanent link">&para;</a></h5>
<p><strong>Abstract</strong> In many cancers, incidence, treatment efficacy and overall prognosis vary between geographic populations. Studies disentangling the contributing factors may help in both understanding cancer biology and tailoring therapeutic interventions. Ancestry estimation in such studies should preferably be driven by genomic data, due to frequently missing or erroneous self-reported or inferred metadata. While respective algorithms have been demonstrated for baseline genomes, such a strategy has not been shown for cancer genomes carrying a substantial somatic mutation load. We have developed a bioinformatics tool for the assignment of population groups from genome profiling data for both unaltered and cancer genomes.
<a href="2020-03-16-snp2pop-scientificreports/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      January 11, 2020
      
    </span>
</aside>

<h3 class="title_news">Geographic assessment of cancer genome profiling studies</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="paula-carrio-cordo-elise-acheson-qingyao-huang-and-michael-baudis-2020">Paula Carrio Cordo, Elise Acheson, Qingyao Huang and Michael Baudis (2020)<a class="headerlink" href="#paula-carrio-cordo-elise-acheson-qingyao-huang-and-michael-baudis-2020" title="Permanent link">&para;</a></h4>
<h5 id="biorxiv-2020-11-01-doi-101101827683">bioRxiv, 2020-11-01. <a href="https://doi.org/10.1101/827683">DOI 10.1101/827683</a><a class="headerlink" href="#biorxiv-2020-11-01-doi-101101827683" title="Permanent link">&para;</a></h5>
<p><strong>Abstract</strong> Cancers arise from the accumulation of somatic genome mutations, which can be influenced by inherited genomic variants and external factors such as environmental or lifestyle-related exposure. Due to the heterogeneity of cancers, precise information about the genomic composition of germline and malignant tissues has to be correlated with morphological, clinical and extrinsic features to advance medical knowledge and treatment options. With global differences in cancer frequencies and disease types, geographic data is of importance to understand the interplay between genetic ancestry and environmental influence in cancer incidence, progression and treatment outcome.
<a href="2020-01-11-publication-geography-assesment-biorxiv/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      August 27, 2019
      
    </span>
</aside>

<h3 class="title_news">Leveraging European infrastructures to access 1 million human genomes by 2022</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="gary-saunders-michael-baudis-regina-becker-sergi-beltran-christophe-beroud-ewan-birney-cath-brooksbank-sren-brunak-marc-van-den-bulcke-rachel-drysdale-salvador-capella-gutierrez-paul-flicek-niklas-blomberg-and-serena-scollen">Gary Saunders, Michael Baudis, Regina Becker, Sergi Beltran, Christophe Béroud, Ewan Birney, Cath Brooksbank, Søren Brunak, Marc Van den Bulcke, Rachel Drysdale, Salvador Capella-Gutierrez, Paul Flicek, ..., Niklas Blomberg, and Serena Scollen<a class="headerlink" href="#gary-saunders-michael-baudis-regina-becker-sergi-beltran-christophe-beroud-ewan-birney-cath-brooksbank-sren-brunak-marc-van-den-bulcke-rachel-drysdale-salvador-capella-gutierrez-paul-flicek-niklas-blomberg-and-serena-scollen" title="Permanent link">&para;</a></h4>
<h5 id="nature-reviews-genetics-volume-20-pages693701-2019">Nature Reviews Genetics volume 20, pages693–701 (2019)<a class="headerlink" href="#nature-reviews-genetics-volume-20-pages693701-2019" title="Permanent link">&para;</a></h5>
<p><strong>Abstract</strong> Human genomics is undergoing a step change from being a predominantly research-driven activity to one driven through health care as many countries in Europe now have nascent precision medicine programmes. To maximize the value of the genomic data generated, these data will need to be shared between institutions and across countries. In recognition of this challenge, 21 European countries recently signed a declaration to transnationally share data on at least 1 million human genomes by 2022. In this Roadmap, we identify the challenges of data sharing across borders and demonstrate that European research infrastructures are well-positioned to support the rapid implementation of widespread genomic data access.</p>
<a href="2019-08-27-publication-1million-genomes/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      July 31, 2019
      
    </span>
</aside>

<h3 class="title_news">Minimum Error Calibration and Normalization for Genomic Copy Number Analysis</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="bo-gao-and-michael-baudis-2019">Bo Gao and Michael Baudis (2019)<a class="headerlink" href="#bo-gao-and-michael-baudis-2019" title="Permanent link">&para;</a></h4>
<h5 id="biorxiv-2019-07-31-doi-101101720854">bioRxiv, 2019-07-31. <a href="https://doi.org/10.1101/720854">DOI 10.1101/720854</a><a class="headerlink" href="#biorxiv-2019-07-31-doi-101101720854" title="Permanent link">&para;</a></h5>
<p><strong>Abstract</strong> Copy number variations (CNV) are regional deviations from the normal autosomal bi-allelic DNA content. While germline CNVs are a major contributor to genomic syndromes and inherited diseases, the majority of cancers accumulate extensive “somatic” CNV (sCNV or CNA) during the process of oncogenetic transformation and progression. While specific sCNV have closely been associated with tumorigenesis, intriguingly many neoplasias exhibit recurrent sCNV patterns beyond the involvement of a few cancer driver genes.
<a href="2019-07-31-publication-Mecan4CNA-biorxiv/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      January 23, 2019
      
    </span>
</aside>

<h3 class="title_news">Federated discovery and sharing of genomic data using Beacons</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="miroslav-cupak-stephen-keenan-jordi-rambla-sabela-de-la-torre-stephanie-dyke-anthony-brookes-knox-carey-david-lloyd-peter-goodhand-maximilian-haeussler-michael-baudis-heinz-stockinger-lena-dolman-ilkka-lappalainen-juha-tornroos-mikael-linden-john-spalding-saif-ur-rehman-angela-page-paul-flicek-susheel-varma-gary-saunders-serena-scollen-stephen-sherry-david-haussler-beacon-project-team">Miroslav Cupak , Stephen Keenan , Jordi Rambla , Sabela de la Torre , Stephanie Dyke , Anthony Brookes , Knox Carey , David Lloyd , Peter Goodhand , Maximilian Haeussler , Michael Baudis , Heinz Stockinger , Lena Dolman , Ilkka Lappalainen , Juha Törnroos , Mikael Linden , John Spalding , Saif Ur-Rehman , Angela Page , Paul Flicek , Susheel Varma , Gary Saunders , Serena Scollen , Stephen Sherry , David Haussler , Beacon Project Team<a class="headerlink" href="#miroslav-cupak-stephen-keenan-jordi-rambla-sabela-de-la-torre-stephanie-dyke-anthony-brookes-knox-carey-david-lloyd-peter-goodhand-maximilian-haeussler-michael-baudis-heinz-stockinger-lena-dolman-ilkka-lappalainen-juha-tornroos-mikael-linden-john-spalding-saif-ur-rehman-angela-page-paul-flicek-susheel-varma-gary-saunders-serena-scollen-stephen-sherry-david-haussler-beacon-project-team" title="Permanent link">&para;</a></h4>
<h5 id="nat-biotechnol-2019-accepted-2019-01-23">Nat Biotechnol (2019), accepted 2019-01-23<a class="headerlink" href="#nat-biotechnol-2019-accepted-2019-01-23" title="Permanent link">&para;</a></h5>
<p><em>Abstract</em> The Beacon Project (github.com/ga4gh-beacon/) is a GA4GH initiative that is developing an open specification for genetic variation discovery and sharing. The project is demonstrating the willingness of international organizations to work together to define standards for, and actively engage in, genomic data sharing. In the two years since the project’s inception, over 90 Beacons have been lit by 35 organizations serving over 200 datasets.
<a href="2019-01-23-publication-beacon/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      January 19, 2019
      
    </span>
</aside>

<h3 class="title_news">DNA copy number imbalances in primary cutaneous lymphomas (PCL)</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="gug-g-huang-q-chiticariu-e-solovan-c-and-baudis-m-2019">Gug G, Huang Q, Chiticariu E, Solovan C and Baudis M  (2019)<a class="headerlink" href="#gug-g-huang-q-chiticariu-e-solovan-c-and-baudis-m-2019" title="Permanent link">&para;</a></h4>
<h5 id="jeadv-2019-01-19-doiorg101111jdv15442">JEADV, 2019-01-19. <a href="https://doi.org/10.1111/jdv.15442">doi.org/10.1111/jdv.15442</a><a class="headerlink" href="#jeadv-2019-01-19-doiorg101111jdv15442" title="Permanent link">&para;</a></h5>
<p>The article has been published with the <em>Journal of the European Academy of Dermatology and Venereology</em> on January 19, 2019. A corresponding preprint can be accessed through <a href="https://www.biorxiv.org/content/early/2018/09/14/417766" target="_blank">[bioRxiv]</a>.</p>
<p><strong>Background</strong></p>
<p>Cutaneous lymphomas (CL) represent a clinically defined group of extran‐ odal non‐Hodgkin lymphomas harbouring heterogeneous and incompletely delineated molecular aberrations. Over the past decades, molecular stud‐ ies have identified several chromosomal aberrations, but the interpreta‐ tion of individual genomic studies can be challenging.</p>
<p><strong>Objective</strong></p>
<p>With a comprehensive meta‐analysis, we aim to delineate genomic alter‐ ations for different types of CL and propose a more accurate classifica‐ tion in line with their various pathogenicity.
<a href="2019-01-19-publication-PCL-JEADV/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      January 14, 2019
      
    </span>
</aside>

<h3 class="title_news">Enabling population assignment from cancer genomes with SNP2pop</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="huang-q-and-baudis-m-2019">Huang Q and Baudis M. (2019)<a class="headerlink" href="#huang-q-and-baudis-m-2019" title="Permanent link">&para;</a></h4>
<h5 id="biorxiv-2019-01-14-doiorg101101368647-first-version-2018-07-14">bioRxiv, 2019-01-14. <a href="https://doi.org/10.1101/368647">doi.org/10.1101/368647</a> (first version 2018-07-14)<a class="headerlink" href="#biorxiv-2019-01-14-doiorg101101368647-first-version-2018-07-14" title="Permanent link">&para;</a></h5>
<p><strong>Abstract</strong> For a variety of human malignancies, incidence, treatment efficacy and overall prognosis show considerable variation between different populations and ethnic groups. Disentangling the effects related to particular population backgrounds can help in both understanding cancer biology and in tailoring therapeutic interventions. Because self-reported or inferred patient data can be incomplete or misleading due to migration and genomic admixture, a data-driven ancestry estimation should be preferred. While algorithms to analyze ancestry structure from healthy individuals have been developed, an easy-to-use tool to assign population groups based on genotyping data from SNP profiles is still missing and benchmarking for the validity of population assignment strategy for aberrant cancer genomes was not tested.
<a href="2019-01-14-publication-popmap-biorxiv-v2/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      August 02, 2018
      
    </span>
</aside>

<h3 class="title_news">Registered access: authorizing data access</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="dyke-som-linden-m-lappalainen-i-de-argila-jr-carey-k-lloyd-d-spalding-jd-cabili-mn-kerry-g-foreman-j-cutts-t-shabani-m-rodriguez-ll-haeussler-m-walsh-b-jiang-x-wang-s-perrett-d-boughtwood-t-rehm-hl-baudis-m-sherry-st-kato-k-knoppers-bm-baker-d-and-flicek-p">Dyke SOM, Linden M, Lappalainen I, De Argila JR, Carey K, Lloyd D, Spalding JD, Cabili MN, Kerry G, Foreman J, Cutts T, Shabani M, Rodriguez LL, Haeussler M, Walsh B, Jiang X, Wang S, Perrett D, Boughtwood T, ..., Rehm HL, Baudis M, Sherry ST, Kato K, Knoppers BM, Baker D, and Flicek P<a class="headerlink" href="#dyke-som-linden-m-lappalainen-i-de-argila-jr-carey-k-lloyd-d-spalding-jd-cabili-mn-kerry-g-foreman-j-cutts-t-shabani-m-rodriguez-ll-haeussler-m-walsh-b-jiang-x-wang-s-perrett-d-boughtwood-t-rehm-hl-baudis-m-sherry-st-kato-k-knoppers-bm-baker-d-and-flicek-p" title="Permanent link">&para;</a></h4>
<h5 id="european-journal-of-human-genetics-2018">European Journal of Human Genetics (2018)<a class="headerlink" href="#european-journal-of-human-genetics-2018" title="Permanent link">&para;</a></h5>
<p><em>Abstract</em> The Global Alliance for Genomics and Health (GA4GH) proposes a data access policy model—“registered access”—to increase and improve access to data requiring an agreement to basic terms and conditions, such as the use of DNA sequence and health data in research. A registered access policy would enable a range of categories of users to gain access, starting with researchers and clinical care professionals. It would also facilitate general use and reuse of data but within the bounds of consent restrictions and other ethical obligations. In piloting registered access with the Scientific Demonstration data sharing projects of GA4GH, we provide additional ethics, policy and technical guidance to facilitate the implementation of this access model in an international setting.</p>
<a href="2018-08-02-publication-registered-access/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      July 31, 2018
      
    </span>
</aside>

<h3 class="title_news">Mountains and Chasms - Surveying the Oncogenomic Publication Landscape</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="carrio-cordo-p-and-baudis-m-2018">Carrio Cordo P and Baudis M. (2018)<a class="headerlink" href="#carrio-cordo-p-and-baudis-m-2018" title="Permanent link">&para;</a></h4>
<h5 id="preprints-2018-2018070618-doi-1020944preprints2018070618v1">Preprints 2018, 2018070618 (doi: 10.20944/preprints201807.0618.v1).<a class="headerlink" href="#preprints-2018-2018070618-doi-1020944preprints2018070618v1" title="Permanent link">&para;</a></h5>
<h5 id="oncology-2018-online-oct-26"><em>Oncology</em> (2018; online Oct 26)<a class="headerlink" href="#oncology-2018-online-oct-26" title="Permanent link">&para;</a></h5>
<p><strong>Abstract</strong> Cancers arise from the accumulation of somatic genome mutations, with varying contributions of intrinsic (i.e. genetic predisposition) and extrinsic (i.e. environmental) factors. For the understanding of malignant clones, precise information about their genomic composition has to be correlated with morphological, clinical and individual features, in the context of the available medical knowledge.
<a href="2018-07-31-publication-landscape-preprint.org/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      July 14, 2018
      
    </span>
</aside>

<h3 class="title_news">Population assignment from cancer genome profiling data</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="huang-q-and-baudis-m-2018">Huang Q and Baudis M. (2018)<a class="headerlink" href="#huang-q-and-baudis-m-2018" title="Permanent link">&para;</a></h4>
<h5 id="biorxiv-2018-07-14-doi101101368647">bioRxiv, 2018-07-14. doi:10.1101/368647<a class="headerlink" href="#biorxiv-2018-07-14-doi101101368647" title="Permanent link">&para;</a></h5>
<p><strong>Abstract</strong> For a variety of human malignancies, incidence, treatment efficacy and overall prognosis show considerable variation between different populations and ethnic groups. Disentangling the effects related to particular population backgrounds can help in both understanding cancer biology and in tailoring therapeutic interventions. Because self-reported or inferred patient data can be incomplete or misleading due to migration and genomic admixture, a data-driven ancestry estimation should be preferred. While tools to map and utilize ancestry information from healthy individuals have been introduced, a population assignment based on genotyping data from somatic variation profiling of cancer samples is still missing.
<a href="2018-07-15-publication-popmap-biorxiv/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      July 12, 2018
      
    </span>
</aside>

<h3 class="title_news">A harmonized meta-knowledgebase of clinical interpretations of cancer genomic variants</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="wagner-ah-walsh-b-mayfield-g-tamborero-d-sonkin-d-krysiak-k-deu-pons-j-duren-r-gao-j-mcmurry-j-patterson-s-del-vecchio-fitz-c-sezerman-ou-warner-j-rieke-dt-aittokallio-t-cerami-e-ritter-d-schriml-lm-haendel-m-raca-g-madhavan-s-baudis-m-griffith-m-griffith-ol-and-margolin-a">Wagner AH, Walsh B, Mayfield G, Tamborero D, Sonkin D, Krysiak K, Deu Pons J, Duren R, Gao J, McMurry J, Patterson S, Del Vecchio Fitz C, Sezerman OU, Warner J, Rieke DT, Aittokallio T, Cerami E, Ritter D, Schriml LM, Haendel M, Raca G, Madhavan S, Baudis M, ..., Griffith M, Griffith OL, and Margolin A<a class="headerlink" href="#wagner-ah-walsh-b-mayfield-g-tamborero-d-sonkin-d-krysiak-k-deu-pons-j-duren-r-gao-j-mcmurry-j-patterson-s-del-vecchio-fitz-c-sezerman-ou-warner-j-rieke-dt-aittokallio-t-cerami-e-ritter-d-schriml-lm-haendel-m-raca-g-madhavan-s-baudis-m-griffith-m-griffith-ol-and-margolin-a" title="Permanent link">&para;</a></h4>
<h5 id="biorxiv-doi101101366856">bioRxiv. doi:10.1101/366856<a class="headerlink" href="#biorxiv-doi101101366856" title="Permanent link">&para;</a></h5>
<p>Precision oncology relies on the accurate discovery and interpretation of genomic variants to enable individualized therapy selection, diagnosis, or prognosis. However, knowledgebases containing clinical interpretations of somatic cancer variants are highly disparate in interpretation content, structure, and supporting primary literature, reducing consistency and impeding consensus when evaluating variants and their relevance in a clinical settin
<a href="2018-07-12-publication-vicc-biorxiv/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      May 24, 2018
      
    </span>
</aside>

<h3 class="title_news">Krüppel-Like Factor 10 participates in cervical cancer immunoediting through transcriptional regulation of Pregnancy-Specific Beta-1 Glycoproteins</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="marrero-rodriguez-d-taniguchi-ponciano-k-subramaniam-m-hawse-jr-pitel-ks-arreola-de-la-cruz-h-huerta-padilla-v-ponce-navarrete-g-figueroa-corona-mdp-gomez-virgilio-l-martinez-cuevas-ti-mendoza-rodriguez-m-rodriguez-esquivel-m-romero-morelos-p-ramirez-salcedo-j-baudis-m-meraz-rios-m-jimenez-vega-f-salcedo-m">Marrero-Rodríguez D, Taniguchi-Ponciano K, Subramaniam M, Hawse JR, Pitel KS, Arreola-De la Cruz H, Huerta-Padilla V, Ponce-Navarrete G, Figueroa-Corona MDP, Gomez-Virgilio L, Martinez-Cuevas TI, Mendoza-Rodriguez M, Rodriguez-Esquivel M, Romero-Morelos P, Ramirez-Salcedo J, Baudis M, Meraz-Rios M, Jimenez-Vega F, Salcedo M.<a class="headerlink" href="#marrero-rodriguez-d-taniguchi-ponciano-k-subramaniam-m-hawse-jr-pitel-ks-arreola-de-la-cruz-h-huerta-padilla-v-ponce-navarrete-g-figueroa-corona-mdp-gomez-virgilio-l-martinez-cuevas-ti-mendoza-rodriguez-m-rodriguez-esquivel-m-romero-morelos-p-ramirez-salcedo-j-baudis-m-meraz-rios-m-jimenez-vega-f-salcedo-m" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> Cervical cancer (CC) is associated with alterations in immune system balance, which is primarily due to
a shift from Th1 to Th2 and the unbalance of Th17/Treg cells. Using in silico DNA copy number analysis, we have demonstrated that ~20% of CC samples exhibit gain of 8q22.3 and 19q13.31; the regions of the genome that encodes the KLF10 and PSG genes, respectively.
<a href="2018-05-24-publications-mbaudis-KLF10/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      March 14, 2018
      
    </span>
</aside>

<h3 class="title_news">segment_liftover...</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h3 id="segment_liftover-a-python-tool-to-convert-segments-between-genome-assemblies">segment_liftover : a Python tool to convert segments between genome assemblies.<a class="headerlink" href="#segment_liftover-a-python-tool-to-convert-segments-between-genome-assemblies" title="Permanent link">&para;</a></h3>
<h4 id="gao-b-huang-q-and-baudis-m">Gao B, Huang Q and Baudis M<a class="headerlink" href="#gao-b-huang-q-and-baudis-m" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> The process of assembling a species’ reference genome may be performed in a number of iterations, with subsequent genome assemblies differing in the coordinates of mapped elements. The conversion of genome coordinates between different assemblies is required for many integrative and comparative studies. While currently a number of bioinformatics tools are available to accomplish this task, most of them are tailored towards the conversion of single genome coordinates. When converting the boundary positions of segments spanning larger genome regions, segments may be mapped into smaller sub-segments if the original segment’s continuity is disrupted in the target assembly. Such a conversion may lead to a relevant degree of data loss in some circumstances such as copy number variation (CNV) analysis, where the quantitative representation of a genomic region takes precedence over base-specific accuracy. segment_liftover aims at continuity-preserving remapping of genome segments between assemblies and provides features such as approximate locus conversion, automated batch processing and comprehensive logging to facilitate processing of datasets containing large numbers of structural genome variation data.</p>
<h5 id="online">Online<a class="headerlink" href="#online" title="Permanent link">&para;</a></h5>
<ul>
<li><a href="https://f1000research.com/articles/7-319/v2"><em>F1000</em></a></li>
<li><a href="https://www.biorxiv.org/content/early/2018/03/01/274084">bioRxiv</a></li>
</ul>
<a href="2018-03-14-segment_liftover/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      October 01, 2017
      
    </span>
</aside>

<h3 class="title_news">Integrated Molecular...</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h3 id="integrated-molecular-meta-analysis-of-1000-pediatric-high-grade-and-diffuse-intrinsic-pontine-glioma">Integrated Molecular Meta-Analysis of 1,000 Pediatric High-Grade and Diffuse Intrinsic Pontine Glioma.<a class="headerlink" href="#integrated-molecular-meta-analysis-of-1000-pediatric-high-grade-and-diffuse-intrinsic-pontine-glioma" title="Permanent link">&para;</a></h3>
<h4 id="mackay-a-burford-a-carvalho-d-izquierdo-e-fazal-salom-j-taylor-kr-bjerke-l-clarke-m-vinci-m-nandhabalan-m-temelso-s-popov-s-molinari-v-raman-p-waanders-aj-han-hj-gupta-s-marshall-l-zacharoulis-s-vaidya-s-mandeville-hc-bridges-lr-martin-aj-al-sarraj-s-chandler-c-ng-hk-li-x-mu-k-trabelsi-s-brahim-dh-kisljakov-an-konovalov-dm-moore-as-carcaboso-am-sunol-m-de-torres-c-cruz-o-mora-j-shats-li-stavale-jn-bidinotto-lt-reis-rm-entz-werle-n-farrell-m-cryan-j-crimmins-d-caird-j-pears-j-monje-m-debily-ma-castel-d-grill-j-hawkins-c-nikbakht-h-jabado-n-baker-sj-pfister-sm-jones-dtw-fouladi-m-von-bueren-ao-baudis-m-resnick-a-jones-c">Mackay A, Burford A, Carvalho D, Izquierdo E, Fazal-Salom J, Taylor KR, Bjerke L, Clarke M, Vinci M, Nandhabalan M, Temelso S, Popov S, Molinari V, Raman P, Waanders AJ, Han HJ, Gupta S, Marshall L, Zacharoulis S, Vaidya S, Mandeville HC, Bridges LR, Martin AJ, Al-Sarraj S, Chandler C, Ng HK, Li X, Mu K, Trabelsi S, Brahim DH, Kisljakov AN, Konovalov DM, Moore AS, Carcaboso AM, Sunol M, de Torres C, Cruz O, Mora J, Shats LI, Stavale JN, Bidinotto LT, Reis RM, Entz-Werle N, Farrell M, Cryan J, Crimmins D, Caird J, Pears J, Monje M, Debily MA, Castel D, Grill J, Hawkins C, Nikbakht H, Jabado N, Baker SJ, Pfister SM, Jones DTW, Fouladi M, von Bueren AO, Baudis M, Resnick A, Jones C.<a class="headerlink" href="#mackay-a-burford-a-carvalho-d-izquierdo-e-fazal-salom-j-taylor-kr-bjerke-l-clarke-m-vinci-m-nandhabalan-m-temelso-s-popov-s-molinari-v-raman-p-waanders-aj-han-hj-gupta-s-marshall-l-zacharoulis-s-vaidya-s-mandeville-hc-bridges-lr-martin-aj-al-sarraj-s-chandler-c-ng-hk-li-x-mu-k-trabelsi-s-brahim-dh-kisljakov-an-konovalov-dm-moore-as-carcaboso-am-sunol-m-de-torres-c-cruz-o-mora-j-shats-li-stavale-jn-bidinotto-lt-reis-rm-entz-werle-n-farrell-m-cryan-j-crimmins-d-caird-j-pears-j-monje-m-debily-ma-castel-d-grill-j-hawkins-c-nikbakht-h-jabado-n-baker-sj-pfister-sm-jones-dtw-fouladi-m-von-bueren-ao-baudis-m-resnick-a-jones-c" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> We collated data from 157 unpublished cases of pediatric high-grade glioma and diffuse intrinsic pontine glioma and 20 publicly available datasets in an integrated analysis of &gt;1,000 cases. We identified co-segregating mutations in histone-mutant subgroups including loss of FBXW7 in H3.3G34R/V, TOP3A rearrangements in H3.3K27M, and BCOR mutations in H3.1K27M. Histone wild-type subgroups are refined by the presence of key oncogenic events or methylation profiles more closely resembling lower-grade tumors. Genomic aberrations increase with age, highlighting the infant population as biologically and clinically distinct. Uncommon pathway dysregulation is seen in small subsets of tumors, further defining the molecular diversity of the disease, opening up avenues for biological study and providing a basis for functionally defined future treatment stratification.</p>
<a href="2017-10-01-Integrated-Molecular/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      October 01, 2016
      
    </span>
</aside>

<h3 class="title_news">CNARA: reliability assessment for genomic copy number profiles</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="ai-n-cai-h-solovan-c-baudis-m">Ai N, Cai H, Solovan C, Baudis M.<a class="headerlink" href="#ai-n-cai-h-solovan-c-baudis-m" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> DNA copy number profiles from microarray and sequencing experiments sometimes contain wave artefacts which may be introduced during sample preparation and cannot be removed completely by existing preprocessing methods. Besides, large derivative log ratio spread (DLRS) of the probes correlating with poor DNA quality is sometimes observed in genome screening experiments and may lead to unreliable copy number profiles. Depending on the extent of these artefacts and the resulting misidentification of copy number alterations/variations (CNA/CNV), it may be desirable to exclude such samples from analyses or to adapt the downstream data analysis strategy accordingly.Here, we propose a method to distinguish reliable genomic copy number profiles from those containing heavy wave artefacts and/or large DLRS. We define four features that adequately summarize the copy number profiles for reliability assessment, and train a classifier on a dataset of 1522 copy number profiles from various microarray platforms. The method can be applied to predict the reliability of copy number profiles irrespective of the underlying microarray platform and may be adapted for those sequencing platforms from which copy number estimates could be computed as a piecewise constant signal. Further details can be found at <a href="https://github.com/baudisgroup/CNARA">github.com/baudisgroup/CNARA</a> .We have developed a method for the assessment of genomic copy number profiling data, and suggest to apply the method in addition to and after other state-of-the-art noise correction and quality control procedures. CNARA could be instrumental in improving the assessment of data used for genomic data mining experiments and support the reliable functional attribution of copy number aberrations especially in cancer research.</p>
<a href="2016-10-01-CNARA-reliability/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      September 01, 2016
      
    </span>
</aside>

<h3 class="title_news">PKCα and HMGB1 antagonistically control hydrogen peroxide-induced poly-ADP-ribose formation</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h3 id="pkc-and-hmgb1-antagonistically-control-hydrogen-peroxide-induced-poly-adp-ribose-formation">PKCα and HMGB1 antagonistically control hydrogen peroxide-induced poly-ADP-ribose formation.<a class="headerlink" href="#pkc-and-hmgb1-antagonistically-control-hydrogen-peroxide-induced-poly-adp-ribose-formation" title="Permanent link">&para;</a></h3>
<h4 id="andersson-a-bluwstein-a-kumar-n-teloni-f-traenkle-j-baudis-m-altmeyer-m-hottiger-mo">Andersson A, Bluwstein A, Kumar N, Teloni F, Traenkle J, Baudis M, Altmeyer M, Hottiger MO.<a class="headerlink" href="#andersson-a-bluwstein-a-kumar-n-teloni-f-traenkle-j-baudis-m-altmeyer-m-hottiger-mo" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> Harmful oxidation of proteins, lipids and nucleic acids is observed when reactive oxygen species (ROS) are produced excessively and/or the antioxidant capacity is reduced, causing 'oxidative stress'. Nuclear poly-ADP-ribose (PAR) formation is thought to be induced in response to oxidative DNA damage and to promote cell death under sustained oxidative stress conditions. However, what exactly triggers PAR induction in response to oxidative stress is incompletely understood. Using reverse phase protein array (RPPA) and in-depth analysis of key stress signaling components, we observed that PAR formation induced by H2O2 was mediated by the PLC/IP3R/Ca(2+)/PKCα signaling axis. Mechanistically, H2O2-induced PAR formation correlated with Ca(2+)-dependent DNA damage, which, however, was PKCα-independent. In contrast, PAR formation was completely lost upon knockdown of PKCα, suggesting that DNA damage alone was not sufficient for inducing PAR formation, but required a PKCα-dependent process. Intriguingly, the loss of PAR formation observed upon PKCα depletion was overcome when the chromatin structure-modifying protein HMGB1 was co-depleted with PKCα, suggesting that activation and nuclear translocation of PKCα releases the inhibitory effect of HMGB1 on PAR formation. Together, these results identify PKCα and HMGB1 as important co-regulators involved in H2O2-induced PAR formation, a finding that may have important relevance for oxidative stress-associated pathophysiological conditions.</p>
<h5 id="links">Links<a class="headerlink" href="#links" title="Permanent link">&para;</a></h5>
<ul>
<li><a href="http://europepmc.org/abstract/MED/27198223"><em>EPMC</em></a></li>
<li><a href="http://internal.baudisgroup.org/public/pdf/27198223.pdf"><em>PDF</em></a></li>
</ul>
<a href="2016-09-01-PKC%CE%B1-and-HMGB1/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      January 01, 2016
      
    </span>
</aside>

<h3 class="title_news">The SIB Swiss Institute of Bioinformatics' resources: focus on curated databases</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="sib-swiss-institute-of-bioinformatics-members">SIB Swiss Institute of Bioinformatics Members.<a class="headerlink" href="#sib-swiss-institute-of-bioinformatics-members" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> The SIB Swiss Institute of Bioinformatics (<a href="http://www.isb-sib.ch">www.isb-sib.ch</a>) provides world-class bioinformatics databases, software tools, services and training to the international life science community in academia and industry. These solutions allow life scientists to turn the exponentially growing amount of data into knowledge. Here, we provide an overview of SIB's resources and competence areas, with a strong focus on curated databases and SIB's most popular and widely used resources. In particular, SIB's Bioinformatics resource portal ExPASy features over 150 resources, including UniProtKB/Swiss-Prot, ENZYME, PROSITE, neXtProt, STRING, UniCarbKB, SugarBindDB, SwissRegulon, EPD, arrayMap, Bgee, SWISS-MODEL Repository, OMA, OrthoDB and other databases, which are briefly described in this article.</p>
<a href="2016-01-01-The-SIB-Swiss/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      September 01, 2015
      
    </span>
</aside>

<h3 class="title_news">A Biobank Supporting Rare Disease Research In Dermatopathology. Our Experience In Establishing A Biobank.</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h3 id="beleut-m-seclaman-e-baudis-m-nicula-a-and-solovan-c">Beleut M, Seclaman E, Baudis M, Nicula A, and Solovan C<a class="headerlink" href="#beleut-m-seclaman-e-baudis-m-nicula-a-and-solovan-c" title="Permanent link">&para;</a></h3>
<h4 id="rojced-2-202-206-2015">RoJCED 2, 202-206 (2015)<a class="headerlink" href="#rojced-2-202-206-2015" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> Biobanks of human patient sample tissues and blood fractions are increasingly recognized as major assets in disease research. We aim to identify DNA copy number and gene expression aberrations typical of different cutaneous pathologies. Another goal is the identification of circulating biomarkers both as prognostic, therapy- responsive and/or therapy-monitoring factors and as disease classifiers and subclassifiers.</p>

<a href="2008-2015/2015-09-01-beleut-biobank/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      January 01, 2015
      
    </span>
</aside>

<h3 class="title_news">Genomic instability of osteosarcoma cell lines in culture: impact on the prediction of metastasis relevant genes</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="muff-r-rath-p-ram-kumar-rm-husmann-k-born-w-baudis-m-fuchs-b">Muff R, Rath P, Ram Kumar RM, Husmann K, Born W, Baudis M, Fuchs B.<a class="headerlink" href="#muff-r-rath-p-ram-kumar-rm-husmann-k-born-w-baudis-m-fuchs-b" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> Osteosarcoma is a rare but highly malignant cancer of the bone. As a consequence, the number of established cell lines used for experimental in vitro and in vivo osteosarcoma research is limited and the value of these cell lines relies on their stability during culture. Here we investigated the stability in gene expression by microarray analysis and array genomic hybridization of three low metastatic cell lines and derivatives thereof with increased metastatic potential using cells of different passages.The osteosarcoma cell lines showed altered gene expression during in vitro culture, and it was more pronounced in two metastatic cell lines compared to the respective parental cells. Chromosomal instability contributed in part to the altered gene expression in SAOS and LM5 cells with low and high metastatic potential. To identify metastasis-relevant genes in a background of passage-dependent altered gene expression, genes involved in "Pathways in cancer" that were consistently regulated under all passage comparisons were evaluated. Genes belonging to "Hedgehog signaling pathway" and "Wnt signaling pathway" were significantly up-regulated, and IHH, WNT10B and TCF7 were found up-regulated in all three metastatic compared to the parental cell lines.Considerable instability during culture in terms of gene expression and chromosomal aberrations was observed in osteosarcoma cell lines. The use of cells from different passages and a search for genes consistently regulated in early and late passages allows the analysis of metastasis-relevant genes despite the observed instability in gene expression in osteosarcoma cell lines during culture.</p>
<a href="2008-2015/2015-01-01-Genomic-instability/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      January 01, 2015
      
    </span>
</aside>

<h3 class="title_news">arrayMap 2014: an updated cancer genome resource</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="cai-h-gupta-s-rath-p-ai-n-baudis-m">Cai H, Gupta S, Rath P, Ai N, Baudis M.<a class="headerlink" href="#cai-h-gupta-s-rath-p-ai-n-baudis-m" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> Somatic copy number aberrations (CNA) represent a mutation type encountered in the majority of cancer genomes. Here, we present the 2014 edition of arrayMap (<a href="http://www.arraymap.org">www.arraymap.org</a>), a publicly accessible collection of pre-processed oncogenomic array data sets and CNA profiles, representing a vast range of human malignancies. Since the initial release, we have enhanced this resource both in content and especially with regard to data mining support. The 2014 release of arrayMap contains more than 64,000 genomic array data sets, representing about 250 tumor diagnoses.
<a href="2008-2015/2015-01-01-arrayMap-2014-an/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      January 01, 2014
      
    </span>
</aside>

<h3 class="title_news">Biopsying parapsoriasis: quo vadis? Are morphological stains enough or are ancillary tests needed?</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="baderca-f-chiticariu-e-baudis-m-solovan-c">Baderca F, Chiticariu E, Baudis M, Solovan C.<a class="headerlink" href="#baderca-f-chiticariu-e-baudis-m-solovan-c" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> BACKGROUND: Parapsoriasis represents a group of cutaneous disorders that shows variable clinical aspects somehow resembling to psoriasis, how is reflecting by its name. It was first named by Brocq, in 1902, as an entity with three components: pityriasis lichenoides, small plaque parapsoriasis and large plaque parapsoriasis. Nowadays, under the name of parapsoriasis are included only the last two categories, that are considered disorders characterized by the presence of a mononuclear infiltrate in the dermis, composed of T-cells. Until now, there were not established pathognomonic histopathological features to diagnose parapsoriasis.
<a href="2008-2015/2014-01-01-Biopsying-parapsoria/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      January 01, 2014
      
    </span>
</aside>

<h3 class="title_news">Chromothripsis-like patterns are recurring but heterogeneously distributed features in a survey of 22,347 cancer genome screens</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="cai-h-kumar-n-bagheri-hc-von-mering-c-robinson-md-baudis-m">Cai H, Kumar N, Bagheri HC, von Mering C, Robinson MD, Baudis M.<a class="headerlink" href="#cai-h-kumar-n-bagheri-hc-von-mering-c-robinson-md-baudis-m" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> BACKGROUND: Chromothripsis is a recently discovered phenomenon of genomic rearrangement, possibly arising during a single genome-shattering event. This could provide an alternative paradigm in cancer development, replacing the gradual accumulation of genomic changes with a "one-off" catastrophic event. However, the term has been used with varying operational definitions, with the minimal consensus being a large number of locally clustered copy number aberrations. The mechanisms underlying these chromothripsis-like patterns (CTLP) and their specific impact on tumorigenesis are still poorly understood.
<a href="2008-2015/2014-01-01-Chromothripsis-like/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      January 01, 2014
      
    </span>
</aside>

<h3 class="title_news">Progenetix: 12 years of oncogenomic data curation</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="cai-h-kumar-n-ai-n-gupta-s-rath-p-baudis-m">Cai H, Kumar N, Ai N, Gupta S, Rath P, Baudis M.<a class="headerlink" href="#cai-h-kumar-n-ai-n-gupta-s-rath-p-baudis-m" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> DNA copy number aberrations (CNAs) can be found in the majority of cancer genomes and are crucial for understanding the potential mechanisms underlying tumor initiation and progression. Since the first release in 2001, the Progenetix project (<a href="http://www.progenetix.org">www.progenetix.org</a>) has provided a reference resource dedicated to provide the most comprehensive collection of genome-wide CNA profiles. Reflecting the application of comparative genomic hybridization techniques to tens of thousands of cancer genomes, over the past 12 years our data curation efforts have resulted in a more than 60-fold increase in the number of cancer samples presented through Progenetix.
<a href="2008-2015/2014-01-01-Progenetix-12-years/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      December 01, 2013
      
    </span>
</aside>

<h3 class="title_news">SIL1 mutations and clinical spectrum in patients with Marinesco-Sjogren syndrome</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="krieger-m-roos-a-stendel-c-claeys-kg-sonmez-fm-baudis-m-bauer-p-bornemann-a-de-goede-c-dufke-a-finkel-rs-goebel-hh-haussler-m-kingston-h-kirschner-j-medne-l-muschke-p-rivier-f-rudnik-schoneborn-s-spengler-s-inzana-f-stanzial-f-benedicenti-f-synofzik-m-lia-taratuto-a-pirra-l-tay-sk-topaloglu-h-uyanik-g-wand-d-williams-d-zerres-k-weis-j-senderek-j">Krieger M, Roos A, Stendel C, Claeys KG, Sonmez FM, Baudis M, Bauer P, Bornemann A, de Goede C, Dufke A, Finkel RS, Goebel HH, Häussler M, Kingston H, Kirschner J, Medne L, Muschke P, Rivier F, Rudnik-Schöneborn S, Spengler S, Inzana F, Stanzial F, Benedicenti F, Synofzik M, Lia Taratuto A, Pirra L, Tay SK, Topaloglu H, Uyanik G, Wand D, Williams D, Zerres K, Weis J, Senderek J.<a class="headerlink" href="#krieger-m-roos-a-stendel-c-claeys-kg-sonmez-fm-baudis-m-bauer-p-bornemann-a-de-goede-c-dufke-a-finkel-rs-goebel-hh-haussler-m-kingston-h-kirschner-j-medne-l-muschke-p-rivier-f-rudnik-schoneborn-s-spengler-s-inzana-f-stanzial-f-benedicenti-f-synofzik-m-lia-taratuto-a-pirra-l-tay-sk-topaloglu-h-uyanik-g-wand-d-williams-d-zerres-k-weis-j-senderek-j" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> Marinesco-Sjögren syndrome is a rare autosomal recessive multisystem disorder featuring cerebellar ataxia, early-onset cataracts, chronic myopathy, variable intellectual disability and delayed motor development. More recently, mutations in the SIL1 gene, which encodes an endoplasmic reticulum resident co-chaperone, were identified as the main cause of Marinesco-Sjögren syndrome. Here we describe the results of SIL1 mutation analysis in 62 patients presenting with early-onset ataxia, cataracts and myopathy or combinations of at least two of these. We obtained a mutation detection rate of 60% (15/25) among patients with the characteristic Marinesco-Sjögren syndrome triad (ataxia, cataracts, myopathy) whereas the detection rate in the group of patients with more variable phenotypic presentation was below 3% (1/37). We report 16 unrelated families with a total of 19 different SIL1 mutations.
<a href="2008-2015/2013-12-01-SIL1-mutations-and/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      August 01, 2013
      
    </span>
</aside>

<h3 class="title_news">Recurrent loss of heterozygosity in 1p36 associated with TNFRSF14 mutations in IRF4 translocation negative pediatric follicular lymphomas</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="martin-guerrero-i-salaverria-i-burkhardt-b-szczepanowski-m-baudis-m-bens-s-de-leval-l-garcia-orad-a-horn-h-lisfeld-j-pellissery-s-klapper-w-oschlies-i-siebert-r">Martin-Guerrero I, Salaverria I, Burkhardt B, Szczepanowski M, Baudis M, Bens S, de Leval L, Garcia-Orad A, Horn H, Lisfeld J, Pellissery S, Klapper W, Oschlies I, Siebert R.<a class="headerlink" href="#martin-guerrero-i-salaverria-i-burkhardt-b-szczepanowski-m-baudis-m-bens-s-de-leval-l-garcia-orad-a-horn-h-lisfeld-j-pellissery-s-klapper-w-oschlies-i-siebert-r" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> Pediatric follicular lymphoma is a rare disease that differs genetically and clinically from its adult counterpart. With the exception of pediatric follicular lymphoma with IRF4-translocation, the genetic events associated with these lymphomas have not yet been defined. We applied array-comparative genomic hybridization and molecular inversion probe assay analyses to formalin-fixed paraffin-embedded tissues from 18 patients aged 18 years and under with IRF4 translocation negative follicular lymphoma.
<a href="2008-2015/2013-08-01-Recurrent-loss-of/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      February 01, 2013
      
    </span>
</aside>

<h3 class="title_news">High resolution copy number analysis of IRF4 translocation-positive diffuse large B-cell and follicular lymphomas</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="salaverria-i-martin-guerrero-i-burkhardt-b-kreuz-m-zenz-t-oschlies-i-arnold-n-baudis-m-bens-s-garcia-orad-a-lisfeld-j-schwaenen-c-szczepanowski-m-wessendorf-s-pfreundschuh-m-trumper-l-klapper-w-siebert-r">Salaverria I, Martin-Guerrero I, Burkhardt B, Kreuz M, Zenz T, Oschlies I, Arnold N, Baudis M, Bens S, García-Orad A, Lisfeld J, Schwaenen C, Szczepanowski M, Wessendorf S, Pfreundschuh M, Trümper L, Klapper W, Siebert R.<a class="headerlink" href="#salaverria-i-martin-guerrero-i-burkhardt-b-kreuz-m-zenz-t-oschlies-i-arnold-n-baudis-m-bens-s-garcia-orad-a-lisfeld-j-schwaenen-c-szczepanowski-m-wessendorf-s-pfreundschuh-m-trumper-l-klapper-w-siebert-r" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> Translocations affecting chromosome subband 6p25.3 containing the IRF4 gene have been recently described as characteristic alterations in a molecularly distinct subset of germinal center B-cell-derived lymphomas. Secondary changes have yet only been described in few of these lymphomas. Here, we performed array-comparative genomic hybridization and molecular inversion probe microarray analyses on DNA from 12 formalin-fixed paraffin-embedded and two fresh-frozen IRF4 translocation-positive lymphomas, which together with the previously published data on nine cases allowed the extension of copy number analyses to a total of 23 of these lymphomas.
<a href="2008-2015/2013-02-01-High-resolution-copy/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      January 01, 2013
      
    </span>
</aside>

<h3 class="title_news">PKC signaling prevents irradiation-induced apoptosis of primary human fibroblasts</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="bluwstein-a-kumar-n-leger-k-traenkle-j-oostrum-jv-rehrauer-h-baudis-m-hottiger-mo">Bluwstein A, Kumar N, Léger K, Traenkle J, Oostrum Jv, Rehrauer H, Baudis M, Hottiger MO.<a class="headerlink" href="#bluwstein-a-kumar-n-leger-k-traenkle-j-oostrum-jv-rehrauer-h-baudis-m-hottiger-mo" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> Primary cells respond to irradiation by activation of the DNA damage response and cell cycle arrest, which eventually leads to senescence or apoptosis. It is not clear in detail which signaling pathways or networks regulate the induction of either apoptosis or senescence. Primary human fibroblasts are able to withstand high doses of irradiation and to prevent irradiation-induced apoptosis. However, the underlying regulatory basis for this phenotype is not well understood.
<a href="2008-2015/2013-01-01-PKC-signaling/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      November 01, 2012
      
    </span>
</aside>

<h3 class="title_news">Molecular karyotyping as a relevant diagnostic tool in children with growth retardation with Silver-Russell features</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="spengler-s-begemann-m-ortiz-bruchle-n-baudis-m-denecke-b-kroisel-pm-oehl-jaschkowitz-b-schulze-b-raabe-meyer-g-spaich-c-blumel-p-jauch-a-moog-u-zerres-k-eggermann-t">Spengler S, Begemann M, Ortiz Brüchle N, Baudis M, Denecke B, Kroisel PM, Oehl-Jaschkowitz B, Schulze B, Raabe-Meyer G, Spaich C, Blümel P, Jauch A, Moog U, Zerres K, Eggermann T.<a class="headerlink" href="#spengler-s-begemann-m-ortiz-bruchle-n-baudis-m-denecke-b-kroisel-pm-oehl-jaschkowitz-b-schulze-b-raabe-meyer-g-spaich-c-blumel-p-jauch-a-moog-u-zerres-k-eggermann-t" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> OBJECTIVE: To determine the contribution of submicroscopic chromosomal imbalances to the etiology of Silver-Russell syndrome (SRS) and SRS-like phenotypes. STUDY DESIGN: We performed molecular karyotyping in 41 patients with SRS or SRS-like features without known chromosome 7 and 11 defects using the Affymetrix SNP Array 6.0 system (Affymetrix, High Wycombe, United Kingdom).
<a href="2008-2015/2012-11-01-Molecular-karyotypin/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      October 01, 2012
      
    </span>
</aside>

<h3 class="title_news">2p21 Deletions in hypotonia-cystinuria syndrome</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="eggermann-t-spengler-s-venghaus-a-denecke-b-zerres-k-baudis-m-ensenauer-r">Eggermann T, Spengler S, Venghaus A, Denecke B, Zerres K, Baudis M, Ensenauer R.<a class="headerlink" href="#eggermann-t-spengler-s-venghaus-a-denecke-b-zerres-k-baudis-m-ensenauer-r" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> The significant role of the SLC3A1 gene in the aetiology of cystinuria is meanwhile well established and more than 130 point mutations have been reported. With the reports on genomic deletions including at least both SLC3A1 and the neighboured PREPL gene the spectrum of cystinuria mutations and of clinical symptoms could recently be enlarged: patients homozygous for these deletions suffer from a general neonatal hypotonia and growth retardation in addition to cystinuria. The hypotonia in these hypotonia-cystinuria (HCS) patients has been attributed to the total loss of the PREPL protein.
<a href="2008-2015/2012-10-01-2p21-Deletions-in/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      September 01, 2012
      
    </span>
</aside>

<h3 class="title_news">DNA copy number alterations in central primitive neuroectodermal tumors and tumors of the pineal region: an international individual patient data meta-analysis</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="von-bueren-ao-gerss-j-hagel-c-cai-h-remke-m-hasselblatt-m-feuerstein-bg-pernet-s-delattre-o-korshunov-a-rutkowski-s-pfister-sm-baudis-m">von Bueren AO, Gerss J, Hagel C, Cai H, Remke M, Hasselblatt M, Feuerstein BG, Pernet S, Delattre O, Korshunov A, Rutkowski S, Pfister SM, Baudis M.<a class="headerlink" href="#von-bueren-ao-gerss-j-hagel-c-cai-h-remke-m-hasselblatt-m-feuerstein-bg-pernet-s-delattre-o-korshunov-a-rutkowski-s-pfister-sm-baudis-m" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> Little is known about frequency, association with clinical characteristics, and prognostic impact of DNA copy number alterations (CNA) on survival in central primitive neuroectodermal tumors (CNS-PNET) and tumors of the pineal region. Searches of MEDLINE, Pubmed, and EMBASE--after the original description of comparative genomic hybridization in 1992 and July 2010--identified 15 case series of patients with CNS-PNET and tumors of the pineal region whose tumors were investigated for genome-wide CNA. One additional case study was identified from contact with experts. Individual patient data were extracted from publications or obtained from investigators, and CNAs were converted to a digitized format suitable for data mining and subgroup identification. Summary profiles for genomic imbalances were generated from case-specific data. Overall survival (OS) was estimated using the Kaplan-Meier method, and by univariable and multivariable Cox regression models.
<a href="2008-2015/2012-09-01-DNA-copy-number/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      September 01, 2012
      
    </span>
</aside>

<h3 class="title_news">Improved multiplex...</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h3 id="improved-multiplex-ligation-dependent-probe-amplification-analysis-identifies-a-deleterious-pms2-allele-generated-by-recombination-with-crossover-between-pms2-and-pms2cl">Improved multiplex ligation-dependent probe amplification analysis identifies a deleterious PMS2 allele generated by recombination with crossover between PMS2 and PMS2CL.<a class="headerlink" href="#improved-multiplex-ligation-dependent-probe-amplification-analysis-identifies-a-deleterious-pms2-allele-generated-by-recombination-with-crossover-between-pms2-and-pms2cl" title="Permanent link">&para;</a></h3>
<h4 id="wernstedt-a-valtorta-e-armelao-f-togni-r-girlando-s-baudis-m-heinimann-k-messiaen-l-staehli-n-zschocke-j-marra-g-wimmer-k">Wernstedt A, Valtorta E, Armelao F, Togni R, Girlando S, Baudis M, Heinimann K, Messiaen L, Staehli N, Zschocke J, Marra G, Wimmer K.<a class="headerlink" href="#wernstedt-a-valtorta-e-armelao-f-togni-r-girlando-s-baudis-m-heinimann-k-messiaen-l-staehli-n-zschocke-j-marra-g-wimmer-k" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> Heterozygous PMS2 germline mutations are associated with Lynch syndrome. Up to one third of these mutations are genomic deletions. Their detection is complicated by a pseudogene (PMS2CL), which--owing to extensive interparalog sequence exchange--closely resembles PMS2 downstream of exon 12. A recently redesigned multiplex ligation-dependent probe amplification (MLPA) assay identifies PMS2 copy number alterations with improved reliability when used with reference DNAs containing equal numbers of PMS2- and PMS2CL-specific sequences. We selected eight such reference samples--all publicly available--and used them with this assay to study 13 patients with PMS2-defective colorectal tumors.
<a href="2008-2015/2012-09-01-Improved-multiplex/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      January 01, 2012
      
    </span>
</aside>

<h3 class="title_news">Integrative genome-wide expression profiling identifies three distinct molecular subgroups of renal cell carcinoma with different patient outcome</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="beleut-m-zimmermann-p-baudis-m-bruni-n-buhlmann-p-laule-o-luu-vd-gruissem-w-schraml-p-moch-h">Beleut M, Zimmermann P, Baudis M, Bruni N, Bühlmann P, Laule O, Luu VD, Gruissem W, Schraml P, Moch H.<a class="headerlink" href="#beleut-m-zimmermann-p-baudis-m-bruni-n-buhlmann-p-laule-o-luu-vd-gruissem-w-schraml-p-moch-h" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> BACKGROUND: Renal cell carcinoma (RCC) is characterized by a number of diverse molecular aberrations that differ among individuals. Recent approaches to molecularly classify RCC were based on clinical, pathological as well as on single molecular parameters. As a consequence, gene expression patterns reflecting the sum of genetic aberrations in individual tumors may not have been recognized. In an attempt to uncover such molecular features in RCC, we used a novel, unbiased and integrative approach.
<a href="2008-2015/2012-01-01-Integrative/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      January 01, 2012
      
    </span>
</aside>

<h3 class="title_news">Losses at chromosome...</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h3 id="losses-at-chromosome-4q-are-associated-with-poor-survival-in-operable-ductal-pancreatic-adenocarcinoma">Losses at chromosome 4q are associated with poor survival in operable ductal pancreatic adenocarcinoma.<a class="headerlink" href="#losses-at-chromosome-4q-are-associated-with-poor-survival-in-operable-ductal-pancreatic-adenocarcinoma" title="Permanent link">&para;</a></h3>
<h4 id="luebke-am-baudis-m-matthaei-h-vashist-yk-verde-pe-hosch-sb-erbersdobler-a-klein-ca-izbicki-jr-knoefel-wt-stoecklein-nh">Luebke AM, Baudis M, Matthaei H, Vashist YK, Verde PE, Hosch SB, Erbersdobler A, Klein CA, Izbicki JR, Knoefel WT, Stoecklein NH.<a class="headerlink" href="#luebke-am-baudis-m-matthaei-h-vashist-yk-verde-pe-hosch-sb-erbersdobler-a-klein-ca-izbicki-jr-knoefel-wt-stoecklein-nh" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> Here we tested the prognostic impact of genomic alterations in operable localized pancreatic ductal adenocarcinoma (PDAC). Fifty-two formalin-fixed and paraffin-embedded primary PDAC were laser micro-dissected and were investigated by comparative genomic hybridization after whole genome amplification using an adapter-linker PCR. Chromosomal gains and losses were correlated to clinico-pathological parameters and clinical follow-up data. The most frequent aberration was loss on chromosome 17p (65%) while the most frequent gains were detected at 2q (41%) and 8q (41%), respectively. The concomitant occurrence of losses at 9p and 17p was found to be statistically significant. Higher rates of chromosomal losses were associated with a more advanced primary tumor stage and losses at 9p and 18q were significantly associated with presence of lymphatic metastasis (chi-square: p = 0.03, p = 0.05, respectively). Deletions on chromosome 4 were of prognostic significance for overall survival and tumor recurrence (Cox-multivariate analysis: p = 0.026 and p = 0.021, respectively). In conclusion our data suggest the common alterations at chromosome 8q, 9p, 17p and 18q as well as the prognostic relevant deletions on chromosome 4q as relevant for PDAC progression. Our comprehensive data from 52 PDAC should provide a basis for future studies with a higher resolution to discover the relevant genes located within the chromosomal aberrations identified.</p>
<a href="2008-2015/2012-01-01-Losses-at-chromosome/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      January 01, 2012
      
    </span>
</aside>

<h3 class="title_news">Specific genomic regions are differentially affected by copy number alterations across distinct cancer types, in aggregated cytogenetic data</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="kumar-n-cai-h-von-mering-c-baudis-m">Kumar N, Cai H, von Mering C, Baudis M.<a class="headerlink" href="#kumar-n-cai-h-von-mering-c-baudis-m" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> BACKGROUND: Regional genomic copy number alterations (CNA) are observed in the vast majority of cancers. Besides specifically targeting well-known, canonical oncogenes, CNAs may also play more subtle roles in terms of modulating genetic potential and broad gene expression patterns of developing tumors. Any significant differences in the overall CNA patterns between different cancer types may thus point towards specific biological mechanisms acting in those cancers. In addition, differences among CNA profiles may prove valuable for cancer classifications beyond existing annotation systems. PRINCIPAL FINDINGS: We have analyzed molecular-cytogenetic data from 25579 tumors samples, which were classified into 160 cancer types according to the International Classification of Disease (ICD) coding system.
<a href="2008-2015/2012-01-01-Specific-genomic/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      January 01, 2012
      
    </span>
</aside>

<h3 class="title_news">arrayMap: a reference resource for genomic copy number imbalances in human malignancies</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="cai-h-kumar-n-baudis-m">Cai H, Kumar N, Baudis M.<a class="headerlink" href="#cai-h-kumar-n-baudis-m" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> BACKGROUND: The delineation of genomic copy number abnormalities (CNAs) from cancer samples has been instrumental for identification of tumor suppressor genes and oncogenes and proven useful for clinical marker detection. An increasing number of projects have mapped CNAs using high-resolution microarray based techniques. So far, no single resource does provide a global collection of readily accessible oncogenomic array data. METHODOLOGY/PRINCIPAL FINDINGS: We here present arrayMap, a curated reference database and bioinformatics resource targeting copy number profiling data in human cancer.
<a href="2008-2015/2012-01-01-arrayMap-a/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      July 01, 2011
      
    </span>
</aside>

<h3 class="title_news">Silver-Russell patients showing a broad range of ICR1 and ICR2 hypomethylation in different tissues</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="begemann-m-spengler-s-kanber-d-haake-a-baudis-m-leisten-i-binder-g-markus-s-rupprecht-t-segerer-h-fricke-otto-s-muhlenberg-r-siebert-r-buiting-k-eggermann-t">Begemann M, Spengler S, Kanber D, Haake A, Baudis M, Leisten I, Binder G, Markus S, Rupprecht T, Segerer H, Fricke-Otto S, Mühlenberg R, Siebert R, Buiting K, Eggermann T.<a class="headerlink" href="#begemann-m-spengler-s-kanber-d-haake-a-baudis-m-leisten-i-binder-g-markus-s-rupprecht-t-segerer-h-fricke-otto-s-muhlenberg-r-siebert-r-buiting-k-eggermann-t" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> In all known congenital imprinting disorders an association with aberrant methylation or mutations at specific loci was well established. However, several patients with transient neonatal diabetes mellitus (TNDM), Silver-Russell syndrome (SRS) and Beckwith-Wiedemann syndrome (BWS) exhibiting multilocus hypomethylation (MLH) have meanwhile been described. Whereas TNDM patients with MLH show clinical symptoms different from carriers with isolated 6q24 aberrations, MLH carriers diagnosed as BWS or SRS present only the syndrome-specific features.
<a href="2008-2015/2011-07-01-Silver-Russell/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      January 01, 2011
      
    </span>
</aside>

<h3 class="title_news">CDCOCA: a statistical method to define complexity dependence of co-occuring chromosomal aberrations</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="kumar-n-rehrauer-h-cai-h-baudis-m">Kumar N, Rehrauer H, Cai H, Baudis M.<a class="headerlink" href="#kumar-n-rehrauer-h-cai-h-baudis-m" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> BACKGROUND: Copy number alterations (CNA) play a key role in cancer development and progression. Since more than one CNA can be detected in most tumors, frequently co-occurring genetic CNA may point to cooperating cancer related genes. Existing methods for co-occurrence evaluation so far have not considered the overall heterogeneity of CNA per tumor, resulting in a preferential detection of frequent changes with limited specificity for each association due to the high genetic instability of many samples.
<a href="2008-2015/2011-01-01-CDCOCA-a-statistica/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      September 01, 2010
      
    </span>
</aside>

<h3 class="title_news">MUC1 oncogene amplification correlates with protein overexpression in invasive breast carcinoma cells</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="lacunza-e-baudis-m-colussi-ag-segal-eiras-a-croce-mv-abba-mc">Lacunza E, Baudis M, Colussi AG, Segal-Eiras A, Croce MV, Abba MC.<a class="headerlink" href="#lacunza-e-baudis-m-colussi-ag-segal-eiras-a-croce-mv-abba-mc" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> The MUC1 gene is aberrantly overexpressed in approximately 90% of human breast cancers. Several studies have shown that MUC1 overexpression is due to transcriptional regulatory events. However, the importance of gene amplification as a mechanism leading to the increase of MUC1 expression in breast cancer has been poorly characterized. The aim of this study was to evaluate the role of MUC1 gene amplification and protein expression in human breast cancer development. By means of real-time quantitative polymerase chain reaction and immunohistochemical methods, 83 breast tissue samples were analyzed for MUC1 gene amplification and protein expression.
<a href="2008-2015/2010-09-01-MUC1-oncogene/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      June 01, 2010
      
    </span>
</aside>

<h3 class="title_news">Chromosome 11p15 duplication in Silver-Russell syndrome due to a maternally inherited translocation t(11;15)</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="eggermann-t-spengler-s-bachmann-n-baudis-m-mau-holzmann-ua-singer-s-rossier-e">Eggermann T, Spengler S, Bachmann N, Baudis M, Mau-Holzmann UA, Singer S, Rossier E.<a class="headerlink" href="#eggermann-t-spengler-s-bachmann-n-baudis-m-mau-holzmann-ua-singer-s-rossier-e" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> The role of 11p15 disturbances in the aetiology of Silver-Russell syndrome (SRS) is well established: in addition to hypomethylation of the H19/IGF2 differentially methylated regions, five patients with a duplication of maternal 11p15 material have been described. We report on a boy with SRS carrying a maternally inherited duplication of chromosome 11p15.
<a href="2008-2015/2010-06-01-Chromosome-11p15/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      June 01, 2010
      
    </span>
</aside>

<h3 class="title_news">Increased expression...</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h3 id="increased-expression-of-cellular-retinol-binding-protein-1-in-laryngeal-squamous-cell-carcinoma">Increased expression of cellular retinol-binding protein 1 in laryngeal squamous cell carcinoma.<a class="headerlink" href="#increased-expression-of-cellular-retinol-binding-protein-1-in-laryngeal-squamous-cell-carcinoma" title="Permanent link">&para;</a></h3>
<h4 id="peralta-r-baudis-m-vazquez-g-juarez-s-ortiz-r-decanini-h-hernandez-d-gallegos-f-valdivia-a-pina-p-salcedo-m">Peralta R, Baudis M, Vazquez G, Juárez S, Ortiz R, Decanini H, Hernandez D, Gallegos F, Valdivia A, Piña P, Salcedo M.<a class="headerlink" href="#peralta-r-baudis-m-vazquez-g-juarez-s-ortiz-r-decanini-h-hernandez-d-gallegos-f-valdivia-a-pina-p-salcedo-m" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> To investigate the genomic alterations in larynx carcinomas (LaCa) tissues and its prognostics values in predicting survival.To analyse the aberrations in the genome of LaCa patients, we used array comparative genomic hybridization in 19 human laryngeal tumour samples. DNA samples were also subjected to detect human papillomavirus (HPV) sequences by polymerase chain reaction (PCR). Copy number gain was confirmed by real-time PCR. The cellular retinol-binding protein 1 (CRBP-1) gene expression was also confirmed by immunohistochemistry assay on LaCa tissues. To identify prognostic feature, CRBP-1 gene gain was correlated to patient survival.The most common gains were detected for CRBP-1 and EGFR genes, while DNA lost in RAF-1 gene. Immunohistochemistry assay was revealed strong expression of CRBP1 protein in those cases with CRBP-1 gene gain. The CRBP-1 gene gain and its expression correlated significantly with survival (P = 0.003). Cox regression analysis indicated that CRBP-1 expression level was a factor of survival (P = 0.008). HPV sequences were detected in 42% of the samples, and did not show any relationship with specific gene alterations.Our data shows that CRBP-1 gene gain can be determined by immunohistochemistry on routinely processed tissue specimens, and could support as a potential novel marker for long-term survival in laryngeal squamous cell carcinoma.</p>
<a href="2008-2015/2010-06-01-Increased-expression/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      May 01, 2010
      
    </span>
</aside>

<h3 class="title_news">Submicroscopic chromosomal imbalances in idiopathic Silver-Russell syndrome (SRS): the SRS phenotype overlaps with the 12q14 microdeletion syndrome</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="spengler-s-schonherr-n-binder-g-wollmann-ha-fricke-otto-s-muhlenberg-r-denecke-b-baudis-m-eggermann-t">Spengler S, Schönherr N, Binder G, Wollmann HA, Fricke-Otto S, Mühlenberg R, Denecke B, Baudis M, Eggermann T.<a class="headerlink" href="#spengler-s-schonherr-n-binder-g-wollmann-ha-fricke-otto-s-muhlenberg-r-denecke-b-baudis-m-eggermann-t" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> Silver-Russell syndrome (SRS) is a heterogeneous disorder associated with intrauterine and postnatal growth restriction, body asymmetry, a relative macrocephaly, a characteristic triangular face and further dysmorphisms. In about 50% of patients, genetic/epigenetic alterations can be detected: &gt;38% of patients show a hypomethylation of the IGF2/H19 imprinting region in 11p15, whereas the additional 10% carry a maternal uniparental disomy of chromosome 7.
<a href="2008-2015/2010-05-01-Submicroscopic/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      February 01, 2010
      
    </span>
</aside>

<h3 class="title_news">Identification of a...</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h3 id="identification-of-a-21q22-duplication-in-a-silver-russell-syndrome-patient-further-narrows-down-the-down-syndrome-critical-region">Identification of a 21q22 duplication in a Silver-Russell syndrome patient further narrows down the Down syndrome critical region.<a class="headerlink" href="#identification-of-a-21q22-duplication-in-a-silver-russell-syndrome-patient-further-narrows-down-the-down-syndrome-critical-region" title="Permanent link">&para;</a></h3>
<h4 id="eggermann-t-schonherr-n-spengler-s-jager-s-denecke-b-binder-g-baudis-m">Eggermann T, Schönherr N, Spengler S, Jäger S, Denecke B, Binder G, Baudis M.<a class="headerlink" href="#eggermann-t-schonherr-n-spengler-s-jager-s-denecke-b-binder-g-baudis-m" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> Several duplications of chromosome 21q helped to narrow down the Down syndrome (DS) critical region (DSCR) to chromosomal band 21q22 with an approximate length of 5.4 Mb. Recently, it has been suggested that the facial gestalt of DS has been linked to the distal part of the DSCR whereas the proximal region harboring DSCR1/RCAN and DSCAM should be associated with the cardiac abnormalities. Here, we report on a patient with Silver-Russell syndrome (SRS) and a paternally inherited 0.46 Mb duplication in 21q22 affecting the KCNE1 and DSCR1/RCAN genes. The identification of an involvement of KCNE1 was interesting because it encodes the beta-subunit of the KvLQT1 channel as the slow component of the cardiac delayed rectifier K(+) current. Since duplication of the KCNQ1 gene encoding the alpha-subunit of the same channel was reported recently in another SRS patient, we screened both genes for mutations in a cohort of SRS patients without detecting pathologic variants. We presume that the duplication of the two functionally linked genes in different patients with the same disorder is a coincidental finding. However, the lack of DS typical clinical features in our case allows us to further narrow down the DSCR in 21q22. We conclude that DSCR1/RCAN is not sufficient for generating phenotypic features associated with DS but our observation does not contradict a possible role for DSCR1/RCAN in mediating DYRK1A-based effects.</p>
<a href="2008-2015/2010-02-01-Identification-of-a/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      November 01, 2009
      
    </span>
</aside>

<h3 class="title_news">Quantifying cancer progression with conjunctive Bayesian networks</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="gerstung-m-baudis-m-moch-h-beerenwinkel-n">Gerstung M, Baudis M, Moch H, Beerenwinkel N.<a class="headerlink" href="#gerstung-m-baudis-m-moch-h-beerenwinkel-n" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> MOTIVATION: Cancer is an evolutionary process characterized by accumulating mutations. However, the precise timing and the order of genetic alterations that drive tumor progression remain enigmatic. RESULTS: We present a specific probabilistic graphical model for the accumulation of mutations and their interdependencies. The Bayesian network models cancer progression by an explicit unobservable accumulation process in time that is separated from the observable but error-prone detection of mutations.
<a href="2008-2015/2009-11-01-Quantifying-cancer/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      September 01, 2009
      
    </span>
</aside>

<h3 class="title_news">Inferring progressio...</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h3 id="inferring-progression-models-for-cgh-data">Inferring progression models for CGH data.<a class="headerlink" href="#inferring-progression-models-for-cgh-data" title="Permanent link">&para;</a></h3>
<h4 id="liu-j-bandyopadhyay-n-ranka-s-baudis-m-kahveci-t">Liu J, Bandyopadhyay N, Ranka S, Baudis M, Kahveci T.<a class="headerlink" href="#liu-j-bandyopadhyay-n-ranka-s-baudis-m-kahveci-t" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> MOTIVATION: One of the mutational processes that has been monitored genome-wide is the occurrence of regional DNA copy number alterations (CNAs), which may lead to deletion or over-expression of tumor suppressors or oncogenes, respectively. Understanding the relationship between CNAs and different cancer types is a fundamental problem in cancer studies. RESULTS: This article develops an efficient method that can accurately model the progression of the cancer markers and reconstruct evolutionary relationship between multiple types of cancers using comparative genomic hybridization (CGH) data. Such modeling can lead to better understanding of the commonalities and differences between multiple cancer types and potential therapies. We have developed an automatic method to infer a graph model for the markers of multiple cancers from a large population of CGH data. Our method identifies highly related markers across different cancer types. It then builds a directed acyclic graph that shows the evolutionary history of these markers based on how common each marker is in different cancer types. We demonstrated the use of this model in determining the importance of markers in cancer evolution. We have also developed a new method to measure the evolutionary distance between different cancers based on their markers. This method employs the graph model we developed for the individual markers to measure the distance between pairs of cancers. We used this measure to create an evolutionary tree for multiple cancers. Our experiments on Progenetix database show that our markers are largely consistent to the reported hot-spot imbalances and most frequent imbalances. The results show that our distance measure can accurately reconstruct the evolutionary relationship between multiple cancer types.</p>
<a href="2008-2015/2009-09-01-Inferring-progressio/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      July 01, 2009
      
    </span>
</aside>

<h3 class="title_news">Recurrent loss, but...</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h3 id="recurrent-loss-but-lack-of-mutations-of-the-smarcb1-tumor-suppressor-gene-in-t-cell-prolymphocytic-leukemia-with-tcl1a-tcrad-juxtaposition">Recurrent loss, but lack of mutations, of the SMARCB1 tumor suppressor gene in T-cell prolymphocytic leukemia with TCL1A-TCRAD juxtaposition.<a class="headerlink" href="#recurrent-loss-but-lack-of-mutations-of-the-smarcb1-tumor-suppressor-gene-in-t-cell-prolymphocytic-leukemia-with-tcl1a-tcrad-juxtaposition" title="Permanent link">&para;</a></h3>
<h4 id="bug-s-durig-j-oyen-f-klein-hitpass-l-martin-subero-ji-harder-l-baudis-m-arnold-n-kordes-u-duhrsen-u-schneppenheim-r-siebert-r">Bug S, Dürig J, Oyen F, Klein-Hitpass L, Martin-Subero JI, Harder L, Baudis M, Arnold N, Kordes U, Dührsen U, Schneppenheim R, Siebert R.<a class="headerlink" href="#bug-s-durig-j-oyen-f-klein-hitpass-l-martin-subero-ji-harder-l-baudis-m-arnold-n-kordes-u-duhrsen-u-schneppenheim-r-siebert-r" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> In T-cell prolymphocytic leukemia (T-PLL), chromosomal imbalances affecting the long arm of chromosome 22 are regarded as typical chromosomal aberrations secondary to a TCRAD-TCL1A fusion due to inv(14) or t(14;14). We analyzed recently obtained data from conventional karyotyping, SNP-chip array copy number mapping, genome-wide expression profiling, and interphase fluorescence in situ hybridization (FISH) of inv(14)-positive T-PLL with respect to structural aberrations on chromosome 22. Combined gene chip and interphase FISH analyses revealed interstitial deletions on 22q in 4 of 12 cases, with one case additionally showing a terminal copy number gain. A minimally deleted region of approximately 9.1 Mb was delineated, from 16.2 Mb (22cen) to 25.3 Mb (22q12.1). The distal borders of copy number alterations spread over a region of approximately 8.8 Mb, from 25.2 Mb (22q12.1) to 34 Mb (22q12.3). Mutation screening of candidate tumor suppressor genes SMARCB1 and CHEK2 mapping to the minimally deleted and the breakpoint regions, respectively, in cases with hemizygous deletion, revealed no inactivating mutations. With gene expression profiling, no significantly downregulated genes were identified in the minimally deleted region. We therefore assume that haploinsufficiency or alternative pathomechanisms underlie chromosome 22 aberrations in T-PLL.</p>
<a href="2008-2015/2009-07-01-Recurrent-loss%2C-but/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      February 01, 2009
      
    </span>
</aside>

<h3 class="title_news">Translocations involving 8q24 in Burkitt lymphoma and other malignant lymphomas: a historical review of cytogenetics in the light of todays knowledge</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="boerma-eg-siebert-r-kluin-pm-baudis-m">Boerma EG, Siebert R, Kluin PM, Baudis M.<a class="headerlink" href="#boerma-eg-siebert-r-kluin-pm-baudis-m" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> Burkitt lymphoma (BL) has a characteristic clinical presentation, morphology, immunophenotype and primary chromosomal aberration, that is, the translocation t(8;14)(q24;q32) or its variants. However, diagnostic dilemmas may arise in daily practice due to overlap of BL with subsets of other aggressive, mature B-cell lymphomas such as diffuse large B-cell lymphomas (DLBCL). Recently, two gene expression studies have described a distinct molecular profile for BL, but also showed the persistence of some cases intermediate between BL and DLBCL. An alternative approach to define BL is to consider (cyto)genetic data, in particular chromosomal abnormalities other than the t(8;14) or its variants. In this review the 'Mitelman Database of Chromosome Aberrations in Cancer,' harboring the majority of all published neoplasia-related karyotypes, was explored to define a cytogenetic profile of 'true' BL.
<a href="2008-2015/2009-02-01-Translocations/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      December 01, 2008
      
    </span>
</aside>

<h3 class="title_news">Chromosomal changes characterize head and neck cancer with poor prognosis</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="bauer-vl-braselmann-h-henke-m-mattern-d-walch-a-unger-k-baudis-m-lassmann-s-huber-r-wienberg-j-werner-m-zitzelsberger-hf">Bauer VL, Braselmann H, Henke M, Mattern D, Walch A, Unger K, Baudis M, Lassmann S, Huber R, Wienberg J, Werner M, Zitzelsberger HF.<a class="headerlink" href="#bauer-vl-braselmann-h-henke-m-mattern-d-walch-a-unger-k-baudis-m-lassmann-s-huber-r-wienberg-j-werner-m-zitzelsberger-hf" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> It is well established that genetic alterations may be associated to prognosis in tumor patients. This study investigates chromosomal changes that predict the clinical outcome of head and neck squamous cell carcinoma (HNSCC) and correlate to characteristic clinicopathological parameters. We applied comparative genomic hybridization (CGH) to tissue samples from 117 HNSCC patients scheduled for radiotherapy. Genomic aberrations occurring in more than five patients were studied for impact on locoregional progression (LRP)-free survival.
<a href="2008-2015/2008-12-01-Chromosomal-changes/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      September 01, 2008
      
    </span>
</aside>

<h3 class="title_news">A 10.7 Mb interstitial deletion of 13q21 without phenotypic effect defines a further non-pathogenic euchromatic variant</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="roos-a-elbracht-m-baudis-m-senderek-j-schonherr-n-eggermann-t-schuler-hm">Roos A, Elbracht M, Baudis M, Senderek J, Schönherr N, Eggermann T, Schüler HM.<a class="headerlink" href="#roos-a-elbracht-m-baudis-m-senderek-j-schonherr-n-eggermann-t-schuler-hm" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> Chromosome 13 deletions are associated with widely varying phenotypes but the clinical picture nearly almost includes mental and growth retardation, craniofacial dysmorphisms, and/or malformations. Several attempts have been made to link monosomy 13q intervals with specific clinical features, but a genotype-phenotype correlation could not be delineated. We report on a woman with a normal phenotype and intelligence referred for chromosomal analysis because of recurrent abortions followed by reproductive loss.
<a href="2008-2015/2008-09-01-A-10-7-Mb-interstiti/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      July 01, 2008
      
    </span>
</aside>

<h3 class="title_news">Comprehensive characterization of genomic aberrations in gangliogliomas by CGH, array-based CGH and interphase FISH</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="hoischen-a-ehrler-m-fassunke-j-simon-m-baudis-m-landwehr-c-radlwimmer-b-lichter-p-schramm-j-becker-aj-weber-rg">Hoischen A, Ehrler M, Fassunke J, Simon M, Baudis M, Landwehr C, Radlwimmer B, Lichter P, Schramm J, Becker AJ, Weber RG.<a class="headerlink" href="#hoischen-a-ehrler-m-fassunke-j-simon-m-baudis-m-landwehr-c-radlwimmer-b-lichter-p-schramm-j-becker-aj-weber-rg" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> Gangliogliomas are generally benign neuroepithelial tumors composed of dysplastic neuronal and neoplastic glial elements. We screened 61 gangliogliomas [World Health Organization (WHO) grade I] for genomic alterations by chromosomal and array-based comparative genomic hybridization (CGH). Aberrations were detected in 66% of gangliogliomas (mean +/- SEM = 2.5 +/- 0.5 alterations/tumor). Frequent gains were on chromosomes 7 (21%), 5 (16%), 8 (13%), 12 (12%); frequent losses on 22q (16%), 9 (10%), 10 (8%). Recurrent partial imbalances comprised the minimal overlapping regions dim(10)(q25) and enh(12)(q13.3-q14.1).
<a href="2008-2015/2008-07-01-Comprehensive/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      June 01, 2008
      
    </span>
</aside>

<h3 class="title_news">Recurrent loss of the Y chromosome and homozygous deletions within the pseudoautosomal region 1: association with male predominance in mantle cell lymphoma</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="nielander-i-martin-subero-ji-wagner-f-baudis-m-gesk-s-harder-l-hasenclever-d-klapper-w-kreuz-m-pott-c-martinez-climent-ja-dreyling-m-arnold-n-siebert-r">Nieländer I, Martín-Subero JI, Wagner F, Baudis M, Gesk S, Harder L, Hasenclever D, Klapper W, Kreuz M, Pott C, Martinez-Climent JA, Dreyling M, Arnold N, Siebert R.<a class="headerlink" href="#nielander-i-martin-subero-ji-wagner-f-baudis-m-gesk-s-harder-l-hasenclever-d-klapper-w-kreuz-m-pott-c-martinez-climent-ja-dreyling-m-arnold-n-siebert-r" title="Permanent link">&para;</a></h4>
<p>Mantle cell lymphoma (MCL) is a B-cell lymphoproliferative disorder which predominantly affects men. In a large retrospective survey of the European MCL Network including 304 patients, median age was 63 years at first diagnosis with a male preponderance of 76%.1 The genetic hallmark of MCL is the translocation t(11;14)(q13;q32) which leads to overexpression of the CCND1 gene encoding Cyclin D1. Although recent studies revealed a number of genomic alterations and differentially expressed genes in MCL, the causes for the male predominance are still unknown. Hormonal differences might contribute to this gender imbalance.
<a href="2008-2015/2008-06-01-Recurrent-loss-of/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      October 01, 2007
      
    </span>
</aside>

<h3 class="title_news">Combined single nucleotide polymorphism-based genomic mapping and global gene expression profiling identifies novel chromosomal imbalances, mechanisms and candidate genes important in the pathogenesis of T-cell prolymphocytic leukemia with inv(14)(q11q32)</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h3 id="combined-single-nucleotide-polymorphism-based-genomic-mapping-and-global-gene-expression-profiling-identifies-novel-chromosomal-imbalances-mechanisms-and-candidate-genes-important-in-the-pathogenesis-of-t-cell-prolymphocytic-leukemia-with-inv14q11q32">Combined single nucleotide polymorphism-based genomic mapping and global gene expression profiling identifies novel chromosomal imbalances, mechanisms and candidate genes important in the pathogenesis of T-cell prolymphocytic leukemia with inv(14)(q11q32).<a class="headerlink" href="#combined-single-nucleotide-polymorphism-based-genomic-mapping-and-global-gene-expression-profiling-identifies-novel-chromosomal-imbalances-mechanisms-and-candidate-genes-important-in-the-pathogenesis-of-t-cell-prolymphocytic-leukemia-with-inv14q11q32" title="Permanent link">&para;</a></h3>
<h4 id="durig-j-bug-s-klein-hitpass-l-boes-t-jons-t-martin-subero-ji-harder-l-baudis-m-duhrsen-u-siebert-r">Dürig J, Bug S, Klein-Hitpass L, Boes T, Jöns T, Martin-Subero JI, Harder L, Baudis M, Dührsen U, Siebert R.<a class="headerlink" href="#durig-j-bug-s-klein-hitpass-l-boes-t-jons-t-martin-subero-ji-harder-l-baudis-m-duhrsen-u-siebert-r" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> T-cell prolymphocytic leukemia (T-PLL) is a rare aggressive lymphoma derived from mature T cells, which is, in most cases, characterized by the presence of an inv(14)(q11q32)/t(14;14)(q11;q32) and a characteristic pattern of secondary chromosomal aberrations. DNA microarray technology was employed to compare the transcriptomes of eight immunomagnetically purified CD3+ normal donor-derived peripheral blood cell samples, with five highly purified inv(14)/t(14;14)-positive T-PLL blood samples. Between the two experimental groups, 734 genes were identified as differentially expressed, including functionally important genes involved in lymphomagenesis, cell cycle regulation, apoptosis and DNA repair. Notably, the differentially expressed genes were found to be significantly enriched in genomic regions affected by recurrent chromosomal imbalances. Upregulated genes clustered on chromosome arms 6p and 8q, and downregulated genes on 6q, 8p, 10p, 11q and 18p. High-resolution copy-number determination using single nucleotide polymorphism chip technology in 12 inv(14)/t(14;14)-positive T-PLL including those analyzed for gene expression, refined chromosomal breakpoints as well as regions of imbalances. In conclusion, combined transcriptional and molecular cytogenetic profiling identified novel specific chromosomal loci and genes that are likely to be involved in disease progression and suggests a gene dosage effect as a pathogenic mechanism in T-PLL.</p>
<a href="2001-2007/2007-10-01-Combined-single/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      January 01, 2007
      
    </span>
</aside>

<h3 class="title_news">Genomic imbalances in 5918 malignant epithelial tumors: an explorative meta-analysis of chromosomal CGH data</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="baudis-m">Baudis M.<a class="headerlink" href="#baudis-m" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> BACKGROUND: Chromosomal abnormalities have been associated with most human malignancies, with gains and losses on some genomic regions associated with particular entities. METHODS: Of the 15429 cases collected for the Progenetix molecular-cytogenetic database, 5918 malignant epithelial neoplasias analyzed by chromosomal Comparative Genomic Hybridization (CGH) were selected for further evaluation. For the 22 clinico-pathological entities with more than 50 cases, summary profiles for genomic imbalances were generated from case specific data and analyzed.</p>

<a href="2001-2007/2007-01-01-Genomic-imbalances/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      November 01, 2006
      
    </span>
</aside>

<h3 class="title_news">ABCB1 over-expression and drug-efflux in acute lymphoblastic leukemia cell lines with t(17;19) and E2A-HLF expression</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="baudis-m-prima-v-tung-yh-hunger-sp">Baudis M, Prima V, Tung YH, Hunger SP.<a class="headerlink" href="#baudis-m-prima-v-tung-yh-hunger-sp" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> BACKGROUND: The t(17;19)(q21;p13), which occurs in a small subset of acute lymphoblastic leukemias (ALLs) and is associated with a dismal prognosis, creates a chimeric E2A-HLF transcription factor with transforming properties. PROCEDURE: We used representational difference analysis to identify candidate E2A-HLF target genes. Transient transfection assays and an inducible expression model system were then used to evaluate the ability of E2A-HLF to modulate target gene expression. RESULTS: We identified ABCB1 (MDR1, P-glycoprotein) as a gene differentially expressed in ALL cell lines with and without E2A-HLF expression and demonstrated that t(17;19)+ ALL cell lines expressed high levels of ABCB1 protein and had a drug efflux-positive phenotype. Although ABCB1 transcription is regulated by C/EBPbeta via interaction with a DNA response element that shares significant homology with the optimal E2A-HLF binding site, E2A-HLF did not directly activate transcription of reporter genes under control of ABCB1 promoter elements in transient transfection assays. However, ABCB1 expression was induced in a DNA-binding independent manner by E2A-HLF, E2A-PBX1, and truncated E2A polypeptides consisting of those portions of E2A present in leukemic fusion proteins. CONCLUSIONS: E2A-HLF-mediated over-expression of ABCB1 may play a critical role in defining the clinical phenotype of ALLs with a t(17;19), suggesting pharmacologic modulation of ABCB1 activity as a rational therapeutic strategy for this chemotherapy resistant subtype of ALL.</p>
<a href="2001-2007/2006-11-01-ABCB1-over-expressio/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      August 01, 2006
      
    </span>
</aside>

<h3 class="title_news">Distance-based clustering of CGH data</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="liu-j-mohammed-j-carter-j-ranka-s-kahveci-t-baudis-m">Liu J, Mohammed J, Carter J, Ranka S, Kahveci T, Baudis M.<a class="headerlink" href="#liu-j-mohammed-j-carter-j-ranka-s-kahveci-t-baudis-m" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> MOTIVATION: We consider the problem of clustering a population of Comparative Genomic Hybridization (CGH) data samples. The goal is to develop a systematic way of placing patients with similar CGH imbalance profiles into the same cluster. Our expectation is that patients with the same cancer types will generally belong to the same cluster as their underlying CGH profiles will be similar. RESULTS: We focus on distance-based clustering strategies. We do this in two steps. (1) Distances of all pairs of CGH samples are computed. (2) CGH samples are clustered based on this distance. We develop three pairwise distance/similarity measures, namely raw, cosine and sim. Raw measure disregards correlation between contiguous genomic intervals. It compares the aberrations in each genomic interval separately. The remaining measures assume that consecutive genomic intervals may be correlated. Cosine maps pairs of CGH samples into vectors in a high-dimensional space and measures the angle between them. Sim measures the number of independent common aberrations. We test our distance/similarity measures on three well known clustering algorithms, bottom-up, top-down and k-means with and without centroid shrinking. Our results show that sim consistently performs better than the remaining measures. This indicates that the correlation of neighboring genomic intervals should be considered in the structural analysis of CGH datasets. The combination of sim with top-down clustering emerged as the best approach. AVAILABILITY: All software developed in this article and all the datasets are available from the authors upon request. CONTACT: <a href="&#109;&#97;&#105;&#108;&#116;&#111;&#58;&#106;&#117;&#108;&#105;&#117;&#64;&#99;&#105;&#115;&#101;&#46;&#117;&#102;&#108;&#46;&#101;&#100;&#117;">&#106;&#117;&#108;&#105;&#117;&#64;&#99;&#105;&#115;&#101;&#46;&#117;&#102;&#108;&#46;&#101;&#100;&#117;</a>.</p>
<a href="2001-2007/2006-08-01-Distance-based/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      May 01, 2006
      
    </span>
</aside>

<h3 class="title_news">Allele-specific loss of heterozygosity in multiple colorectal adenomas: toward an integrated molecular cytogenetic map II</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="mao-x-hamoudi-ra-talbot-ic-baudis-m">Mao X, Hamoudi RA, Talbot IC, Baudis M.<a class="headerlink" href="#mao-x-hamoudi-ra-talbot-ic-baudis-m" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> Colorectal cancer (CRC) remains a significant public health challenge despite our increased understanding of the genetic defects underlying the pathogenesis of this common disease. It has been thought that multiple mechanisms lead to the malignant phenotype, with familial predisposition syndromes accounting for only a small proportion of all CRC cases. To identify additional loci likely involved in CRC and to test the hypothesis of allele-specific loss of heterozygosity (LOH) for the localization of CRC susceptibility genes, we initially conducted a genome-wide allelotyping analysis of 48 adenomas from a patient with familial adenomatous polyposis coli (FAP) and 63 adenomas from 7 patients with sporadic CRC using 79 fluorescently tagged oligonucleotide primers amplifying microsatellite loci covering the human genome.
<a href="2001-2007/2006-05-01-Allele-specific-loss/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      March 01, 2006
      
    </span>
</aside>

<h3 class="title_news">Online database and bioinformatics toolbox to support data mining in cancer cytogenetics</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h3 id="online-database-and-bioinformatics-toolbox-to-support-data-mining-in-cancer-cytogenetics">Online database and bioinformatics toolbox to support data mining in cancer cytogenetics.<a class="headerlink" href="#online-database-and-bioinformatics-toolbox-to-support-data-mining-in-cancer-cytogenetics" title="Permanent link">&para;</a></h3>
<h4 id="baudis-m">Baudis M.<a class="headerlink" href="#baudis-m" title="Permanent link">&para;</a></h4>
<a href="2001-2007/2006-03-01-Online-database-and/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      July 01, 2005
      
    </span>
</aside>

<h3 class="title_news">Genetic losses in breast cancer: toward an integrated molecular cytogenetic map</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="mao-x-hamoudi-ra-zhao-p-baudis-m">Mao X, Hamoudi RA, Zhao P, Baudis M.<a class="headerlink" href="#mao-x-hamoudi-ra-zhao-p-baudis-m" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> Breast cancer is the most common malignant disease in Caucasian women, but is less frequent in Chinese women. The molecular basis for such ethnical difference in disease pathogenesis remains unknown. To address this issue, we performed allelotyping analysis of formalin-fixed, paraffin-embedded samples from 21 Chinese patients with breast cancer using 59 fluorescently tagged oligonucleotide primers amplifying microsatellite loci. Loss of heterozygosity (LOH) was found in all tumor samples. Frequent allelic losses were identified at markers D3S1578 (56%); D7S507 (55%); D1S2766 (50%); D17S789 and D17S946 (43% each); D19S814 (35%); D2S162, D13S158 and D13S296 (33% each); D1S551 and D1S2800 (29% each); D3S1597 and D6S260 (22% each); and D1S1588 (21%). To compare our data to previous reports, we determined the band-specific frequency of chromosomal imbalances in breast cancer karyotypes reported in the Mitelman database, and from the CGH results of cases accessible through the Progenetix website. Furthermore, published LOH analyses of breast cancer cases were compared to our own LOH results, demonstrating the most common chromosomal regions affected by allelic losses. The combined results provide a comprehensive view of genetic losses in breast cancers, indicating the comparability of these different techniques and suggesting the presence of a distinct subset of breast cancers with high-frequency LOH at chromosomes 1 and 2p in Chinese patients.</p>
<a href="2001-2007/2005-07-01-Genetic-losses-in/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      April 01, 2005
      
    </span>
</aside>

<h3 class="title_news">Unequivocal delineation of clinicogenetic subgroups and development of a new model for improved outcome prediction in neuroblastoma</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="vandesompele-j-baudis-m-de-preter-k-van-roy-n-ambros-p-bown-n-brinkschmidt-c-christiansen-h-combaret-v-lastowska-m-nicholson-j-omeara-a-plantaz-d-stallings-r-brichard-b-van-den-broecke-c-de-bie-s-de-paepe-a-laureys-g-speleman-f">Vandesompele J, Baudis M, De Preter K, Van Roy N, Ambros P, Bown N, Brinkschmidt C, Christiansen H, Combaret V, Lastowska M, Nicholson J, O'Meara A, Plantaz D, Stallings R, Brichard B, Van den Broecke C, De Bie S, De Paepe A, Laureys G, Speleman F.<a class="headerlink" href="#vandesompele-j-baudis-m-de-preter-k-van-roy-n-ambros-p-bown-n-brinkschmidt-c-christiansen-h-combaret-v-lastowska-m-nicholson-j-omeara-a-plantaz-d-stallings-r-brichard-b-van-den-broecke-c-de-bie-s-de-paepe-a-laureys-g-speleman-f" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> PURPOSE: Neuroblastoma is a genetically heterogeneous pediatric tumor with a remarkably variable clinical behavior ranging from widely disseminated disease to spontaneous regression. In this study, we aimed for comprehensive genetic subgroup discovery and assessment of independent prognostic markers based on genome-wide aberrations detected by comparative genomic hybridization (CGH).
<a href="2001-2007/2005-04-01-Unequivocal/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      January 01, 2005
      
    </span>
</aside>

<h3 class="title_news">Microarray comparative genomic hybridization detection of chromosomal imbalances in uterine cervix carcinoma</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="hidalgo-a-baudis-m-petersen-i-arreola-h-pina-p-vazquez-ortiz-g-hernandez-d-gonzalez-j-lazos-m-lopez-r-perez-c-garcia-j-vazquez-k-alatorre-b-salcedo-m">Hidalgo A, Baudis M, Petersen I, Arreola H, Piña P, Vázquez-Ortiz G, Hernández D, González J, Lazos M, López R, Pérez C, García J, Vázquez K, Alatorre B, Salcedo M.<a class="headerlink" href="#hidalgo-a-baudis-m-petersen-i-arreola-h-pina-p-vazquez-ortiz-g-hernandez-d-gonzalez-j-lazos-m-lopez-r-perez-c-garcia-j-vazquez-k-alatorre-b-salcedo-m" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> BACKGROUND: Chromosomal Comparative Genomic Hybridization (CGH) has been applied to all stages of cervical carcinoma progression, defining a specific pattern of chromosomal imbalances in this tumor. However, given its limited spatial resolution, chromosomal CGH has offered only general information regarding the possible genetic targets of DNA copy number changes. METHODS: In order to further define specific DNA copy number changes in cervical cancer, we analyzed 20 cervical samples (3 pre-malignant lesions, 10 invasive tumors, and 7 cell lines), using the GenoSensor microarray CGH system to define particular genetic targets that suffer copy number changes.
<a href="2001-2007/2005-01-01-Microarray/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      December 01, 2001
      
    </span>
</aside>

<h3 class="title_news">Progenetix.net: an online repository for molecular cytogenetic aberration data.</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="baudis-m-cleary-ml">Baudis M, Cleary ML.<a class="headerlink" href="#baudis-m-cleary-ml" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> Through sequencing projects and, more recently, array-based expression analysis experiments, a wealth of genetic data has become accessible via online resources. In contrast, few of the (molecular-) cytogenetic aberration data collected in the last decades are available in a format suitable for data mining procedures. <a href="http://www.progenetix.net">www.progenetix.net</a> is a new online repository for previously published chromosomal aberration data, allowing the addition of band-specific information about chromosomal imbalances to oncologic data analysis efforts.</p>

<a href="2001-2007/2001-12-01-Progenetix-net-an/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      April 01, 2001
      
    </span>
</aside>

<h3 class="title_news">Gain of chromosome arm 9p is characteristic of primary mediastinal B-cell lymphoma (MBL): comprehensive molecular cytogenetic analysis and presentation of a novel MBL cell line</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="bentz-m-barth-tf-bruderlein-s-bock-d-schwerer-mj-baudis-m-joos-s-viardot-a-feller-ac-muller-hermelink-hk-lichter-p-dohner-h-moller-p">Bentz M, Barth TF, Brüderlein S, Bock D, Schwerer MJ, Baudis M, Joos S, Viardot A, Feller AC, Müller-Hermelink HK, Lichter P, Döhner H, Möller P.<a class="headerlink" href="#bentz-m-barth-tf-bruderlein-s-bock-d-schwerer-mj-baudis-m-joos-s-viardot-a-feller-ac-muller-hermelink-hk-lichter-p-dohner-h-moller-p" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> Primary mediastinal B-cell lymphoma (MBL) is an aggressive Non-Hodgkin's Lymphoma, which has been recognized as a distinct disease entity. We performed a comprehensive molecular cytogenetic study analyzing 43 MBLs. By comparative genomic hybridization (CGH), the most common aberrations were gains of chromosome arms 9p and Xq, which were present in 56% and 40% of cases, respectively. Based on the limited resolution of CGH, this technique may underestimate the real incidence of aberrations.
<a href="2001-2007/2001-04-01-Gain-of-chromosome/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      January 01, 2001
      
    </span>
</aside>

<h3 class="title_news">Comparative genomic hybridization for the analysis of leukemias and lymphomas</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="baudis-m-bentz-m">Baudis M, Bentz M.<a class="headerlink" href="#baudis-m-bentz-m" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> Cytogenetic methods have become increasingly important tools for both research in hematological malignancies and for the diagnostic workup of leukemias and lymphomas. The knowledge about specific chromosomal aberrations has been an essential prerequisite for the identification of pathogenetically relevant genes. Important examples are molecular genetic analyses of the breakpoint regions in chromosomal translocations, which resulted in the detection of protooncogenes such as ABL in chronic myeloid leukemia (CML) and acute lymphoblastic leukemia (ALL) [t(9,22)(q34;q11)], or MYC in Burkitt's lymphoma [t(8;14)(q24;q32); for a review see refs. 1 and 2].</p>
<a href="2001-2007/2001-01-01-Comparative-genomic/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      January 01, 2001
      
    </span>
</aside>

<h3 class="title_news">Potential of chromosomal and matrix-based comparative genomic hybridization for molecular diagnostics in lymphomas</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h3 id="potential-of-chromosomal-and-matrix-based-comparative-genomic-hybridization-for-molecular-diagnostics-in-lymphomas">Potential of chromosomal and matrix-based comparative genomic hybridization for molecular diagnostics in lymphomas.<a class="headerlink" href="#potential-of-chromosomal-and-matrix-based-comparative-genomic-hybridization-for-molecular-diagnostics-in-lymphomas" title="Permanent link">&para;</a></h3>
<h4 id="wessendorf-s-lichter-p-schwanen-c-fritz-b-baudis-m-walenta-k-kloess-m-dohner-h-bentz-m">Wessendorf S, Lichter P, Schwänen C, Fritz B, Baudis M, Walenta K, Kloess M, Döhner H, Bentz M.<a class="headerlink" href="#wessendorf-s-lichter-p-schwanen-c-fritz-b-baudis-m-walenta-k-kloess-m-dohner-h-bentz-m" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> 
Genome research lead to one of the largest scientific achievements of the last decade. Apart from sequencing the human genome, there was also a huge increase of knowledge regarding genomic aberrations in cancer. In Non-Hodgkin’s lymphomas (NHL), some of these findings already are of clinical relevance; specific genomic aberrations are characteristic of distinct subtypes of NHL and correlate with certain morphological, immunological and clinical findings.
<a href="2001-2007/2001-01-01-Potential-of/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      March 01, 2000
      
    </span>
</aside>

<h3 class="title_news">t(11;14)-positive mantle cell lymphomas exhibit complex karyotypes and share similarities with B-cell chronic lymphocytic leukemia</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="bentz-m-plesch-a-bullinger-l-stilgenbauer-s-ott-g-muller-hermelink-hk-baudis-m-barth-tf-moller-p-lichter-p-dohner-h">Bentz M, Plesch A, Bullinger L, Stilgenbauer S, Ott G, Müller-Hermelink HK, Baudis M, Barth TF, Möller P, Lichter P, Döhner H.<a class="headerlink" href="#bentz-m-plesch-a-bullinger-l-stilgenbauer-s-ott-g-muller-hermelink-hk-baudis-m-barth-tf-moller-p-lichter-p-dohner-h" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> Until now, few data on additional chromosomal aberrations in t(11;14)-positive mantle cell lymphomas (MCLs) have been published. We analyzed 39 t(11;14)-positive MCLs by either comparative genomic hybridization (CGH; n = 8), fluorescence in situ hybridization (FISH) with a set of DNA probes detecting the most frequent aberrations in B-cell neoplasms (n = 12), or both techniques (n = 19). The t(11;14) was present in all cases. In 37 of 39 cases, chromosomal imbalances were found.
<a href="1995-2000/2000-03-01-t-11-14-positive/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      December 01, 1997
      
    </span>
</aside>

<h3 class="title_news">Analysis of genomic alterations in benign, atypical, and anaplastic meningiomas: toward a genetic model of meningioma progression</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="weber-rg-bostrom-j-wolter-m-baudis-m-collins-vp-reifenberger-g-lichter-p">Weber RG, Boström J, Wolter M, Baudis M, Collins VP, Reifenberger G, Lichter P.<a class="headerlink" href="#weber-rg-bostrom-j-wolter-m-baudis-m-collins-vp-reifenberger-g-lichter-p" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> Nineteen benign [World Health Organization (WHO) grade I; MI], 21 atypical (WHO grade II; MII), and 19 anaplastic (WHO grade III; MIII) sporadic meningiomas were screened for chromosomal imbalances by comparative genomic hybridization (CGH). These data were supplemented by molecular genetic analyses of selected chromosomal regions and genes. With increasing malignancy grade, a marked accumulation of genomic aberrations was observed; i.e., the numbers (mean +/- SEM) of total alterations detected per tumor were 2.9 +/- 0.7 for MI, 9.2 +/- 1.2 for MII, and 13.3 +/- 1.9 for MIII.
<a href="1995-2000/1997-12-01-Analysis-of-genomic/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      August 01, 1997
      
    </span>
</aside>

<h3 class="title_news">High-level DNA amplifications are common genetic aberrations in B-cell neoplasms</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="werner-ca-dohner-h-joos-s-trumper-lh-baudis-m-barth-tf-ott-g-moller-p-lichter-p-bentz-m">Werner CA, Döhner H, Joos S, Trümper LH, Baudis M, Barth TF, Ott G, Möller P, Lichter P, Bentz M.<a class="headerlink" href="#werner-ca-dohner-h-joos-s-trumper-lh-baudis-m-barth-tf-ott-g-moller-p-lichter-p-bentz-m" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> Gene amplification is one of the molecular mechanisms resulting in the up-regulation of gene expression. In non-Hodgkin's lymphomas, such gene amplifications have been identified rarely. Using comparative genomic hybridization, a technique that has proven to be very sensitive for the detection of high-level DNA amplifications, we analyzed 108 cases of B-cell neoplasms (42 chronic B-cell leukemias, 5 mantle cell lymphomas, and 61 aggressive B-cell lymphomas).
<a href="1995-2000/1997-08-01-High-level-DNA/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      September 01, 1996
      
    </span>
</aside>

<h3 class="title_news">Efficacy of current molecular cytogenetic protocols for the diagnosis of chromosome aberrations in tumor specimens</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="lichter-p-fischer-k-joos-s-fink-t-baudis-m-potkul-rk-ohl-s-solinas-toldo-s-weber-r-stilgenbauer-s-bentz-m-dohner-h">Lichter P, Fischer K, Joos S, Fink T, Baudis M, Potkul RK, Ohl S, Solinas-Toldo S, Weber R, Stilgenbauer S, Bentz M, Döhner H.<a class="headerlink" href="#lichter-p-fischer-k-joos-s-fink-t-baudis-m-potkul-rk-ohl-s-solinas-toldo-s-weber-r-stilgenbauer-s-bentz-m-dohner-h" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> Molecular cytogenetics provides a powerful link between molecular genetic analysis and chromosome morphology, allowing one to pinpoint structurally aberrant chromosome regions on the molecular level. Fluorescence in situ hybridization with selected DNA probes allows the design of efficient and sensitive tools for the diagnosis of chromosomal aberrations present in tumor cells. Comparative genomic hybridization (CGH) allows the identification of chromosomal imbalances in a comprehensive manner, and is applied to solid tumors and hematological malignancies in order to (i) identify clonal differences within a specimen, (ii) contribute to tumor classifications, (iii) identify recurrent chromosomal gains and losses as starting points for the characterization and isolation of pathogenetically relevant genes, such as proto-oncogenes and tumor suppressor genes respectively, (iv) identify imbalances of prognostic relevance, (v) detect high-copy-number amplification and other markers of genetic instability, and (vi) analyze chromosomal imbalances during tumor progression.</p>
<a href="1995-2000/1996-09-01-Efficacy-of-current/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      January 01, 1996
      
    </span>
</aside>

<h3 class="title_news">Chromosome imbalances in papillary renal cell carcinoma and first cytogenetic data of familial cases analyzed by comparative genomic hybridization</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="bentz-m-bergerheim-us-li-c-joos-s-werner-ca-baudis-m-gnarra-j-merino-mj-zbar-b-linehan-wm-lichter-p">Bentz M, Bergerheim US, Li C, Joos S, Werner CA, Baudis M, Gnarra J, Merino MJ, Zbar B, Linehan WM, Lichter P.<a class="headerlink" href="#bentz-m-bergerheim-us-li-c-joos-s-werner-ca-baudis-m-gnarra-j-merino-mj-zbar-b-linehan-wm-lichter-p" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> We used comparative genomic hybridization to analyze 17 tumor samples from 11 patients with papillary renal cell carcinoma (RCC), including three patients with hereditary papillary RCC. Whereas the most frequent aberrations confirmed data obtained by banding analyses, copy number increases on 5q, which previously were considered characteristic of nonpapillary RCC, were identified in two cases.
<a href="1995-2000/1996-01-01-Chromosome/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  
    <aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0M21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      December 01, 1995
      
    </span>
</aside>

<h3 class="title_news">Identification of genetic imbalances in malignant lymphoma using comparative genomic hybridization</h3>
    
<!-- Use a hidden p tag to provide a preview -->
    <h4 id="bentz-m-dohner-h-werner-ca-huck-k-baudis-m-joos-s-schlegelberger-b-trumper-lh-feller-ac-pfreundschuh-m">Bentz M, Döhner H, Werner CA, Huck K, Baudis M, Joos S, Schlegelberger B, Trümper LH, Feller AC, Pfreundschuh M.<a class="headerlink" href="#bentz-m-dohner-h-werner-ca-huck-k-baudis-m-joos-s-schlegelberger-b-trumper-lh-feller-ac-pfreundschuh-m" title="Permanent link">&para;</a></h4>
<p><strong>Abstract</strong> In comparison to leukemias, the clinical relevance of chromosomal aberrations in non-Hodgkin's lymphoma (NHL) is not as well understood. This is primarily due to limitations of chromosomal banding techniques which have been the central methods for cytogenetic analysis. These techniques depend on the availability of fresh tumor tissue and the examination of metaphase cells which may not be representative for the major cell clone in vivo. In contrast, the new technique of comparative genomic hybridization (CGH) allows researchers to obtain a comprehensive view of chromosomal gains and losses by analyzing tumor DNA, which can be prepared from archival tissue samples.
<a href="1995-2000/1995-12-01-Identification-of/">
  <span class="twemoji">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.75.75 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0"/></svg>
  </span>
  Continue reading
</a>

  


              </article>
            </div>
          
          
  <script>var tabs=__md_get("__tabs");if(Array.isArray(tabs))e:for(var set of document.querySelectorAll(".tabbed-set")){var labels=set.querySelector(".tabbed-labels");for(var tab of tabs)for(var label of labels.getElementsByTagName("label"))if(label.innerText.trim()===tab){var input=document.getElementById(label.htmlFor);input.checked=!0;continue e}}</script>

<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      &copy; Copyright 2007 - 2025 Michael Baudis
    </div>
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
        
<div class="md-social">
  
    
    
    
    
      
      
    
    <a href="https://github.com/baudisgroup" target="_blank" rel="noopener" title="github.com" class="md-social__link">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><!--! Font Awesome Free 7.1.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M173.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6m-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3m44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9M252.8 8C114.1 8 8 113.3 8 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C436.2 457.8 504 362.9 504 252 504 113.3 391.5 8 252.8 8M105.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1m-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7m32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1m-11.4-14.7c-1.6 1-1.6 3.6 0 5.9s4.3 3.3 5.6 2.3c1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2"/></svg>
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    
      
      
      <script id="__config" type="application/json">{"annotate": null, "base": "..", "features": ["content.tabs.link", "search.highlight", "search.share", "navigation.instant", "navigation.tracking", "navigation.sections", "toc.integrate"], "search": "../assets/javascripts/workers/search.2c215733.min.js", "tags": null, "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}, "version": null}</script>
    
    
      <script src="../assets/javascripts/bundle.79ae519e.min.js"></script>
      
    
  </body>
</html>